
# evolve.jargon

## Table of Contents
- [2018](#2018)
- [neutral theory](#neutral-theory) ä¸­ç«‹èª¬
- [adaptation](#adaptation) é©å¿œ
- [amelioration](#amelioration)
- [abduction](#abduction)
- [allele](#allele) å¯¾ç«‹éºä¼å­
- [ancestral reconstruction](#ancestral-reconstruction) ç¥–å…ˆæ¨å®š
- [basal](#basal)
- [concerted evolution](#concerted-evolution) å”èª¿é€²åŒ–
- [convergent](#convergent) åæ–‚é€²åŒ–
- [concordance](#concordance)
- [divergence time](#divergence-time) åˆ†å²æ™‚é–“
- [effective population size](#effective-population-size) é›†å›£ã®æœ‰åŠ¹ãªå¤§ãã•
- [homology](#homology) ãƒ›ãƒ¢ãƒ­ã‚¯ã‚™ã€ã‚ªãƒ¼ã‚½ãƒ­ã‚¯ã‚™ã€ãƒã‚šãƒ©ãƒ­ã‚¯ã‚™
- [homoplasy](#homoplasy)
- [overall similarity](#overall-similarity) å…¨ä½“çš„é¡ä¼¼åº¦
- [phylogenetic diversity](#phylogenetic-diversity)
- [recombination](#recombination) çµ„æ›ãˆ
- [Newick](#newick)
- [species](#species)
- [taxon](#taxon)
- [treedist](#treedist)
- [dNdS](#dnds) é©å¿œ
- [RELL](#rell)
- [selective inference](#selective-inference)
- [taxon sampling](#taxon-sampling)
- [revisionterm](#revisionterm)
- [branch length](#branch-length) æé•·
- [LBA](#lba) é•·æèª˜å¼•
- [partition](#partition)
- [microbe](#microbe) å¾®ç”Ÿç‰©
- [constraint](#constraint) åˆ¶ç´„
- [duplication](#duplication) éºä¼å­é‡è¤‡
- [outgroup](#outgroup)
- [root](#root)
- [clade](#clade) åˆ†å²ç¾¤ã€å˜ç³»çµ±ç¾¤
- [unclassified](#unclassified)
- [pcm](#pcm) phylogenetic comparative methods
- [brownian](#brownian) ãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«
- [tree](#tree)
  - [circle tree](#circle-tree)
- [marker](#marker)
- [taxon_sampling](#taxon_sampling) ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°
- [](#)
- [model](#model)
  - [GTR](#gtr)
  - [JTT](#jtt)
- [substitution matrix](#substitution-matrix) ç½®æ›è¡Œåˆ—
- [HGT](#hgt) éºä¼å­æ°´å¹³ä¼æ’­
- [](#)
- [rate_of_evolution](#rate_of_evolution) é€²åŒ–é€Ÿåº¦
- [mutation](#mutation) å¤‰ç•°
- [saturation](#saturation) é£½å’Œ
- [group_selection](#group_selection) ç¾¤é¸æŠ
- [](#)

----------


----------
## group_selection

https://ja.wikipedia.org/wiki/ç¾¤é¸æŠ

2024å¹´7æœˆ7æ—¥ 8:00
https://diamond.jp/articles/-/346050
ã€Œç”Ÿç‰©ãŒéºä¼å­ã‚’æ®‹ãã†ã¨ã™ã‚‹ã®ã¯ç¨®ã®ä¿å­˜ã®ãŸã‚ã€ãŒå®Œå…¨ãªèª¤ã‚Šãªãƒ¯ã‚± | ãƒ‹ãƒ¥ãƒ¼ã‚¹ãªæœ¬ | ãƒ€ã‚¤ãƒ¤ãƒ¢ãƒ³ãƒ‰ãƒ»ã‚ªãƒ³ãƒ©ã‚¤ãƒ³
æ²³ç”°é›…åœ­: æ±åŒ—å¤§å­¦æ•™é¤Šæ•™è‚²é™¢ç·é•·ç‰¹å‘½æ•™æˆã€åèª‰æ•™æˆ
â€»æœ¬ç¨¿ã¯ã€æ²³ç”°é›…åœ­ã€ãƒ€ãƒ¼ã‚¦ã‚£ãƒ³ã®é€²åŒ–è«–ã¯ã©ã“ã¾ã§æ­£ã—ã„ã®ã‹ï¼Ÿ é€²åŒ–ã®ä»•çµ„ã¿ã‚’åŸºç¤ã‹ã‚‰å­¦ã¶ã€ï¼ˆå…‰æ–‡ç¤¾ï¼‰ã®ä¸€éƒ¨ã‚’æŠœç²‹ãƒ»ç·¨é›†ã—ãŸã‚‚ã®ã§ã™ã€‚
https://diamond.jp/articles/-/346050?page=3
æ¬¡ã®ãƒšãƒ¼ã‚¸
ãƒ’ãƒˆã¯ã€Œç¨®æ—ç¶­æŒã®ãŸã‚ã®é€²åŒ–ã€ã¨è€ƒãˆã¦ã—ã¾ã„ãŒã¡
https://diamond.jp/articles/-/346050?page=4
ã€€ã¨ã“ã‚ã§ã€ç¾åœ¨ã€ãƒ‰ãƒ¼ã‚­ãƒ³ã‚¹ã®ã€Œåˆ©å·±çš„éºä¼å­ã€ã¨ã„ã†è¨€è‘‰ã¯ä¸€èˆ¬ã®äººã«ã‚‚æ¯”è¼ƒçš„çŸ¥ã‚‰ã‚Œã¦ã„ã‚‹ã®ã«å¯¾ã—ã€ã€Œç¨®ã®ãŸã‚ã®é€²åŒ–ã¯ç”Ÿã˜ãªã„ã€ã“ã¨ã¯ç†è§£ã•ã‚Œã¦ã„ãªã„ã“ã¨ãŒå¤šã„ã€‚

ã€€ãã®ãŸã‚ã«ã€ã€Œç”Ÿç‰©ãŒè‡ªã‚‰ã®éºä¼å­ã‚’æ®‹ãã†ã¨ã™ã‚‹ã®ã¯ã€ç¨®ã®ä¿å­˜ã®ãŸã‚ã ã€ã¨ã„ã†èª¤è§£ã‚’ã™ã‚‹äººã‚‚å°‘ãªã‹ã‚‰ãšã„ã‚‹ã€‚ã€Œåˆ©å·±çš„éºä¼å­ã€ã®è¦‹æ–¹ãŒé©åˆ‡ã§ã‚ã‚‹ã‹ã©ã†ã‹ã«ã‹ã‹ã‚ã‚‰ãšã€ã“ã®è¡¨ç¾ã¯å…¨ãã®èª¤ã‚Šã§ã‚ã‚‹ã€‚

ã€€ãƒ’ãƒˆã«ã¯ã€Œç¨®æ—ç¶­æŒã®ãŸã‚ã®é€²åŒ–ã€ã¨ã„ã†è€ƒãˆã«é™¥ã‚Šã‚„ã™ã„æ€è€ƒãƒã‚¤ã‚¢ã‚¹ãŒã‚ã‚‹ã®ã‹ã‚‚ã—ã‚Œãªã„ã€‚

2022å¹´3æœˆ20æ—¥ 16:00
https://note.com/masakadokawata/n/n41079da12807
ã€Œç¨®ã®ä¿å­˜ã®ãŸã‚ã®é€²åŒ–ã€ã¯ã©ã“ãŒèª¤ã‚Šãªã®ã‹ï½œæ²³ç”° é›…åœ­

2023.06.07
https://gendai.media/articles/-/111047?page=4
ã€Œç¾ç”Ÿäººé¡ã¯ã€ã„ãšã‚Œç¨®ã¨ã—ã¦ã®å¯¿å‘½ãŒãã¦çµ¶æ»…ã™ã‚‹ã€ï¼Ÿâ€¦ã€Œç¨®ã€ã«ã€Œå¯¿å‘½ã€ã¯ã‚ã‚‹ã‹ã¨ã„ã†å¤§å•é¡Œï¼ˆæ›´ç§‘ åŠŸï¼‰ - 4ãƒšãƒ¼ã‚¸ç›® | ãƒ–ãƒ«ãƒ¼ãƒãƒƒã‚¯ã‚¹ | è¬›è«‡ç¤¾


----------
## tree

https://nextstrain.org/help/general/how-to-read-a-tree
How to interpret the phylogenetic trees

https://artic.network/guides/how-to-read-a-tree
How to read a phylogenetic tree | Tutorial | Phylogenetics | ARTIC network - pathogen genomics from sample to response

https://evolution-outreach.biomedcentral.com/articles/10.1007/s12052-010-0273-6
McLennan, D.A. How to Read a Phylogenetic Tree. Evo Edu Outreach 3, 506â€“519 (2010). https://doi.org/10.1007/s12052-010-0273-6

https://archosaurmusings.wordpress.com/2008/12/18/how-to-read-a-phylogenetic-tree/
How to read a phylogenetic tree | Dave Hone's Archosaur Musings
basal taxa (at the bottom of the tree)

----------

https://www.digitalatlasofancientlife.org/learn/systematics/phylogenetics/reading-trees/
2.1 Reading Trees - Digital Atlas of Ancient Life

2.1 Reading Trees
Chapter contents:
Systematics
â€” 1. Taxonomy
â€” 2. Phylogenetics
â€”â€” 2.1 Reading trees â†
â€”â€” 2.2 Building trees
â€”â€” 2.3 Character mapping
â€”â€” 2.4 Phylogenetic trees and classification

Clades and Sister Groups
A clade (from the Greek klados = branch) is a group that includes an ancestor (node) and all of its descendants (all shallower nodes and terminal taxa that descend from that node) on a phylogenetic tree.

----------
### circle tree

2:30 AM Â· May 14, 2022
https://twitter.com/jilloberski/status/1525166721161523202
Jill Oberski ğŸœ on Twitter: "Maybe a hot take: I think circular phylogenies are a bad choice for manuscript figures. Theyâ€™re hard to read and hard to interpret." / Twitter

6:40 PM Â· Sep 24, 2020
https://twitter.com/copypasteusa/status/1309065045201096704
"Tree A is in polar format (often called a circle tree).
These tree formats are often used to make a big visual impact in papers but generally have reduced readability - it is difficult to compare how far nodes are from the centre. They are best avoided. https://artic.network/how-to-read-a-tree.html

Feb 5, 2021
https://twitter.com/windowmoon/status/1357643379950067715
ã ã‹ã‚‰ã‚ã‚Œã»ã©å††å½¢ç³»çµ±æ¨¹ã¯ä½¿ã†ãªã¨å°ä¸€æ™‚é–“ä»¥ä¸‹ç•¥ã€‚ä»Šå›ã®ä¾‹ã¯ä¸€æ–¹å‘ã®ç³»çµ±æ¨¹ã«ã—ã¦ã‚‚è§£é‡ˆãŒåˆ†ã‹ã‚Œãã†ã€‚ã‚ãŸã—ã¯ã€Œã©ã¡ã‚‰ã¨ã‚‚è¨€ãˆãªã„ã€æ´¾ã‹ãªã€‚
https://twitter.com/mutselbalance/status/1357642470587199490
ä¸¸ã„ç³»çµ±æ¨¹ã§æ›¸ã„ã¦ã—ã¾ã†ã¨ã€äººé–“ã®é è¿‘æ„Ÿè¦šã«ã¯å¤šæ§˜æ€§ãŒã‚ã‚‹ãŸã‚ã€è§£é‡ˆãŒå›°é›£ã«ãªã£ã¦ã—ã¾ã†â€¦â€¦ãã®å¥½ä¾‹ã€‚åƒ•ã«ã¯è¿‘ã„ã‚ˆã†ã«è¦‹ãˆã‚‹â€¦ã€‚
https://twitter.com/Parapriacanthus/status/1357629304788590592
é•·å¯¿ã®æœ¨ã¯é€²åŒ–çš„ã«é›¢ã‚ŒãŸç³»çµ±ã§ã¿ã‚‰ã‚Œã‚‹
https://twitter.com/sPlot_iDiv/status/1357249814178725900
How old do trees grow? Why and where?
All answers in a #TansleyReview by #GianlucaPiovesan and #FrancoBiondi in 
@NewPhyt
https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.17148
On tree longevity

2:02 PM Â· Sep 24, 2020
https://twitter.com/windowmoon/status/1308995216729399296
çª“æœˆï¼ Sç´šåœ¨å®…å£« on Twitter: "ä½•åº¦ã‚‚è¨€ã£ã¦ã¾ã™ãŒã€å††å½¢ç³»çµ±æ¨¹ã¯å®³æ‚ªã§ã™ã€‚ã‹ã£ã“ã„ã„ã ã‘ã§è¦‹ã‚„ã™ããªã„ã€‚ç§‘å­¦è€…ãªã‚‰ä½¿ã†ã¹ãã§ã¯ãªã„ã€‚æœ¬å½“ã«ã‚„ã‚ã¦ã»ã—ã„ã€‚" / Twitter
Most tree plotting programs have "collapse branches" function. Therefore, I recommend to use this function and plot "collapsed" whole tree and several subtrees of "collapsed" branches.

5:35 PM Â· Mar 19, 2019
https://twitter.com/windowmoon/status/1107923607349424129
- ç›¸å¤‰ã‚ã‚‰ãšçš†ã•ã‚“å††å½¢ç³»çµ±æ¨¹å¥½ãã§ã™ã­ã€‚è¦‹ã¥ã‚‰ã„ã‹ã‚‰ã‚„ã‚ã‚ã€‚
- å††å½¢ç³»çµ±æ¨¹ã£ã¦ã€ã€Œãƒ‡ã‚¶ã‚¤ãƒ³çš„ã«ã‚«ãƒƒã‚³ã„ã„ã€ä»¥å¤–ã®ãƒ¡ãƒªãƒƒãƒˆã‚ã‚‹ã‚“ã§ã—ã‚‡ã†ã‹ã­ï¼Ÿ
- ä¸€åˆ‡ãªã„ã¨æ€ã„ã¾ã™ã€‚æœ‰å®³ç„¡ç›Šã€‚ç¦æ­¢ã™ã¹ãã€‚å††ã‚°ãƒ©ãƒ•ã¿ãŸã„ãªã‚‚ã®ã§ã™ã­ã€‚

12:58 AM - 19 Mar 2019
https://twitter.com/Tyu_Shi/status/1107868946269519872
Megaphylogeny resolves global patterns of mushroom evolution https://www.nature.com/articles/s41559-019-0834-1 â€¦ ã‚­ãƒã‚³ã®Timetreeè«–æ–‡ã€‚

https://nam-students.blogspot.com/2013/03/blog-post_2476.html
NAMså‡ºç‰ˆãƒ—ãƒ­ã‚¸ã‚§ã‚¯ãƒˆ: å…¨ç”Ÿç‰©ã®ç³»çµ±æ¨¹ã€å††å½¢ãƒãƒ¼ã‚¸ãƒ§ãƒ³ï¼šãƒ¡ãƒ¢

----------
### gene trees and species trees

https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/188bto4/alma99159251504031
Computational Molecular Evolution
3.1.4 Gene trees and species trees

https://www.kyoritsu-pub.co.jp/book/b10010733.html
åˆ†å­ç³»çµ±å­¦ã¸ã®çµ±è¨ˆçš„ã‚¢ãƒ—ãƒ­ãƒ¼ãƒ - å…±ç«‹å‡ºç‰ˆ
https://kyoritsu-pub.sakura.ne.jp/app/file/goods_contents/1152.pdf
3.1.4 éºä¼å­ç³»çµ±æ¨¹ã¨ç¨®ç³»çµ±æ¨¹

https://x.com/copypasteusa/status/1664499852351508481
éºä¼å­ç³»çµ±æ¨¹ã¨ç¨®ç³»çµ±æ¨¹ãŒç•°ãªã‚‹è¦å› ï¼šæ¨å®šã®èª¤å·®ã€éºä¼å­ã®æ°´å¹³ä¼æ’­ã€éºä¼å­é‡è¤‡ï¼ˆãƒ‘ãƒ©ãƒ­ã‚¬ã‚¹ãªéºä¼å­ã‚’ä½¿ã†ï¼‰ã€ç¥–å…ˆã®å¤šå‹ã‚„ç³»çµ±é¸åˆ¥ï¼ˆlineage sortingï¼‰

----------
## unclassified

https://en.wikipedia.org/wiki/Sister_group
https://ja.wikipedia.org/wiki/å§‰å¦¹ç¾¤

https://ja.wikipedia.org/wiki/å‰µå§‹è€…åŠ¹æœ

https://ja.wikipedia.org/wiki/é©å¿œæ”¾æ•£


----------

http://www.saitama-u.ac.jp/ohnishi/jikken/phylogenetic_methods.htm
ç³»çµ±æ¨¹ã®åŸºç¤çŸ¥è­˜ (å›³å‚ç…§)
Nodeï¼ˆç¯€ï¼‰ã¨Branchï¼ˆæï¼‰ã‹ã‚‰æˆã£ã¦ã„ã‚‹ï¼Nodeã«ã¯External Nodes (å³ç«¯ã®ç¾ç”Ÿã® [ç”Ÿç‰©ã‚ã‚‹ã„ã¯é…åˆ—] ï¼ˆä»¥ä¸‹ã§ã¯é…åˆ—ã¨ã®ã¿è¨˜è¿°ï¼‰) ã¨ç³»çµ±æ¨¹å†…ã®Internal Nodes (éå»ã«å­˜åœ¨ã—ãŸï¼Œã‚ã‚‹ã„ã¯ï¼Œå­˜åœ¨ã—ãŸã¨æ¨å®šã•ã‚Œã‚‹é…åˆ—) ãŒã‚ã‚‹ï¼ˆå›³(1)ï¼‰ï¼Root (R)ã¯ä»®æƒ³çš„ãªå…±é€šç¥–å…ˆ (Common ancestor)ï¼
ã¾ãŸï¼ŒInternal Nodesã«ã¯bifurcating (ï¼’é–åˆ†å²)ã¨multifurcating (å¤šé–åˆ†å²) ã™ã‚‹ã‚‚ã®ã®ï¼’ç¨®é¡ãŒã‚ã‚‹ï¼åŸå‰‡çš„ã«ã¯ï¼Œï¼’é–åˆ†å²ã®ã¯ãšã ãŒï¼Œåˆ†å²ã®é †åºãŒæ­£ç¢ºã«æ±ºã¾ã‚‰ãªã„å ´åˆã¯ï¼Œmultifurcatingã®å½¢ã«è¡¨ç¤ºã•ã‚Œã‚‹ï¼(ä¸‹å›³ (2)å³ã®N1)

https://www.kochi-u.ac.jp/w3museum/Fish_Labo/Member/Endoh/animal_taxonomy/termonology01.html
ï¼‘ï¼åˆ†é¡å­¦ã«é–¢ã™ã‚‹ãŠã‚‚ãªç”¨èªã€€æ›´æ–°æ—¥ï¼š2017.4.14

ï¼’ï¼ç³»çµ±å­¦ã«é–¢ã™ã‚‹ç”¨èª

phenetics, phenetic classificationï¼šè¡¨å½¢å­¦ï¼Œè¡¨å½¢åˆ†é¡ã€€ï¼Šphenetists è¡¨å½¢å­¦è€… 
A system of classification in which the organisms are grouped together on the basis of their overall similarity*.  *ç·é¡ä¼¼åº¦ï¼Œç·ä½“çš„é¡ä¼¼åº¦ã€€ï¼Šphenotype ã¯è¡¨ç¾å½¢ 

numerical taxonomyï¼šæ•°é‡åˆ†é¡å­¦ï¼›numerical pheneticsï¼šæ•°é‡è¡¨å½¢å­¦ã€€ 
â˜…è¡¨å½¢åˆ†é¡ã§ã¯åŸºæœ¬çš„ã«ã™ã¹ã¦ã®å½¢è³ªã‚’ç­‰ä¾¡ã¨è¦‹ãªã—ï¼Œå½¢è³ªã®å…±æœ‰ã‚’æ•°é‡çš„ã«å‡¦ç†ã—ã¦ã‚°ãƒ«ãƒ¼ãƒ”ãƒ³ã‚°ã™ã‚‹ãŸã‚ï¼Œæ•°é‡åˆ†é¡å­¦ã¨ã‚‚å‘¼ã°ã‚Œã‚‹ï¼ 

cladistics, cladistic classificationï¼šåˆ†å²å­¦ï¼Œåˆ†å²åˆ†é¡ã€€ï¼Šcladists åˆ†å²å­¦è€… 
A system of classification in which the only groups formally recognized are clades.
â˜…åˆ†å²åˆ†é¡ã§ã¯ï¼Œå…±æœ‰æ´¾ç”Ÿå½¢è³ªã§æ”¯æŒã•ã‚Œã‚‹å˜ç³»çµ±ç¾¤ã«åŸºã¥ã„ã¦ã‚°ãƒ«ãƒ¼ãƒ”ãƒ³ã‚°ã™ã‚‹ï¼ç¥–å…ˆå½¢è³ªã®å…±æœ‰ã¯ç³»çµ±é–¢ä¿‚ã‚’ç¤ºã•ãªã„ï¼å†…ç¾¤ã¨å¤–ç¾¤ã‚’å«ã‚ãŸã‚°ãƒ«ãƒ¼ãƒ—ã‚’æœ€ç¯€ç´„æ³•ã§è§£æã—ï¼Œç³»çµ±é–¢ä¿‚ã‚’æ¨å®šã™ã‚‹ï¼ˆå†…ç¾¤ã®å½¢è³ªã®æ¥µæ€§ã‚‚æ¨å®šã•ã‚Œã‚‹ï¼‰ï¼ 

https://quizlet.com/153288496/chpt-261-263-flash-cards/
Chpt. 26.1-26.3å˜èªã‚«ãƒ¼ãƒ‰ | Quizlet
branch points
-in picture, point 3 is common ancestor of A, B, C
-4 shows that taxa B and C diverged after their shared lineage split from lineage leading to A

![](https://quizlet.com/cdn-cgi/image/f=auto,fit=cover,h=200,onerror=redirect,w=240/https://o.quizlet.com/2o6hNR7P.YrVZJ7Uty5mmA.png)

http://lbm.ab.a.u-tokyo.ac.jp/~omori/phylogeny/txt/phylogeny_txt.html
Morecular phylogeny
2015å¹´åº¦ç”Ÿç‰©æ¸¬å®šå­¦å¿œç”¨å®Ÿé¨“
åˆ†å­ç³»çµ±æ¨¹
æ±äº¬å¤§å­¦å¤§å­¦é™¢è¾²å­¦ç”Ÿå‘½ç§‘å­¦ç ”ç©¶ç§‘ã€€å¤§æ£®å®
2016å¹´ 1æœˆ 4æ—¥
1. å¡©åŸºç½®æ›ã®ç¢ºç‡ãƒ¢ãƒ‡ãƒ«
2. ç³»çµ±æ¨¹ä½œæˆæ‰‹æ³•
å‚è€ƒæ–‡çŒ®
åˆ†å­é€²åŒ–éºä¼å­¦ï¼Œæ ¹äº•æ­£åˆ©ï¼Œ(Molecular Evolutionary Genetics, äº”æ¢å €å­ãƒ»æ–ç±æˆä¹Ÿã€€å…±è¨³ï¼Œæ ¹äº•æ­£åˆ©ã€€ç›£è¨³)ï¼Œ åŸ¹é¢¨é¤¨ï¼Œ1990ï¼

http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
ç³»çµ±æ¨å®šã®åŸºæœ¬ç”¨èª

https://ja.wikipedia.org/wiki/é€€åŒ–
ä¸€èˆ¬èªã¨ã—ã¦ã®é€€åŒ–ã¯é€²åŒ–ã®å¯¾ç¾©èªã¨ä½ç½®ã¥ã‘ã‚‰ã‚Œå¾—ã‚‹[2]ãŒã€ç”Ÿç‰©å­¦ã«ãŠã„ã¦é€€åŒ–ã¯é€²åŒ–ã®ä¸€å´é¢ã§ã‚ã‚Šã€å¯¾ç¾©èªã§ã¯ãªã„[3]ã€‚

Tomoaki NISHIYAMA
Wed Dec 27 18:49:55 JST 2000
http://www.nibb.ac.jp/~tomoaki/protocols/genetree/words
ç”¨èªé›†

![](http://www.nibb.ac.jp/~tomoaki/protocols/genetree/three-topologies.gif)



https://ja.wikipedia.org/wiki/éåŠ é‡çµåˆæ³•
ï¼ˆUnweighted Pair Group Method with Arithmetic meanã€UPGMAã¨ç•¥ã™)ã¯ç³»çµ±æ¨¹ã‚’ä½œè£½ã™ã‚‹ãŸã‚ã®ãƒœãƒˆãƒ ã‚¢ãƒƒãƒ—å¼ã®ã‚¯ãƒ©ã‚¹ã‚¿è§£ææ³•ã§ã‚ã‚‹ã€‚å…¥åŠ›ãƒ‡ãƒ¼ã‚¿ã¯å¯¾è±¡ã®å„ãƒšã‚¢é–“ã®è·é›¢ã§ã‚ã‚Šã€æœ‰æ ¹ç³»çµ±æ¨¹ãŒä½œè£½ã•ã‚Œã‚‹ã€‚é€²åŒ–é€Ÿåº¦ãŒä¸€å®šï¼ˆåˆ†å­æ™‚è¨ˆä»®èª¬ï¼‰ã¨ä»®å®šã—ã¦æœ‰æ ¹ç³»çµ±æ¨¹ã‚’ä½œè£½ã™ã‚‹ã®ã«ã¨ãã©ãç”¨ã„ã‚‰ã‚Œã‚‹ã€‚
UPGMAã¯é€²åŒ–é€Ÿåº¦ä¸€å®šã®ä»®å®šã‚’ç”¨ã„ã¦ã„ã‚‹ãŸã‚ã€ã“ã‚ŒãŒå¯¾è±¡ã«é–¢ã—ã¦æ­£ã—ã„ã“ã¨ãŒç¤ºã•ã‚Œãªã„é™ã‚Šã€ç³»çµ±æ¨¹ã®æ¨å®šã«é©ã—ãŸæ–¹æ³•ã§ã¯ãªã„ã€‚
ã‚„ã¯ã‚Šè·é›¢ã‚’ç”¨ã„ã‚‹æ–¹æ³•ã§ã‚ã‚‹ãŒä¸Šè¨˜ã®ä»®å®šã‚’è¦ã—ãªã„ã‚‚ã®ã«è¿‘éš£çµåˆæ³•ï¼ˆNJæ³•ï¼‰ãŒã‚ã‚‹ã€‚
https://ja.wikipedia.org/wiki/è¿‘éš£çµåˆæ³•

https://shorebird.hatenablog.com/entry/20180601/1527842963
ã€Œç³»çµ±ä½“ç³»å­¦ã®ä¸–ç•Œã€ - shorebirdã€€é€²åŒ–å¿ƒç†å­¦ä¸­å¿ƒã®æ›¸è©•ãªã©






----------



----------
## video

https://www.youtube.com/playlist?list=PL0uaKHgcG00atli2VM1tKkE7F0Lr7VI0h
ç¬¬222å›è¾²æ—äº¤æµã‚»ãƒ³ã‚¿ãƒ¼ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ— åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨

https://doi.org/10.7875/togotv.2019.193
MEGA X ã‚’ç”¨ã„ãŸåˆ†å­ç³»çµ±è§£æ @ åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨ ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—

----------
## workshop

https://github.com/haruosuz/bioinfo/blob/master/2019/CaseStudy.md#ws222
ç¬¬222å›è¾²æ—äº¤æµã‚»ãƒ³ã‚¿ãƒ¼ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—ã€Œåˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•ï¼šç†è«–ã¨å¿œç”¨ã€

2019
https://www.affrc.maff.go.jp/tsukuba/top/event/workshop/ws_2019/222ws.html
ç¬¬222å›è¾²æ—äº¤æµã‚»ãƒ³ã‚¿ãƒ¼ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—ï¼šè¾²æ—æ°´ç”£æŠ€è¡“ä¼šè­°
åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•ï¼šç†è«–ã¨å¿œç”¨
2019å¹´10æœˆ23æ—¥ï¼ˆæ°´æ›œæ—¥ï¼‰-10æœˆ25æ—¥ï¼ˆé‡‘æ›œæ—¥ï¼‰

https://twitter.com/hashtag/ws222

https://www.fifthdimension.jp/documents/molphytextbook/
åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ - ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§

https://www.fifthdimension.jp/documents/molphytextbook/answers.pdf
åˆ†å­ç³»çµ±è§£æã«ãŠã‘ã‚‹æ§˜ã€…ãªå•é¡Œã«ã¤ã„ã¦
```
ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°æ³•

â— å…¨ç¨®ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã¯å¿…ãšã—ã‚‚è‰¯ããªã„
â— ç³»çµ±æ¨¹ä¸Šã®åˆ†å²ç‚¹ãƒ»ç«¯ç‚¹ã®å¯†åº¦ãŒã§ãã‚‹ã ã‘åã‚‰ãªã„æ–¹ãŒè‰¯ã„
â€“ åŒä¸€é…åˆ—ã‚„è¿‘ç¸é…åˆ—ãŒä¸€éƒ¨ã§ã¯å¤šãä¸€éƒ¨ã§ã¯å°‘ãªã„ã®ã¯Ã—
```


https://togetter.com/li/742642
2014å¹´åº¦ãƒ»ç¬¬188å›è¾²æ—äº¤æµã‚»ãƒ³ã‚¿ãƒ¼ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—ã€ˆåˆ†å­ç³»çµ±å­¦ã®ç†è«–ã¨å®Ÿç¿’ã€‰ãƒ„ã‚¤ãƒ¼ãƒˆã¾ã¨ã‚

https://togetter.com/li/587872
2013å¹´åº¦ãƒ»ç¬¬176å›è¾²æ—äº¤æµã‚»ãƒ³ã‚¿ãƒ¼ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—ã€ˆåˆ†å­ç³»çµ±å­¦ã®ç†è«–ã¨å®Ÿç¿’ã€‰ãƒ„ã‚¤ãƒ¼ãƒˆã¾ã¨ã‚

https://togetter.com/li/399359
2012å¹´åº¦ãƒ»ç¬¬166å›è¾²æ—äº¤æµã‚»ãƒ³ã‚¿ãƒ¼ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—ã€ˆåˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•ï¼šç†è«–ã¨å¿œç”¨ã€‰ ãƒ„ã‚¤ãƒ¼ãƒˆã¾ã¨ã‚

----------
## basal

https://en.wikipedia.org/wiki/Basal_(phylogenetics)
deep-branching or early-branching are similar in meaning.

https://quizlet.com/97829330/phylogenetic-terms-flash-cards/
Phylogenetic Termså˜èªã‚«ãƒ¼ãƒ‰ | Quizlet

Root
the most basal internode at the bottom of a tree. This is the common ancestor.

### 2025-03

10:48 AM Â· May 30, 2025
https://x.com/Takashirouzu/status/1928267286265094463
ç™½æ°´ã€€è²´ on X: "èŒå­¦ã«ãŠã‘ã‚‹è„±â€œé€²æ­©ã®æ¢¯å­â€è«–æ–‡ï¼èŒé¡ã®é€²åŒ–ç ”ç©¶ã«ãŠã‘ã‚‹â€œearly divergingâ€ã‚„â€œbasalâ€ï¼Œâ€œlowerâ€ãªã©ã®ç”¨èªã®ä½¿ç”¨ã¯èª¤è§£ã‚’æ‹›ãã‚„ã™ãä¸æ­£ç¢ºã§ã‚ã‚‹ï¼Œã¨ã™ã‚‹ï¼ç‰¹å®šã®ç³»çµ±ãŒåˆæœŸçš„ã¾ãŸã¯åŸå§‹çš„ã«è¦‹ãˆãŸã¨ã—ã¦ã‚‚ãã‚Œã¯éŒ¯è¦šã‹èª¤è§£ï¼å½¢è³ªã«åŸºã¥ãè¨˜è¿°ã‚„åˆ†é¡ç¾¤åã‚’ç”¨ã„ã‚‹ã¹ãã¨çµè«–ï¼ https://t.co/EZZLwlDWEj" / X

2:23 PM Â· Mar 1, 2025
https://x.com/Hoyaboya11/status/1895706496253784118
Hoyaboya on X: "James &amp; Rokas (2025) äºŒæ ¸èŒé¡Dikaryaä»¥å¤–ã®èŒé¡ã‚’â€œearly diverging fungiâ€ï¼ˆåˆæœŸã«åˆ†å²ã—ãŸèŒé¡ï¼‰ã‚„ â€œbasal fungiâ€ï¼ˆåŸºç›¤çš„ãªèŒé¡ï¼‰ãªã©ã¨å‘¼ã¶ã®ã¯å³å¯†ã«ã¯é–“é•ã£ã¦ã„ã‚‹ä¸Šã€èª¤è§£ã‚’æ‹›ããŸã‚ã€åˆ†é¡ç¾¤ã®åå‰ã‚’ç›´æ¥è¡¨è¨˜ã—ã¾ã›ã‚“ã‹ã¨ä¸»å¼µã—ã¦ã„ã¾ã™ã€‚ã†ãƒ¼ã‚“ã€åŸºç›¤çš„ã¯è¨±ã—ã¦æ¬²ã—ã„ã‘ã©ãªã‚ã€‚" / X

https://pubmed.ncbi.nlm.nih.gov/40020155/
Mycologia
. 2025 Mar-Apr;117(2):246-254. doi: 10.1080/00275514.2025.2460003. Epub 2025 Feb 28.
Use their names: there are no basal, lower, or early diverging fungi
Timothy Y James 1, Antonis Rokas 2
https://www.tandfonline.com/doi/full/10.1080/00275514.2025.2460003

### 2018

2018/6/7
http://darwin.c.u-tokyo.ac.jp/sesj2018/symposium/
Basal lineageã¯ã€ŒåŸå§‹çš„ã€ã‹ï¼Ÿï¼šç”Ÿç‰©ç•Œã¨åˆ†é‡ã‚’è¶…ãˆã¦
http://darwin.c.u-tokyo.ac.jp/sesj2018/wp/wp-content/uploads/2018/08/5d5ef9dd328e3e7158e5fbaff5ce1d98.pdf
ç‰¹å®šã®ç”Ÿç‰©ç¾¤ã«ãŠã„ã¦æ—©ã„æ™‚æœŸã«åˆ†å²ã—ãŸç³»çµ±ã¯å¾€ã€…ã«ã—ã¦ã€Œbasal lineageã€ã¨å‘¼ã°ã‚Œã€ãã®ç”Ÿç‰©ç¾¤ã®åŸå§‹
çš„ãªè¡¨ç¾å‹ã‚’æŒã¤ã¨è€ƒãˆã‚‰ã‚Œã¦ã„ã‚‹ã“ã¨ãŒå¤šã„ã€‚è¿‘å¹´ã€ã‚²ãƒãƒ æƒ…å ±ã‚’åˆ©ç”¨ã—ãŸéºä¼å­æ©Ÿèƒ½ã®åˆ†åŒ–ãƒ¡ã‚«ãƒ‹ã‚ºãƒ 
ã®ç ”ç©¶ãªã©ã‹ã‚‰ã€ã€Œbasal lineage ã¯åŸå§‹çš„ã§ã‚ã‚‹ã€ã¨ã„ã†è¦³å¿µã®å†æ¤œè¨ã®å¿…è¦ãŒéœ²ã‚ã¨ãªã£ã¦ã„ã‚‹ã€‚å¤šè§’çš„
ã«ã“ã®å•é¡Œã‚’è­°è«–ã™ã‚‹ãŸã‚ã€ç”Ÿç‰©ç•Œã‚’è¶…ãˆãŸ basal lineage ã®ä¾‹ã«ã¤ã„ã¦è§¦ã‚Œã€åˆ†é‡ã‚’ã¾ãŸã„ã§è­°è«–ã™ã‚‹ã€‚

Aug 22, 2018
https://www.slideshare.net/leeswijzer/minaka-sesj2018-slideshare
Minaka sesj2018 slideshare
è¬›æ¼”ã‚¹ãƒ©ã‚¤ãƒ‰â€”æ—¥æœ¬é€²åŒ–å­¦ä¼šç¬¬20å›å¤§ä¼šã‚·ãƒ³ãƒã‚¸ã‚¦ãƒ ãƒ»S5ã€ˆBasal lineageã¯ã€ŒåŸå§‹çš„ã€ã‹ï¼Ÿï¼šç”Ÿç‰©ç•Œã¨åˆ†é‡ã‚’è¶…ãˆã¦ã€‰ï¼šä¸‰ä¸­ä¿¡å®ã€ŒBasal Lineage ã®å¹»æƒ³ã¨å®Ÿæ…‹ï¼šç³»çµ±æ¨¹ã®æœ‰æ ¹åŒ–ã¨å½¢è³ªçŠ¶æ…‹å¾©å…ƒã®è¦³ç‚¹ã‹ã‚‰ã€2018å¹´8æœˆ22æ—¥ï¼ˆæ°´ï¼‰15:50ã€œ17:50ï¼ æ±äº¬å¤§å­¦é§’å ´ã‚­ãƒ£ãƒ³ãƒ‘ã‚¹5å·é¤¨ï¼ˆRoom 5ï¼‰

- https://twitter.com/search?q=Basal%20Lineage%20lang%3Aja
  - https://twitter.com/copypasteusa/status/1032263315752284160
#sesj2018 #æ—¥æœ¬é€²åŒ–å­¦ä¼šç¬¬20å›å¤§ä¼š #é€²åŒ–å­¦ä¼š
Basal lineageã¯ã€Œãªã„ã€
Sister taxaã¯ã€Œã‚ã‚‹ã€

2022/11/30
https://www.youtube.com/watch?v=lAWAdZA38aU
DNAæƒ…å ±ã‹ã‚‰è¦‹ã‚‹å‹•ç‰©ã®é€²åŒ–ï¼šå·¥æ¨‚ æ¨¹æ´‹ æ•™æˆã€éºä¼ç ”å…¬é–‹è¬›æ¼”ä¼š2022ã€‘
https://youtu.be/lAWAdZA38aU?t=645

https://researchmap.jp/genombaum/misc/40127747
2017å¹´
å††å£é¡ãƒŒã‚¿ã‚¦ãƒŠã‚®ã¨ãƒ¤ãƒ„ãƒ¡ã‚¦ãƒŠã‚®ã®åˆ†å­é€²åŒ–å­¦
æœˆåˆŠ æµ·æ´‹
ãƒ€ã‚¦ãƒ³ãƒ­ãƒ¼ãƒ‰

å›³ 2C ã«ãŠã„ã¦ãƒ’ãƒˆãŒåŸå§‹çš„ã¨ã„ãˆãªã„ã®ã¨åŒæ§˜ã«ï¼ŒçŸ­çµ¡çš„ã«ãƒ¤ãƒ„ãƒ¡ã‚¦ãƒŠã‚®ãŒåŸå§‹çš„ã¨è€ƒãˆã‚‹ã®ã¯å±é™ºã§ã‚ã‚‹ã€‚ã“ã®å±é™ºã‚’æŒ‡æ‘˜ã™ã‚‹ã®ã« primitive lineage fallacy ã¨ã„ã†ãƒ•ãƒ¬ãƒ¼ã‚ºãŒä½¿ã‚ã‚Œã‚‹ã“ã¨ã‚‚ã‚ã‚‹ (Omland et al., 2008)

å›³ 2 è¦‹ã›æ–¹ã«ã‚ˆã£ã¦å°è±¡ãŒå¤§ããå¤‰ã‚ã‚Šã†ã‚‹ç³»çµ±æ¨¹ã€‚Kuraku et al., 2016 ã‚ˆã‚Šæ”¹å¤‰

https://pubmed.ncbi.nlm.nih.gov/18693264/
Bioessays
. 2008 Sep;30(9):854-67. doi: 10.1002/bies.20794.
Tree thinking for all biology: the problem with reading phylogenies as ladders of progress
Kevin E Omland 1, Lyn G Cook, Michael D Crisp
https://onlinelibrary.wiley.com/doi/10.1002/bies.20794
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_69433249
Wiley Online Library Database Model 

https://pubmed.ncbi.nlm.nih.gov/26818824/
Review Dev Growth Differ
. 2016 Jan;58(1):131-42. doi: 10.1111/dgd.12258. Epub 2016 Jan 5.
Incorporating tree-thinking and evolutionary time scale into developmental biology
Shigehiro Kuraku 1, Nathalie Feiner 2, Sean D Keeley 1, Yuichiro Hara 1
https://onlinelibrary.wiley.com/doi/10.1111/dgd.12258

https://archosaurmusings.wordpress.com/2008/12/18/how-to-read-a-phylogenetic-tree/
How to read a phylogenetic tree | Dave Hone's Archosaur Musings
basal taxa (at the bottom of the tree)

14 June 2004
https://resjournals.onlinelibrary.wiley.com/doi/10.1111/j.0307-6970.2004.00262.x
Which side of the tree is more basal?
Frank-T Krell, Peter S. Cranston

----------
## saturation
Substitution saturation
é£½å’Œ
transition/transversion

https://en.wikipedia.org/wiki/Genetic_saturation

### saturation_PhyKIT

https://www.preprints.org/manuscript/202404.1514/v1
PhyKIT: A Multitool for Phylogenomics
https://www.preprints.org/manuscript/202404.1514/v1/download
preprints202404.1514.v1.pdf
|
Figure 18. Cartoon depiction of saturation.

https://pubmed.ncbi.nlm.nih.gov/33560364/
Bioinformatics
. 2021 Aug 25;37(16):2325-2331. doi: 10.1093/bioinformatics/btab096.
PhyKIT: a broadly applicable UNIX shell toolkit for processing and analyzing phylogenomic data
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8388027/
(11)Saturation. Saturation refers to when an MSA contains many sites that have experienced multiple substitutions in individual taxa. Saturation is estimated from the slope of the regression line between patristic distances and pairwise identities. Saturated MSAs have reduced phylogenetic information and can result in issues of long branch attraction (Lake, 1991; Philippe et al., 2011).

https://jlsteenwyk.com/PhyKIT/usage/index.html#saturation
Calculate saturation for a given tree and alignment.

Saturation is defined as sequences in multiple sequence alignments that have undergone numerous substitutions such that the distances between taxa are underestimated.

Data with no saturation will have a value of 1. The closer the value is to 1, the less saturated the data.

Saturation is calculated following Philippe et al., PLoS Biology (2011), 

https://jlsteenwyk.com/PhyKIT/tutorials/index.html#saturation
Saturation in a multiple sequence alignments is driven by sites with multiple substitutions and results in the alignment underestimating real genetic distances among taxa. Values of 1 have no saturation and values of 0 are completely saturated by multiple substitutions (Philippe et al. 2011).

https://pubmed.ncbi.nlm.nih.gov/21423652/
PLoS Biol
. 2011 Mar;9(3):e1000602. doi: 10.1371/journal.pbio.1000602. Epub 2011 Mar 15.
Resolving difficult phylogenetic questions: why more sequences are not enough
HervÃ© Philippe 1, Henner Brinkmann, Dennis V Lavrov, D Timothy J Littlewood, Michael Manuel, Gert WÃ¶rheide, Denis Baurain
https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1000602
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3057953/
Saturation: When sequences in a multiple alignment have undergone so many multiple substitutions that apparent distances largely underestimate the real genetic distances, the alignment is said to be saturated. Phylogenetic inference works best with datasets that are only slightly saturated. Owing to their reduced state space (four possible bases), nucleotide sequences saturate more rapidly than protein sequences (20 possible amino acids).

Depending on both functional constraints and evolutionary trajectory, different genes can include positions subject to different ranges of multiple substitutions, i.e., they may display variable levels of saturation. To estimate the saturation in the three datasets [2]â€“[4], we used the comparison of patristic and uncorrected distances [47]. As shown by the slope of the regression line (data without any saturation have slopeâ€Š=â€Š1; see [12]), the three datasets (Figure 5) are different, with that of Schierwater et al. being the most saturated (slopeâ€Š=â€Š0.38) and that of Philippe et al. the least affected by multiple substitutions (slopeâ€Š=â€Š0.53). 

Figure 5. Saturation levels of datasets from Schierwater et al., Dunn et al., and Philippe et al.
The level of saturation was estimated for each dataset by computing the slope of the regression line of patristic distances (y-axis) versus uncorrected distances (x-axis), as previously described [12]. 

01 March 1994
https://academic.oup.com/jeb/article-abstract/7/2/247/7322433
https://onlinelibrary.wiley.com/doi/pdf/10.1046/j.1420-9101.1994.7020247.xFig. 6. (A) Comparison, for SOD data, between the number of substitutions estimated from the most parsimonious tree (X-axis) and the number of observed differences (Y-axis). (B) Comparison, for our rRNA data setbetween the number of substitutions estimated from the most parsimonious tree (X axis) and the number of observed differences (Y axis).
### saturation_PhyloSuite
2023/03/03
https://www.youtube.com/watch?v=qtAL8X3314g
4:58

https://onlinelibrary.wiley.com/doi/10.1002/imt2.87
Using PhyloSuite for molecular phylogeny and treeâ€based analyses - Xiang - 2023 - iMeta - Wiley Online Library

Substitution saturation analysis
What is substitution saturation?

Box 9: Why analyze substitution saturation?

Substitution saturation is often strongly pronounced in datasets comprising distantly-related lineages or rapidly evolving sequences (sites). Identification and removal of loci that exhibit substitution saturation can improve the reliability of phylogenetic tree reconstruction [72, 73].

How to analyze substitution saturation in PhyloSuite?

### 2022-04-19
https://pubmed.ncbi.nlm.nih.gov/34508605/
Syst Biol
. 2022 Apr 19;71(3):676-689. doi: 10.1093/sysbio/syab075.
Excluding Loci With Substitution Saturation Improves Inferences From Phylogenomic Data
David A DuchÃªne 1, Niklas Mather 2, Cara Van Der Wal 2, Simon Y W Ho 2
https://academic.oup.com/sysbio/article/71/3/676/6368692
https://pmc.ncbi.nlm.nih.gov/articles/PMC9016599/

### 2021-07-26
https://www.frontiersin.org/articles/10.3389/fmars.2021.696523/full
Frontiers | Commentary: Unbiasing Genome-Based Analyses of Selection: An Example Using Iconic Shark Species
Nicholas J. Marra1 Michael J. Stanhope2 Nathaniel K. Jue3 Vincent P. Richards4 Stephen J. O'Brien5,6 Agostinho Antunes7,8 Mahmood S. Shivji5,9*

Response To Investigation of Substitution Saturation

### 2021-03-10
https://www.frontiersin.org/articles/10.3389/fmars.2021.573853/full
Frontiers | Unbiasing Genome-Based Analyses of Selection: An Example Using Iconic Shark Species
Kazuaki Yamaguchi Shigehiro Kuraku*

FIGURE 1
(B) Substitution saturation plots for transition and transversion of the coding region of the Fgg, Mdm4, Chek2, and Dtl genes chosen from those previously regarded as positively selected (Marra et al., 2019).

Investigation of Substitution Saturation

To investigate possible substitution saturation, we further analyzed the Fgg, Mdm4, Chek2, and Dtl genes. Indices of substitution saturation (Iss), introduced by Xia (2009), were computed for the first, second, and third codon positions of ortholog sequences using the DAMBE program (Xia, 2018). 

### 2017
https://digitalcommons.usu.edu/biology_posters/
PDF
Multiple Sequence Alignment While Assessing Saturation Across Sequence Data, K. Bodie Weedop and William D. Pearse
https://digitalcommons.usu.edu/biology_posters/165/

### 2012
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/fb8nrm/alma99186639204031
SpringerLink Books (Series & Reference Works)
https://link.springer.com/book/10.1007/978-1-4614-1743-9
Analysis of Phylogenetics and Evolution with R | SpringerLink
https://link.springer.com/chapter/10.1007/978-1-4614-1743-9_5
130 5 Phylogeny Estimation
5.1.2 Exploring and Assessing Distances
|
On the other hand, large distances will lead to a similar problem because of mutation saturation: the JC69 distance cannot exceed 0.75, but in practice distances greater than 0.3 lead to difficulties.
A second useful, though very simple, graphical exploration tool is to plot two sets of distances computed on the same data with the goal to contrast the effect of different methods. A form of this approach is well-known as the â€˜saturation plotâ€™ where the number of transitions or transversions is plotted against the K80 distance (see p. 191), but this can be generalized. 

5.8 Case Studies 189 
Table 5.5. Some points to be considered in phylogeny estimation
|
Histogram and saturation plots

190 5 Phylogeny Estimation
5.8.1 Sylvia Warblers
|
This shows that the GG95 distances differ substantially from the others. Note that a perfect correlation does not guarantee that the distances are the same: some graphical analyses are needed to check this. We do this to examine the saturation of substitutions in the sequences. We first compute the distances using the JC69 model and the raw distance (i.e., proportion of different sites):

5.8 Case Studies 191
|
Fig. 5.15. Saturation plots for the cytochrome b sequences of 25 species of Sylvia showing the effects of multiple substitutions (left) and of the transition/transversion ratio (right)

This analysis, though informative, is not what is usually called â€œsaturation plotsâ€ in the literature. The latter is a plot, eventually for each codon position, of the numbers of transitions and transversions on the x-axis against the K80 distance on the y-axis. 

http://www.iu.a.u-tokyo.ac.jp/lectures/AG01/180502/20180502.pdf
Microsoft PowerPoint - 2018_ç”Ÿç‰©é…åˆ—è§£æåŸºç¤_3å›ç›®.pptx
ï¼‘ã¤ã®éºä¼â¼¦ã®ç³»çµ±è§£æã ã‘ã§ã¯é€²åŒ–ã®æ­´å²ã‚’èª¿ã¹ã‚‹ã®ã«ä¸â¼—åˆ†ãªå ´åˆãŒã‚ã‚‹
- ç•°ãªã‚‹éºä¼â¼¦ã‚’â½¤ã„ã¦ç³»çµ±æ¨¹ã‚’ä½œæˆã—ãŸå ´åˆã«ï¼Œãƒˆãƒãƒ­ã‚¸ãƒ¼ãŒâ¼€è‡´ã—ãªã„ã“ã¨ãŒã‚ã‚‹
- ã“ã‚Œã«ã¯ï¼Œéºä¼â¼¦ã®â½”å¹³ç§»å‹•ï¼Œåˆ†å²å¹´ä»£ã®è¿‘ã•ï¼Œå¡©åŸºãƒ»ã‚¢ãƒŸãƒé…¸ç½®æ›ã®é£½å’Œï¼Œå€‹ã€…ã®éºä¼â¼¦ã«ã‹ã‹ã‚‹é¸æŠåœ§ã®é•ã„ãªã©ï¼Œæ§˜ã€…ãªåŸå› ãŒè€ƒãˆã‚‰ã‚Œã‚‹

2014å¹´01æœˆ21æ—¥
https://bookclub.kodansha.co.jp/product?item=0000194810
ã€åˆ†å­ã‹ã‚‰ã¿ãŸç”Ÿç‰©é€²åŒ–ã€€ï¼¤ï¼®ï¼¡ãŒæ˜ã‹ã™ç”Ÿç‰©ã®æ­´å²ã€ï¼ˆå®®ç”°ã€€éš†ï¼‰
p. 326, 358
é£½å’Œ

http://www.fish-evol.org/DatabaseEnglish.html
ç³»çµ±è§£æã®è‹±èªä¾‹æ–‡
2012 å¹´ 7 æœˆ 16 æ—¥
äº•ä¸Š æ½¤ã€€

3rd ãƒã‚¸ã‚·ãƒ§ãƒ³
ã“ã®çµæœã¯ï¼Œçµ±è¨ˆçš„æœ‰ç‚ºã•ã«æ‚ªå½±éŸ¿ã‚’åŠã¼ã™é«˜åº¦ã®é£½å’Œã‚’ç¤ºã—ã¦ã„ã‚‹ï¼ã“ã®ãŸã‚ï¼Œè§£æã‹ã‚‰é™¤å¤–ã™ã‚‹ï¼
This preliminary result indicates
a high degree of saturation that severely affected the statistical significance of the estimates (results not shown) and,
thus, we excluded these sites from the analysis (Schrago and Russo, 2003).

é£½å’Œã—ã¦ã„ã‚‹ãŒï¼Œ3rd ã«ã¯ç³»çµ±æƒ…å ±ãŒå«ã¾ã‚Œã¦ã„ã‚‹ã“ã¨ãŒã‚ã‚‹ï¼
Third codon positions for both mitochondrial genes,
despite obvious potential saturation problems (Cao et al. 1994),
often contain phylogenetic information, even among distantly related species (Zardoya and Meyer, 1996).

ç³»çµ±æ¨¹ãŒè§£ã‘ãªã„ã¨ãã®ç†ç”±
3rd ã® TV ã§ã‚‚ï¼Œã„ãã‚‰ã‹é£½å’ŒãŒã‚ã‚‹ã®ã¯æ˜ã‚‰ã‹ï¼
It was apparent that some degree of saturation also occurred in 3rd-codons TVs (Yamanoue et al. 2009).

https://pubmed.ncbi.nlm.nih.gov/26609078/
Mol Biol Evol
. 2016 Mar;33(3):595-602. doi: 10.1093/molbev/msv274. Epub 2015 Nov 25.
On the Causes of Evolutionary Transition:Transversion Bias
Arlin Stoltzfus 1, Ryan W Norris 2
https://academic.oup.com/mbe/article/33/3/595/2579658
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107541/

https://pubmed.ncbi.nlm.nih.gov/25886870/
BMC Evol Biol
. 2015 Mar 11;15:36. doi: 10.1186/s12862-015-0312-6.
Declining transition/transversion ratios through time reveal limitations to the accuracy of nucleotide substitution models
SebastiÃ¡n DuchÃªne 1, Simon Y W Ho 2, Edward C Holmes 3 4
https://bmcecolevol.biomedcentral.com/articles/10.1186/s12862-015-0312-6
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4358783/

https://pubmed.ncbi.nlm.nih.gov/17274688/
PLoS Genet
. 2007 Feb 2;3(2):e22. doi: 10.1371/journal.pgen.0030022.
Transition-transversion bias is not universal: a counter example from grasshopper pseudogenes
Irene Keller 1, Douda Bensasson, Richard A Nichols
https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030022
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1790724/

https://pubmed.ncbi.nlm.nih.gov/10093216/
J Mol Evol
. 1999 Mar;48(3):274-83. doi: 10.1007/pl00006470.
Estimation of the transition/transversion rate bias and species sampling
Z Yang 1, A D Yoder
https://link.springer.com/article/10.1007%2FPL00006470

----------
## rate_of_evolution
é€²åŒ–é€Ÿåº¦

https://en.wikipedia.org/wiki/Rate_of_evolution

https://pubmed.ncbi.nlm.nih.gov/31570957/
J Mol Evol
. 2019 Dec;87(9-10):317-326. doi: 10.1007/s00239-019-09912-5. Epub 2019 Sep 30.
Investigating Evolutionary Rate Variation in Bacteria
Beth Gibson 1, Adam Eyre-Walker 2
https://link.springer.com/article/10.1007/s00239-019-09912-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858405/
 Across species, we find that accumulation rates vary by over 3700-fold. We investigate whether accumulation rates are associated to a number potential correlates including genome size, GC content, measures of the natural selection and the time frame over which the accumulation rates were estimated. After controlling for phylogenetic non-independence, we find that the accumulation rate is not significantly correlated to any factor. Furthermore, contrary to previous results, we find that it is not impacted by the time frame of which the estimate was made.

We compiled estimates of the accumulation rate for 34 species of bacteria. These vary by over 3700 fold (Fig. 1.), but the majority of species accumulate mutations at rates of between 1â€‰Ã—â€‰10âˆ’6 and 2â€‰Ã—â€‰10âˆ’6 per site per year. 

Fig.  1
Distribution of accumulation rate estimates for 34 species of bacteria

DuchÃªne S, et al. Genome-scale rates of evolutionary change in bacteria. Microb Gen. 2016;2(11):e000094. [PMC free article] [PubMed] [Google Scholar]

2016-11-30
https://pubmed.ncbi.nlm.nih.gov/28348834/
Microb Genom
. 2016 Nov 30;2(11):e000094. doi: 10.1099/mgen.0.000094. eCollection 2016 Nov.
Genome-scale rates of evolutionary change in bacteria
Sebastian DuchÃªne 1 2 3, Kathryn E Holt 2 3, FranÃ§ois-Xavier Weill 4, Simon Le Hello 4, Jane Hawkey 2 3, David J Edwards 2 3, Mathieu Fourment 1, Edward C Holmes 1
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000094#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320706/
The robustly estimated evolutionary rates spanned several orders of magnitude, from approximately 10âˆ’5 to 10âˆ’8 nucleotide substitutions per site yearâˆ’1. 

Despite the wealth of sequence data, genome-wide rates of evolutionary change in bacteria are often uncertain. At one end of the spectrum, rates as high as ~10âˆ’5 nucleotide substitutions per site yearâˆ’1 have been reported for Neisseria gonorrhoeae (PÃ©rez-Losada et al., 2007). In contrast, genome-wide rates of only ~10âˆ’9 substitutions per site yearâˆ’1 have been observed in Mycobacterium tuberculosis (Comas et al., 2013). Importantly, however, these estimates are not always readily comparable because they use different methods and sources of data. 

To provide a comprehensive picture of genomic-scale evolutionary rates in bacteria and their temporal dynamics, particularly the extent of time-dependency in the data, we analysed, using a variety of phylogenetic methods, 36 publically available whole genome SNP data sets from bacterial pathogens associated with human disease sampled over periods extending over 2000 years.

https://pubmed.ncbi.nlm.nih.gov/33560364/
PhyKIT
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8388027/
estimation of mutation rate

https://jlsteenwyk.com/PhyKIT/usage/index.html#evolutionary-rate
Calculate a tree-based estimation of the evolutionary rate of a gene.
Evolutionary rate is the total tree length divided by the number of terminals.
Calculate evolutionary rate following Telford et al., Proceedings of the Royal Society B (2014).

https://jlsteenwyk.com/PhyKIT/usage/index.html#covarying-evolutionary-rates
Determine if two genes have a signature of covariation with one another. Genes that have covarying evolutionary histories tend to have similar functions and expression levels.

2015/06/12
https://www.youtube.com/watch?v=2zPXHmqM3rk&list=RDLV6_XMKmFQ_w8
2. Phylogenetics & Phylogeography(lecture-part 2)
a.Why consider time?

| | RNA Viruses | DNA Viruses | Bacteria |
|:---------|:---------|:---------|:---------|
| Mutations per year | 10-100 | 1-20 | 0-1 |

6:01 AM Â· Jul 18, 2018
https://twitter.com/ASMicrobiology/status/1019326186625912832
ASM on Twitter: "Complexities of viral mutation rates, including 3 hypotheses for why mutation rates have not evolved to be zero, in a new @JVirology Gem: https://t.co/bWwl3Xa6Ti https://t.co/SnGlN65JxL" / Twitter

https://pubmed.ncbi.nlm.nih.gov/29720522/
Review J Virol
. 2018 Jun 29;92(14):e01031-17. doi: 10.1128/JVI.01031-17. Print 2018 Jul 15.
Complexities of Viral Mutation Rates
Kayla M Peck 1, Adam S Lauring 2 3
https://journals.asm.org/doi/10.1128/JVI.01031-17
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6026756/
substitutions per nucleotide site per cell infection (s/n/c)

https://covid-19chronicles.cseas.kyoto-u.ac.jp/post-041-jp-html/
æ±å—ã‚¢ã‚¸ã‚¢ã«ãŠã‘ã‚‹æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ã®çªç„¶å¤‰ç•°ã¨æ‹¡æ•£â”€â”€ä»Šå¾Œã®ãƒ¯ã‚¯ãƒãƒ³ã®è¡Œæ–¹â”€â”€ | Corona Chronicles: Voices from the Field

å›³3 RNAã‚¦ã‚¤ãƒ«ã‚¹ã®é€²åŒ–é€Ÿåº¦ã®æ¯”è¼ƒã€‚é€²åŒ–é€Ÿåº¦ï¼ˆs/n/yã€å¡©åŸºç½®æ›ã‚’å¡©åŸºã‚ãŸã‚Šã€å¹´ã‚ãŸã‚Šã§ç¤ºã™ï¼‰ã¯yè»¸ä¸Šã€ã‚²ãƒãƒ ã‚µã‚¤ã‚ºï¼ˆntã€å¡©åŸºæ•°ã«åŸºã¥ãï¼‰ã¯xè»¸ä¸Šã«ç¤ºã™ã€‚ã‚ˆã‚Šå¤§ããªã‚²ãƒãƒ ã‚’æŒã¤ç”Ÿå‘½ä½“ã¯ã€å¤‰ç•°é€Ÿåº¦ãŒã‚ˆã‚Šé…ã„å‚¾å‘ã«ã‚ã‚‹ã€‚SARS-CoV-2ã‚’å«ã‚€ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ç§‘ã¯ã€30,000ntã¨ã„ã†å·¨å¤§ãªã‚²ãƒãƒ ã‚’æŒã¡ã€ãã®ä»–ã®RNAã‚¦ã‚¤ãƒ«ã‚¹ã«æ¯”ã¹ã¦é…ã„å¤‰ç•°ã‚’ç¤ºã™ã€‚DNAã‚¦ã‚¤ãƒ«ã‚¹ã®å˜ç´”ãƒ˜ãƒ«ãƒšã‚¹ã‚¦ã‚¤ãƒ«ã‚¹ã¯ã€æ¯”è¼ƒã®ãŸã‚ã«ç¤ºã™ã‚‚ã®ã€‚ãƒ‡ãƒ¼ã‚¿ã¯è¤‡æ•°ã®è³‡æ–™ã‚’åŸºã«ç·¨é›†ã—ãŸ16,17,19â€“22ã€‚


https://www.nig.ac.jp/museum/evolution-x/01_c.html
éºä¼å­¦é›»å­åšç‰©é¤¨

åˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬ã€€ï½æœ¨æ‘è³‡ç”Ÿã¨ä¸­ç«‹èª¬
(1) ä¸­ç«‹èª¬ã®èª•ç”Ÿ

(2)ä¸­ç«‹èª¬ã®å±•é–‹

ã€Œæ©Ÿèƒ½çš„ã«é‡è¦ã§ãªã„åˆ†å­(ã¾ãŸã¯åˆ†å­å†…ã®é‡è¦ã§ãªã„éƒ¨åˆ†)ã»ã©ã€ãã†ã§ãªã„ã‚‚ã®ã‚ˆã‚Šé€²åŒ–ã®éç¨‹ã§ã‚¢ãƒŸãƒé…¸ã‚„DNAå¡©åŸºã®ç½®æ›ãŒæ€¥é€Ÿã«èµ·ã“ã‚Šã€ç½®æ›ç‡(é€²åŒ–é€Ÿåº¦)ã®æœ€é«˜ã¯çªç„¶å¤‰ç•°ç‡ã§æ±ºã¾ã‚‹ã€ã¨ã„ã†è€ƒãˆã«é”ã—ãŸã®ã¯1973å¹´é ƒã§ã€ã“ã‚ŒãŒä¼çµ±çš„ãªé€²åŒ–éºä¼å­¦ã‹ã‚‰è¦‹ã‚Œã°å…¨ãã®ç•°ç«¯çš„ãªè€ƒãˆã§ã‚ã‚‹ã“ã¨ã¯ååˆ†æ‰¿çŸ¥ã—ã¦ã„ãŸã€‚

(3)ä¸­ç«‹èª¬ã®ç¢ºç«‹
ç­†è€…ã¯1977å¹´ã«ã€ã‚‚ã—ã€é€²åŒ–é€Ÿåº¦ã®ä¸Šé™ãŒçªç„¶å¤‰ç•°ç‡ã§æ±ºã¾ã‚‹ã¨ã„ã†çµæœãŒåˆ†å­é€²åŒ–ã®ç ”ç©¶ã‹ã‚‰å°†æ¥å‡ºã‚Œã°ã€ãã‚Œã¯ä¸­ç«‹èª¬ã‚’æ”¯æŒã™ã‚‹ã“ã¨ã«ãªã‚‹ã¨ã„ã†ä¸»å¼µã‚’ç™ºè¡¨ã—ãŸãŒã€ã—ã°ã‚‰ãã—ã¦ã€äºˆæƒ³ã‚‚ã—ãªã‹ã£ãŸã¨ã“ã‚ã‹ã‚‰ã“ã‚Œã‚’æ”¯æŒã™ã‚‹åŠ‡çš„ãªãƒ‡ãƒ¼ã‚¿ãŒå‡ºã¦ããŸã€‚ãã‚Œã¯ãƒã‚¦ã‚¹ã§è¦‹ã„å‡ºã•ã‚ŒãŸãƒ˜ãƒ¢ã‚°ãƒ­ãƒ“ãƒ³ã®å½éºä¼å­ã«é–¢ã™ã‚‹ã‚‚ã®ã§ã‚ã‚‹ã€‚ã“ã“ã§ã€å½éºä¼å­(pseudogene)ã¨ã¯ã€æ—¢çŸ¥ã®æ­£å¸¸éºä¼å­ã¨å¡©åŸºé…åˆ—ã®ä¸Šã§éå¸¸ã«ä¼¼ã¦ã„ã‚‹ãŒã€æ­£å¸¸éºä¼å­ã‹ã‚‰é‡è¤‡ã«ã‚ˆã£ã¦ç”Ÿã¾ã‚ŒãŸå¾Œã€ä½•ã‚‰ã‹ã®ç†ç”±ã§éºä¼å­ã¨ã—ã¦ã®æ©Ÿèƒ½ã‚’å¤±ã£ãŸã‚‚ã®ã§ã€ã€Œæ­»ã‚“ã éºä¼å­ã€ã¨ã‚ˆã°ã‚Œã‚‹ã“ã¨ã‚‚ã‚ã‚‹ã‚‚ã®ã§ã‚ã‚‹ã€‚ä¹å·å¤§å­¦ã®å®®ç”°éš†åšå£«ã¯ä¸–ç•Œã«å…ˆãŒã‘ã€ã“ã®é€²åŒ–é€Ÿåº¦ã®æ¨å®šã«æˆåŠŸã—ãŸã€‚å¾—ã‚‰ã‚ŒãŸå€¤ã¯DNAã®å¡©åŸºç½®æ›ç‡ã¨ã—ã¦ã¯æœ€é«˜ã¨è€ƒãˆã‚‰ã‚Œã‚‹å€¤ã§ã‚ã£ãŸã€‚

(4) ä¸­ç«‹èª¬ã¨åˆ†å­é€²åŒ–é€Ÿåº¦
4-1ï¼šé€Ÿåº¦ã®ä¸€å®šæ€§ã®èª¬æ˜

ã€€åˆ†å­é€²åŒ–ã«ã¯äºŒã¤ã®å¤§ããªç‰¹å¾´ã€ã™ãªã‚ã¡å¹´ã‚ãŸã‚Šã®ã€Œé€Ÿåº¦ã®ä¸€å®šæ€§ã€ã¨ã€å¤‰åŒ–æ§˜å¼ã®ã€Œä¿å®ˆæ€§ã€ãŒã‚ã‚‹ã€‚ã“ã‚Œã‚‰ã®ç‰¹å¾´ã¯ä¸­ç«‹èª¬ã«ã‚ˆã£ã¦ã©ã®ã‚ˆã†ã«èª¬æ˜ã•ã‚Œã‚‹ã§ã‚ã‚ã†ã‹ã€‚

ã€€ ä¸­ç«‹çªç„¶å¤‰ç•°ã«å¯¾ã—ã¦ã¯ã€Œé€²åŒ–é€Ÿåº¦=çªç„¶å¤‰ç•°ç‡ã€ã¨ã„ã†ç°¡å˜ãªæ³•å‰‡ãŒæˆã‚Šç«‹ã¤ã€‚ã—ãŸãŒã£ã¦ã€é€²åŒ–é€Ÿåº¦ã®å¹´ã‚ãŸã‚Šã®ä¸€å®šæ€§ã¯ä¸­ç«‹çªç„¶å¤‰ç•°ã®å‡ºç¾ç‡ãŒç‰¹å®šã®éºä¼å­(ã¾ãŸã¯ãã‚Œã‹ã‚‰ä½œã‚‰ã‚Œã‚‹ã‚¿ãƒ³ãƒ‘ã‚¯è³ª)ã«ã¤ã„ã¦å„ç¨®ã®ç”Ÿç‰©ã®ç³»çµ±ã§å¹´ã‚ãŸã‚Šä¸€å®šã§ã‚ã‚‹ã¨ä»®å®šã—ã¦èª¬æ˜ã•ã‚Œã‚‹ã€‚ã—ãŸãŒã£ã¦ã€ä¸­ç«‹èª¬ãŒæ­£ã—ã‘ã‚Œã°ã€åˆ†å­ãƒ¬ãƒ™ãƒ«ã®çªç„¶å¤‰ç•°(DNAå¡©åŸºã®å¤‰åŒ–)ã€ç‰¹ã«ä¸­ç«‹çªç„¶å¤‰ç•°ã®èµ·ã“ã‚‹ç‡ã¯ç’°å¢ƒæ¡ä»¶ã€é›†å›£ã®å¤§ãã•ã€ä¸€ä¸–ä»£ã®é•·ã•ã»ã¨ã‚“ã©å½±éŸ¿ã•ã‚Œãªã„ã¯ãšã§ã‚ã‚‹ã€‚

4-2ï¼šå˜ä½ã¯ä¸–ä»£ã‹ç‰©ç†çš„æ™‚é–“ã‹

4-3ï¼šçªç„¶å¤‰ç•°ç‡ã¯ç”Ÿæ®–ç´°èƒã®åˆ†è£‚å›æ•°ã«æ¯”ä¾‹

4-5ï¼šä¸–ä»£ã‚ãŸã‚Šã§ä¸€å®šã®çªç„¶å¤‰ç•°

è³‡æ–™ï¼šæœ¨æ‘è³‡ç”Ÿã€Œç”Ÿç‰©é€²åŒ–ã‚’è€ƒãˆã‚‹ã€å²©æ³¢æ–°æ›¸,1988

2005å¹´4æœˆ1æ—¥
https://www.brh.co.jp/research/formerlab/miyata/2005/post_000003.php
ã€ãƒ‘ãƒ©ãƒ€ã‚¤ãƒ ã‚·ãƒ•ãƒˆï¼šåˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬ã€‘ | JTç”Ÿå‘½èªŒç ”ç©¶é¤¨

ã‚‚ã—ã“ã®é–¢ä¿‚ã‚’çŸ¥ã£ã¦ã„ã‚Œã°ã€ç½®æ›æ•°ã‹ã‚‰åˆ†å²å¹´ä»£ãŒç°¡å˜ã«æ¨å®šã§ãã‚‹ã“ã¨ã«ãªã‚‹ã€‚
ã€€åˆ†å­æ™‚è¨ˆã¯ã€è¨€ã„æ–¹ã‚’å¤‰ãˆã‚Œã°ã€åˆ†å­é€²åŒ–é€Ÿåº¦ã®ä¸€å®šæ€§ã§ã‚ã‚‹ã€‚ã“ã®çµæœã¯ä¸­ç«‹èª¬ã§å®¹æ˜“ã«ç†è§£ã§ãã‚‹ã€‚ä¸­ç«‹èª¬ã«ã‚ˆã‚‹ã¨ã€åˆ†å­é€²åŒ–é€Ÿåº¦ï½‹ã¯ç·çªç„¶å¤‰ç•°ç‡Î¼ã«å¯¾ã™ã‚‹ä¸­ç«‹ãªçªç„¶å¤‰ç•°ã®å‰²åˆï½†ã«æ¯”ä¾‹ã™ã‚‹ã€‚
ã€€ã€€ã€€ã€€ã€€ã€€ã€€ã€€ã€€ã€€k ï¼ fãƒ»Î¼ã€€ã€€ã€€ã€€ã€€ã€€ã€€ã€€ã€€ã€€(1)

2005å¹´8æœˆ1æ—¥
https://www.brh.co.jp/research/formerlab/miyata/2005/post_000006.php
ã€€DNAã€RNAã€ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã¨ã„ã£ãŸåˆ†å­ã®é€²åŒ–é€Ÿåº¦ã¯æ©Ÿèƒ½çš„åˆ¶ç´„ã®å¼·ã•ã«ç›´æ¥ä¾å­˜ã™ã‚‹ã€‚åˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬ã«ã‚ˆã‚‹ã¨ã€åˆ†å­é€²åŒ–é€Ÿåº¦kã¯ä¸­ç«‹ãªçªç„¶å¤‰ç•°ç‡ã§æ±ºã¾ã‚‹ã€‚ã™ãªã‚ã¡ã€k=fãƒ»Î¼ã§ã‚ã‚‹ï¼ˆæœ¬ã‚·ãƒªãƒ¼ã‚ºã€Œãƒ‘ãƒ©ãƒ€ã‚¤ãƒ ã‚·ãƒ•ãƒˆï¼šåˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬ã€ã‚’å‚ç…§ï¼‰ã€‚ã“ã“ã§ã€Î¼ã¯ç·çªç„¶å¤‰ç•°ç‡ã§ã€fã¯ãã®ã†ã¡ã®ä¸­ç«‹ãªå¤‰ç•°ã®å‰²åˆã§ã‚ã‚‹ã€‚çªç„¶å¤‰ç•°ã¯å¤§ã–ã£ã±ã«æœ‰åˆ©ã€ä¸åˆ©ã€åŠã³ä¸­ç«‹ãªå¤‰ç•°ã«åˆ†ã‘ã‚‰ã‚Œã‚‹ãŒã€æœ‰åˆ©ãªå¤‰ç•°ã¯æ•°ã®ä¸Šã§éå¸¸ã«å°‘ãªã„ã®ã§ç„¡è¦–ã™ã‚‹ã¨ã€1-fã¯ä¸åˆ©ãªå¤‰ç•°ã®å‰²åˆã«ãªã‚Šã€æ©Ÿèƒ½çš„åˆ¶ç´„ã®å¤§ãã•ã«æ¯”ä¾‹ã™ã‚‹é‡ã§ã‚ã‚‹ã€‚å¾“ã£ã¦æ©Ÿèƒ½çš„åˆ¶ç´„ãŒå¤§ãããªã‚‹ã¨fãŒå°ã•ããªã‚Šã€é€²åŒ–é€Ÿåº¦kãŒå°ã•ããªã‚‹ã€‚ã“ã®ã“ã¨ã‚’å®Ÿéš›ã®ä¾‹ã§ç¢ºã‹ã‚ã¦ã¿ã‚ˆã†ã€‚

----------
## mutation

https://en.wikipedia.org/wiki/Mutation_bias

2020å¹´05æœˆ01æ—¥
https://hc.nikkan-gendai.com/articles/272230?page=2
ï¼²ï¼®ï¼¡ã‚¦ã‚¤ãƒ«ã‚¹ã¯å¡©åŸºé…åˆ—ãŒå¤‰ã‚ã‚Šã‚„ã™ãã€å¤‰ç•°ãŒè“„ç©ã™ã‚‹é€Ÿåº¦ã¯ãƒ’ãƒˆã®æ ¸ã‚²ãƒãƒ ã®ï¼¤ï¼®ï¼¡ã¨æ¯”ã¹ã¦ï¼‘ï¼ï¼ä¸‡å€é€Ÿã„ã¨ã„ã‚ã‚Œã¦ã„ã¾ã™ã€‚
æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ã®é€²åŒ–é€Ÿåº¦ã¯ã€ï¼³ï¼¡ï¼²ï¼³ã€ï¼­ï¼¥ï¼²ï¼³ã¨æ¯”è¼ƒã—ã¦ã€ã»ã¼åŒã˜ã§æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ã®ï¼²ï¼®ï¼¡ã‚²ãƒãƒ ã«ï¼‘å¹´é–“ã§è“„ç©ã•ã‚Œã‚‹å¡©åŸºå¤‰ç•°ã¯ï¼“ä¸‡å€‹ã®å¡©åŸºã®ã†ã¡ï¼’ï¼”å€‹ç¨‹åº¦ã¨æ¨å®šã•ã‚Œã¦ã„ã¾ã™ã€‚
https://www.yodosha.co.jp/jikkenigaku/special/SARS-CoV-2.html
æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹SARS-CoV-2ã®æ¯”è¼ƒã‚¦ã‚¤ãƒ«ã‚¹å­¦ã¨æ¯”è¼ƒã‚²ãƒãƒ è§£æï½œ2020å¹´5æœˆå·ï½œå®Ÿé¨“åŒ»å­¦onlineï¼šç¾ŠåœŸç¤¾ - ç¾ŠåœŸç¤¾
SARS-CoV-2ã®å¤‰ç•°
RNAã‚¦ã‚¤ãƒ«ã‚¹ã®è¤‡è£½ã‚’æ‹…ã†RNAãƒãƒªãƒ¡ãƒ©ãƒ¼ã‚¼ã¯æ ¡æ­£æ©Ÿæ§‹ã‚’æœ‰ã•ãªã„ãŸã‚ï¼Œçªç„¶å¤‰ç•°ç‡ãŒéå¸¸ã«é«˜ãï¼Œãã®ãŸã‚å¡©åŸºç½®æ›é€Ÿåº¦ï¼ˆé€²åŒ–é€Ÿåº¦ï¼‰ã‚‚éå¸¸ã«é€Ÿã„ï¼ãŸã ã—ï¼Œã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ã«ã¤ã„ã¦ã¯ä»–ã®RNAã‚¦ã‚¤ãƒ«ã‚¹ã¨ç•°ãªã‚Šï¼Œè¤‡è£½æ™‚ã®ã‚¨ãƒ©ãƒ¼ã‚’æ ¡æ­£ã™ã‚‹é…µç´ ã‚’ã‚‚ã¡15ï¼‰ï¼ŒSARS-CoV-2ã§ã‚‚æ©Ÿèƒ½ã—ã¦ã„ã‚‹ã¨è€ƒãˆã‚‰ã‚Œã‚‹ï¼2020å¹´3æœˆ6æ—¥ã«ã‚¨ã‚¸ãƒ³ãƒãƒ©å¤§å­¦ã®Andrew Rambautã‚‰ãŒå ±å‘Šã—ãŸSARS-CoV-2ã®176é…åˆ—ï¼ˆé€²åŒ–é€Ÿåº¦è§£æã«ä½¿ç”¨ã—ãŸé…åˆ—ã¯86é…åˆ—ï¼‰ã‚’ç”¨ã„ãŸè§£æçµæœã«ã‚ˆã‚‹ã¨ï¼Œé€²åŒ–é€Ÿåº¦ã¯0.80Ã—10ï¼3 substitution/site/yearã¨æ¨å®šã•ã‚ŒãŸï¼ˆ95ï¼…ä¿¡é ¼åŒºé–“ï¼š0.14ã€œ1.31Ã—10ï¼3ï¼‰ï¼ˆhttp://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356ï¼‰ï¼SARS-CoV-2ã®ã‚²ãƒãƒ ã‚µã‚¤ã‚ºãŒãŠã‚ˆã30Ã—103å¡©åŸºã¨ã„ã†ã“ã¨ã‚’è€ƒãˆã‚‹ã¨ï¼Œ1å¹´ã§ãŠã‚ˆã24ç®‡æ‰€ãã‚‰ã„ã®å¡©åŸºç½®æ›ãŒè“„ç©ã—ã¦ã„ãã¨è€ƒãˆã‚‰ã‚Œã‚‹â€»3ï¼
https://sites.google.com/site/sonakagawa/etc/sars-cov-2_yodosha_etc
ã€Œæ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹SARS-CoV-2ã®æ¯”è¼ƒã‚¦ã‚¤ãƒ«ã‚¹å­¦ã¨æ¯”è¼ƒã‚²ãƒãƒ è§£æã€åŸ·ç­†ã®ã‚ˆã‚‚ã‚„ã¾è©± - So Nakagawa Website
ãƒ»å¤‰ç•°ã«ã¤ã„ã¦ã€å…·ä½“çš„ã«1å¹´ã§24ç®‡æ‰€ãã‚‰ã„ã®å¤‰ç•°ã‚’è“„ç©ã™ã‚‹ã¨ã„ã†æ›¸ãæ–¹ã§ã™ãŒã€ã“ã‚Œã¯ã©ã®ãã‚‰ã„ã®å¤‰ç•°ãŒã‚ã‚‹ã®ã‹ã‚’æƒ³åƒã—ã¦ã»ã—ãã¦å…·ä½“çš„ãªæ•°å­—ã‚’ä¸Šã’ãŸã‚‚ã®ã§ã™ã€‚ã“ã‚Œã¯ã€ã‚¦ã‚¤ãƒ«ã‚¹é›†å›£ã®ãªã‹ã§æ®‹ã£ã¦ã„ã‚‹æ ªãŒãã‚Œãã‚Œã‚ªãƒªã‚¸ãƒŠãƒ«ã¨æ¯”è¼ƒã™ã‚‹ã¨24ç¨‹åº¦ã®çªç„¶å¤‰ç•°ãŒæ®‹ã£ã¦ã„ã‚‹ã¨ã„ã†ã ã‘ã§ã€é›†å›£ä¸­ã«ã¯ã‚‚ã£ã¨å¤šãã®çªç„¶å¤‰ç•°ãŒå­˜åœ¨ã—ã¾ã™ï¼ˆã“ã‚Œã¯é›†å›£ã‚µã‚¤ã‚ºã«ä¾å­˜ã—ã¾ã™ï¼‰ã€‚

https://sites.google.com/site/nosada17/Home/osada_text?authuser=0
Naoki Osada (CV) - ç”Ÿå‘½æƒ…å ±è§£æå­¦ãƒ†ã‚­ã‚¹ãƒˆ
https://drive.google.com/file/d/12TOTW2kOEtqC2G7vZRqRswH_5bNTXG5F/view
ç¬¬2ç«  é›†å›£éºä¼å­¦

ä¸–ä»£ã‚ãŸ ã‚Šã«å›ºå®šã™ã‚‹å¤‰ç•°ã®æ•°ã¯ Î¼ ã¨ãªã‚Šï¼Œçªç„¶å¤‰ç•°ç‡ã¨ç­‰ã—ããªã‚‹.ä¸–ä»£ã‚ãŸã‚Šã«å›ºå®šã™ã‚‹å¤‰ç•° ã®æ•°ã®ã“ã¨ã‚’åˆ†å­é€²åŒ–é€Ÿåº¦(rate of molecular evolution)ã¨å‘¼ã¶.ä¸­ç«‹ãªå¤‰ç•°ã§ã¯åˆ†å­é€²åŒ–é€Ÿ åº¦ã¯çªç„¶å¤‰ç•°ç‡ã«ç­‰ã—ããªã‚‹.ã“ã®é–¢ä¿‚ãŒåˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬(the neutral theory of molecular evolution)ã¨å‘¼ã°ã‚Œã‚‹èª¬ã®ä¸­å¿ƒã«ãªã‚‹.

https://drive.google.com/file/d/1gHH9qLdrOrOF5ptLaJCBIg_LJ25UfVFa/view?usp=sharing
ç¬¬3ç«  åˆ†å­é€²åŒ–
- ä¸­ç«‹ãªå¤‰ç•°ã®ç½® æ›é€Ÿåº¦ã¯çªç„¶å¤‰ç•°ç‡(Î¼)ã«ç­‰ã—ããªã‚‹ã“ã¨ã¯ç¬¬ 2 ç« ã¦ã‚™æ—¢ã«è¿°ã¸ã‚™ãŸ.
- Kimuraâ€™s 2-parameter model
å‰ç« ã¦ã‚™ã‚‚è¿°ã¸ã‚™ãŸã¨ãŠã‚Šï¼Œå¡©åŸºé–“ã®çªç„¶å¤‰ç•°ç‡ã¯(JC ãƒ¢ãƒ†ã‚™ãƒ«ã¦ã‚™ä»®å®šã—ã¦ã„ã‚‹ã‚ˆã†ã«)å‡ä¸€ã¦ã‚™ ã¯ãªã„.ä¸€èˆ¬çš„ã«ã¯ãƒˆãƒ©ãƒ³ã‚·ã‚™ãƒƒã‚·ãƒ§ãƒ³ã¨å‘¼ã¯ã‚™ã‚Œã‚‹ãƒ•ã‚šãƒªãƒ³ï¼Œãƒ’ã‚šãƒªãƒŸã‚·ã‚™ãƒ³åŒå£«ã®å¤‰ç•°ã‹ã‚™ãƒˆãƒ©ãƒ³ã‚¹ ãƒã‚™ãƒ¼ã‚·ã‚™ãƒ§ãƒ³ã¨å‘¼ã¯ã‚™ã‚Œã‚‹ãƒ•ã‚šãƒªãƒ³ï¼Œãƒ’ã‚šãƒªãƒŸã‚·ã‚™ãƒ³é–“ã¦ã‚™ã®å¤‰ç•°ã‚ˆã‚Šã‚‚ 2 å€ç¨‹åº¦èµ·ã“ã‚Šã‚„ã™ã„ã“ã¨ã‹ã‚™ çŸ¥ã‚‰ã‚Œã¦ã„ã‚‹.ã“ã®é•ã„ã‚’è€ƒæ…®ã«å…¥ã‚Œã¦ã„ã‚‹ã®ã‹ã‚™ï¼Œæœ¨æ‘ã® 2 ãƒã‚šãƒ©ãƒ¡ãƒ¼ã‚¿ãƒ¢ãƒ†ã‚™ãƒ«(Kimuraâ€™s 2- parameter model)ã¦ã‚™ã‚ã‚‹(
- ã‚µã‚¤ãƒˆã«ã‚ˆã‚‹ç½®æ›é€Ÿåº¦ã®é•ã„
ã“ã‚Œã¾ã¦ã‚™ã®ãƒ¢ãƒ†ã‚™ãƒ«ã¦ã‚™ã¯ã‚µã‚¤ãƒˆã“ã‚™ã¨ã®ç½®æ›é€Ÿåº¦ã¯ä¸€å®šã¨ä»®å®šã•ã‚Œã¦ã„ãŸ.ã¨ã“ã‚ã‹ã‚™ï¼Œéºä¼å­ã® ä¸­ã¦ã‚™ã‚‚å¡©åŸºé…åˆ—ã®ç½®æ›é€Ÿåº¦ã«ã¯å·®ã‹ã‚™ã‚ã‚‹ã“ã¨ã‹ã‚™ã‚ˆãçŸ¥ã‚‰ã‚Œã¦ã„ã‚‹.ã“ã‚Œã¯çªç„¶å¤‰ç•°ç‡ã‹ã‚™ã‚µ ã‚¤ãƒˆã«ã‚ˆã£ã¦é•ã£ã¦ã„ã‚‹ã“ã¨ã‹ã‚™åŸå› ã®ä¸€ã¤ãŸã‚™ã‚ã†.ä¾‹ãˆã¯ã‚™å‰è¿°ã—ãŸã‚ˆã†ã« CpG ã‚µã‚¤ãƒˆã® C ã¯å¤‰ç•°ç‡ã‹ã‚™ä»–ã«æ¯”ã¸ã‚™ã¦ã¨ã¦ã‚‚é«˜ã„.ç¨‹åº¦ã®å·®ã“ãã‚ã‚Œï¼Œã“ã®ã‚ˆã†ãªç½®æ›é€Ÿåº¦ã®ã¯ã‚™ã‚‰ã¤ãã‹ã‚™å­˜ åœ¨ã™ã‚‹.ã“ã®å•é¡Œã‚’è§£æ±ºã™ã‚‹ãŸã‚ã«æ§˜ã€…ãªæ–¹æ³•ã‹ã‚™ææ¡ˆã•ã‚Œã¦ããŸã‹ã‚™ï¼Œã‚ˆãç”¨ã„ã‚‰ã‚Œã¦ã„ã‚‹ã® ã‹ã‚™ï¼Œã‚µã‚¤ãƒˆã“ã‚™ã¨ã®ç½®æ›é€Ÿåº¦ã®ã¯ã‚™ã‚‰ã¤ãã‚’ã‚«ã‚™ãƒ³ãƒåˆ†å¸ƒ(gamma distribution)ã¨ä»®å®šã™ã‚‹ãƒ¢ãƒ†ã‚™ãƒ« ã¦ã‚™ã‚ã‚‹ (Yang1993).
- åŒç¾©ãƒ»éåŒç¾©ã‚µã‚¤ãƒˆ
éã‚³ãƒ¼ãƒˆã‚™é ˜åŸŸã«ãŠã„ã¦ã¯ï¼Œã€Œã‚µã‚¤ãƒˆå½“ãŸã‚Šã®çªç„¶å¤‰ç•°ç‡ã€ãªã¨ã‚™ã®ã‚ˆã†ã«å¡©åŸºã‚µã‚¤ãƒˆã®å®šç¾©ã¯ æ˜ç¢ºã¦ã‚™ã‚ã£ãŸ.ãã‚Œã¦ã‚™ã¯ã‚³ãƒˆã‚™ãƒ³ã‚’è€ƒãˆãŸå ´åˆã®åŒç¾©ãƒ»éåŒç¾©ã‚µã‚¤ãƒˆã®å®šç¾©ã¯ã¨ã‚™ã†ãªã‚‹ãŸã‚™ã‚ã†.
- éåŒç¾©ç½®æ›ã¨åŒç¾©ç½®æ›ã®æ¯”
éåŒç¾©ç½®æ›ç‡ã‚’ KA ã¾ãŸã¯ dNï¼ŒåŒç¾©ç½®æ›ç‡ã‚’ KS ã¾ãŸã¯ dS ã¨è¡¨è¨˜ã™ã‚‹ã“ã¨ã‹ã‚™å¤šã„.ã“ã‚Œã‚‰ã® æ¯”(ã‚³ãƒˆã‚™ãƒ³ãƒ¢ãƒ†ã‚™ãƒ«ã¦ã‚™ã¯ Ï‰ ã¨ã—ã¦æ¨å®šã•ã‚Œã‚‹ãƒã‚šãƒ©ãƒ¡ãƒ¼ã‚¿)ã¯éºä¼å­ã“ã‚™ã¨ã®ã‚³ãƒˆã‚™ãƒ³ã«å¯¾ã™ã‚‹è‡ª ç„¶é¸æŠã‚’è¡¨ã™æŒ‡æ¨™ã¨ã—ã¦ç”¨ã„ã‚‰ã‚Œã¦ã„ã‚‹.åŒç¾©ç½®æ›ã«ã¯è‡ªç„¶é¸æŠã‹ã‚™åƒã‹ãªã„ã¨ä»®å®šã™ã‚‹ã¨ï¼Œ Ks ã¯ãã®é ˜åŸŸã¦ã‚™ã®çªç„¶å¤‰ç•°ç‡ã‚’åæ˜ ã—ã¦ã„ã‚‹ã¨è€ƒãˆã‚‰ã‚Œã‚‹.

http://www.iu.a.u-tokyo.ac.jp/lectures/AG01/170517/2017_ç”Ÿç‰©é…åˆ—è§£æåŸºç¤_3å›ç›®_è³‡æ–™.pdf
åˆ†å­ç³»çµ±æ¨¹ 12
DNAã«è“„ç©ã™ã‚‹å¤‰ç•°ã¯ä¸€å®šã®å‰²åˆã¦ã‚™èµ·ã“ã£ã¦ãŠã‚Šã€ãã®ã»ã¨ã‚“ã¨ã‚™ã‹ã‚™è‡ªç„¶é¸æŠã¨ã¯ç„¡é–¢ ä¿‚ãªä¸­ç«‹ã®å¤‰ç•°ã¦ã‚™ã‚ã‚‹
DNAã®é…åˆ—ã‹ã‚™ã¨ã‚™ã®ãã‚‰ã„ä¼¼ã¦ã„ã‚‹ã‹ã‚’èª¿ã¸ã‚™ã‚‹ã“ã¨ã«ã‚ˆã£ã¦ã€é€²åŒ–çš„ã«ã¨ã‚™ã®ç¨‹åº¦è¿‘ç¸ã¦ã‚™ ã‚ã‚‹ã‹ã‚’çŸ¥ã‚‹ã“ã¨ã‹ã‚™ã¦ã‚™ãã‚‹

https://kimuraseminar.wordpress.com/2017å¹´8æœˆ3æ—¥-ç³»çµ±å­¦2/
2017å¹´8æœˆ3æ—¥ ç³»çµ±å­¦2
ç³»çµ±å­¦ï¼’ï¼ˆå²¸é‡æ´‹ä¹…ï¼‰
é›†å›£ã«å›ºå®šã•ã‚Œãªã„æœ‰å®³ãªå¤‰ç•°ã¯è¦³æ¸¬ã•ã‚Œãªã„ã®ã§ã€åˆ†å­é€²åŒ–é€Ÿåº¦ã¯çªç„¶å¤‰ç•°ç‡ã¨ä¸­ç«‹ãªå¤‰ç•°ã®å‰²åˆã§è¡¨ç¾ã•ã‚Œã¾ã™ã€‚åˆ†å­é€²åŒ–é€Ÿåº¦ã®ä¸€å®šæ€§ã®ä½œã‚Šå‡ºã™åˆ†å­æ™‚è¨ˆã‹ã‚‰ã¯ã€ã“ã‚Œã‚‰äºŒã¤ã®è¦ç´ ãŒå®‰å®šã—ã¦ãŠã‚Šã€ã‚ã¾ã‚Šå¤‰åŒ–ã—ãªã„ã“ã¨ãŒä¼ºã‚ã‚Œã¾ã™ã€‚

2008
https://www.ism.ac.jp/editsec/toukei/abstract/56-1j.html
ã‚³ãƒ‰ãƒ³ãƒ¢ãƒ‡ãƒ«ã‚’ç”¨ã„ãŸåˆ†å²å¹´ä»£ã®ãƒ™ã‚¤ã‚ºæ¨å®š
https://www.ism.ac.jp/editsec/toukei/pdf/56-1-037.pdf
ä¸­ç«‹èª¬ã®ä¸‹ã§ã¯ f = 1
2N ã¨ãªã‚Šï¼Œåˆ†å­é€²åŒ–é€Ÿåº¦ã¯é›†å›£ã®å¤§ãã•ã«å·¦å³ã•ã‚Œãšï¼Œéºä¼
å­ã®çªç„¶å¤‰ç•°ç‡ã«ãªã‚‹ï¼å¾“ã£ã¦ï¼Œçªç„¶å¤‰ç•°ç‡ãŒä¸€å®šã§ã‚ã‚‹ã‚ˆã†ãªçŠ¶æ³ã§ã¯ï¼Œåˆ†å­æ™‚è¨ˆãŒæˆç«‹ã™
ã‚‹ã“ã¨ã«ãªã‚‹ï¼é€²åŒ–é€Ÿåº¦ã®ä¸€å®šæ€§ãŒèªã‚ã‚‰ã‚Œã‚‹å ´åˆã«ãŠã„ã¦ã¯ï¼ŒåŒ–çŸ³ãƒ‡ãƒ¼ã‚¿ã¨ä¸€éƒ¨ã®é…åˆ—ãƒ‡ãƒ¼
ã‚¿ã‹ã‚‰æ¨å®šã•ã‚ŒãŸé€²åŒ–é€Ÿåº¦ã‚’å…¨ç³»çµ±æ¨¹ã«é©ç”¨ã—ï¼Œç³»çµ±æ¨¹ã®å…¨ã¦ã®åˆ†å²ç‚¹ã®å¹´ä»£ã‚’æ¨å®šã™ã‚‹ã“ã¨
ãŒã§ãã‚‹ï¼

2002
https://www.ism.ac.jp/editsec/toukei/pdf/50-1-017.pdf
ä¸­ç«‹èª¬ã®ä¸‹ã§ã¯ï¼Œçªç„¶å¤‰ç•°ç‡ãŒä¸€å®šã§ã‚ã‚Œã°ï¼Œåˆ†å­é€²åŒ–é€Ÿåº¦ã¯
ä¸€å®šã¨ãªã‚Šï¼Œã„ã‚ã‚†ã‚‹åˆ†å­æ™‚è¨ˆãŒæˆç«‹ã™ã‚‹ãŸã‚ï¼Œã“ã‚Œã‚’æ¤œå®šã™ã‚‹æ–¹å¼ãŒã„ãã¤ã‹æå”±ã•ã‚Œã¦ã
ãŸï¼ˆ

----------


----------
## microbe

æœ€çµ‚æ›´æ–°æ—¥: 2020.05.07
https://www.sbj.or.jp/sbj/sbj_yomoyama_2.html
ç”Ÿç‰©å·¥å­¦ä¼šèªŒ â€“ã€ç¶šãƒ»ç”Ÿç‰©å·¥å­¦åŸºç¤è¬›åº§ï¼ãƒã‚¤ã‚ªã‚ˆã‚‚ã‚„ã¾è©±ï¼ã€ | å…¬ç›Šç¤¾å›£æ³•äºº æ—¥æœ¬ç”Ÿç‰©å·¥å­¦ä¼š
å¾®ç”Ÿç‰©ã®ç³»çµ±æ¨¹ï¼Œã©ã†æãã®ï¼Ÿ	é£¯é‡ éš†å¤«ãƒ»ä¼Šè—¤ éš†	91-10-576
https://www.sbj.or.jp/wp-content/uploads/file/sbj/9110/9110_yomoyama.pdf

https://www.sbj.or.jp/sbj/sbj_yomoyama.html
ç”Ÿç‰©å·¥å­¦åŸºç¤è¬›åº§ï¼ãƒã‚¤ã‚ªã‚ˆã‚‚ã‚„ã¾è©±ï¼ï¼ˆ2011å¹´89å·»4å·ï½2013å¹´91å·»3å·æ²è¼‰ï¼‰ã€€ | å…¬ç›Šç¤¾å›£æ³•äºº æ—¥æœ¬ç”Ÿç‰©å·¥å­¦ä¼š
çŸ¥ã£ã¦ãŠããŸã„æ®ºèŒãƒ»é™¤èŒãƒ»æ»…èŒæŠ€è¡“	æ¾æ‘ å‰ä¿¡ãƒ»ä¸­ç”° è¨“æµ©	89â€“12â€“739
https://www.sbj.or.jp/wp-content/uploads/file/sbj/8912/8912_yomoyama_1.pdf
ä½•ã‹ã‚‰å§‹ã‚ã‚ˆã†ã€€å¾®ç”Ÿç‰©ã®åŒå®šï¼ç´°èŒãƒ»ã‚¢ãƒ¼ã‚­ã‚¢ç·¨ï¼	æµœç”° ç››ä¹‹ãƒ»éˆ´æœ¨ å¥ä¸€æœ—	89â€“12â€“744
https://www.sbj.or.jp/wp-content/uploads/file/sbj/8912/8912_yomoyama_2.pdf

https://jcm.brc.riken.jp/ja/
å¾®ç”Ÿç‰©ææ–™é–‹ç™ºå®¤ (JCM) (RIKEN BRC)

----------
## taxon_sampling
ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°

https://www.fifthdimension.jp/documents/molphytextbook/answers.pdf
åˆ†å­ç³»çµ±è§£æã«ãŠã‘ã‚‹æ§˜ã€…ãªå•é¡Œã«ã¤ã„ã¦

2019/04/20
https://sites.google.com/view/enter-the-fungi/phylogenetic-analysis/pre-phylogenetic-analysis
Enter the FUNGI - ç³»çµ±è§£æã®å‰æº–å‚™ã®è©±

https://twitter.com/i/events/939216389797068800
Nov 3, 2017
BAMMã‚’ä½¿ã£ã¦ã¿ã¦æ„Ÿã˜ãŸã“ã¨ã¯
1. çµæœãŒã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã«å·¦å³ã•ã‚Œã‚‹ã€è§£æææ–™ãŒç³»çµ±æ¨¹ã ã‘ãªã®ã§ã€‚

http://setoblo.blogspot.com/2013/10/
ã›ã¨ãƒ–ãƒ­ â€ ç€¬æˆ¸è‡¨æµ·å®Ÿé¨“æ‰€å…¬å¼ãƒ–ãƒ­ã‚°: October 2013
ç¥ï¼ã€€å²¡è¥¿æ”¿å…¸ã•ã‚“ã€€è«–æ–‡å‡ºç‰ˆï¼
ã¾ãŸæœ¬ç ”ç©¶ã§ã¯ã€ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã¨é©åˆ‡ãªéºä¼å­ã®é¸æŠãŒåˆ†å­ç³»çµ±è§£æã«ä¸ãˆã‚‹å½±éŸ¿ã«ã¤ã„ã¦ã‚‚è¨€åŠã—ã¦ã„ã¾ã™ã€‚

2010-08-17
https://blog.goo.ne.jp/acornworm/e/e286ffe827f0270406c8410b17a85b9f
ç¥–å…ˆçš„ãªå‹•ç‰©ã®ç³»çµ±é–¢ä¿‚ - æ·±æµ·ç”Ÿç‰©ç ”ç©¶è€…ã®æ—¥å¸¸
åˆ†å­ã®æ•°ã‚’å¢—ã‚„ã™ã‚ˆã‚Šã‚‚ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã®æ–¹ãŒé‡è¦
Long branch attractionã«ã‚‚æ°—ã‚’ã¤ã‘ã‚

https://www.ikushimo.com/news/2007/02/10.html
â—† [Science] Detecting the Node-Density Artifact in Phylogeny Reconstruction - 09:53:39
ã€€ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°å¯†åº¦ãŒé«˜ã‘ã‚Œã°é«˜ã„ã»ã©ã€æœ€ç¯€ç´„ãƒ»æœ€å°¤ãƒ»ãƒ™ã‚¤ã‚¸ã‚¢ãƒ³ç³»çµ±æ¨å®šã«ãŠã‘ã‚‹å¤šé‡ç½®æ›æ¤œå‡ºç‡ã¯ä¸Šæ˜‡ã™ã‚‹ã€‚ã¤ã¾ã‚Šã€ã‚¿ã‚¯ã‚½ãƒ³ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ãŒç²—ãªåˆ†é¡ç¾¤ã¨å¯†ãªåˆ†é¡ç¾¤ã§ã¯æé•·ã®æ¨å®šç²¾åº¦ãŒç•°ãªã‚‹ã€‚å¤šé‡ç½®æ›æ¤œå‡ºç‡ãŒä½ã‘ã‚Œã°ä½ã„ã»ã©ã€Œè¦‹ã‹ã‘ã®é€²åŒ–é€Ÿåº¦ã€ãŒé…ããªã‚Šã€é€†ã¯é€Ÿããªã‚‹ã€‚ã“ã“ã§ã€ç¾åœ¨ã®ç³»çµ±æ¨å®šæ³•ã§ã¯é€²åŒ–ã®é€Ÿã„åˆ†é¡ç¾¤ã©ã†ã—ãŒå¼•ãã¤ã‘åˆã†Long-Branch Attractionã¨ã„ã†åŠ¹æœãŒçŸ¥ã‚‰ã‚Œã¦ã„ã‚‹ã€‚æ•…ã«ã€å¯†ã«ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã•ã‚ŒãŸåˆ†é¡ç¾¤ã§ã¯å¤šé‡ç½®æ›ãŒã‚ˆãæ¤œå‡ºã•ã‚Œã¦é€²åŒ–ãŒè¦‹ã‹ã‘ä¸Šé«˜é€Ÿã«ãªã‚Šã€ãã®ã‚ˆã†ãªåˆ†é¡ç¾¤ã©ã†ã—ãŒLong-Branch Attractionã«ã‚ˆã£ã¦å¼•ãã¤ã‘åˆã†åŠ¹æœãŒç”Ÿã¾ã‚Œã‚‹ã€‚ã“ã‚ŒãŒç³»çµ±æ¨å®šã‚’èª¤ã‚‹åŸå› ã«ãªã‚Šå¾—ã‚‹ã€‚

----------
## marker

https://ja.wikipedia.org/wiki/éºä¼å­ãƒãƒ¼ã‚«ãƒ¼
ã‚‚ã—ãã¯ç³»çµ±ï¼ˆå€‹äººã®ç‰¹å®šã€è¦ªå­ãƒ»è¦ªæ—é–¢ä¿‚ã€è¡€çµ±ã‚ã‚‹ã„ã¯å“ç¨®ãªã©ï¼‰ã®ç›®å°ã¨ãªã‚‹ã€ã¤ã¾ã‚Šã‚ã‚‹æ€§è³ªã‚’ã‚‚ã¤å€‹ä½“ã«ç‰¹æœ‰ã®ã€DNAé…åˆ—ã‚’ã„ã†[1]ã€‚
ã¾ãŸç³»çµ±ã®è§£æã«éºä¼å­ãƒãƒ¼ã‚«ãƒ¼ã‚’åˆ©ç”¨ã™ã‚‹æ–¹æ³•ã¯ã€ä¸€èˆ¬ã«DNAå‹é‘‘å®šãªã©ã®åã§å‘¼ã°ã‚Œã¦ã„ã‚‹ã€‚ã“ã‚Œã«ã¯ãƒã‚¤ã‚¯ãƒ­ã‚µãƒ†ãƒ©ã‚¤ãƒˆãªã©ãŒãƒãƒ¼ã‚«ãƒ¼ã¨ã—ã¦ç”¨ã„ã‚‰ã‚Œã¦ã„ã‚‹ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/24146954
PLoS One. 2013 Oct 17;8(10):e77033. doi: 10.1371/journal.pone.0077033. eCollection 2013.
Systematic identification of gene families for use as "markers" for phylogenetic and phylogeny-driven ecological studies of bacteria and archaea and their major subgroups.
Wu D1, Jospin G, Eisen JA.

https://github.com/haruosuz/microbe/blob/master/references/README.microbiome.md#16s

https://www.aist.go.jp/aist_j/press_release/pr2017/pr20170830/pr20170830.html
ç”£ç·ç ”ï¼šé€²åŒ–ç³»çµ±åˆ†é¡ã®æŒ‡æ¨™ã¨ãªã‚‹16S rRNAéºä¼å­ã®é€²åŒ–çš„ãªä¸­ç«‹æ€§ã‚’å®Ÿé¨“çš„ã«è¨¼æ˜
ç”Ÿç‰©ã®ç³»çµ±é€²åŒ–ã‚’æ­£ç¢ºã«åæ˜ ã™ã‚‹ã€Œåˆ†å­ãƒãƒ¼ã‚«ãƒ¼ï¼ˆåˆ†å­æ™‚è¨ˆï¼‰ã€ã®è¦ä»¶ã¯ã€
åˆ†å­æ™‚è¨ˆï¼ˆé€²åŒ–ç³»çµ±è§£æã«ãŠã‘ã‚‹åˆ†å­ãƒãƒ¼ã‚«ãƒ¼ï¼‰ã¨å‘¼ã¶ã€‚

----------
## model

model-selection

ç³»çµ±ï¼ˆæ¨¹å½¢ã‚„ç¥–å…ˆé…åˆ—ï¼‰æ¨å®šã«ãŠã„ã¦ã€é€²åŒ–ãƒ¢ãƒ‡ãƒ«é¸æŠã¯ä¸è¦ãªæ‰‹é †ã§ã‚ã‚Šã€æœ€ã‚‚è¤‡é›‘ãªãƒ¢ãƒ‡ãƒ«(GTR+I+G)ã‚’ä½¿ç”¨ã™ã‚‹ã“ã¨ã§å›é¿ã§ãã‚‹ã€‚

https://www.nature.com/articles/s41467-019-08822-w
Model selection may not be a mandatory step for phylogeny reconstruction | Nature Communications

https://natureecoevocommunity.nature.com/users/207830-shiran-abadi/posts/44527-is-model-selection-a-mandatory-step-for-phylogeny-reconstruction
Is Model Selection a mandatory step for phylogeny reconstruction? | Nature Research Ecology & Evolution Community
```
We reconstructed all phylogenies with the most complex model, GTR+I+G, regardless of the Model Selection methods, and the inferences were even better than those of the models selected by the Model Selection methods.
any model could serve just as well as the best fitted one.
Model Selection is an unnecessary step and can be avoided by employing the most complex model.

ãƒ¢ãƒ‡ãƒ«é¸æŠæ³•ã«é–¢ä¿‚ãªãã€æœ€ã‚‚è¤‡é›‘ãªãƒ¢ãƒ‡ãƒ«GTR+I+Gã‚’ä½¿ç”¨ã—ã¦ã™ã¹ã¦ã®ç³»çµ±ã‚’å†æ§‹ç¯‰ã—ãŸã¨ã“ã‚ã€ãƒ¢ãƒ‡ãƒ«é¸æŠæ³•ã«ã‚ˆã£ã¦é¸æŠã•ã‚ŒãŸãƒ¢ãƒ‡ãƒ«ã‚ˆã‚Šã‚‚æ¨è«–ãŒå„ªã‚Œã¦ã„ã¾ã—ãŸã€‚
ã©ã®ãƒ¢ãƒ‡ãƒ«ã‚‚ã€æœ€é©ãªãƒ¢ãƒ‡ãƒ«ã¨åŒæ§˜ã«æ©Ÿèƒ½ã—ã¾ã™ã€‚
ãƒ¢ãƒ‡ãƒ«é¸æŠã¯ä¸è¦ãªæ‰‹é †ã§ã‚ã‚Šã€æœ€ã‚‚è¤‡é›‘ãªãƒ¢ãƒ‡ãƒ«ã‚’ä½¿ç”¨ã™ã‚‹ã“ã¨ã§å›é¿ã§ãã¾ã™ã€‚
```

https://twitter.com/fburki/status/1100003521569660929
Fabien Burki on Twitter: "This will likely stir some S: Model selection may not be a mandatory step for phylogeny reconstruction https://t.co/p0XYygiJxM"
7:04 AM - 25 Feb 2019

https://twitter.com/3rdreviewer/status/1100151955110809600
Matthew Hahn on Twitter: "This seems like a big deal. Hope to hear from some people who know this stuff better (cc: @roblanfear) Model selection may not be a mandatory step for phylogeny reconstruction https://t.co/In76QkR5yH"
4:54 PM - 25 Feb 2019

### GTR

http://cse.naro.affrc.go.jp/minaka/cladist/NOTES/stochastic.html
ç¢ºç‡éç¨‹æ¦‚è«–ï¼ˆåˆ†å­é€²åŒ–ãƒ¢ãƒ‡ãƒ«ï¼‰
ãƒãƒ«ã‚³ãƒ•éç¨‹ã«ã¤ã„ã¦ã®æ¦‚è«–
ã€Œä¸€èˆ¬æ™‚é–“é€†è»¢ãƒ¢ãƒ‡ãƒ«ã€ï¼ˆï¼§ï¼´ï¼²ï¼šgeneral time-reversible modelï¼‰

http://cse.naro.affrc.go.jp/minaka/cladist/NOTES/modeltest.html
â—æœ€å°¤æ³•ã§ã®ãƒ¢ãƒ‡ãƒ«é¸æŠå•é¡Œï¼ˆModelTestã‚’ç”¨ã„ã¦ï¼‰
ã“ã“ã§ã¯å¡©åŸºç½®æ›ãƒ¢ãƒ‡ãƒ«ã«ç„¦ç‚¹ã‚’å½“ã¦ã¦ãƒ¢ãƒ‡ãƒ«é¸æŠå•é¡Œã‚’è€ƒãˆã¦ã¿ã¾ã™ï¼ä¸€èˆ¬æ™‚é–“é€†è»¢ãƒ¢ãƒ‡ãƒ«ï¼ˆï¼§ï¼´ï¼²ï¼‰ã‚’ã‚‚ã£ã¨ã‚‚ç·©ã„ãƒ¢ãƒ‡ãƒ«ï¼ŒJukes-Cantorãƒ¢ãƒ‡ãƒ«ï¼ˆï¼ªï¼£ï¼‰ã‚’ã‚‚ã£ã¨ã‚‚å³ã—ã„ãƒ¢ãƒ‡ãƒ«ã¨ã™ã‚‹ã¨ï¼Œç¾åœ¨ç”¨ã„ã‚‰ã‚Œã¦ã„ã‚‹å¡©åŸºç½®æ›ãƒ¢ãƒ‡ãƒ«ã¯ï¼§ï¼´ï¼²ã¨ï¼ªï¼£ã‚’ä¸¡æ¥µç«¯ã¨ã™ã‚‹éšå±¤ã®ã„ãšã‚Œã‹ã®å ´æ‰€ã«ä½ç½®ã¥ã‘ã‚‰ã‚Œã¾ã™ï¼

http://www.tezuru-mozuru.com/?cat=200
ç³»çµ±è§£æ â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹

2017å¹´3æœˆ9æ—¥
http://www.tezuru-mozuru.com/?p=9843
MEGAã«ã‚ˆã‚‹ãƒ¢ãƒ‡ãƒ«ãƒ†ã‚¹ãƒˆ â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹
ä»Šå›ã®è§£æã§ã¯ï¼ŒGTR+G+IãŒãƒ™ã‚¹ãƒˆãªãƒ¢ãƒ‡ãƒ«ã¨ã—ã¦é¸æŠã•ã‚Œã¾ã—ãŸï¼

2016-10-12
http://www.geocities.jp/ancientfishtree/RAxML
RAxML - äº•ä¸Š æ½¤

-m GTRCAT
ãƒ¢ãƒ‡ãƒ«ã®é¸æŠï¼ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—è§£æã§ã¯ CAT-based ã®ãƒ¢ãƒ‡ãƒ« (å°¤åº¦ã‚’æœ€å¤§åŒ–ã™ã‚‹ã®ã§ã¯ãªãï¼Œãƒ™ã‚¹ãƒˆã®ç³»çµ±æ¨¹ã‚’æ¢ç´¢ã™ã‚‹ã“ã¨ã«çš„ã‚’çµã£ãŸãƒ¢ãƒ‡ãƒ«) ãŒãƒãƒ‹ãƒ¥ã‚¢ãƒ«ã§ã¯æ¨å¥¨ã•ã‚Œã¦ã„ã¾ã™ï¼
ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—è§£æã¯ GTRCAT ã§è¡Œã£ãŸæ–¹ãŒã‚ˆã„ã¨æ€ã„ã¾ã™ï¼

https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§
ç”°è¾ºæ™¶å²
2015/10/20
ç¬¬2ç« 
åˆ†å­é€²åŒ–ãƒ¢ãƒ‡ãƒ«ã®åŸºç¤
2.1 å¡©åŸºç½®æ›ãƒ¢ãƒ‡ãƒ«
ä¸€èˆ¬æ™‚é–“åè»¢å¯èƒ½(general time-reversibleã‚’ç•¥ã—ã¦GTR)ãƒ¢ãƒ‡ãƒ«
2.2 ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ‡ãƒ«
2.3 ã‚ˆã‚Šè¤‡é›‘ãªãƒ¢ãƒ‡ãƒ«
ãªãŠã€RAxMLã¨ã„ã†ã‚½ãƒ•ãƒˆã«ã‚‚CATãƒ¢ãƒ‡ãƒ«ã¨ã„ã†ã‚‚ã®ãŒå®Ÿè£…ã•ã‚Œã¦ã„ã¾ã™ãŒã€åå‰ãŒåŒã˜ãªã ã‘ã§å…¨ãã®åˆ¥ç‰©ã§ã™ã€‚ã“ã¡ã‚‰ã®CATãƒ¢ãƒ‡ãƒ«ã¯ã€Î“åˆ†å¸ƒã‚’ä½¿ã‚ãšã«ã€åº§ä½ã‚’ä»»æ„ã®æ•°ã®é€Ÿåº¦ã‚«ãƒ†ã‚´ãƒªã«åˆ†ã‘ã¦å°¤åº¦ã‚’è¨ˆç®—ã™ã‚‹ASRVãƒ¢ãƒ‡ãƒ«ã§ã™ã€‚+ G ãƒ¢ãƒ‡ãƒ«ã®é«˜é€Ÿãªè¿‘ä¼¼æ³•ã¨ã—ã¦ç”¨ã„ã‚‰ã‚Œã¦ã„ã¾ã™ã€‚

2011-05-07
https://anond.hatelabo.jp/20110507155609
ãƒ¢ãƒ†ã‚‹ç³»çµ±è§£æç³»å¥³å­åŠ›ã‚’ç£¨ããŸã‚ã®4ã¤ã®å¿ƒå¾—
ã€ŒRAxMLã§ã—ã‚‡ï¼Ÿã‹ãªã‚ŠçŸ­æ™‚é–“ã§æœ€å°¤æ³•ã‚’çµ‚ã‚ã‚‰ã›ã‚‹ã“ã¨ãŒã§ã§ãã‚‹ã‘ã©ã€åŸºæœ¬çš„ã«GTRãƒ¢ãƒ‡ãƒ«ã‹CATãƒ¢ãƒ‡ãƒ«ã§ãªã„ã¨ã‚ã‚“ã©ãã•ã„ã“ã¨ã«ãªã‚‹ã¿ãŸã„ã ã‚ˆã€‚

### JTT

https://en.wikipedia.org/wiki/Substitution_model
The Dayhoff PAM matrices were based on relatively few alignments (since not more were available at that time), but in the 1990s, new matrices were estimated using almost the same methodology, but based on the large protein databases available then (,[16][17] the latter being known as "JTT" matrices).

http://motdb.dbcls.jp/?plugin=attach&pcmd=open&file=20130730ajacs41_onami.pdf&refer=AJACS41
T1R3ã‚¢ãƒŸãƒé…¸é…åˆ—ã®ç³»çµ±æ¨¹æ§‹ç¯‰ 2.æœ€å°¤æ³•(ç³»çµ±æ¨¹æ¨å®š)

48ä»¶ã®ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ†ã‚™ãƒ«ã®å†…ã€BIC(Bayesian Information Criteria)ã‹ã‚™æœ€ã‚‚ä½ã„ãƒ¢ãƒ†ã‚™ãƒ«ã¯ã€ŒJTT+Gã€

â– Model/Method â†’ã€ŒJonesâ€Taylorâ€Thornton(JTT) modelã€â– Rates among Sites â†’ã€ŒGamma Distributed (G)ã€
ä¸Šè¨˜äºŒã‹æ‰€ã®ãƒã‚šãƒ©ãƒ¡ãƒ¼ã‚¿ã‚’ã€ŒJTT+Gã€ã«åˆã‚ã›ã¦è¨­å®šã—ã€ ã€ŒComputeã€

https://www35.atwiki.jp/kojimakk/pages/21.html
ModelGenerator - å°å³¶ç”Ÿç‰©å­¦å¾¡ç ”ç©¶å®¤ @ ã‚¦ã‚£ã‚­ - ã‚¢ãƒƒãƒˆã‚¦ã‚£ã‚­

JTT(Jones, Taylor, Thornton)ï¼šDayhoffã®PAMè¡Œåˆ—ã®æ”¹è‰¯ã€‚

WAGï¼šæœ€å°¤æ¨å®šã«ã‚ˆã‚‹JTTã®æ”¹è‰¯ç‰ˆï¼ˆJTTã¯æœ€ç¯€ç´„çš„ãªç½®æ›æ¨å®šï¼‰ã€çƒçŠ¶ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®ç½®æ›é »åº¦ã‹ã‚‰è¡Œåˆ—ã‚’ç”Ÿæˆã€‚(Whelan and Goldman, 2001)

DcMutï¼šDayhoffã¨JTTã®äºœå‹ã‚’çµ±åˆã—ãŸã‚‚ã®ã€‚(Kosiol and Goldman, 2005)

çµ±è¨ˆæ•°ç†(2008)
https://www.ism.ac.jp/editsec/toukei/abstract/56-1j.html
è¤‡æ•°éºä¼å­ã®çµåˆãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ãåˆ†å­ç³»çµ±æ¨¹ã®æ¨æ¸¬
â€”çœŸæ ¸ç”Ÿç‰©ã®å¤§ç³»çµ±ã®è§£æã‚’ä¾‹ã¨ã—ã¦â€”
http://www.ism.ac.jp/editsec/toukei/pdf/56-1-145.pdf
ä»Š,ã‚¢ãƒŸãƒé…¸ãƒ¬ãƒ˜ã‚™ãƒ«ã®è§£æã‚’è¡Œã†ã‚‚ã®ã¨ã—,çµŒé¨“çš„ãªã‚¢ãƒŸãƒé…¸ç½®æ›ç¢ºç‡(PAM ãƒ¢ãƒ†ã‚™ ãƒ«(Dayhoff et al., 1978),JTT ãƒ¢ãƒ†ã‚™ãƒ«(Jones et al., 1992),WAG ãƒ¢ãƒ†ã‚™ãƒ«(Whelan and Goldman, 2001)ãªã¨ã‚™)ã‚’ç”¨ã„,ã‚¢ãƒŸãƒé…¸çµ„æˆã‚’ãƒ†ã‚™ãƒ¼ã‚¿ã‹ã‚‰æ¨å®šã™ã‚‹ã‚‚ã®ã¨ã—,

RAxML ãƒ•ã‚šãƒ­ã‚¯ã‚™ãƒ© ãƒ ã«ã‚ˆã‚Š,ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ†ã‚™ãƒ«ã¨ã—ã¦ JTT ã‚’ç”¨ã„,ã‚¢ãƒŸãƒé…¸çµ„æˆã‚’ãƒ†ã‚™ãƒ¼ã‚¿ã‹ã‚‰æ¨å®š(F ã‚ªãƒ•ã‚šã‚·ãƒ§ ãƒ³)ã—,åº§ä½é–“ã®é€²åŒ–é€Ÿåº¦ã®ä¸å‡ä¸€æ€§ã‚’é›¢æ•£ Î“ åˆ†å¸ƒã¦ã‚™è¿‘ä¼¼ã—ã¦è§£æã—ãŸçµæœ,å›³ 2 ã«ç¤ºã™ç³»çµ± æ¨¹ã‹ã‚™æœ€è‰¯ãªç³»çµ±æ¨¹ã¨ã—ã¦é¸æŠã•ã‚ŒãŸ.

ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ†ã‚™ãƒ«ã¨ã—ã¦ JTT + Î“ ã‚’ç”¨ã„,ã‚¢ãƒŸãƒé…¸ã®çµ„æˆå€¤ã¯ãƒ†ã‚™ãƒ¼ã‚¿ã‹ã‚‰æ¨å®šã—ãŸ.

ä¸€ æ–¹,ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ†ã‚™ãƒ«ã¨ã—ã¦,JTT(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ«ã®ä»£ã‚ã‚Šã« WAG(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ«,PAM(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ«ã‚’ç”¨ã„ã¦åŒæ§˜ã®è§£æã‚’è¡Œã£ãŸ(ãƒ†ã‚™ãƒ¼ã‚¿ä¸è¡¨ç¤º).å…¨èˆ¬çš„ã« JTT(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ« ã«æ¯”ã¸ã‚™ã¦ PAM(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ«ã¯ã‚„ã‚„é«˜ã„å¯¾æ•°å°¤åº¦ã®å€¤ã‚’ä¸ãˆ,WAG(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ«ã¯ã•ã‚‰ ã«é«˜ã„å¯¾æ•°å°¤åº¦ã®å€¤ã‚’ä¸ãˆãŸ.ã—ã‹ã—ãªã‹ã‚™ã‚‰,ç³»çµ±æ¨¹ã®é¸æŠã¨ã„ã†è¦³ç‚¹ã¦ã‚™ã¯,ã„ã™ã‚™ã‚Œã®ãƒ¢ãƒ†ã‚™ãƒ« ã«ã‚ˆã‚‹çµæœã‚‚ JTT(F)+ Î“ ãƒ¢ãƒ†ã‚™ãƒ«ã«ã‚ˆã‚‹çµæœã¨å…¨ãåŒæ§˜ã¦ã‚™ã‚ã£ãŸ.ã“ã®ã‚ˆã†ã«,ç½®æ›ãƒ¢ãƒ†ã‚™ãƒ« ã‚’å¤‰ãˆã¦ã‚‚çµåˆãƒ†ã‚™ãƒ¼ã‚¿è§£æã®ãƒ¢ãƒ†ã‚™ãƒ«ã‚’å¤‰ãˆã¦ã‚‚æ¨æ¸¬ã®çµæœã«å¤§å·®ã‹ã‚™ãªã„ã¨ã„ã†ã“ã¨ã‹ã‚‰,ãƒªãƒ›ã‚™ ã‚½ãƒ¼ãƒ ã‚¿ãƒ³ãƒã‚šã‚¯è³ªã«ã‚ˆã‚‹ä»Šå›ã®ãƒ†ã‚™ãƒ¼ã‚¿è§£æã®çµæœã¯éå¸¸ã«é ‘å¥ãªã‚‚ã®ã¦ã‚™ã‚ã‚‹ã¨è€ƒãˆã‚‰ã‚ŒãŸ.

----------
## substitution matrix
ç½®æ›è¡Œåˆ—

https://en.wikipedia.org/wiki/BLOSUM

https://github.com/haruosuz/r4bioinfo/blob/master/references/R.bio.md#subvis

https://github.com/haruosuz/r4bioinfo/blob/master/R_Avril_Coghlan/README.md#pairwise-global-alignment-of-dna-sequences-using-the-needleman-wunsch-algorithm
scoring matrix (substitution matrix) 
Biostringsãƒ‘ãƒƒã‚±ãƒ¼ã‚¸ã®nucleotideSubstitutionMatrix()é–¢æ•°ã§ã‚¹ã‚³ã‚¢ãƒãƒˆãƒªãƒƒã‚¯ã‚¹(ç½®æ›è¡Œåˆ—)ã‚’ä½œã‚‹:

https://github.com/haruosuz/r4bioinfo/blob/master/R_Avril_Coghlan/README.md#pairwise-global-alignment-of-protein-sequences-using-the-needleman-wunsch-algorithm
ã‚¢ãƒŸãƒé…¸ç½®æ›è¡Œåˆ— BLOSUM (BLOcks SUbstitution Matrix)

https://github.com/haruosuz/bioinfo/blob/master/2019/CaseStudy.md#akifumi-tanabe
https://doi.org/10.7875/togotv.2019.195
2019-12-08 åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ - ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§ @ åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨ ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—
1:07:45 / 3:22:38
ã‚¢ãƒŸãƒé…¸ ç½®æ›é€Ÿåº¦ è¡Œåˆ—
https://www.fifthdimension.jp/documents/molphytextbook/
åˆ†å­é€²åŒ–ã®çµ±è¨ˆãƒ¢ãƒ‡ãƒªãƒ³ã‚°ã¨ãƒ¢ãƒ‡ãƒ«é¸æŠ
https://www.fifthdimension.jp/documents/molphytextbook/modelselection_lecture.pdf
20x20 ã®ç½®æ›é€Ÿåº¦è¡Œåˆ—
â— 189 ã® rXY ã¨ 19 ã® Ï€X ã‚’ãƒ‘ãƒ©ãƒ¡ãƒ¼ã‚¿ã¨ã—ã¦æŒã¤
è¨ˆç®—å›°é›£
â— Empirical Model
â€“ æ—¢çŸ¥ã®ç³»çµ±æ¨¹ã¨å¤§é‡ã®ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰æ¨å®šã—ãŸå€¤ã«å›ºå®š
â€“ ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰æ¨å®šã™ã‚‹ãƒ‘ãƒ©ãƒ¡ãƒ¼ã‚¿æ•°ã¯ 0

https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
ç¬¬2ç« 
åˆ†å­é€²åŒ–ãƒ¢ãƒ‡ãƒ«ã®åŸºç¤

2.1 å¡©åŸºç½®æ›ãƒ¢ãƒ‡ãƒ«
2.1.1 å¡©åŸºç½®æ›é€Ÿåº¦è¡Œåˆ—
å¡©åŸºç½®æ›é€Ÿåº¦è¡Œåˆ—(nucleotide substitution rate matrix)ã¯ã€åº§ä½(site)å†…ã«ãŠã‘ã‚‹ã€å½¢è³ªçŠ¶æ…‹(character state)é–“ã®ç§»è¡Œé€Ÿåº¦ã®ä¸å‡è³ªæ€§(heterogeneity)ã‚’è¡¨ç¾ã™ã‚‹ã‚‚ã®ã§ã™ã€‚

2.2 ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ‡ãƒ«
2.2.1 Empirical model
å¡©åŸºç½®æ›é€Ÿåº¦è¡Œåˆ—ã¯4x4ã®è¡Œåˆ—ã§ã—ãŸãŒã€ã‚¢ãƒŸãƒé…¸ç½®æ›é€Ÿåº¦è¡Œåˆ—ã¯20x20ã®è¡Œåˆ—ã¨ãªã‚‹ãŸã‚ã€RateXYã¨FreqXã®æ•°ã¯æ™‚é–“åè»¢å¯èƒ½ãƒ¢ãƒ‡ãƒ«ã§ã‚‚ 190 + 20 = 210 ã¨ãªã‚Šè†¨å¤§ã§ã™ã€‚ãã“ã§ã€æ—¢ã«ç³»çµ±é–¢ä¿‚ã®åˆ†ã‹ã£ã¦ã„ã‚‹åˆ†é¡ç¾¤é–“ã®ç³»çµ±æ¨¹ã«ãŠã„ã¦ã€å¤§é‡ã®ãƒ‡ãƒ¼ã‚¿ã‚’ç”¨ã„ã¦ã‚ã‚‰ã‹ã˜ã‚æ¨å®šã•ã‚ŒãŸRateXY FreqXã®å€¤ã‚’ç”¨ã„ãŸãƒ¢ãƒ‡ãƒ«ã‚’ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ‡ãƒ«ã¨ã—ã¦ç”¨ã„ã¾ã™ã€‚

2.3 ã‚ˆã‚Šè¤‡é›‘ãªãƒ¢ãƒ‡ãƒ«

https://bi.biopapyrus.jp/seq/
2018.08.17
https://bi.biopapyrus.jp/seq/score-matrix.html
ç½®æ›è¡Œåˆ— | ã‚¹ã‚³ã‚¢ãƒãƒˆãƒªãƒƒã‚¯ã‚¹ã®ä½œã‚Šæ–¹
ç½®æ›ã‚¹ã‚³ã‚¢ã¯ã€ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã‹ã‚‰æ±‚ã‚ã‚‹ã€‚ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã¯å¿…è¦ã«å¿œã˜ã¦ã€è¿‘ç¸ç¨®ã®å¡©åŸºé…åˆ—ã‚„ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’ç”¨ã„ãŸã‚Šã€é ç¸ç¨®ã®ãã‚Œã‚’ç”¨ã„ãŸã‚Šã™ã‚‹ã€‚

https://bioinformaticshome.com/bioinformatics_tutorials/sequence_alignment/substitution_matrices.html
Bioinformatics tutorial: Construction of substitution matrices 2020


http://feynmanino.watson.jp/10717_220MolecularPhylogenetics.html
220ãƒ»åˆ†å­ç³»çµ±å­¦ã®æ‰‹æ³•
- BLOSUM62ã¯blastpã®ãƒ‡ãƒ•ã‚©ãƒ«ãƒˆã§ä½¿ã‚ã‚Œã¦ã„ã‚‹ç½®æ›ã‚¹ã‚³ã‚¢è¡Œåˆ—ã§ã€62%ä»¥ä¸Šã®é…åˆ—ä¸€è‡´æ€§ã‚’ç¤ºã™ã‚¢ãƒŸãƒé…¸é…åˆ—é–“ã§è¦³æ¸¬ã•ã‚ŒãŸç½®æ›ã‚’åŸºã«ç®—å‡ºã—ã¦å¾—ã‚‰ã‚ŒãŸã‚¢ãƒŸãƒé…¸ã®ç½®æ›è¡Œåˆ—ã€ã¨ã„ã†æ„å‘³ã§ã™ï¼ˆhttps://slidesplayer.net/slide/11313941/ï¼‰ã€‚
- PAMè¡Œåˆ—ï¼ˆDayhoffã®ã‚¢ãƒŸãƒé…¸ç½®æ›è¡Œåˆ—ï¼‰ã¯ã€ã‚¢ãƒŸãƒé…¸ã®å¤‰åŒ–ç‡ãŒåˆè¨ˆã§ï¼‘%ã®å¤‰åŒ–ã®å‰²åˆã«ãªã‚‹ã¨ãã®ç›¸å¯¾çš„ãªå‰²åˆã‚’ç¤ºã™è¡Œåˆ—ãŒPAM1è¡Œåˆ—ã§ã€ãã‚Œã‚’Nä¹—ï¼ˆPAMï¼ˆ250ï¼‰è¡Œåˆ—ãªã‚‰250ä¹—ï¼‰ã™ã‚‹ã“ã¨ã§å¾—ã‚‰ã‚ŒãŸé·ç§»è¡Œåˆ—ã§ã™ï¼ˆhttps://www.slideserve.com/claus/pam250-matrixï¼‰ã€‚

2.3 years ago
https://www.biostars.org/p/291408/
Creating a custom substitution matrix from multiple alignments/trees?

You can use BayesTraits in Multistate mode

21 months ago
https://www.biostars.org/p/322615/
Creating your own substitution matrix from an alignment

https://www.ncbi.nlm.nih.gov/pubmed/28583067
BMC Bioinformatics. 2017 Jun 5;18(1):293. doi: 10.1186/s12859-017-1703-z.
PFASUM: a substitution matrix from Pfam structural alignments.
Keul F1, Hess M2, Goesele M3, Hamacher K1.

http://goto.kuicr.kyoto-u.ac.jp/lecture/2_homology_search.pdf
2016å¹´åº¦ã€Œãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹ã€
1. é…åˆ—ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆï¼ˆäº”æ–—ï¼‰
ã‚¹ã‚³ã‚¢ãƒãƒˆãƒªãƒƒã‚¯ã‚¹ï¼ˆã‚¢ãƒŸãƒé…¸ç½®æ›è¡Œåˆ—ï¼‰
- PAM (Accepted Point Mutations)
- BLOSUM (BLOck SUbstitution Matrices) 

https://www.ncbi.nlm.nih.gov/pubmed/20808876
PLoS Comput Biol. 2010 Aug 19;6(8). pii: e1000885. doi: 10.1371/journal.pcbi.1000885.
CodonTest: modeling amino acid substitution preferences in coding sequences.
Delport W1, Scheffler K, Botha G, Gravenor MB, Muse SV, Kosakovsky Pond SL.

2009å¹´9æœˆ12æ—¥
https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/
ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹ã®åŸºç¤ï¼šé…åˆ—è§£æ
è¬›å¸«ï¼šå·ç«¯ã€€çŒ›
è¬›ç¾©è³‡æ–™1 (PDF;2241 KB)
https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/h21_3_1.pdf

https://www.youtube.com/watch?v=VpYaLtaU_dw
ã€ã‚²ãƒãƒ ãƒªãƒ†ãƒ©ã‚·ãƒ¼è¬›åº§ã€‘é…åˆ—è§£æåŸºç¤ï¼ˆè¬›ç¾©1ï¼‰ - YouTube
19:33 / 1:08:21


http://www.fish-evol.org/AAmatrix.html
äº•ä¸Šæ½¤ï¼šã‚¢ãƒŸãƒé…¸ç½®æ›è¡Œåˆ—
Note on amino acid R matrix
2007 å¹´ 11 æœˆ 3 æ—¥ã€€æ”¹è¨‚
äº•ä¸Š æ½¤
ã‚¢ãƒŸãƒé…¸ç½®æ›è¡Œåˆ—ã«é–¢ã™ã‚‹å€‹äººçš„ãªãƒ¡ãƒ¢ã§ã™ï¼

5 January 2005
http://abacus.gene.ucl.ac.uk/software/pamlFAQs.pdf
How can I estimate an amino acid substitution matrix from my own data, like mtmam.dat and wag.dat? 


https://assets.geneious.com/manual/2020.0/static/GeneiousManualsu114.html
12.3.4 Distance models or molecular evolution models for Amino Acid sequences

Jukes-Cantor
This is the simplest substitution model. It assumes that all amino acids have the same equilibrium base frequency, i.e., each amino acid occurs with a frequency of 0.05 in protein sequences. This model also assumes that all amino acid substitutions occur at equal rates.

https://www.megasoftware.net/mega1_manual/Contents.html
Contents --MEGA manual
```
4. Distance Estimation
4.1 Nucleotide Substitutions
4.2 Synonymous and Nonsynonymous Substitutions
4.3 Amino Acid Substitutions
```
https://www.megasoftware.net/mega1_manual/Distance.html#4-3

https://www.megasoftware.net/webhelp/contexthelp_hc/hc_estimate_substitution_matrix_ml_.htm
Estimate Substitution Matrix (ML)
Models | Estimate Substitution Matrix
This option estimates and displays the nucleotide substitution rate matrix using the Maximum Likelihood method for the current data set and evolutionary model selected. This method finds the set of values for the substitution rate matrix parameters that maximizes the probability (likelihood) of the data. This is applicable only to nucleotide data (coding or non-coding).


----------
## HGT
Horizontal gene transfer
éºä¼å­æ°´å¹³ä¼æ’­

https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/horizontal-gene-transfer

### HGT_japanese

https://ja.wikipedia.org/wiki/éºä¼å­ã®æ°´å¹³ä¼æ’­
Horizontal gene transferï¼ˆHGTï¼‰ã¾ãŸã¯Lateral gene transferï¼ˆLGTï¼‰

2017å¹´8æœˆ18æ—¥
http://www.nikkei-science.com/page/sci_book/bessatu/51221.htmll
å¾®ç”Ÿç‰©ã®é©šç•° | æ—¥çµŒã‚µã‚¤ã‚¨ãƒ³ã‚¹

ã€åˆ¥å†Š221ã€€å¾®ç”Ÿç‰©ã®è„…å¨ã€€ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ã‹ã‚‰å¤šå‰¤è€æ€§èŒã¾ã§ã€‘è¦ªã‹ã‚‰å­ã¸éºä¼æƒ…å ±ãŒå—ã‘ç¶™ãŒã‚Œã‚‹ã¨ã„ã†éºä¼å­¦ã®åŸºæœ¬åŸç†ã§ã¯èª¬æ˜ã§ããªã„éºä¼æƒ…å ±ã®ã‚„ã‚Šã¨ã‚Šã€Œæ°´å¹³ä¼æ’­ã€ã®ç™ºè¦‹ã®ãã£ã‹ã‘ã¨ãã®å¾Œã®å±•é–‹ã€‚ã€Œè‡ªç„¶ç•Œã‚’æ¸¡ã‚Šæ­©ãç´°èŒã®DNAã€ã€€ http://www.nikkei-science.com/page/magazine/9804/DNA.html â€¦

http://www.nikkei-science.com/?p=14437
ç´°èŒã®éºä¼å­äº¤æ›ï¼Œãšã£ã¨é »ç¹ï½æ—¥çµŒã‚µã‚¤ã‚¨ãƒ³ã‚¹2011å¹´6æœˆå·ã‚ˆã‚Š
ã€Œæ°´å¹³ä¼æ’­ã€ã¨ã„ã†ç¾è±¡ãŒé€²åŒ–ã‚’ä¿ƒã—ã¦ã„ã‚‹

2015-12-12
http://horikawad.hatenadiary.com/entry/2015/12/12/014019
ã€Œã‚¯ãƒãƒ ã‚·ã«å¤–æ¥éºä¼å­17%ã€ã¯çœŸå®Ÿã‹ - ã‚¯ãƒãƒ ã‚·åšå£«ã®ã‚€ã—ãƒ–ãƒ­

2013-04-05
http://horikawad.hatenadiary.com/entry/20130405/1365159629
ãƒ‘ãƒ¯ãƒ¼ã‚¢ãƒƒãƒ—ã—ãŸéºä¼å­ã‚³ãƒ¬ã‚¯ã‚¿ãƒ¼ - ã‚¯ãƒãƒ ã‚·åšå£«ã®ã‚€ã—ãƒ–ãƒ­
ã“ã®è—»é¡ã«ã¯ATPã‚¢ãƒ¼ã‚¼éºä¼å­ã¯å¤šæ•°ã®ã‚³ãƒ”ãƒ¼ãŒã‚ã‚Šã€æ°´å¹³ä¼æ¬ã§å–è¾¼ã‚“ã å¾Œã«é‡è¤‡åŒ–ãŒé€²ã‚“ã ã‚‚ã®ã¨è¦‹ã‚‰ã‚Œã‚‹ã€‚ATPã‚¢ãƒ¼ã‚¼éºä¼å­ãŒå¤šãã‚ã‚‹ã»ã©ç†±ã«è€æ€§ã‚’ã‚‚ã¤ã‚ˆã†ã«ãªã‚‹ã“ã¨ãŒçŸ¥ã‚‰ã‚Œã¦ãŠã‚Šã€å¤ç´°èŒã‹ã‚‰ã‚³ãƒ¬ã‚¯ã‚·ãƒ§ãƒ³ã—ãŸã“ã®ATPã‚¢ãƒ¼ã‚¼éºä¼å­ã‚’å¤šãã‚‚ã¤ã“ã¨ã«ã‚ˆã‚Šã€ã“ã®è—»é¡ãŒé«˜æ¸©è€æ€§ã‚’èº«ã«ã¤ã‘ã€ç†±æ°´ç’°å¢ƒã«é©å¿œã—ãŸã®ã ã‚ã†ã€‚
ä»–ã«ã‚‚ã€é«˜æ¿ƒåº¦ã®é‡é‡‘å±ã«å¯¾å‡¦ã™ã‚‹ãŸã‚ã«é‡è¦ãªæ©Ÿèƒ½ã‚’æ‹…ã†éºä¼å­ã‚’ç´°èŒã‹ã‚‰å–ã‚Šå…¥ã‚ŒãŸã“ã¨ã‚‚ç¤ºå”†ã•ã‚Œã¦ã„ã‚‹ã€‚ãƒ’ç´ ã‚’ç´°èƒå¤–ã«æ’å‡ºã™ã‚‹ãƒãƒ³ãƒ—ã‚’ã¤ãã‚‹éºä¼å­ã¯ã€å¥½ç†±æ€§ç´°èŒã‹ã‚‰å–è¾¼ã‚“ã ã‚ˆã†ã ã€‚ã“ã®ã‚ˆã†ã«ã—ã¦ã€é‡é‡‘å±ãŒè±Šå¯Œãªã‚¹ãƒ¼ãƒ—ã®ä¸­ã§ã‚‚å…ƒæ°—ã«å¢—æ®–ã§ãã‚‹ã‚‚ã®ã¨æ€ã‚ã‚Œã‚‹ã€‚

### HGT_2019

ãƒ¡ã‚¿ã‚²ãƒãƒ ã«ãŠã‘ã‚‹éºä¼å­æ°´å¹³ä¼æ’­ã‚¤ãƒ™ãƒ³ãƒˆåŒå®šæ‰‹æ³•ã«é–¢ã™ã‚‹ç·èª¬
https://pubmed.ncbi.nlm.nih.gov/31504488/
Review Genome Biol Evol
. 2019 Oct 1;11(10):2750-2766. doi: 10.1093/gbe/evz184.
Current and Promising Approaches to Identify Horizontal Gene Transfer Events in Metagenomes
Gavin M Douglas 1, Morgan G I Langille 1
https://academic.oup.com/gbe/article/11/10/2750/5554466
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6777429/

https://twitter.com/MicrobiomeJ/status/1102948766972145664
Microbiome on Twitter: "MetaCHIP: community-level horizontal gene transfer identification through the combination of best-match and phylogenetic approaches https://t.co/eFx9Jk1jdU"
10:07 AM - 5 Mar 2019
https://github.com/songweizhi/MetaCHIP
songweizhi/MetaCHIP: Horizontal gene transfer (HGT) identification pipeline

### HGT_2018

å…±ç”Ÿç´°èŒãƒœãƒ«ãƒã‚­ã‚¢ã‹ã‚‰å®¿ä¸»ã¸ã®éºä¼å­æ°´å¹³ä¼æ’­
9:13 AM Â· Oct 14, 2018
https://twitter.com/KentsisResearch/status/1051264658030714890
KentsisResearchGroup on Twitter: "Horizontal gene transfer in eukaryotes: The first draft genomes of the ant Formica exsecta, and its Wolbachia endosymbiont reveal extensive gene transfer from endosymbiont to host. https://t.co/ZakV1uyz8B"

https://twitter.com/TetYahara/status/1083497343897333760
Tetsukazu Yahara on Twitter: "å¯„ç”Ÿæ¤ç‰©ã®ãƒ¤ã‚»ã‚¦ãƒ„ãƒœã¨ãƒŠãƒ³ãƒãƒ³ã‚®ã‚»ãƒ«ã§ã€å®¿ä¸»ã¸ã®éºä¼å­ã®æ°´å¹³ç§»å‹•ã‚’ç¢ºèªã—ãŸç ”ç©¶ã€‚ã‚¤ãƒ³ãƒˆãƒ­ãƒ³ãŒä¿æŒã•ã‚Œã¦ã„ã‚‹ã®ã§ã€RNAã§ã¯ãªãDNAãã®ã‚‚ã®ãŒç§»ã£ãŸã€‚ç§»ã£ãŸå¾Œã®éºä¼å­ã®é€²åŒ–ã‚‚èµ·ãã¦ã„ã‚‹ã€‚ã“ã‚“ãªã“ã¨ãŒã‚ã‹ã‚‹æ™‚ä»£ã«ãªã£ãŸã“ã¨ã«é©šãã€‚â€¦ "
5:54 PM - 10 Jan 2019
https://twitter.com/kfuku0502/status/1083325395657723904
ãƒãƒã‚¦ãƒ„ãƒœç§‘ï¼•ç¨®ã®ãƒˆãƒ©ãƒ³ã‚¹ã‚¯ãƒªãƒ—ãƒˆãƒ¼ãƒ ã‚’ä½¿ã£ã¦å¯„ä¸»æ¤ç‰©ã‹ã‚‰ã®æ°´å¹³ä¼æ’­éºä¼å­ã‚’æ¢ç´¢ã€‚çµ¶å¯¾å¯„ç”Ÿã®ï¼’ç¨®ã®ã¿ã§æ°´å¹³ä¼æ’­ãŒç¢ºå®Ÿãã†ãªã‚‚ã®ãŒè¦‹ã¤ã‹ã‚‹ã€‚
ã“ã®æ‰‹ã®ç ”ç©¶ã®èª²é¡Œã¯ã„ã‹ã«ã‚³ãƒ³ã‚¿ãƒŸã‚’é™¤å¤–ã™ã‚‹ã‹ã ã‘ã©ã€ã‚²ãƒãƒ ã‚‚ä¸€ç·’ã«èª­ã‚“ã§ãã®å¯èƒ½æ€§ã‚’é™¤å¤–ã—ã¦ã„ã‚‹ã€‚éºä¼å­ãŒæœ¬å½“ã«æ°´å¹³ä¼æ’­ã—ã¦å¯„ç”Ÿæ¤ç‰©ã‚²ãƒãƒ ã«æŒ¿å…¥ã•ã‚Œã¦ã„ã‚‹ãªã‚‰k-meré »åº¦ãŒsingle-copy regionã®ãã‚Œã¨ä¸€è‡´ã™ã‚‹ã€‚ã“ã‚Œã‚’æ°´å¹³ä¼æ’­ã®è¨¼å·¦ã¨ã—ã¦ã„ã‚‹ã€‚ã“ã‚Œãªã‚‰ã‚²ãƒãƒ ã‚¢ã‚»ãƒ³ãƒ–ãƒªãƒ¼ã¯å¿…è¦ãªã„ã€‚

https://twitter.com/Greeeening/status/1075562589382311936
å°æ— åº·ä¸€ (K. Kobayashi) on Twitter: "ãƒã‚¯ãƒ†ãƒªã‚¢ã®åˆ†é¡ãƒ»åŒå®šã«é–¢ã—ã¦ã¯è©³ã—ããªã„ã®ã ã‘ã©ã€ä»¥å‰ã«ç”Ÿç‰©å…¨ä½“ã®ãƒ˜ãƒ ä»£è¬ç³»ã®ã‚²ãƒãƒ è§£æã‚’ä½è—¤ç›´æ¨¹å…ˆç”Ÿã¨ä¸€ç·’ã«ã‚„ã‚‰ã›ã¦ã‚‚ã‚‰ã£ãŸã¨ãã¯ã™ã”ãå‹‰å¼·ã«ãªã£ãŸã€‚https://t.co/kYsNeienRZ ãƒã‚¯ãƒ†ãƒªã‚¢ã¯éºä¼å­ã®æ°´å¹³ä¼æ’­ãŒå¤šãã¦ã€tree of lifeã¨ã„ã†ã‚ˆã‚Šã€ring of lifeã®æ§˜ç›¸ã§ã‚ã‚‹ã“ã¨ã‚’çŸ¥ã£ãŸã€‚"
8:24 PM - 19 Dec 2018

https://pubmed.ncbi.nlm.nih.gov/31828235/
Annu Rev Biomed Data Sci
. 2018 Jul;1:93-114. doi: 10.1146/annurev-biodatasci-080917-013431. Epub 2018 Apr 25.
Alignment-Free Sequence Analysis and Applications
Jie Ren 1, Xin Bai 1 2, Yang Young Lu 1, Kujin Tang 1, Ying Wang 3, Gesine Reinert 4, Fengzhu Sun 1 2
https://www.annualreviews.org/doi/10.1146/annurev-biodatasci-080917-013431
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6905628/
https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013431

https://www.ncbi.nlm.nih.gov/pubmed/29713314
Front Microbiol. 2018 Apr 16;9:711. doi: 10.3389/fmicb.2018.00711. eCollection 2018.
Background Adjusted Alignment-Free Dissimilarity Measures Improve the Detection of Horizontal Gene Transfer.
Tang K1, Lu YY1, Sun F1,2.

https://www.ncbi.nlm.nih.gov/pubmed/29176581
Nat Rev Microbiol. 2018 Feb;16(2):67-79. doi: 10.1038/nrmicro.2017.137. Epub 2017 Nov 27.
Functional horizontal gene transfer from bacteria to eukaryotes.
Husnik F1,2,3, McCutcheon JP2.

### HGT_2017

https://www.ncbi.nlm.nih.gov/pubmed/27894995
Mol Phylogenet Evol. 2017 Feb;107:338-344. doi: 10.1016/j.ympev.2016.11.016. Epub 2016 Nov 26.
Short branches lead to systematic artifacts when BLAST searches are used as surrogate for phylogenetic reconstruction.
Dick AA1, Harlow TJ2, Gogarten JP3.
https://linkinghub.elsevier.com/retrieve/pii/S1055-7903(16)30377-3

Short Branch Attraction (SBA)

SBA is an intended feature of BLAST searches, but becomes an issue, when top scoring BLAST hit analyses are used to infer Horizontal Gene Transfers (HGTs), assign taxonomic category with environmental sequence data in phylotyping, or gather homologous sequences for building gene families.

Therefore, one should look for this phenomenon when conducting best BLAST hit analyses as a surrogate method to identify HGTs, in phylotyping, or when using BLAST to gather homologous sequences.

It is well established that the top scoring BLAST hit does not necessarily represent the nearest phylogenetic neighbor (Eisen, 2000, Koski and Golding, 2001) and this is indeed expected, because the nearest neighbor is not always the most similar. 

### HGT_2016

https://pubmed.ncbi.nlm.nih.gov/27189546/
Mol Biol Evol
. 2016 Jul;33(7):1843-57. doi: 10.1093/molbev/msw062. Epub 2016 Apr 6.
Estimating the Frequency of Horizontal Gene Transfer Using Phylogenetic Models of Gene Gain and Loss
Seyed Alireza Zamani-Dahaj 1, Mohamed Okasha 1, Jakub Kosakowski 1, Paul G Higgs 2
https://academic.oup.com/mbe/article/33/7/1843/2579271

A gene gain could occur by horizontal transfer or by origin of a gene within the lineage being studied.
éºä¼å­ã®ç²å¾—ã¯ã€æ°´å¹³ä¼æ’­ã«ã‚ˆã£ã¦èµ·ã“ã‚‹å ´åˆã¨ã€ç ”ç©¶å¯¾è±¡ã®ç³»çµ±å†…ã§éºä¼å­ãŒç™ºç”Ÿã™ã‚‹å ´åˆã¨ãŒã‚ã‚‹ã€‚

The distribution of rates of gene loss is very broad, which explains why many genes follow a treelike pattern of vertical inheritance, despite the presence of a significant minority of genes that undergo horizontal transfer.
éºä¼å­æ¶ˆå¤±ã®å‰²åˆã®åˆ†å¸ƒã¯éå¸¸ã«åºƒãã€æ°´å¹³è»¢ç§»ã‚’ã™ã‚‹éºä¼å­ã¯ã‹ãªã‚Šå°‘æ•°ã§ã‚ã‚‹ã«ã‚‚ã‹ã‹ã‚ã‚‰ãšã€å¤šãã®éºä¼å­ãŒæ¨¹çŠ¶éºä¼ã®ãƒ‘ã‚¿ãƒ¼ãƒ³ã‚’ã¨ã£ã¦ã„ã‚‹ã“ã¨ãŒèª¬æ˜ã•ã‚Œã‚‹ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/27508073
F1000Res. 2016 Jul 25;5. pii: F1000 Faculty Rev-1805. doi: 10.12688/f1000research.8737.1. eCollection 2016.
Horizontal gene transfer: essentiality and evolvability in prokaryotes, and roles in evolutionary transitions.
Koonin EV1.
Quantitatively, however, horizontal gene transfer (HGT) dominates microbial evolution, with the rate of gene gain and loss being comparable to the rate of point mutations and much greater than the duplication rate.

https://www.ncbi.nlm.nih.gov/pubmed/27303384
Front Microbiol. 2016 May 31;7:797. doi: 10.3389/fmicb.2016.00797. eCollection 2016.
Gene Loss and Horizontal Gene Transfer Contributed to the Genome Evolution of the Extreme Acidophile "Ferrovum".
Ullrich SR1, GonzÃ¡lez C2, Poehlein A3, Tischler JS1, Daniel R3, SchlÃ¶mann M1, Holmes DS4, MÃ¼hling M1.
Furthermore, analysis of their genome synteny provides first insights into their genome evolution, suggesting that horizontal gene transfer and genome reduction in the group 2 strains by loss of genes encoding complete metabolic pathways or physiological features contributed to the observed diversification.


2016-08-20
https://katosei.jsbba.or.jp/view_html.php?aid=652
ç´°èƒå¤–æ ¸é…¸ã‚’åˆ©ç”¨ã—ãŸç°¡ä¾¿ã§è¿…é€Ÿãªå½¢è³ªè»¢æ›ç³»ã®ç¢ºç«‹
å¾®ç”Ÿç‰©é–“ã§ç”Ÿã˜ã‚‹DNAã®æ°´å¹³ä¼æ’­ï¼ˆHorizontal Gene Transfer; HGTï¼‰ã¯è‡ªç„¶ç•Œã§ã®å¾®ç”Ÿç‰©ã®å¤šæ§˜æ€§ã«å¤§ããè²¢çŒ®ã—ã¦ã„ã‚‹ã“ã¨ãŒçŸ¥ã‚‰ã‚Œã¦ã„ã‚‹(1, 2)ï¼HGTã®åˆ†å­ãƒ¡ã‚«ãƒ‹ã‚ºãƒ ã¯ï¼Œãƒ•ã‚¡ãƒ¼ã‚¸ã«ã‚ˆã‚‹ã€Œå½¢è³ªå°å…¥ï¼ˆtransductionï¼‰ã€ï¼Œtype IV secretion systemã«ã‚ˆã‚‹ã€Œæ¥åˆä¼é”ï¼ˆconjugationï¼‰ã€ï¼Œãã—ã¦ç´°èƒå¤–æ ¸é…¸ã«ã‚ˆã‚‹ã€Œè‡ªç„¶å½¢è³ªè»¢æ›ï¼ˆnatural genetic transformationï¼‰ã€ã®3ã¤ã«å¤§åˆ¥ã•ã‚Œã‚‹ï¼


### HGT_2015

https://www.ncbi.nlm.nih.gov/pubmed/26439115
PLoS Comput Biol. 2015 Oct 6;11(10):e1004408.
Detecting Horizontal Gene Transfer between Closely Related Taxa.

https://www.ncbi.nlm.nih.gov/pubmed/26184597
Nat Rev Genet. 2015 Aug;16(8):472-82.
Horizontal gene transfer: building the web of life.

24 August 2015.
https://www.jstage.jst.go.jp/article/jjprotozool/48/1-2/48_31/_article/-char/ja/
Review
å¾®ç”Ÿç‰©ç”Ÿæ…‹ç³»ã«ãŠã‘ã‚‹ç´°èŒã®éºä¼å­æ°´å¹³ä¼æ’­ç¾è±¡ - æ—¥æœ¬åŸç”Ÿç”Ÿç‰©å­¦ä¼š
æ¾äº•ä¸€å½° è‘—
Bacterial HGTs are mediated by one of three mechanisms: transformation, conjugation, or transduction.
https://www.jstage.jst.go.jp/article/jjprotozool/48/1-2/48_31/_pdf
http://protistology.jp/journal/jjp48/JJP48MATSUI.pdf
å½¢è³ªå°å…¥ã«ã‚ˆã‚‹éºä¼å­æ°´å¹³ä¼æ’­ã«é–¢ã‚ã‚‹å› å­

ã¾ãŸæµ·æ´‹ã‚’æ¨¡ã—
ãŸå½¢è³ªå°å…¥ã®å®Ÿé¨“çµæœã‚’å…ƒã«ï¼ŒTampa Bay ã®æ±½æ°´åŸŸ
ï¼ˆã‚¢ãƒ¡ãƒªã‚«ãƒ•ãƒ­ãƒªãƒ€å·ï¼‰ã«ãŠã‘ã‚‹å½¢è³ªå°å…¥é »åº¦ã‚’æ±‚
ã‚ãŸç ”ç©¶ã§ã¯ï¼Œä¸€å¹´é–“ã« 1.3 Ã— 1014 å›ã®å½¢è³ªå°å…¥ãŒ
Tampa Bay ã§èµ·ã“ã£ã¦ã„ã‚‹ã¨è¦‹ç©ã‚‚ã‚‰ã‚Œã¦ã„ã‚‹
ï¼ˆJiang and Paul, 1998ï¼‰ï¼

éºä¼å­æ°´å¹³ä¼æ’­ã®æ¤œå‡ºã«ã¯ã€Œç³»çµ±æƒ…å ±ã€ã‚„ã€Œå¡©åŸºçµ„æˆã€ã«åŸºã¥ãæ‰‹æ³•ãŒç”¨ã„ã‚‰ã‚Œã‚‹ã€‚
https://en.wikipedia.org/wiki/Inferring_horizontal_gene_transfer
http://topicpageswiki.plos.org/wiki/Inferring_horizontal_gene_transfer
https://www.ncbi.nlm.nih.gov/pubmed/26020646
PLoS Comput Biol. 2015 May 28;11(5):e1004095.
Inferring horizontal gene transfer.

2014å¹´6æœˆ16æ—¥
https://www.brh.co.jp/salon/shinka/2014/post_000009.php
æ°´å¹³éºä¼å­ä¼æ¬ | JTç”Ÿå‘½èªŒç ”ç©¶é¤¨

2004 June
http://www.origin-life.gr.jp/3202/3202121/3202121.html
ç”Ÿæ…‹ç³»ã«ãŠã‘ã‚‹åºƒå®¿ä¸»åŸŸéºä¼å­ä¼é”ç²’å­ã®æ„ç¾©
1. è‡ªç„¶ç•Œã§ã®éºä¼å­ä¼æ’­å½¢å¼ã®ç”Ÿç‰©å­¦ã¨ï¼Œãã®ç”Ÿæ…‹ç³»ã¸ã®å¯„ä¸
i. å½¢è³ªè»¢æ›ï¼ˆTransformationï¼‰ï¼š
ii. æ¥åˆï¼ˆConjugationï¼‰ï¼š
iii. Transductionï¼ˆå½¢è³ªå°å…¥ï¼‰ï¼š

http://www.au-techno.com/microbio/microbio_body.htm
å¾®ç”Ÿç‰©å­¦ç”¨èª
å½¢è³ªè»¢æ› (transformationï¼šã‘ã„ã—ã¤ã¦ã‚“ã‹ã‚“)
å½¢è³ªå°å…¥ (transductionï¼šã‘ã„ã—ã¤ã©ã†ã«ã‚…ã†)
æ¥åˆ (conjugationï¼šã›ã¤ã”ã†)

### HGT_2014

https://www.ncbi.nlm.nih.gov/pubmed/24387194
BMC Genomics. 2014 Jan 3;15:8. doi: 10.1186/1471-2164-15-8.
ITEP: an integrated toolkit for exploration of microbial pan-genomes.
Benedict MN, Henriksen JR, Metcalf WW, Whitaker RJ, Price ND1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3890548/
Analysis of core and variable gene content
Studying gene gain and loss and examining the core (conserved) and variable (non-conserved) genes in a collection of organisms can provide insights into the plasticity of cellular functions and can be used to identify genes that define a clade [47]. 
Ochman H, Lerat E, Daubin V. Examining bacterial species under the specter of gene transfer and exchange. Proc Natl Acad Sci USA. 2005;15(Suppl 1):6595â€“6599. [PMC free article] [PubMed]

https://www.ncbi.nlm.nih.gov/pubmed/21170026
Nature. 2011 Jan 6;469(7328):93-6. doi: 10.1038/nature09649. Epub 2010 Dec 19.
Rapid evolutionary innovation during an Archaean genetic expansion.
David LA1, Alm EJ.
gene birth, transfer, duplication and loss events
http://almlab.mit.edu/angst.html
the proposal of a series of horizontal gene transfer (HGT), gene duplication (DUP) and gene loss (LOS) events to explain observed distributions of orthologous genes amongst extant prokaryotic genomes. 
![](http://almlab.mit.edu/img/29_AnGST.png)

### HGT_2010

https://www.ncbi.nlm.nih.gov/pubmed/20551134
Bioinformatics. 2010 Aug 1;26(15):1910-2. doi: 10.1093/bioinformatics/btq315. Epub 2010 Jun 15.
Count: evolutionary analysis of phylogenetic profiles with parsimony and likelihood.
CsurÃ¶s M1.
It implements popular methods employed in gene content analysis such as Dollo and Wagner parsimony, propensity for gene loss, as well as probabilistic methods involving a phylogenetic birth-and-death model.
http://www.iro.umontreal.ca/~csuros/gene_content/count.html

https://www.ncbi.nlm.nih.gov/pubmed/20376325
PLoS One. 2010 Apr 1;5(4):e9989. doi: 10.1371/journal.pone.0009989.
A benchmark of parametric methods for horizontal transfers detection.
Becq J1, Churlaud C, Deschavanne P.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848678/
Some methods such as GCtotal, GC1-GC3, 

https://pubmed.ncbi.nlm.nih.gov/19864285/
Review Proc Biol Sci
. 2010 Mar 22;277(1683):819-27. doi: 10.1098/rspb.2009.1679. Epub 2009 Oct 28.
Horizontal gene transfer in evolution: facts and challenges
Luis Boto 1
https://royalsocietypublishing.org/doi/10.1098/rspb.2009.1679
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842723/
secondly, about the extent of horizontal gene transfer towards different evolutionary times.

For example, it has been shown that the integration of a single transferred gene into regulatory interaction networks is very slow (Lercher & PÃ¡l 2008) in the case of genes providing the receptor with new functions, and it is dependent on the number of partners for the gene product in the regulatory network, according to the complexity hypothesis. 
ä¾‹ãˆã°ã€å—å®¹ä½“ã«æ–°ã—ã„æ©Ÿèƒ½ã‚’ä¸ãˆã‚‹éºä¼å­ã®å ´åˆã€1ã¤ã®è»¢ç§»éºä¼å­ãŒèª¿ç¯€ç›¸äº’ä½œç”¨ãƒãƒƒãƒˆãƒ¯ãƒ¼ã‚¯ã«çµ±åˆã•ã‚Œã‚‹ã®ã¯éå¸¸ã«é…ãï¼ˆLercher & PÃ¡l 2008ï¼‰ã€è¤‡é›‘æ€§ä»®èª¬ã«ã‚ˆã‚Œã°ã€èª¿ç¯€ãƒãƒƒãƒˆãƒ¯ãƒ¼ã‚¯ã«ãŠã‘ã‚‹ãã®éºä¼å­ç”£ç‰©ã®ãƒ‘ãƒ¼ãƒˆãƒŠãƒ¼ã®æ•°ã«ä¾å­˜ã™ã‚‹ã“ã¨ãŒç¤ºã•ã‚Œã¦ã„ã‚‹ã€‚

æ°´å¹³ä¼æ’­ã—ãŸéºä¼å­ãŒèª¿ç¯€ç›¸äº’ä½œç”¨ãƒãƒƒãƒˆãƒ¯ãƒ¼ã‚¯ã«çµ±åˆã•ã‚Œã‚‹ã«ã¯ã€ä½•ç™¾ä¸‡å¹´ã‚‚ã‹ã‹ã‚‹
https://pubmed.ncbi.nlm.nih.gov/18158322/
Mol Biol Evol
. 2008 Mar;25(3):559-67. doi: 10.1093/molbev/msm283. Epub 2007 Dec 24.
Integration of horizontally transferred genes into regulatory interaction networks takes many million years
Martin J Lercher 1, Csaba PÃ¡l
https://academic.oup.com/mbe/article/25/3/559/1060663
Fine-tuned integration of horizontally transferred genes into the regulatory network spans more than 8-22 million years and encompasses accelerated evolution of regulatory regions, stabilization of protein-protein interactions, and changes in codon usage.
æ°´å¹³ä¼æ’­ã—ãŸéºä¼å­ã¯ã€800ä¸‡å¹´ã‹ã‚‰2200ä¸‡å¹´ã®é–“ã«ã€åˆ¶å¾¡é ˜åŸŸã®åŠ é€Ÿçš„é€²åŒ–ã€ã‚¿ãƒ³ãƒ‘ã‚¯è³ªé–“ç›¸äº’ä½œç”¨ã®å®‰å®šåŒ–ã€ã‚³ãƒ‰ãƒ³ä½¿ç”¨é‡ã®å¤‰åŒ–ãªã©ã‚’çµŒã¦ã€åˆ¶å¾¡ãƒãƒƒãƒˆãƒ¯ãƒ¼ã‚¯ã«ãã‚ç´°ã‹ãçµ±åˆã•ã‚ŒãŸã€‚




https://www.ncbi.nlm.nih.gov/pubmed/18632554
Proc Natl Acad Sci U S A. 2008 Jul 22;105(29):10039-44. doi: 10.1073/pnas.0800679105. Epub 2008 Jul 16.
Modular networks and cumulative impact of lateral transfer in prokaryote genome evolution.
Dagan T1, Artzy-Randrup Y, Martin W.

https://www.ncbi.nlm.nih.gov/pubmed/18366724
BMC Genomics. 2008 Mar 24;9:136. doi: 10.1186/1471-2164-9-136.
Estimating the extent of horizontal gene transfer in metagenomic sequences.
Tamames J1, Moya A.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324111/
We have created two different methods that are suitable for the study of HGT in metagenomic samples. The methods are based on phylogenetic and DNA compositional approaches, and have allowed us to assess the extent of possible HGT events in metagenomes for the first time. The methods are shown to be compatible and quite precise, although they probably underestimate the number of possible events. Our results show that the phylogenetic method detects HGT in between 0.8% and 1.5% of the sequences, while DNA compositional methods identify putative HGT in between 2% and 8% of the sequences. These ranges are very similar to these found in complete genomes by related approaches. Both methods act with a different sensitivity since they probably target HGT events of different ages: the compositional method mostly identifies recent transfers, while the phylogenetic is more suitable for the detections of older events. 

https://www.ncbi.nlm.nih.gov/pubmed/18174121
Microbiology. 2008 Jan;154(Pt 1):1-15. doi: 10.1099/mic.0.2007/011833-0.
Are all horizontal gene transfers created equal? Prospects for mechanism-based studies of HGT patterns.
Zaneveld JR1, Nemergut DR, Knight R.
http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.2007/011833-0#tab2
Mechanisms of intercellular DNA transfer include conjugation, phage transduction, and transformation.


https://www.ncbi.nlm.nih.gov/pubmed/15653627
Nucleic Acids Res. 2005 Jan 13;33(1):e6.
Detection and characterization of horizontal transfers in prokaryotes using genomic signature.
Dufraigne C1, Fertil B, Lespinats S, Giron A, Deschavanne P.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC546175/


https://pubmed.ncbi.nlm.nih.gov/12902542/
Review Proc Natl Acad Sci U S A
. 2003 Aug 19;100(17):9658-62. doi: 10.1073/pnas.1632870100. Epub 2003 Aug 5.
Horizontal gene transfer: a critical view
C G Kurland 1, B Canback, Otto G Berg
https://www.pnas.org/doi/full/10.1073/pnas.1632870100
In E. coli the overall rate of import has been estimated at 10â€“6 genes per cell per generation (6, 37, 39). If one in a thousand is a possible replacement and one in a hundred of these is neutral, the rate of gene replacement in E. coli would be one per 109 years.
å¤§è…¸èŒã§ã¯ã€å…¨ä½“ã®è¼¸å…¥é€Ÿåº¦ã¯1ä¸–ä»£ã‚ãŸã‚Šç´°èƒã‚ãŸã‚Š10-6å€‹ã®éºä¼å­ã¨è¦‹ç©ã‚‚ã‚‰ã‚Œã¦ã„ã‚‹(6, 37, 39)ã€‚1000å€‹ã«1å€‹ãŒç½®æ›ã®å¯èƒ½æ€§ãŒã‚ã‚Šã€ãã®ã†ã¡100å€‹ã«1å€‹ãŒä¸­ç«‹ã§ã‚ã‚‹ã¨ã™ã‚‹ã¨ã€å¤§è…¸èŒã«ãŠã‘ã‚‹éºä¼å­ã®ç½®æ›é€Ÿåº¦ã¯109å¹´ã«1å€‹ã¨ã„ã†ã“ã¨ã«ãªã‚‹ã€‚
The residence time of a neutral coding sequence in E. coli seems to be <1 million years (6, 37, 39). 
å¤§è…¸èŒã«ãŠã‘ã‚‹ä¸­æ€§ã‚³ãƒ¼ãƒ‰é…åˆ—ã®æ»ç•™æ™‚é–“ã¯100ä¸‡å¹´æœªæº€ã¨æ€ã‚ã‚Œã‚‹(6, 37, 39)ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/11076857
Genome Res. 2000 Nov;10(11):1719-25.
Horizontal gene transfer in bacterial and archaeal complete genomes.
Garcia-VallvÃ© S1, Romeu A, Palau J.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310969/
total and positional G+C contents (G+C[T], G+C[1], G +C[2] and G+C[3]),
We considered genes as extraneous in terms of the G+C content if their G+C(T) content deviated by >1.5Ïƒ from the mean value of their genome or if deviations of G+C(1) and G+C(3) were of the same sign and at least one was >1.5Ïƒ.


[Karlin S (2001) Trends Microbiol. "Detecting anomalous gene clusters and pathogenicity islands in diverse bacterial genomes."](https://www.ncbi.nlm.nih.gov/pubmed/11435108) | [pdf](https://eclass.uoa.gr/modules/document/file.php/D473/Î’Î¹Î²Î»Î¹Î¿Î³ÏÎ±Ï†Î¯Î±/DNA%20Composition/Karlin_2001.pdf)



http://virtualgenomeproject.blogspot.com/
Virtual Genome Project Blog

http://virtualgenomeproject.blogspot.com/2008/12/functions-of-horizontally-transferred.html
Virtual Genome Project Blog: Functions of horizontally transferred genes

http://virtualgenomeproject.blogspot.com/2008/09/contribution-of-horizontal-gene.html
Virtual Genome Project Blog: Contribution of horizontal gene transfer to microbial evolution

http://virtualgenomeproject.blogspot.com/2008/07/reticulate-classification-of-mobile.html
Virtual Genome Project Blog: Reticulate classification of mobile genetic elements



----------

10:03 PM Â· Mar 6, 2023
https://twitter.com/M123Takahashi/status/1632728666873696261
é«˜æ©‹å°†å®œ Masayoshi Takahashi on X: "ç¢ºã‹ã«tæ¤œå®šã€åˆ†æ•£åˆ†æã€å›å¸°åˆ†æã‚’åŒã˜æ çµ„ã¿ã§è€ƒãˆã‚‰ã‚Œã‚‹ã“ã¨ã«è¨€åŠã—ã¦ã„ã‚‹æ›¸ç±ã¯å°‘ãªã„ã‹ã‚‚ã—ã‚Œã¾ã›ã‚“ã­ã€‚ãªãŠã€æ‹™è‘—ã€çµ±è¨ˆçš„å› æœæ¨è«–ã®ç†è«–ã¨å®Ÿè£…ã€p.43ã¨å²©å´å…ˆç”Ÿã®ã€çµ±è¨ˆçš„å› æœæ¨è«–ã€p.50ã§ã¯ã€èª¬æ˜å¤‰æ•°ã«ãƒ€ãƒŸãƒ¼å¤‰æ•°ã®ã¿ã‚’ç”¨ã„ãŸå›å¸°åˆ†æã¯2æ¨™æœ¬tæ¤œå®šã¨åŒã˜ã§ã‚ã‚‹ã“ã¨ã«è¨€åŠã—ã¦ã„ã¾ã™ã€‚" / X

6:35 AM Â· Feb 25, 2020
https://twitter.com/gekkou583/status/1232056543006183424
ãƒˆãƒªã‚¦ãƒ on Twitter: "ã„ã‚ã‚†ã‚‹ã€ã‚¾ã‚¦ã®æ™‚é–“ã€€ãƒã‚ºãƒŸã®æ™‚é–“ã€èª¬ã«å¯¾ã™ã‚‹ç–‘ç¾©ã€‚å†…æ¸©æ€§å¤–æ¸©æ€§å«ã‚€é™¸ä¸Šæ€§è„Šæ¤å‹•ç‰©4100ç¨®ã®ãƒ‡ãƒ¼ã‚¿æ¯”è¼ƒã€ä»£è¬é€Ÿåº¦ã¨å¯¿å‘½ã«è² ã®ç›¸é–¢ãŒã‚ã‚‹ã¨ã„ã†â€˜rateâ€ofâ€livingâ€™ ç†è«–ã¯è¨¼æ˜ã•ã‚Œãšã€ãã®å¹³å‡å¯¿å‘½ã¯å¤–å› ã«ã‚ˆã‚‹ã¨ã“ã‚ãŒå¤§ãã„ã¨ã®ã“ã¨ã€‚ https://t.co/Du5H6OmwQw" / Twitter
https://onlinelibrary.wiley.com/doi/full/10.1111/geb.13069
No evidence for the â€˜rateâ€ofâ€livingâ€™ theory across the tetrapod tree of life

10:53 AM Â· Dec 10, 2019
https://twitter.com/JunShimizu/status/1204217634662731776
Junichi Shimizu / æ¸…æ°´æº–ä¸€ on Twitter: "æ°—ã‚’ã¤ã‘ãŸã„ã€‚ã€€é–¢è¥¿å­¦é™¢å¤§å­¦ã®æ¸…æ°´å…ˆç”Ÿã®ãƒ–ãƒ­ã‚°ã€Œéƒ½é“åºœçœŒå˜ä½ã®åˆ†æã€å›½å˜ä½ã®åˆ†æã¯ã€ã„ã‚ã„ã‚ç½ ãŒã‚ã‚‹ã®ã§æ°—ã‚’ã¤ã‘ã‚ˆã†ã­ã€ã¨ã„ã†ãŠè©±ã§ã—ãŸã€‚ã€ ç´è±†ã¨ç‰›è‚‰ã®ã€Œã‚¤ã‚±ãƒŠã‚¤ã€é–¢ä¿‚ï¼šç©ºé–“çš„è‡ªå·±ç›¸é–¢ã®ãƒ¢ãƒ‡ãƒªãƒ³ã‚° | Sunny side up! https://t.co/JvGfvTWF2Y" / Twitter

9:10 AM Â· Dec 10, 2019
https://x.com/simizu706/status/1204191564378988544
Hiroshi Shimizu on X: "Stan Advent Calendar2019ã®10æ—¥ç›®ã®è¨˜äº‹ã§ã™ã€‚ ã€Œç´è±†ã¨ç‰›è‚‰ã®ã€Œã‚¤ã‚±ãƒŠã‚¤ã€é–¢ä¿‚ï¼šç©ºé–“çš„è‡ªå·±ç›¸é–¢ã®ãƒ¢ãƒ‡ãƒªãƒ³ã‚°ã€ ã„ã‚ã‚†ã‚‹ã€Galtonå•é¡Œã‚’çµ±è¨ˆãƒ¢ãƒ‡ãƒªãƒ³ã‚°ã§è§£æ±ºã—ã¦ã¿ã‚ˆã†ã€ã¨ã„ã†è¨˜äº‹ã§ã™ã€‚ https://t.co/QW9Pf2lHGn" / X

2019å¹´12æœˆ10æ—¥ by norimune
http://norimune.net/3313
ç´è±†ã¨ç‰›è‚‰ã®ã€Œã‚¤ã‚±ãƒŠã‚¤ã€é–¢ä¿‚ï¼šç©ºé–“çš„è‡ªå·±ç›¸é–¢ã®ãƒ¢ãƒ‡ãƒªãƒ³ã‚° | Sunny side up!
Galtonå•é¡Œã‚’ç°¡å˜ã«è¨€ã†ã¨ã€å›½åˆ¥ã€åœ°åŸŸåˆ¥ã®æ–‡åŒ–ãƒ‡ãƒ¼ã‚¿é–“ã®ç›¸é–¢ã¯ã€å›½ã‚„åœ°åŸŸãŒæ­´å²ã‚’å…±æœ‰ã—ã¦ã„ã‚‹ï¼ˆæ–‡åŒ–ä¼æ¬ãŒèµ·ãã¦ã„ã‚‹ï¼‰ã“ã¨ã«ã‚ˆã‚‹ã‚‚ã®ãªã®ã‹ã€ãã®æ–‡åŒ–çš„ãªæ©Ÿèƒ½ã«é–¢é€£æ€§ãŒã‚ã‚‹ã®ã‹ãŒåŒºåˆ¥ã§ããªã„ã€ã¨ã„ã†ã“ã¨ã§ã™ã€‚


æ“¬ä¼¼åå¾©
å½åå¾©
https://github.com/haruosuz/statistics/blob/master/README.md#pseudoreplicate

----------
## pcm
phylogenetic comparative methods

https://en.wikipedia.org/wiki/Phylogenetic_comparative_methods
Phylogenetic comparative methods (PCMs) use information on the historical relationships of lineages (phylogenies) to test evolutionary hypotheses.

https://hydrodictyon.eeb.uconn.edu/people/plewis/courses/phylogenetics/lectures/2022/IndependentContrasts.pdf
Fig. 5 from Felsenstein (1985)
Fig. 6 from Felsenstein (1985b)
Fig. 7 from Felsenstein (1985b)

https://www.jstor.org/stable/2461605
Joseph Felsenstein (1985) Phylogenies and the Comparative Method

FIG. 7.-The same data set, with the points distinguished to show the members of the 2
monophyletic taxa. It can immediately be seen that the apparently significant relationship of
fig. 6 is illusory

http://web.biosci.utexas.edu/bio386/Readings/Literature/Quantitative%20Char/Felsenstein.AmNat.125.1.pdf

https://pubmed.ncbi.nlm.nih.gov/34750532/
Nat Ecol Evol
. 2021 Dec;5(12):1624-1636. doi: 10.1038/s41559-021-01573-2. Epub 2021 Nov 8.
Plasmids do not consistently stabilize cooperation across bacteria but may promote broad pathogen host-range
Anna E Dewar # 1, Joshua L Thomas # 2, Thomas W Scott 2, Geoff Wild 3, Ashleigh S Griffin 2, Stuart A West 2, Melanie Ghoul 2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612097/

Statistical tests typically assume that data points are independent, and even slight non-independence can lead to heavily biased results (type I errors)21,22. There is an extensive literature in the field of evolutionary biology showing that species share characteristics inherited though common descent, rather than through independent evolution, and so cannot be considered independent data points23â€“25. Genomes are nested within species, and genes are nested within genomes, multiplying this problem of non-independence, analogous to the problem of pseudoreplication in experimental studies26â€“29.

Why does using bacterial genomes as independent data points lead to a significant result? By using a Wilcoxon signed-rank test, at the level of the genome, we are implicitly assuming that all the genomes analysed are: (i) independent from one another; (ii) a representative sample of bacteria in nature. Neither of these are true for multi-species genomic datasets. First, due to shared ancestry, species are not independent from one another, and so neither are genomes in such analyses24,42. Even a slight lack of independence can lead to heavily biased results in statistical analyses and spurious conclusions21. Second, genomic databases tend to have a disproportionate abundance of certain species and genera. This will bias the results towards commonly sequenced species.


8:02 AM Â· Aug 6, 2020
https://twitter.com/wsdewitt/status/1291147602507862016
William DeWitt on Twitter: "Genome of the whale shark, Earth's largest fishâ€”contributed by @geochurch and colleagues. Ostensibly extremely significant correlations are reported between physiological and genomic features across taxa, interpreted as evincing scaling laws. https://t.co/9EvT18ClPR (1/n)" / Twitter

For example, here are the first couple panels of Fig 2. Impressive p-values. But the species (points) in these correlations are phylogenetically related, so they are not independent samples, as it seems the simple correlation analysis done here assumes. History is shared. (2/n)

Not accounting for the null covariance structure induced by phylogeny seems like a problem. The effective number of independent samples is smaller, so statistical significance is inflated. Shared evolutionary history means independent traits will appear linked (3/n)

Joe Felsenstein showed how to do this correctly in 1985 using phylogenetically independent contrasts. It's a widely known paper (although perhaps not widely enough). American Naturalist has a recent piece on the history and impact of Joe's paper: https://journals.uchicago.edu/doi/10.1086/703055 (4/n)

To demonstrate the problem, Joe considers two independent traits randomly evolving on this phylogeny (according to independent Brownian motion for each trait) (5/n)

If you measure the two traits for each species (the tips of the tree) and make a scatter plot, it looks like they're correlated. (5/n)

But this is an artifact of the phylogeny. If we annotate the points by which clade they came from, the correlation disappears. The traits are independent, but the samples are not. Samples within a clade share most of their random history. (6/n)

Although the correlation analysis and p-values in this new paper are not meaningful, it would be interesting to see if the scaling relationships held up to a phylogenetically correct analysis. Also, having the whale shark genome is totally awesome (ğŸ¦ˆ/n)


10:22 AM Â· Dec 26, 2021
https://twitter.com/9w9w9w92/status/1474913385733312517
å°¾ä¸Šæ­£äºº on Twitter: "ã€Œã²ã¨ã¤ã®ç¾ç”Ÿç¨®ã‚’ã²ã¨ã¤ã®ãƒ‡ãƒ¼ã‚¿ç‚¹ã ã¨å˜ç´”ã«è¦‹ã¦ç›¸é–¢ãªã©ã‚’èª¿ã¹ã‚‹ã“ã¨ã«ã‚ˆã‚Šå½¢è³ªã®é€²åŒ–ã‚’æ¤œè¨ã™ã‚‹ã¨ã€ã‚ã‚‹æã§èµ·ã“ã£ãŸåŒã˜é€²åŒ–çš„å¤‰åŒ–ã‚’åˆ¥ã®ç¾ç”Ÿç¨®ã‚’è¦‹ã¦ä½•åº¦ã‚‚ãã‚Šã‹ãˆã—è©•ä¾¡ã—ã¦ã—ã¾ã†ã“ã¨ã«ãªã‚‹ã€‚çµ±è¨ˆçš„ãªè¨€ã„æ–¹ã§ã¯ã€èª¤ã£ã¦ãƒ‡ãƒ¼ã‚¿ç‚¹ã™ãªã‚ã¡ç¾ç”Ÿç¨®ãŒç‹¬ç«‹ã§ã‚ã‚‹ã¨ä»®å®šã—ã¦ã—ã¾ã£ã¦ã„ã‚‹ã€58é " / Twitter
9:17 PM Â· Dec 24, 2021
https://twitter.com/9w9w9w92/status/1474353642673426432
çŸ³å·çµ±ãƒ»æ–è—¤æˆä¹Ÿãƒ»ä½è—¤çŸ©è¡Œãƒ»é•·è°·å·çœç†å­ç·¨ï¼ˆ2005ï¼‰ã€é€²åŒ–å­¦ã®æ–¹æ³•ã¨æ­´å²ã€€ã‚·ãƒªãƒ¼ã‚ºé€²åŒ–å­¦ï¼—ã€å²©æ³¢æ›¸åº—


6:42 AM Â· Jan 13, 2020
https://twitter.com/vsbuffalo/status/1216475530066743296
Vince Buffalo on Twitter: "A question for comparative methods folks: when we think about patterns across extremely broad taxonomical groups, such as the body length and generation time (Bonner 1983) and body size and pop density (Damuth 1987), does it make sense to correct for phylogeny? https://t.co/0Ggr9uEAJA" / Twitter

2019-09-25
https://eprints.lib.hokudai.ac.jp/dspace/bitstream/2115/76072/1/Yusaku_OHKUBO_abstract.pdf
Author(s) å¤§ä¹…ä¿, ç¥ä½œ
Citation åŒ—æµ·é“å¤§å­¦. åšå£«(ç’°å¢ƒç§‘å­¦) ç”²ç¬¬13743å·
On the phylogenetic comparative analysis of directional evolution by Approximate Bayesian Computation
ï¼ˆè¿‘ä¼¼ãƒ™ã‚¤ã‚ºè¨ˆç®—ã‚’ç”¨ã„ãŸæ–¹å‘æ€§é€²åŒ–ã®åˆ†æã«ã¤ã„ã¦ï¼‰
ç³»çµ±æ¯”è¼ƒæ³• (Phylogenetic Comparative Method; PCM ç³»çµ±ç¨®é–“æ¯”è¼ƒã¨ã‚‚å‘¼ã°ã‚Œã‚‹)ã¯ã€ç”Ÿç‰©ã®å½¢è³ªé€²åŒ–ã‚’
åˆ†æã™ã‚‹çµ±è¨ˆçš„ãªæ‰‹æ³•ã§ã‚ã‚Šã€è¿‘å¹´ç”Ÿæ…‹å­¦ã‚„é€²åŒ–å­¦ã‚’ä¸­å¿ƒã«éå¸¸ã«é‡è¦ãªå½¹å‰²ã‚’æœãŸã—ã¦ã„ã‚‹ã€‚ã—ã‹ã—ã“ã‚Œ
ã¾ã§å¤šãã®ç³»çµ±æ¯”è¼ƒæ³•ã¯ã€å½¢è³ªã®å¾“ã†é€²åŒ–ã¨ã—ã¦ãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«ã‚„Ornstein-Uhlenbeckãƒ¢ãƒ‡ãƒ«ãªã©ã®ã”
ãé™ã‚‰ã‚ŒãŸãƒ—ãƒ­ã‚»ã‚¹ã‚’ä»®å®šã—ã¦ãŠã‚Šã€ç”Ÿç‰©é€²åŒ–ã®ä¸­ã§æœ€ã‚‚é‡è¦ãªãƒ—ãƒ­ã‚»ã‚¹ã®ï¼‘ã¤ã§ã‚ã‚‹æ–¹å‘æ€§é€²åŒ–ã‚’ã©ã®ã‚ˆ
ã†ã«åˆ†æã™ã‚‹ã¹ãã‹ååˆ†æ¤œè¨ã•ã‚Œã¦ã“ãªã‹ã£ãŸã€‚ãã“ã§æœ¬ç ”ç©¶ã¯ã€æ—¢å­˜ã®PCMã«ãŠã‘ã‚‹æ–¹å‘æ€§é€²åŒ–ã®åˆ†æã«
ãŠã‘ã‚‹å•é¡Œç‚¹ã‚’æŒ‡æ‘˜ã—ã€æ–°ãŸãªåˆ†ææ–¹æ³•ã‚’ææ¡ˆã—ãŸã€‚ã¾ãŸã€ãã®æˆæœã‚’å…ˆè¡Œç ”ç©¶ã®å†åˆ†æã¨ã‚·ãƒŸãƒ¥ãƒ¬ãƒ¼ã‚·ãƒ§
ãƒ³å®Ÿé¨“ã§ç¢ºèªã—ãŸã€‚


https://pubmed.ncbi.nlm.nih.gov/27536807/
J Evol Biol
. 2016 Dec;29(12):2422-2435. doi: 10.1111/jeb.12966. Epub 2016 Sep 9.
A comparative perspective on longevity: the effect of body size dominates over ecology in moths
S Holm 1, R B Davis 1, J JavoiÅ¡ 1, E Ã•unap 1 2, A Kaasik 1, F Molleman 1 3 4, T Tammaru 1
https://onlinelibrary.wiley.com/doi/full/10.1111/jeb.12966
Phylogenetic comparative analyses were applied to study variation in speciesâ€specific values of lifespan and to reveal its ecological and lifeâ€history correlates.

https://pubmed.ncbi.nlm.nih.gov/27499839/
Methods Ecol Evol
. 2016 Jun;7(6):693-699. doi: 10.1111/2041-210X.12533. Epub 2016 Jun 13.
Shedding light on the 'dark side' of phylogenetic comparative methods
Natalie Cooper 1, Gavin H Thomas 2, Richard G FitzJohn 3
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4957270/
https://www.nhm.ac.uk/our-science/our-work/origins-evolution-and-futures/phylogenetic-comparative-methods.html
Phylogenetic comparative methods can be used to investigate the relationships between body size, life span and flight

https://www.esj.ne.jp/meeting/abst/62/T02-4.html
æ—¥æœ¬ç”Ÿæ…‹å­¦ä¼šç¬¬62å›å…¨å›½å¤§ä¼š (2015å¹´3æœˆã€é¹¿å…å³¶) 
ç³»çµ±ç¨®é–“æ¯”è¼ƒæ³•ï¼ˆPCM: phylogenetic comparative methodsï¼‰ã¯ç¨®é–“ã®ç³»çµ±é–¢ä¿‚ã‚’è€ƒæ…®ã—ãŸå½¢è³ªæ¯”è¼ƒã‚’è¡Œã†æ‰‹æ³•ã§ã‚ã‚Šï¼Œé©å¿œé€²åŒ–ã‚’æ¤œè¨¼ã™ã‚‹ã†ãˆã§æœ‰åŠ›ãªã‚¢ãƒ—ãƒ­ãƒ¼ãƒã§ã‚ã‚‹ï¼ã“ã‚Œã¾ã§ï¼Œé‡çš„å½¢è³ªã«é–¢ã™ã‚‹åˆ†æã§ã¯ç³»çµ±ä¸€èˆ¬åŒ–æœ€å°äºŒä¹—æ³•ï¼ˆPGLS: phylogenetic generalized least squaresï¼‰ãŒç”¨ã„ã‚‰ã‚Œã‚‹ã“ã¨ãŒå¤šã‹ã£ãŸï¼ã—ã‹ã—ï¼ŒPGLSã¯å½¢è³ªé€²åŒ–ãŒåŸºæœ¬çš„ã«ãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ï¼ˆãƒ©ãƒ³ãƒ€ãƒ æµ®å‹•ï¼‰ã«å¾“ã†ã“ã¨ã‚’ä»®å®šã™ã‚‹ãŸã‚ï¼Œæ¤œè¨¼ã§ãã‚‹ãƒ¢ãƒ‡ãƒ«ãŒé™ã‚‰ã‚Œã‚‹ï¼ä¸€æ–¹ã§ï¼Œè¿‘å¹´é–‹ç™ºã•ã‚ŒãŸè¿‘ä¼¼ãƒ™ã‚¤ã‚ºè¨ˆç®—ï¼ˆABC: approximate Bayesian computationï¼‰ã‚’ç”¨ã„ãŸç³»çµ±ç¨®é–“æ¯”è¼ƒï¼ˆABC-PCM; Kutsukake & Innan 2013ï¼‰ã§ã¯ã‚ˆã‚ŠæŸ”è»Ÿãªãƒ¢ãƒ‡ãƒ«ã‚‚æ¤œè¨¼ã§ãã‚‹ï¼

https://www.fifthdimension.jp/wiki.cgi?page=FrontPage&file=20100522BiometricsJapanPreprint%2Epdf&action=ATTACH
ç”°è¾ºæ™¶å², 2010, "ãƒ™ã‚¤ã‚¸ã‚¢ãƒ³MCMCã«ã‚ˆã‚‹ç”Ÿç‰©é–“ç³»çµ±é–¢ä¿‚ã®æ¨å®šæ³•"

ç”Ÿç‰©å­¦ã«ãŠã‘ã‚‹ç³»çµ±æ¨¹ã®å¿…è¦æ€§

ç³»çµ±æ¨¹ã¯ã€è¡¨ç¾å‹å½¢è³ªãªã©ã‚’ãã®ä¸Šã«é…ç½® (ç¥–å…ˆå½¢è³ªå¾© å…ƒ) ã—ãŸã‚Šã™ã‚‹ã“ã¨ã§é€²åŒ–ã®éç¨‹ã‚’æ˜ã‚‰ã‹ã«ã™ã‚‹ã“ã¨ãŒã§ ãã‚‹ã€‚

ãã‚‚ãã‚‚ã€ã“ã‚Œã¾ã§ã«ã€Œç³»çµ±çš„ã«ç‹¬ç«‹ã—ãŸç”Ÿç‰©ã€ã¯ç¢ºèª ã•ã‚Œã¦ã„ãªã„ã€‚ãã®ãŸã‚ã€çµ±è¨ˆè§£æä¸Šã®æœ€ã‚‚é‡è¦ãªä»®å®šã®ä¸€ã¤ã§ã‚ã‚‹ã€Œã‚µãƒ³ãƒ—ãƒ«é–“ã®ç‹¬ç«‹ æ€§ã€ãŒç”Ÿç‰©ã§ã¯æˆç«‹ã—å¾—ãªã„ã“ã¨ã¯æ˜ç™½ã§ã‚ã‚‹ã€‚
ç³»çµ±é–¢ä¿‚=ã‚µãƒ³ãƒ—ãƒ«é–“ã®éä¾å­˜æ€§ã‚’è€ƒæ…®ã—ã¦çµ±è¨ˆè§£æã‚’è¡Œã†ã“ã¨ã§ã“ã®ã‚ˆã†ãªå•é¡Œã‚’ è§£æ±ºã—ã‚ˆã†ã¨ã™ã‚‹æ‰‹æ³•ãŒã‚ã‚Šã€ç³»çµ±çš„ç‹¬ç«‹æ¯”è¼ƒæ³•ãªã©ã¨å‘¼ã°ã‚Œã¦ã„ã‚‹ (Felsenstein, 1985; Grafen, 1989)ã€‚ã“ã‚ŒãŒç³»çµ±æ¨¹ã®ç¬¬ 2 ã®ç”¨é€”ã§ã‚ã‚‹ã€‚

ç³»çµ±æ¨¹ã®ç¬¬ 3 ã®ç”¨é€”ã¯ã€å¤šæ§˜æ€§ã®å®šé‡åŒ–ã§ã‚ã‚‹ã€‚

https://www.ikushimo.com/news/2009/03/23.html
â—† [Science] ç©ºé–“è‡ªå·±ç›¸é–¢ã¨æ™‚é–“è‡ªå·±ç›¸é–¢ã€ãã—ã¦ç³»çµ±è‡ªå·±ç›¸é–¢ - 00:45:32
ã€€ç©ºé–“è‡ªå·±ç›¸é–¢ãŒã‚ã‚‹(ç©ºé–“çš„ã«è¿‘ã„ç‚¹ã¯äº’ã„ã«ä¼¼ã¦ã„ã‚‹)ãƒ‡ãƒ¼ã‚¿ã‚’ç‹¬ç«‹ãƒ‡ãƒ¼ã‚¿ã¨è¦‹ãªã—ã¦è§£æã—ã¦ã¯ã„ã‘ãªã„ã€ã¨ã„ã†ã®ãŒã€Œè‡ªå·±ç›¸é–¢ã¨å½åå¾©ã€ä¼ç”»é›†ä¼šã®è¶£æ—¨ã§ã—ãŸã€‚ã‚‚ã¡ã‚ã‚“ã€è‡ªå·±ç›¸é–¢ãŒã‚ã‚‹ã®ã¯ç©ºé–“ã ã‘ã§ã¯ãªãã€æ™‚é–“ã«ã‚‚ã‚ã‚Šã¾ã™ã€‚æ™‚é–“è‡ªå·±ç›¸é–¢ã‚’è€ƒæ…®ã—ãŸè§£æã¯æ˜”ã‹ã‚‰ã‚ã‚Šã€æ™‚ç³»åˆ—è§£æã¨å‘¼ã°ã‚Œã‚‹è§£ææ–¹æ³•ç¾¤ãŒãã‚Œã«å½“ãŸã‚Šã¾ã™ã€‚
ã€€ã¾ãŸã€æ™‚é–“è‡ªå·±ç›¸é–¢ã®1ç¨®ã¨è¨€ãˆã‚‹ã‹ã‚‚ã—ã‚Œã¾ã›ã‚“ãŒã€ç³»çµ±çš„ã«è¿‘ã„ç”Ÿç‰©ã¯äº’ã„ã«ä¼¼ã¦ã„ã‚‹ã¨ã„ã†ç³»çµ±è‡ªå·±ç›¸é–¢ã‚‚å­˜åœ¨ã—ã¾ã™ã€‚ã“ã‚Œã‚’ã¡ã‚ƒã‚“ã¨è€ƒæ…®ã«å…¥ã‚Œã¦ã‚„ã‚ã†ã¨ã„ã†ã®ãŒã€ç³»çµ±è‡ªç”±é›†ä¼šã§ã‚‚ã†ä¸€äººã®æ¼”è€…ãŒè©±ã—ãŸã€Œç³»çµ±çš„ç‹¬ç«‹æ¯”è¼ƒã€(Phylogenetic Independent Contrast)ã§ã™ã€‚
ã€€ã—ã‹ã—ã€ç©ºé–“è‡ªå·±ç›¸é–¢ã«ã¯å®Ÿã¯ä»–ã®äºŒã¤ã¨ã¯å¤§ããç•°ãªã‚‹ç‚¹ãŒã‚ã‚Šã¾ã™ã€‚ãã‚Œã¯ã€ã€Œè·³èºã€ãŒã‚ã‚Šå¾—ã‚‹ã¨ã„ã†ç‚¹ã§ã™ã€‚

http://leeswijzer.org/files/seibutsu-keitou.html
ã€€ã€€ã€€ï¼•ï¼ï¼‘ï¼ï¼‘ï¼ï¼’ï¼šæ‡Šæ‚©ï¼çµ±è¨ˆçš„ç‹¬ç«‹ã®å•é¡Œ 334
çµ±è¨ˆçš„ç‹¬ç«‹ã®å•é¡Œã€‚ç¨®é–“ã«ç³»çµ±é–¢ä¿‚ãŒã‚ã‚‹ãªã‚‰ã°ã€çµ±è¨ˆå­¦çš„ã«ã¿ã¦ãƒ‡ãƒ¼ã‚¿ç‚¹ã¯ç‹¬ç«‹ã§ã¯ã‚ã‚Šå¾—ãªã„ã€‚ï¼ˆä¸‰ä¸­ä¿¡å®ã€Œç”Ÿç‰©ç³»çµ±å­¦ã€334ãƒšãƒ¼ã‚¸ï¼‰
FIG.2. ãƒ‡ãƒ¼ã‚¿ç‚¹ãŒç‹¬ç«‹
FIG.3. ãƒ‡ãƒ¼ã‚¿ç‚¹ãŒéç‹¬ç«‹

----------
### lukejharmon

- https://lukejharmon.github.io/pcm/
- https://lukejharmon.github.io/pcm/chapters/
- https://lukejharmon.github.io/pcm/chapter1_introduction/
Introduction Â· Phylogenetic Comparative Methods

https://lukejharmon.github.io/pcm/pdf/phylogeneticComparativeMethods.pdf
Phylogenetic Comparative Methods
Luke J. Harmon 2019-3-15

https://lukejharmon.github.io/pcm/chapter6_beyondbm/
Beyond Brownian motion Â· Phylogenetic Comparative Methods
https://lukejharmon.github.io/pcm/chapter6_beyondbm/#section-6.3a-rate-tests-using-phylogenetic-independent-contrasts

Section 6.3a: Rate tests using phylogenetic independent contrasts

In particular, Garland (1992) suggests using a t-test, as long as the absolute value of independent contrasts are approximately normally distributed. However, under a Brownian motion model, the contrasts themselves â€“ but not the absolute values of the contrasts â€“ should be approximately normal, so it is quite likely that absolute values of contrasts will strongly violate the assumptions of a t-test.

A t-test is not significant (Welch two-sample t-test Pâ€„=â€„0.42), but we also can see that the distribution of PIC absolute values is strongly skewed (Figure 6.2C).

Alternatively, we can again follow Garlandâ€™s (1992) suggestion and use a Mann-Whitney U-test, the nonparametric equivalent of a t-test, on the absolute values of the contrasts.

https://ja.wikipedia.org/wiki/Tæ¤œå®š
æ¨™æœ¬ãŒç‹¬ç«‹ã§ã€ç­‰åˆ†æ•£æ€§ãŒä»®å®šã§ããªã„ï¼ˆç•°åˆ†æ•£ï¼‰å ´åˆã€‚ã“ã‚Œã¯æ­£ç¢ºã«ã¯ã‚¦ã‚§ãƒ«ãƒã®tæ¤œå®šã¨å‘¼ã°ã‚Œã‚‹ã€‚
tæ¤œå®šã¯æ¯é›†å›£ã®æ­£è¦åˆ†å¸ƒã‚’å‰æã¨ã™ã‚‹ãƒ‘ãƒ©ãƒ¡ãƒˆãƒªãƒƒã‚¯æ¤œå®šã§ã‚ã‚‹ãŒã€ã“ã®æ¡ä»¶ãŒæº€ãŸã•ã‚Œãšã€ã•ã‚‰ã«æ¨™æœ¬ã‚µã‚¤ã‚ºãŒå°ã•ã„ã¨ã€tæ¤œå®šã§è¿‘ä¼¼ã™ã‚‹ã“ã¨ã‚‚å›°é›£ã¨ãªã‚‹ã€‚ãã†ã„ã£ãŸå ´åˆã«ã¯ãƒãƒ³ãƒ‘ãƒ©ãƒ¡ãƒˆãƒªãƒƒã‚¯æ¤œå®šã‚’ç”¨ã„ã‚‹æ–¹æ³•ãŒã‚ã‚‹ã€‚
æ¨™æœ¬ãŒç‹¬ç«‹ãªã‚‰ã°ãƒãƒ³ãƒ»ãƒ›ã‚¤ãƒƒãƒˆãƒ‹ãƒ¼ã®Uæ¤œå®šãªã©

----------
## brownian

Brownian motion
ãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«

http://www.phytools.org/***SanJuan2016/ex/5/Fitting-BM.html
Exercise 5.2: Fitting Brownian motion models

https://lukejharmon.github.io/ilhabela/instruction/2015/07/04/multi-regime-models/
Multi-rate, multi-regime, and multivariate models for continuous traits
Jul 4, 2015
This tutorial is about fitting multi-rate Brownian motion models (using phytools), multi-regime OU models (using the OUwie package), and multivariate Brownian models (using phytools).
Written by Liam J. Revell. Last updated July 4, 2015

http://phytools.org/eqg/Exercise_4.1/
Simulating Brownian motion in R
Written by Liam J. Revell. Last updated Aug. 8, 2013


6:59 PM Â· Jul 6, 2018
https://twitter.com/tharano34/status/1015173437885538305
ç³»çµ±ç¨®é–“æ¯”è¼ƒã§ã¯ã€å½¢è³ªã®é€²åŒ–ã«ãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«ã‚’ä»®å®šã™ã‚‹ã®ãŒä¸€èˆ¬çš„ã§ã‚ã‚‹ã€‚ãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«ã§ã¯ã€é€²åŒ–ã®æ–¹å‘ã¨å¤‰åŒ–é‡ãŒãƒ©ãƒ³ãƒ€ãƒ ã«å¤‰åŒ–ã™ã‚‹ã€‚ã“ã®ãƒ¢ãƒ‡ãƒ«ã«å¯¾ã—ã¦ã€ãƒã‚³ç§‘ç³»çµ±æ¨¹å†…ã§ã‚¦ãƒ³ãƒ”ãƒ§ã‚¦ã€ã‚¹ãƒŸãƒ­ãƒ‰ãƒ³ã€ãƒ›ãƒ¢ãƒ†ãƒªã‚¦ãƒ ã®ãã‚Œãã‚Œã«è‡³ã‚‹æã«ã€æ–¹å‘æ€§é¸æŠã®åŠ¹æœã‚’è¡¨ã™ãƒ‘ãƒ©ãƒ¡ãƒ¼ã‚¿ã‚’çµ„ã¿è¾¼ã‚“ã ã€‚


20090903EvoJapanPoster.pdf
Untitled - Akifumi S. Tanabehttps://www.fifthdimension.jp â€º wiki
PDF
å¾“æ¥ã®åˆ†å­é€²åŒ–é€Ÿåº¦ãŒãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ã™ã‚‹ã¨ä»®å®šã—ãŸåˆ†å²å¹´ä»£æ¨å®šæ³•ã§ã¯åˆ†å²ã®ä¸€æ–¹ã§ã¯åŠ é€Ÿã€ ... å¾“æ¥æ³•ã¨ã€åˆ†å²ã®ä¸¡æ–¹ã§åŠ é€Ÿã‚„ä¸¡æ–¹ã§æ¸›é€Ÿã‚’æ¡ç”¨ã—ã‚„ã™ã„ã€Œâ€‹å®šå‘é€²åŒ–çš„ãƒ¢ãƒ‡ãƒ«ã€ã‚’ä»®å®šã—ãŸ ... ç³»çµ±æ¨¹:æœ‰èƒç›¤å“ºä¹³é¡ã®æœ€å°¤ç³»çµ±æ¨¹ (Murphy et alâ€‹.

2008å¹´ ç¦å²¡
https://www.esj.ne.jp/meeting/abst/55/P3-159.html
æ—¥æœ¬ç”Ÿæ…‹å­¦ä¼šå…¨å›½å¤§ä¼š ESJ55 è¬›æ¼”è¦æ—¨
ä¸€èˆ¬è¬›æ¼”ï¼ˆãƒã‚¹ã‚¿ãƒ¼ç™ºè¡¨ï¼‰ P3-159
å½¢è³ªé€²åŒ–ã«ãŠã‘ã‚‹è‡ªç„¶é¸æŠã¨éºä¼çš„æµ®å‹•ã®ç›¸å¯¾çš„ãªé‡è¦æ€§ï¼šãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«ã¨ã®æ¯”è¼ƒ
*é å±±å¼˜æ³•ï¼ˆä¹å¤§ãƒ»ç†ãƒ»ç”Ÿæ…‹ï¼‰, çŸ¢åŸå¾¹ä¸€ï¼ˆä¹å¤§ãƒ»ç†ãƒ»ç”Ÿæ…‹ï¼‰
é¸æŠã¨æµ®å‹•ã®ç›¸å¯¾çš„ãªé‡è¦æ€§ã‚’æ˜ã‚‰ã‹ã«ã™ã‚‹ãŸã‚ã«ã€ç³»çµ±æ¨¹ã‚’åŸºã«ã—ãŸè§£æã‚’è¡Œã£ãŸã€‚å…·ä½“çš„ã«ã¯ã€ç³»çµ±æ¨¹ã¨å½¢è³ªå€¤ã‹ã‚‰æ¨å®šã•ã‚Œã‚‹å¯¾æ¯”ã‚’ç”¨ã„ã¦ã€ä¸­ç«‹ï¼ˆãƒ–ãƒ©ã‚¦ãƒ³é‹å‹•ãƒ¢ãƒ‡ãƒ«ï¼‰ã‚’ä»®å®šã—ãŸå½¢è³ªé€²åŒ–ã‚’10000å›ã‚·ãƒŸãƒ¥ãƒ¬ãƒ¼ã‚·ãƒ§ãƒ³ã—ã€å®Ÿãƒ‡ãƒ¼ã‚¿ã¨ã®æ¯”è¼ƒã‚’è¡Œã£ãŸã€‚

----------
## outgroup

https://en.wikipedia.org/wiki/Computational_phylogenetics#Using_outgroups

https://en.wikipedia.org/wiki/Outgroup_(cladistics)
In cladistics or phylogenetics, an outgroup[1] is a more distantly related group of organisms that serves as a reference group when determining the evolutionary relationships of the ingroup, the set of organisms under study, 

https://en.wikipedia.org/wiki/Outgroup_(cladistics)#Choice_of_outgroup

To qualify as an outgroup, a taxon must satisfy the following two characteristics:

- It must not be a member of the ingroup.
- It must be related to the ingroup, closely enough for meaningful comparisons to the ingroup.

https://ja.wikipedia.org/wiki/å¤–ç¾¤
å¤–ç¾¤ã¯ã€å†…ç¾¤ã®ä»–ã®ã‚°ãƒ«ãƒ¼ãƒ—ã«è¿‘ç¸ã§ã‚ã‚‹ãŒã€ã—ã‹ã—ä»–ã®ã©ã®å†…ç¾¤ã®ã‚°ãƒ«ãƒ¼ãƒ—ã‚ˆã‚Šã‚‚äº’ã„ã«é ç¸ã§ã‚ã‚‹ã¨è€ƒãˆã‚‰ã‚Œã‚‹ã‚‚ã®ã‚’é¸æŠã™ã‚‹ã€‚
å¤–ç¾¤ã¯å†…ç¾¤ã®å§‰å¦¹ç¾¤ã€ã¾ãŸã¯ã‚ˆã‚Šé ç¸ãªã‚‚ã®ã§ã‚ã‚‹[2]ã€‚

å¤–ç¾¤ã®é¸æŠ
æœ€é©ãªå¤–ç¾¤ã¯ä»¥ä¸‹ã®2ã¤ã®æ¡ä»¶ã‚’æº€è¶³ã—ãªã‘ã‚Œã°ãªã‚‰ãªã„ã€‚
1. å†…ç¾¤ã«å«ã¾ã‚Œã¦ã¯ãªã‚‰ãªã„
2. å†…ç¾¤ã¨æ„å‘³ã®ã‚ã‚‹æ¯”è¼ƒãŒã§ãã‚‹ã»ã©å†…ç¾¤ã«è¿‘ç¸ã§ãªã‘ã‚Œã°ãªã‚‰ãªã„

http://nesseiken.info/Chiba_lab/index.php?æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
- å†…ç¾¤ï¼ˆingroupï¼‰ã€€ä»Šã€ç³»çµ±æ¨å®šã®å¯¾è±¡ã¨ã—ã¦ã„ã‚‹ã‚°ãƒ«ãƒ¼ãƒ—ã®ã“ã¨ã€‚
- å¤–ç¾¤ (outgoup)ã€€å†…ç¾¤ã«å«ã¾ã‚Œãªã„åˆ†é¡ç¾¤ã¯ã™ã¹ã¦å¤–ç¾¤ï¼ˆoutgroupï¼‰ã«ãªã‚‹ã€‚å¤–ç¾¤ã¯é€šå¸¸ã€ç³»çµ±æ¨¹ã«æ ¹ã‚’ã¤ã‘ã‚‹ã¨ãã«ä½¿ã‚ã‚Œã€å†…ç¾¤ã®å§‰å¦¹ç¾¤ã‹ã‚‰è¤‡æ•°ã®ã‚‚ã®ã‚’ç”¨ã„ã‚‹ã“ã¨ãŒå¤šã„ã€‚

2023/05/24
https://www.iu.a.u-tokyo.ac.jp/textbook/chapter6.html
Webé€£æºãƒ†ã‚­ã‚¹ãƒˆ ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
ç¬¬6ç«  ç”Ÿç‰©é…åˆ—çµ±è¨ˆå­¦

page174
å¤–ç¾¤(outgroup)
ç³»çµ±é–¢ä¿‚ã‚’çŸ¥ã‚ŠãŸã„ç”Ÿç‰©(ã“ã‚Œã‚’å†…ç¾¤ã¨ã„ã„ã¾ã™)ã®ç³»çµ±çš„ä½ç½®é–¢ä¿‚ã‚’æ±ºå®šã™ã‚‹ã«å‚ç…§ã™ã‚‹ç¾¤ã®ã“ã¨ã§ã™ã€‚å†…ç¾¤ã®ä»–ã®ã‚°ãƒ«ãƒ¼ãƒ—ã«è¿‘ç¸ã§ã‚ã‚Šã€ä»–ã®ã©ã®å†…ç¾¤ã®ã‚°ãƒ«ãƒ¼ãƒ—ã‚ˆã‚Šã‚‚äº’ã„ã«é ç¸ã§ã‚ã‚‹ã¨è€ƒãˆã‚‰ã‚Œã‚‹ã‚‚ã®ã‚’é¸æŠã™ã‚‹ã®ãŒåŸºæœ¬ã®ã‚ˆã†ã§ã™ã€‚

æœ‰èƒç›¤é¡
å“ºä¹³é¡ã¨æœ‰è¢‹é¡ã®ä¸­ã§ã€ç‰¹ã«èƒç›¤ã‚’æœ‰ã™ã‚‹å‹•ç‰©ã®ç·ç§°ã¨ç†è§£ã™ã‚Œã°ã‚ˆã„ã§ã™ã€‚æœ‰è¢‹é¡ã®ã‚ªãƒãƒƒã‚µãƒ ã¯ã€èƒç›¤ã‚’æŒãŸãªã„(è‚²å…åš¢ã‚’ã‚‚ã¤)ã®ã§å¤–ç¾¤(outgroup)ã¨ã—ã¦å–ã‚Šæ‰±ãˆã¾ã™ã€‚

6.6 åˆ†å­ç³»çµ±æ¨¹

6.6.1 ç„¡æ ¹ç³»çµ±æ¨¹ã¨å¤–ç¾¤ã€ãƒˆãƒãƒ­ã‚¸ãƒ¼ã¨æã®é•·ã•

2023å¹´1æœˆ20æ—¥
https://note.com/fish_and_worms/n/n67cd44ebcebd
Outgroupã¯ä½•ã§ã‚‚ã‚ˆã„ã‚ã‘ã§ã¯ã‚ã‚Šã¾ã›ã‚“ã€‚â€å°‘ã—é•ã†â€ãã‚‰ã„ãŒã¡ã‚‡ã†ã©ã‚ˆã„ã§ã™ã€‚ä¸Šã®å›³ã¯ã€ã‚­ãƒ³ã‚®ãƒ§ã¨ãƒ•ãƒŠã‚’æ¯”è¼ƒã™ã‚‹ã®ã§ã€Outgroupã¨ã—ã¦åŒã˜ã‚³ã‚¤ç§‘ã®ã‚³ã‚¤ã‚’ä½¿ã†æ„Ÿã˜ã§ã™ã€‚

2012-10-31
https://togetter.com/li/399359?page=3
#166ws Outgroupã®é¸å®šã‚‚é‡è¦ã€‚ä¸€æœ¬ã ã‘ã¨ã„ã†ã®ã¯ã‚ã¾ã‚Šã‚ˆããªã„ã“ã¨ãŒå¤šã„ã€‚
#166ws ãƒ˜ã‚¿ã«é ã„å¤–ç¾¤ã‚’ãƒãƒ„ãƒ³ã¨å…¥ã‚Œã‚‹ã¨ã€Œé•·æèª˜å¼•ã€ã®å¼Šå®³ãŒå¿ƒé…ï¼

2011-07-13
https://togetter.com/li/161578?page=3
ã‚ã‚‹ç„¡æ ¹ç³»çµ±æ¨¹ã«ã€Œæ ¹ã€ã®æƒ…å ±ã‚’ä»˜ã‘åŠ ãˆã‚‹ã“ã¨ã§ã€è¤‡æ•°ã®æœ‰æ ¹ç³»çµ±æ¨¹ã‚’å°ãã“ã¨ãŒã§ãã¾ã™ã€‚ã—ãŸãŒã£ã¦ã€ç„¡æ ¹ç³»çµ±æ¨¹ã¯æœ‰æ ¹ç³»çµ±æ¨¹ã®ã€Œé›†åˆã€ã¨ã¿ãªã™ã“ã¨ãŒå¯èƒ½ã§ã™ã€‚é€†ã«ã„ãˆã°ã€ç„¡æ ¹ç³»çµ±æ¨¹ã¯ã€Œç¥–å…ˆã€ã®æ¦‚å¿µã‚’æŠœã‹ã‚ŒãŸç³»çµ±æ¨¹ã¨ã„ã†ã“ã¨ã‚‚ã§ãã¾ã™ã€‚ï¼ˆä¸‰ä¸­ 2006: 171ï¼‰
ç”Ÿç‰©ã®ç³»çµ±æ¨¹ã‚’æ¨å®šã™ã‚‹å ´åˆã«ã¯ã€å¯¾è±¡ã¨ãªã‚‹ç”Ÿç‰©ç¾¤ã®ä¸­ã«å°‘ãªãã¨ã‚‚ã²ã¨ã¤ã¯ã€Œé ç¸ã§ã‚ã‚‹ã¨ä»®å®šã•ã‚ŒãŸç”Ÿç‰©ã€ã‚’å«ã‚ã¦ãŠãã¾ã™ã€‚ã“ã®ç”Ÿç‰©ã‚’å¤–ç¾¤ã¨å‘¼ã³ã¾ã™ã€‚ãŸã¨ãˆã°ã€ãƒ’ãƒˆã‚’å«ã‚€éœŠé•·é¡ã®ç³»çµ±é–¢ä¿‚ã‚’æ¨å®šã™ã‚‹ã¨ãã«ã¯ã€éœŠé•·é¡ã«å«ã¾ã‚Œãªã„ã‚µãƒ«ãŒå¤–ç¾¤ã«æŒ‡å®šã•ã‚Œã‚‹ã§ã—ã‚‡ã†ã€‚ï¼ˆä¸‰ä¸­ 2006: 172ï¼‰
ã“ã®ã‚ˆã†ãªå¤–ç¾¤ã¯ã€å˜ã«ã€Œé ç¸ã§ã‚ã‚‹ã¨ä»®å®šã€ã•ã‚Œã¦ã„ã‚‹ã ã‘ã§ã™ãŒã€ãã®ä»®å®šã‚’ç½®ãã“ã¨ã§å¯¾è±¡ç”Ÿç‰©ç¾¤â€•â€•å¤–ç¾¤ã«å¯¾ã—ã¦ã€Œå†…ç¾¤ï¼ˆingroupï¼‰ã€ã¨å‘¼ã°ã‚Œã¾ã™â€•â€•ã®ç³»çµ±æ¨¹ã«æ ¹ã‚’ã¤ã‘ã‚‹ã“ã¨ãŒã§ãã¾ã™ã€‚ï¼ˆä¸‰ä¸­ 2006: 173ï¼‰

ï¼’ï¼ï¼ï¼™å¹´ï¼•æœˆï¼‘ï¼’æ—¥ï¼ˆç«ï¼‰
http://isw3.naist.jp/IS/Kawabata-lab/LECDOC_KINDAI/2009/phylo_09May12.pdf
ãƒ»æ ¹ã¯é©å½“ãªå¤–ç¾¤(out group)ã®é¸æŠã§æ±ºã‚ã‚‹ã€‚
å¤–ç¾¤ï¼šä»–ã®å…¨ã¦ã®OTUã¨ååˆ†é ã„ã¨è€ƒãˆã‚‰ã‚Œã‚‹OTU

ãƒ»ç„¡æ ¹ç³»çµ±æ¨¹ã‹ã‚‰æœ‰æ ¹ç³»çµ±æ¨¹ã¸ã®å¤‰æ›ï¼šOTUã®ä¸­ã‹ã‚‰é©åˆ‡ãªå¤–ç¾¤(out group)ã‚’é¸ã¹ã°ã‚ˆã„ã€‚
å¤–ç¾¤ã®é¸æŠåŸºæº–ï¼šï¼ˆï¼‘ï¼‰ä»–ã®å…¨ã¦ã®OTUã¨ç›¸åŒã€(2)ä»–ã®ã©ã®OTUã¨ã‚‚ååˆ†é ç¸


----------
## root
ç³»çµ±æ¨¹ã«æ ¹(root)ã‚’ã¤ã‘ã‚‹

2016-04-21
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7149615/
Rooting Trees, Methods for

(Brady et al., 2011, 

Brady S.G., Litman J.R., Danforth B.N. Rooting phylogenies using gene duplications: An empirical example from the bees (Apoidea) Molecular Phylogenetics and Evolution. 2011;60(3):295â€“304. [PubMed]
https://pubmed.ncbi.nlm.nih.gov/21600997/
outgroup versus paralogous rooting

2015-01-05
https://phylobotanist.blogspot.com/2015/01/how-to-root-phylogenetic-tree-outgroup.html
How to root a phylogenetic tree: outgroup, midpoint and other methods
- Outgroup rooting
- Midpoint rooting
- Ultrametric trees
- Asymmetric step-matrices
- Gene duplication events

2012-06-07
http://cabbagesofdoom.blogspot.com/2012/06/how-to-root-phylogenetic-tree.html
How to root a phylogenetic tree
Unrooted. 
Midpoint Rooting.
Outgroup Rooting.
Why does the root matter?

https://www.fifthdimension.jp/documents/molphytextbook/modelselection_lecture.pdf
åˆ†å­é€²åŒ–ã®çµ±è¨ˆãƒ¢ãƒ‡ãƒªãƒ³ã‚°ã¨ãƒ¢ãƒ‡ãƒ«é¸æŠ
æ™‚é–“åè»¢å¯èƒ½ãƒ¢ãƒ‡ãƒ«
æ™‚é–“åè»¢ä¸èƒ½ãƒ¢ãƒ‡ãƒ«

2021-05-18
IQ-TREE Non-Reversible Models æ™‚é–“åè»¢ä¸èƒ½ãƒ¢ãƒ‡ãƒ«ã‚’ç”¨ã„ã¦ã€å¤–ç¾¤ãªã—ã§æœ‰æ ¹ç³»çµ±æ¨¹ã‚’æ¨å®š
https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#iqtree_rootstrap

2021-05-01
root_digger
https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#root_digger

FastTreeã®ãƒ«ãƒ¼ãƒˆã®ä½ç½®ã¯ç”Ÿç‰©å­¦çš„ã«æ„å‘³ãŒãªã„ã€‚
https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#fasttree
http://www.microbesonline.org/fasttree/
Output formats: FastTree outputs trees in Newick format. The placement of the root is not biologically meaningful. 
http://www.microbesonline.org/fasttree/#FAQ
What does the rooting of the tree mean?
FastTree's placement of the root is arbitrary and is not biologically meaningful. To place the root accurately, you need outside information beyond what is in the alignment. If you do not have such information, another common approach is to use the midpoint, so that the maximum distance from the root to any leaf is minimized. This makes sense if the sequences are evolving according to a molecular clock. Most tree viewers can reroot a tree, but for huge trees I use the stat/reroot.pl script in MicrobesOnline's code base.

https://github.com/haruosuz/DS4GD/blob/master/2021/CaseStudy.md#mega
Barry G. Hall (2017) Phylogenetic Trees Made Easy: A How-To Manual (5th edition)
- p.79-80 Rooted and Unrooted Trees | For display purposes, MEGA has put a bend in one branch or another by the midpoint rooting method, but that does not accurately root an unrooted tree. (see Chapter 7 for a more detailed discussion of rooting trees.)
- p.112 Rooting a Tree | The rectangular format is obtained by selecting a branch of an unrooted tree and placing an interior nodeâ€”the rootâ€”within that branch. Some tree-draw-ing programs simply place the apparent root in the branch that leads to the first sequence listed in the alignment. MEGA improves upon that by placing the apparent root at the midpoint of the NJ tree, in the branch located midway between the two most distant sequences. If the rate of evolution is roughly con- stant along all branches, then this midpoint rooting will have placed the root correctly. More often, however, midpoint rooting places the root incorrectly, so it should not be trusted. So how can we find the root of the tree?

### 2023-05-29

https://github.com/deropi/PhyloRooting

https://pubmed.ncbi.nlm.nih.gov/37247390/
Genome Biol Evol
. 2023 Jun 1;15(6):evad096. doi: 10.1093/gbe/evad096.
Phylogenomic Testing of Root Hypotheses
Fernando D K Tria 1, Giddy Landan 1, Devani Romero Picazo 1, Tal Dagan 1
https://academic.oup.com/gbe/article/15/6/evad096/7185701
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10262964/
Data Availability
The source code to run the phylogenomic rooting as well as the unrooted trees with AD values used in this study are found in the following repository: https://github.com/deropi/PhyloRooting.git. Additionally, R code to replicate some of the figures in this paper is also provided.

5:55 AM Â· May 31, 2023
https://twitter.com/DaganLab/status/1663650390057930752
DaganLab on Twitter: "Phylogenomic Testing of Root Hypotheses. Extremely happy to have this work out. We reformulate root inference in the framework of statistical hypothesis testing and outline an analytical procedure to test competing root positions for a group of species. https://t.co/n0Z2WUIt34" / Twitter

### 2023-03-05

https://pubmed.ncbi.nlm.nih.gov/36992393/
Viruses
. 2023 Mar 5;15(3):684. doi: 10.3390/v15030684.
Rooting and Dating Large SARS-CoV-2 Trees by Modeling Evolutionary Rate as a Function of Time
Xuhua Xia 1 2
https://www.mdpi.com/1999-4915/15/3/684
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10057463/
software TRAD [22] freely available at http://dambe.bio.uottawa.ca/TRAD/TRAD.aspx (accessed on 7 May 2022).
http://dambe.bio.uottawa.ca/TRAD/TRAD.aspx

### 2022

### 2022-04

https://pubmed.ncbi.nlm.nih.gov/35143961/
Mol Phylogenet Evol
. 2022 Apr:169:107434. doi: 10.1016/j.ympev.2022.107434. Epub 2022 Feb 7.
The performance of outgroup-free rooting under evolutionary radiations
Alessandra P Lamarca 1, Beatriz Mello 1, Carlos G Schrago 2
https://www.sciencedirect.com/science/article/abs/pii/S1055790322000471
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_2628301396
Elsevier ScienceDirect Journals Complete
https://www-sciencedirect-com.kras.lib.keio.ac.jp/science/article/pii/S1055790322000471?via%3Dihub

In this work we analyzed the performance of two of these methods, the midpoint rooting (MPR) and the minimal ancestor deviation (MAD), in rooting topologies evolved under challenging scenarios of fast evolutionary radiations derived from empirical data, characterized by short internal branches near the crown node. Considering all branch length combinations investigated, both methods exhibited average success rates below 50%, although MAD slightly outperformed MPR. Moreover, tree balance significantly impacted the relative performance of the methods.
èª¿æŸ»ã—ãŸã™ã¹ã¦ã®æé•·ã®çµ„ã¿åˆã‚ã›ã‚’è€ƒæ…®ã™ã‚‹ã¨ã€ä¸¡æ‰‹æ³•ã¨ã‚‚å¹³å‡æˆåŠŸç‡ã¯50%æœªæº€ã€MADã¯MPRã‚’ã‚ãšã‹ã«ä¸Šå›ã£ãŸã€‚ç³»çµ±æ¨¹ã®ãƒãƒ©ãƒ³ã‚¹ã¯æ‰‹æ³•ã®ç›¸å¯¾çš„ãªæ€§èƒ½ã«å¤§ããå½±éŸ¿ã—ãŸã€‚
If predictive models of the outcomes of outgroup-free rooting methods could be inferred for any topology, they would be useful in situations when phylogenetically close outgroups are absent, as is the case in evolutionary analyses of fast evolving pathogens and bacteria (Pipes et al., 2021). 

### 2021-12-09

https://pubmed.ncbi.nlm.nih.gov/34436605/
Mol Biol Evol
. 2021 Dec 9;38(12):5514-5527. doi: 10.1093/molbev/msab254.
Phylogenetic Signal, Congruence, and Uncertainty across Bacteria and Archaea
Carolina A Martinez-Gutierrez 1, Frank O Aylward 1 2
https://academic.oup.com/mbe/article/38/12/5514/6358142?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8662615/
Our results indicate that the Patescibacteria (also called the CPR) are a derived group that is sister to the Chloroflexota, consistent with two recent studies (Coleman et al. 2021). This is in contrast to other studies that placed this group as either basal-branching or falling outside of the Terrabacteria (Hug et al. 2016; Parks et al. 2017; Castelle et al. 2018; MÃ©heust et al. 2019). 

### 2021-05-07

https://scholar.google.co.jp/scholar?cites=15482149728459064999&as_sdt=2005&sciodt=0,5&hl=ja

https://pubmed.ncbi.nlm.nih.gov/33958464/
Comment Science
. 2021 May 7;372(6542):574-575. doi: 10.1126/science.abh2814.
Illuminating the first bacteria
Laura A Katz 1
https://science.sciencemag.org/content/372/6542/574

https://pubmed.ncbi.nlm.nih.gov/33958449/
Science
. 2021 May 7;372(6542):eabe0511. doi: 10.1126/science.abe0511.
A rooted phylogeny resolves early bacterial evolution
Gareth A Coleman # 1, AdriÃ¡n A DavÃ­n # 2, Tara A Mahendrarajah 3, LÃ©nÃ¡rd L SzÃ¡nthÃ³ 4 5, Anja Spang 3 6, Philip Hugenholtz # 7, Gergely J SzÃ¶llÅ‘si # 8 5 9, Tom A Williams # 10
https://science.sciencemag.org/content/372/6542/eabe0511
Our analyses place the root between two major bacterial clades, the Gracilicutes and Terrabacteria. We found no support for a root between the Candidate Phyla Radiation (CPR), a lineage comprising putative symbionts and parasites with small genomes, and all other Bacteria. Instead, the CPR was inferred to be a member of the Terrabacteria and formed a sister lineage to the Chloroflexota and Dormibacterota. 

6:40 PM Â· May 7, 2021
https://twitter.com/BIRDlab_ENS/status/1390602531219312647
Alice Lebreton on Twitter: ""Illuminating the first bacteria": Perspective by Laura A. Katz in @ScienceMagazine on recent research that aim at finding out where the bacterial tree is rooted. https://t.co/iGcpUV1P24 https://t.co/PZztmTbRKU" / Twitter
https://twitter.com/BIRDlab_ENS/status/1390602535124209664
Alice Lebreton on Twitter: "The original research: "A rooted phylogeny resolves early bacterial evolution", by Coleman, DavÃ­n et al. in @ScienceMagazine. https://t.co/uz1IuHs3g1 https://t.co/ipg4dH9GCR" / Twitter

11:36 AM Â· May 7, 2021
https://twitter.com/takatoh_life/status/1390495662588923906
æœ¬ç ”ç©¶ã§ã¯ç‹¬è‡ªé–‹ç™ºã—ãŸæ‰‹æ³•ã«ã‚ˆã‚Šã€éºä¼å­ã®å‚ç›´ä¼æ¬ã ã‘ã§ãªãæ°´å¹³ä¼æ¬ã‚„éºä¼å­ã®é‡è¤‡ã‚„æ¶ˆå¤±ã‚’è€ƒæ…®ã—ã¦ã€ç´°èŒã®ç³»çµ±æ¨¹ã®æ ¹å…ƒã‚’æ¨å®šã—ãŸã€‚ãã®çµæœã€éºä¼å­ã®ä¼é”ã®2/3ã¯å‚ç›´æ–¹å‘ã§ã‚ã‚Šã€ãã®æ ¹å…ƒã¯ã‚°ãƒ©ãƒ é™°æ€§èŒGracilicutesã¨ã‚°ãƒ©ãƒ é™½æ€§èŒTerrabacteriaã®é–“ã«æ ¹å…ƒãŒã‚ã‚‹å¯èƒ½æ€§ãŒé«˜ã„ã¨æ¨å®šã—ãŸã€‚

### 2021-04-13

https://pubmed.ncbi.nlm.nih.gov/33295605/
Mol Biol Evol
. 2021 Apr 13;38(4):1537-1543. doi: 10.1093/molbev/msaa316.
Assessing Uncertainty in the Rooting of the SARS-CoV-2 Phylogeny
Lenore Pipes 1, Hongru Wang 1, John P Huelsenbeck 1, Rasmus Nielsen 1 2 3
https://academic.oup.com/mbe/article/38/4/1537/6028993
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7798932/

### 2020-08-06

7:09 PM Â· Aug 7, 2020
https://twitter.com/AlexisCompBio/status/1291677772570603522
Alexis Stamatakis on Twitter: "In our latest preprint https://t.co/kg6LYXJmaF we find that phylogenetic analyses of SARS-CoV-2 data are extremely challenging due to weak signal and that rooting the tree is difficult. We provide some recommendations on analyzing SARS-CoV-2 data." / Twitter

https://www.biorxiv.org/content/10.1101/2020.08.05.239046v1
Phylogenetic analysis of SARS-CoV-2 data is difficult | bioRxiv
We further find that rooting the inferred phylogeny with some degree of confidence either via the bat and pangolin outgroups or by applying novel computational methods on the ingroup phylogeny does not appear to be possible.

### 2020-05-15

https://pubmed.ncbi.nlm.nih.gov/32413061/
PLoS One
. 2020 May 15;15(5):e0232950. doi: 10.1371/journal.pone.0232950. eCollection 2020.
Assessing the accuracy of phylogenetic rooting methods on prokaryotic gene families
Taylor Wade 1, L Thiberio Rangel 2, Soumya Kundu 1, Gregory P Fournier 2, Mukul S Bansal 1 3
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0232950
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7228096/
Among other findings, this study identifies parsimonious Duplication-Transfer-Loss (DTL) rooting and Minimal Ancestor Deviation (MAD) rooting as two of the most accurate gene tree rooting methods for prokaryotes and specifies the evolutionary conditions under which these methods are most accurate, demonstrates that DTL rooting is highly sensitive to high evolutionary rates and gene tree error, and that rooting methods based on branch-lengths are generally robust to gene tree reconstruction error.

2:27 PM Â· May 23, 2020
https://twitter.com/MicrobiomDigest/status/1264065321817653248
Elisabeth Bik on Twitter: "Assessing the accuracy of phylogenetic rooting methods on prokaryotic gene families - Taylor Wade et al. * 3098 gene trees from 504 bacterial species * parsimonious Duplication-Transfer-Loss rooting, Minimal Ancestor Deviation rooting most accurate https://t.co/1UKaubCOJS https://t.co/zMma2416gh" / Twitter

### 2019-03-01
https://pubmed.ncbi.nlm.nih.gov/30753429/
Genome Biol Evol
. 2019 Mar 1;11(3):883-898. doi: 10.1093/gbe/evz034.
Investigating the Origins of Membrane Phospholipid Biosynthesis Genes Using Outgroup-Free Rooting
Gareth A Coleman 1, Richard D Pancost 2, Tom A Williams 1
https://academic.oup.com/gbe/advance-article/doi/10.1093/gbe/evz034/5310093
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6431249/

Materials and Methods

Phylogenetics

The trees were rooted with an uncorrelated lognormal relaxed molecular clock (RMC), using the LG model with a discretized Gamma distribution (Yang 1994) with four rate categories, and a Yule tree prior (Stadler 2009; Hartmann et al. 2010) in BEAST (Drummond and Rambaut 2007; Drummond et al. 2012). We also rooted the trees using minimal ancestor deviation (MAD) rooting (Tria et al 2017). 

Results and Discussion

Table 1

We used two complementary approaches to root these single-gene trees: a RMC in BEAST (Drummond and Rambaut 2007; Drummond et al. 2012), and the recently described MAD rooting method of Tria et al. (2017). 

### 2017-06-19
Tria

https://pubmed.ncbi.nlm.nih.gov/29388565/
Nat Ecol Evol
. 2017 Jun 19;1:193. doi: 10.1038/s41559-017-0193.
Phylogenetic rooting using minimal ancestor deviation
Fernando Domingues KÃ¼mmel Tria 1, Giddy Landan 1, Tal Dagan 1
https://www.nature.com/articles/s41559-017-0193
Access options
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/1rcqpmf/cdi_proquest_miscellaneous_1993387992
Tria, Fernando Domingues KÃ¼mmel ; Landan, Giddy ; Dagan, Tal

6:00 AM Â· Mar 23, 2018
https://twitter.com/kfuku0502/status/976926638012211200
Kenji Fukushima (ç¦å³¶ å¥å…) on Twitter: 
minimal ancestor deviationæ³•ã§ã®ãƒ«ãƒ¼ãƒ†ã‚£ãƒ³ã‚°ã€ã„ãã¤ã‹ã®éºä¼å­ç³»çµ±æ¨¹ã§è©¦ã—ã¦ã¿ãŸã‘ã©ã‚ã¡ã‚ƒãã¡ã‚ƒã†ã¾ãã„ãã€‚ï¼’ï¼•ï¼‘æä¸­ï¼‘æã®ã¿ã€ãã“ã§ãƒ«ãƒ¼ãƒ†ã‚£ãƒ³ã‚°ã™ã‚‹ã¨éºä¼å­é‡è¤‡ã‚¤ãƒ™ãƒ³ãƒˆãŒæœ€å°ã¨ãªã‚‹ã‚ˆã†ãªæ¨¹å½¢ã§ã‚‚çš„ç¢ºã«ãã®ï¼‘æã‚’è¦‹ã¤ã‘ã¦ã„ã‚‹ã€‚
é‡è¤‡ã‚¤ãƒ™ãƒ³ãƒˆã®æœ€å°åŒ–ã¨çµ„ã¿åˆã›ãŸã‚‰ç²¾åº¦å¢—ã™ã‚“ã˜ã‚ƒãªã„ã‹ã¨æ€ã£ã¦ãŸã‘ã©ã€MADã ã‘ã§ååˆ†ã‹ã‚‚ã€‚
ä»Šè©¦ã—ã¦ã¿ãŸã‚‰ã“ã®ç³»çµ±æ¨¹ã¯midpoint rootingã§ã‚‚ãã¡ã‚“ã¨éºä¼å­é‡è¤‡æ•°æœ€å°ã®æã§ãƒ«ãƒ¼ãƒ†ã‚£ãƒ³ã‚°ã•ã‚Œã‚‹â€¦ã€‚
ã†ãƒ¼ã‚“ã€ã‚„ã£ã±ã‚Šã„ã‚ã‚“ãªãƒ‡ãƒ¼ã‚¿è¦‹ã¦ã‚‹ã¨æœ€çµ‚çš„ã«ã¯reconciliation-basedã®ãƒ«ãƒ¼ãƒ†ã‚£ãƒ³ã‚°ã«è½ã¡ç€ããã†ãªããŒã™ã‚‹â€¦ã€‚
reconciliation-basedã®æ–¹æ³•ã¯root nodeä»˜è¿‘ãŒduplication nodesã°ã‹ã‚Šã ã¨çˆ†æ­»ã™ã‚‹ã®ã§ã€ãã†ã„ã†å ´åˆã§ãªãŠã‹ã¤reconciliation scoreãŒæœ€å°ã®æã®ä¸­ã«MADã®æãŒã‚ã‚Œã°ãã‚Œã‚’æ¡ç”¨ã€ã¨ã„ã†æ–¹å‘ã§è¡Œã“ã†ã€‚

8:03 AM - 20 Jul 2018
https://twitter.com/NatureEcoEvo/status/1020277970299162624
Phylogenetic rooting using minimal ancestor deviation https://www.nature.com/articles/s41559-017-0193 â€¦ #July2017highlight #ICYMI

### 2009-08-12
https://pubmed.ncbi.nlm.nih.gov/19571238/
Review Philos Trans R Soc Lond B Biol Sci
. 2009 Aug 12;364(1527):2177-85. doi: 10.1098/rstb.2009.0035.
Genome beginnings: rooting the tree of life
James A Lake 1, Ryan G Skophammer, Craig W Herbold, Jacqueline A Servin
https://royalsocietypublishing.org/doi/10.1098/rstb.2009.0035
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC2873003/


### 2015
2015
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/188bto4/alma99275963904031
Welcome to the Microbiome : Getting to Know the Trillions of Bacteria and Other Microbes In, On, and Around You / Rob DeSalle, Susan L. Perkins.
DeSalle, Rob, author. Perkins, Susan L., author. Wynne, Patricia J.
What Is Life? 23
If traces of a single gene family can be found in organisms of all three major groups of cells (Archaea, Eukarya, and Bacteria), then one can assume that the gene duplication event occurred in the common ancestor of all life on the planet, and a gene from one can be used to root genes in the others. The rooting of one gene family with a closely related gene family is called â€œparalog rootingâ€
ãƒ‘ãƒ©ãƒ­ã‚°ãƒ»ãƒ«ãƒ¼ãƒ†ã‚£ãƒ³ã‚°

### MPR
midpoint rooting (MPR)

ä¾‹:

2020
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7738440/ A Novel Family of Acinetobacter Mega-Plasmids Are Disseminating Multi-Drug Resistance Across the Globe While Acquiring Location-Specific Accessory Genes
The resulting tree was annotated with FigTree v.1.4.3 (Rambaut, 2017) with midpoint root.

2019 Jan 8;
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6325168/ Pangenomic Approach To Understanding Microbial Adaptations within a Model Built Environment, the International Space Station, Relative to Human Hosts and Soil
FIG 3
Phylogenetic tree constructed from core gene codon alignment with midpoint rooting.
The Newick trees were processed with Phangorn v2.4.0 (81) for midpoint rooting and plotted with Ape v5.1 (82).

2006 May
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1458961/ Genome Sequence of Rickettsia bellii Illuminates the Role of Amoebae in Gene Exchanges between Intracellular Pathogens
Figure 5
The tree was built using a maximum likelihood method with JTT substitution model and midpoint rooting based on the concatenated sequence alignment of TraDF and TraGF.

### 2007-12
https://pubmed.ncbi.nlm.nih.gov/32287391/
Biol J Linn Soc Lond
. 2007 Dec;92(4):669-674. doi: 10.1111/j.1095-8312.2007.00864.x. Epub 2007 Dec 7.
An empirical test of the midpoint rooting method
Pablo N Hess 1, Claudia A DE Moraes Russo 1
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7110036/
Interestingly, the more consistent the outgroup root is, the more successful MPR appears to be. This is a strong indication that the MPR method is valuable, particularly for cases where a proper outgroup is unavailable.

### root_jp

2023 å¹´ 12 æœˆ 7 æ—¥
äº•ä¸Š æ½¤
https://fish-evol.org/R_JI.html
ape: BS å€¤ä»˜ã tree ã® reroot ã¯ãŠã‹ã—ã„?
reroot ã™ã‚‹ã¨ï¼ŒBS å€¤ã®ä½ç½®ãŒãŠã‹ã—ããªã‚‹ã¨èãã¾ã—ãŸï¼ä»¥ä¸‹ã¯ä¾‹é¡Œã§ã™ï¼å·¦ã® tree ã‚’ d ã¨ã„ã† leaf ã§ rooting ã—ã¦ã„ã¾ã™ï¼ã—ã‹ã—ï¼Œç§ã«ã¯å•é¡Œãªã„ã‚ˆã†ã«æ€ãˆã¾ã™ï¼

2012
http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
- æ ¹ï¼ˆãƒ«ãƒ¼ãƒˆã€rootï¼‰ã€€ä»–ã®å…¨ã¦ã®ç¯€ã«ç¶šãç‰¹å®šã®ç¯€ã®ã“ã¨ã‚’æ ¹ï¼ˆãƒ«ãƒ¼ãƒˆï¼‰ã¨è¨€ã†ã€‚æ ¹ã‚’æŒã¤ç³»çµ±æ¨¹ã‚’æœ‰æ ¹ç³»çµ±æ¨¹(rooted tree), æ ¹ã‚’æŒãŸãªã„ç³»çµ±æ¨¹ã‚’ç„¡æ ¹ç³»çµ±æ¨¹(unrooted tree)ã¨å‘¼ã¶ã€‚ç„¡æ ¹ç³»çµ±æ¨¹ã¯åˆ†é¡ç¾¤ç›¸äº’ã®é–¢é€£ã®ã¿ã‚’ç¤ºã—ã¦ãŠã‚Šã€é€²åŒ–çš„ãªé–¢ä¿‚ï¼ˆæ™‚é–“ã®çµŒéã‚’ä¼´ã†é¡ç¸é–¢ä¿‚ï¼‰ã‚’ç¤ºã—ã¦ã„ãªã„ã€‚ã¤ã¾ã‚Šã€å³å¯†ã«è¨€ã†ãªã‚‰ã°ã€ç„¡æ ¹ç³»çµ±æ¨¹ã¯ç³»çµ±æ¨¹ã¨è¦‹ãªã›ãªã„ã¨ã„ã†è€ƒãˆæ–¹ã‚‚ã§ãã‚‹ã€‚ã€€4ã¤ã®æœ«ç«¯ç¯€ã‹ã‚‰ãªã‚‹ç„¡æ ¹ç³»çµ±æ¨¹ã®ã‚¤ãƒ³ã‚¿ãƒ¼ãƒŠãƒ«ãƒ–ãƒ©ãƒ³ãƒã‚’ã‚»ãƒ³ãƒˆãƒ©ãƒ«ãƒ»ãƒ–ãƒ©ãƒ³ãƒï¼ˆcentral branchï¼‰ã¨å‘¼ã¶ã€‚
- å¤–ç¾¤ (outgoup)ã€€å†…ç¾¤ã«å«ã¾ã‚Œãªã„åˆ†é¡ç¾¤ã¯ã™ã¹ã¦å¤–ç¾¤ï¼ˆoutgroupï¼‰ã«ãªã‚‹ã€‚å¤–ç¾¤ã¯é€šå¸¸ã€ç³»çµ±æ¨¹ã«æ ¹ã‚’ã¤ã‘ã‚‹ã¨ãã«ä½¿ã‚ã‚Œã€å†…ç¾¤ã®å§‰å¦¹ç¾¤ã‹ã‚‰è¤‡æ•°ã®ã‚‚ã®ã‚’ç”¨ã„ã‚‹ã“ã¨ãŒå¤šã„ã€‚

2009
https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/
åˆ†å­ç³»çµ±è§£æï¼ˆè¬›ç¾©ï¼“ï¼‰
è¬›ç¾©è³‡æ–™3
https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/h21_4_3b.pdf
ç„¡æ ¹ç³»çµ±æ¨¹ã«æ ¹ã‚’ã¤ã‘ã‚‹æ–¹æ³•
æ ¹ã‚’ã¤ã‘ã‚‹æ–¹æ³•ã¯ï¼’ã¤
ï¼‘ï¼æœ€ã‚‚é ã„é–¢ä¿‚ã«ã‚ã‚‹ã¨çŸ¥ã‚‰ã‚Œã¦ã„ã‚‹ç”Ÿç‰©ç¨®ã®é…åˆ—ï¼ˆå¤–ç¾¤ã€
outgroupï¼‰ã‚’ï¼‘ã¤ä»¥ä¸Šå«ã‚ã‚‹
ï¼’ï¼æœ€ã‚‚é ã„é–¢ä¿‚ã«ã‚ã‚‹ï¼’ã¤ã®é…åˆ—ã‚’çµã¶æã®ä¸­ç‚¹ã‚’internal
nodeã¨ã™ã‚‹


2018
http://leeswijzer.org/files/activity.html
â—æ—¥æœ¬é€²åŒ–å­¦ä¼šç¬¬20å›å¤§ä¼šãƒ»ã‚·ãƒ³ãƒã‚¸ã‚¦ãƒ ãƒ»S5ã€ˆBasal lineageã¯ã€ŒåŸå§‹çš„ã€ã‹ï¼Ÿï¼šç”Ÿç‰©ç•Œã¨åˆ†é‡ã‚’è¶…ãˆã¦ã€‰2018å¹´8æœˆ22æ—¥ï¼ˆæ°´ï¼‰15:50ã€œ17:50ï¼ æ±äº¬å¤§å­¦é§’å ´ã‚­ãƒ£ãƒ³ãƒ‘ã‚¹5å·é¤¨ï¼ˆRoom 5ï¼‰ï¼šä¸‰ä¸­ä¿¡å®ã€ŒBasal Lineage ã®å¹»æƒ³ã¨å®Ÿæ…‹ï¼šç³»çµ±æ¨¹ã®æœ‰æ ¹åŒ–ã¨å½¢è³ªçŠ¶æ…‹å¾©å…ƒã®è¦³ç‚¹ã‹ã‚‰ã€â†’ Slideshare
https://www.slideshare.net/slideshow/minaka-sesj2018-slideshare/111052581
19. é–“æ¥æ³•ï¼ˆindirect methodï¼‰ï¼å¤–ç¾¤æ¯”è¼ƒæ³• å¯¾è±¡ã¨ãªã‚‹å†…ç¾¤ï¼ˆingroupï¼‰ã«å¯¾ã—ã¦ â€œè¿‘ç¸â€ ã¨ä»®å®šã• ã‚Œã‚‹å¤–ç¾¤ï¼ˆoutgroupï¼‰ã‚’ä»®å®šã—ï¼Œå¤–ç¾¤ã‚’åŸºæº–ã¨ã—ã¦å†… ç¾¤ã®å½¢è³ªçŠ¶æ…‹ã®åŸå§‹æ€§ï¼æ´¾ç”Ÿæ€§ã®æ–¹å‘æ€§ã‚’æ¨å®šã—ï¼Œå†… ç¾¤ã®æœ€ç¯€ç´„åˆ†å²å›³ï¼ˆå¤–ç¾¤æœ‰æ ¹åŒ–ï¼‰ã‚’æ¨å®šã™ã‚‹ï¼ â†“ å†…ç¾¤ï¼‹å¤–ç¾¤ã®å…¨ä½“ã«ã‚ãŸã‚‹å¤§åŸŸçš„æœ€ç¯€ç´„åˆ†æï¼ˆMPï¼‰ã‚’ è¡Œã†ã¨åŒæ™‚ã«ï¼Œç„¡æ ¹æœ€ç¯€ç´„åˆ†å²å›³ã®æ¨¹å½¢æ¨å®šã¨åŒæ™‚ã« åŸå§‹æ€§ï¼æ´¾ç”Ÿæ€§ã®æ–¹å‘æ€§ãŒæ±ºã¾ã‚Šï¼Œå¤–ç¾¤æœ‰æ ¹åŒ–ã•ã‚Œã‚‹ï¼
20. ç›´æ¥æ³•ï¼ˆdirect methodï¼‰ï¼å€‹ä½“ç™ºç”Ÿæ³• å¤–ç¾¤ã®ä»®å®šã‚’ç½®ã‹ãšã«ï¼Œå€‹ä½“ç™ºç”Ÿå½¢è³ªï¼ˆç™ºç”Ÿæ®µéšé·ç§» ã®æƒ…å ±ï¼‰ã‚’ç”¨ã„ã‚Œã°ï¼Œã‚ˆã‚Šç›´æ¥çš„ãªå½¢è³ªçŠ¶æ…‹ã®æ–¹å‘æ€§ï¼ˆåŸ å§‹æ€§ï¼æ´¾ç”Ÿæ€§ï¼‰ã®æ¨å®šã¨åˆ†å²å›³ã®æœ‰æ ¹åŒ–ãŒå¯èƒ½ã ã‚ã† ã¨ã„ã†ä¸»å¼µãŒ 1970 å¹´ä»£ã«åˆ†å²å­¦æ´¾ã®ä¸­ã§æ´»ç™ºã«è­°è«– ã•ã‚ŒãŸï¼ãã®å¾Œï¼Œåå¹´ä»¥ä¸Šã‚‚è«–äº‰ãŒç¶šã„ãŸãŒï¼Œæ–¹å‘æ€§ æ¨å®šã®æ–¹æ³•ã¨ã—ã¦ã¯ï¼Œæœ€çµ‚çš„ã«å¤–ç¾¤æ¯”è¼ƒæ³•ã«æŠ¼ã•ã‚Œã¦ å½±ãŒè–„ããªã£ã¦ã—ã¾ã£ãŸã‚ˆã†ã ï¼

20060715
https://leeswijzer.hatenadiary.com/entry/20060715/1152931775
ã€ç³»çµ±æ¨¹æ€è€ƒã®ä¸–ç•Œï¼šã™ã¹ã¦ã¯ãƒ„ãƒªãƒ¼ã¨ã¨ã‚‚ã«ã€ - leeswijzer: een nieuwe leeszaal van dagboek
ã€€ç¬¬ï¼’ç¯€ï¼šã‚°ãƒ©ãƒ•ã¨ã—ã¦ã®ç³»çµ±æ¨¹â€•â€•ç‚¹ãƒ»è¾ºãƒ»æ ¹ 168
ã€€ã€€ç„¡æ ¹ç³»çµ±æ¨¹ã¨æœ‰æ ¹ç³»çµ±æ¨¹
ã€€ã€€ç¥–å…ˆå­å­«é–¢ä¿‚ã¯åŸç†çš„ã«ä¸å¯çŸ¥ã§ã‚ã‚‹

2010å¹´07æœˆ02æ—¥
http://bbs.jinruisi.net/blog/2010/07/829.html
â– ç„¡æ ¹ç³»çµ±æ¨¹ã¨æœ‰æ ¹ç³»çµ±æ¨¹

![](http://bbs.jinruisi.net/blog/wp-content/uploads/0-2ã®ã‚³ãƒ”ãƒ¼.jpg)

----------
## clade

https://en.wikipedia.org/wiki/Clade
In biological phylogenetics, a clade
 is a grouping of organisms that are monophyletic â€“ that is, composed of a common ancestor and all its lineal descendants â€“ on a phylogenetic tree.

https://en.wikipedia.org/wiki/Monophyly

https://en.wikipedia.org/wiki/Paraphyly

https://ja.wikipedia.org/wiki/ç³»çµ±ç¾¤
Cladeï¼‰ã¨ã¯ã€å…±é€šã®ç¥–å…ˆã‹ã‚‰é€²åŒ–ã—ãŸç”Ÿç‰©ç¾¤ã®ã“ã¨ã€‚å´ç³»çµ±ç¾¤ã€å˜ç³»çµ±ç¾¤ã€å¤šç³»çµ±ç¾¤ãªã©ãŒã‚ã‚‹ã€‚

https://ja.wikipedia.org/wiki/å˜ç³»çµ±ç¾¤
ã¨ã¯ã€ç”Ÿç‰©ã®åˆ†é¡ç¾¤ã®ã†ã¡ã€å˜ä¸€ã®é€²åŒ–çš„ç³»çµ±ã‹ã‚‰ãªã‚Šã€ã—ã‹ã‚‚ãã®ç³»çµ±ã«å±ã™ã‚‹ç”Ÿç‰©ã™ã¹ã¦ã‚’å«ã‚€ã‚‚ã®ã‚’ã„ã†ã€‚

https://ja.wikipedia.org/wiki/åˆ†å²å­¦
åˆ†é¡å­¦ã«ãŠã‘ã‚‹åˆ†é¡ç¾¤ï¼ˆã‚¿ã‚¯ã‚½ãƒ³ï¼‰ã«ã¯ã€å˜ä¸€ã®ç³»çµ±ã‹ã‚‰ãªã‚‹ã€Œå˜ç³»çµ±ç¾¤ã€ï¼ˆä¾‹ãˆã°é³¥é¡ï¼‰ã¨ã€å¤§ããªå˜ç³»çµ±ç¾¤ã‹ã‚‰ä¸€éƒ¨ã®å˜ç³»çµ±ç¾¤ã‚’é™¤ã„ã¦ã¾ã¨ã‚ãŸã€Œå´ç³»çµ±ç¾¤ã€ï¼ˆé³¥é¡ã‚’é™¤ã„ãŸçˆ¬è™«é¡ãªã©ï¼‰ãŒã‚ã‚‹ãŒã€åˆ†å²å­¦ã®ç«‹å ´ã§ã¯å´ç³»çµ±ç¾¤ã¯åˆ†é¡ç¾¤ã¨ã—ã¦èªã‚ã‚‹ã¹ãã§ã¯ãªãã€å˜ç³»çµ±ç¾¤ï¼ˆåˆ†å²å­¦ã§ã¯ã‚¯ãƒ¬ãƒ¼ãƒ‰Cladeã¨ã„ã†ï¼‰ã®ã¿ã‚’èªã‚ã‚‹ã¹ãã ã¨ã„ã†ã“ã¨ã«ãªã‚‹ï¼ˆé€²åŒ–åˆ†é¡å­¦ã¨å‘¼ã°ã‚Œã‚‹è€ƒãˆæ–¹ã§ã¯å´ç³»çµ±ç¾¤ã‚‚èªã‚ã‚‹ï¼‰ã€‚


ã€ã‚«ãƒ©ãƒ¼å›³è§£ã€€é€²åŒ–ã®æ•™ç§‘æ›¸ã€€ç¬¬ï¼“å·»ã€€ç³»çµ±æ¨¹ã‚„ç”Ÿæ…‹ã‹ã‚‰è¦‹ãŸé€²åŒ–ã€
p.22
å›³9-11ã€€ã‚¯ãƒ¬ãƒ¼ãƒ‰ï¼ˆå˜ç³»çµ±ç¾¤ï¼‰ã¯ã€å¤§ããªç³»çµ±æ¨¹ã‹ã‚‰ãƒã‚µãƒŸã‚’ï¼‘å›ã ã‘ä½¿ã£ã¦åˆ‡ã‚Šå–ã‚‹ã“ã¨ãŒã§ãã‚‹ã€‚ã‚‚ã—ãƒã‚µãƒŸã‚’ï¼’å›ä½¿ã†å¿…è¦ãŒã‚ã‚‹ãªã‚‰ã€ãã‚Œã¯ã‚¯ãƒ¬ãƒ¼ãƒ‰ã§ã¯ãªã„ã€‚ã‚¯ãƒ¬ãƒ¼ãƒ‰ã¯ã€ã‚ã‚‹ç”Ÿç‰©ã¨ãã®ã™ã¹ã¦ã®å­å­«ã‹ã‚‰ãªã‚‹ã€‚


https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§
ç”°è¾ºæ™¶å²
2015/10/20
6.1 ã‚¯ãƒ¬ãƒ¼ãƒ‰ãƒ»å˜ç³»çµ±ãƒ»å´ç³»çµ±ãƒ»å¤šç³»çµ±ãƒ»ç¥–å…ˆçš„ãƒ»æ´¾ç”Ÿçš„
ã¾ãšã€ã‚¯ãƒ¬ãƒ¼ãƒ‰(clade)ã«ã¤ã„ã¦ã§ã™ã€‚ã‚¯ãƒ¬ãƒ¼ãƒ‰ã¨ã¯ã€ç³»çµ±æ¨¹ä¸Šã§è¤‡æ•°ã®OTUãŒæ‰€å±ã™ã‚‹éƒ¨åˆ†ç³»çµ±æ¨¹ã®ã“ã¨ã§ã™ã€‚ãŸã ã—ã€æœ‰æ ¹ç³»çµ±æ¨¹ã¨ç„¡æ ¹ç³»çµ±æ¨¹ã§ã¯ã‚„ã‚„æ„å‘³ãŒç•°ãªã‚Šã¾ã™ã€‚ç„¡æ ¹ç³»çµ±æ¨¹ã§ã¯ã€ã‚ã‚‹å†…åˆ†æ(internal/interior branch)ã®ä¸€æ–¹ã®ç«¯ç‚¹ã«æ¥ç¶šã•ã‚Œã¦ã„ã‚‹éƒ¨åˆ†ç³»çµ±æ¨¹ã‚’ã‚¯ãƒ¬ãƒ¼ãƒ‰ã¨è¨€ã„ã¾ã™ãŒã€æœ‰æ ¹ç³»çµ±æ¨¹ã§ã¯å†…åˆ†æã®æ ¹ã‹ã‚‰é ã„å´ã®ç«¯ç‚¹ã«æ¥ç¶šã•ã‚Œã¦ã„ã‚‹éƒ¨åˆ†ç³»çµ±æ¨¹ã‚’æŒ‡ã—ã¾ã™ã€‚ã¤ã¾ã‚Šã€æœ‰æ ¹ç³»çµ±æ¨¹ä¸Šã®ã‚¯ãƒ¬ãƒ¼ãƒ‰ãŒæ ¹ç‚¹ã‚’å«ã‚€ã“ã¨ã¯ã‚ã‚Šã¾ã›ã‚“ã€‚


http://www2.tba.t-com.ne.jp/nakada/takashi/phylogeny/monophyly.html
å˜ç³»çµ±ï¼Œå´ç³»çµ±ï¼Œå¤šç³»çµ±
ä½œæˆï¼šä»²ç”°å´‡å¿—
æ›´æ–°ï¼š2006å¹´10æœˆ13æ—¥
ã€Œå˜ç³»çµ±ã€ï¼ˆmonophylyï¼‰ã‚ã‚‹ã„ã¯ã€Œå˜ç³»çµ±ç¾¤ã€ï¼ˆmonophyletic groupï¼‰
ã€Œåˆ†å²ç¾¤ã€ ï¼ˆcladeï¼›ã€Œã‚¯ãƒ¬ãƒ¼ãƒ‰ã€ã¨å­—è¨³ã•ã‚Œã‚‹ã“ã¨ã‚‚å¤šã„ï¼‰

http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
å˜ç³»çµ±ç¾¤ (monophyletic group), ã‚¯ãƒ¬ãƒ¼ãƒ‰ (clade)ã€€ï¼‘ã¤ã®å…±é€šç¥–å…ˆã¨ã€ãã‚Œã‹ã‚‰æ´¾ç”Ÿã—ãŸåˆ†é¡ç¾¤å…¨ã¦ã‚’å«ã‚€ã‚°ãƒ«ãƒ¼ãƒ—ã®ã“ã¨ã€‚åˆ†å­ç³»çµ±å­¦ã§ã¯ã€ã‚¯ãƒ¬ãƒ¼ãƒ‰ã¨ã„ã†ç”¨èªã‚’ã€ï¼‘ã¤ã®å…±é€šç¥–å…ˆã‹ã‚‰æ´¾ç”Ÿã—ãŸåˆ†é¡ç¾¤ã‹ã‚‰ãªã‚‹ã‚°ãƒ«ãƒ¼ãƒ—ã§ã€ä»–ã®ã‚°ãƒ«ãƒ¼ãƒ—ã®ã‚‚ã®ã¨ã¯ãã®ç¥–å…ˆã‚’å…±æœ‰ã—ãªã„ã‚‚ã®ã«ä½¿ã†ã€‚ç³»çµ±æ¨¹ã‚’è¦‹ã¦ã€åˆ†é¡ç¾¤åŒå£«ã®é–¢ä¿‚ã‚’è­°è«–ã™ã‚‹ã®ã«ã€æœ€ã‚‚æ™®é€šã«ä½¿ã‚ã‚Œã‚‹ç”¨èªã€‚


----------
## constraint
åˆ¶ç´„

http://www.iqtree.org/doc/Advanced-Tutorial#constrained-tree-search
Constrained tree search
IQ-TREE supports constrained tree search via -g option, so that the resulting tree must obey a constraint tree topology. The constraint tree can be multifurcating and need not to contain all species. 

NOTE: While this option helps to enforce the tree based on prior knowledge, it is advised to always perform tree topology tests to make sure that the resulting constrained tree is NOT significantly worse than an unconstrained tree! See tree topology tests and testing constrained tree below for a guide how to check this.

http://www.iqtree.org/doc/Command-Reference#tree-search-parameters

-g	Specify a topological constraint tree file in NEWICK format. The constraint tree can be a multifurcating tree and need not to include all taxa.

Example usages:

Infer an ML tree for an alignment data.phy obeying a topological constraint tree constraint.tree:


https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
RAxMLã§æ¨¹å½¢åˆ¶ç´„(topological constraint)ã‚’èª²ã—ãŸç³»çµ±æ¨¹æ¨å®šã‚’è¡Œã†ã«ã¯ã€ã¾ãšåˆ¶ç´„ã¨ãªã‚‹ç³»çµ±æ¨¹ã‚’ä½œæˆã™ã‚‹å¿…è¦ãŒã‚ã‚Šã¾ã™ã€‚ä¾‹ãˆã°ã€TaxonAï½TaxonEã®5 OTUã®ãƒ‡ãƒ¼ã‚¿ã§TaxonAã¨TaxonBã®å˜ç³»çµ±æ€§(monophyly)ã‚’åˆ¶ç´„ã¨ã—ã¦èª²ã™å ´åˆã€ä»¥ä¸‹ã®ã‚ˆã†ãªç³»çµ±æ¨¹ãƒ•ã‚¡ã‚¤ãƒ«ã‚’ç”¨æ„ã—ã¾ã™ã€‚

ã“ã®ã‚ˆã†ã«ã€ã€Œç‰¹å®šã®ç³»çµ±ä»®èª¬ã‚’æº€ãŸã™ã€æ¨¹å½¢åˆ¶ç´„ã‚’æ­£ã®åˆ¶ç´„(positive constraint)ã¨è¨€ã„ã¾ã™ã€‚æ­£ã®åˆ¶ç´„ä¸‹ã®ç³»çµ±æ¨¹æ¨å®šã§ã¯ã€ãã®ç³»çµ±ä»®èª¬ã¨äº’æ›æ€§ã®ã‚ã‚‹ç³»çµ±æ¨¹ã®ä¸­ã§ãƒ™ã‚¹ãƒˆãªç³»çµ±æ¨¹ã‚’æ¢ç´¢ã™ã‚‹ã“ã¨ã«ãªã‚Šã¾ã™ã€‚ã€Œç‰¹å®šã®ç³»çµ±ä»®èª¬ã‚’æº€ãŸã•ãªã„ã€ã¨ã„ã†åˆ¶ç´„ã‚‚ã‚ã‚Šã€ã“ã‚Œã‚’è² ã®åˆ¶ç´„(negative constraint)ã¨å‘¼ã³ã¾ã™ã€‚è² ã®åˆ¶ç´„ä¸‹ã®ç³»çµ±æ¨¹æ¨å®šã§ã¯ã€ãã®ç³»çµ±ä»®èª¬ã¨äº’æ›æ€§ã®ç„¡ã„ç³»çµ±æ¨¹ã®ä¸­ã§ãƒ™ã‚¹ãƒˆãªç³»çµ±æ¨¹ã‚’æ¢ç´¢ã™ã‚‹ã“ã¨ã«ãªã‚Šã¾ã™ã€‚RAxMLã¯è² ã®åˆ¶ç´„ã«å¯¾å¿œã—ã¦ã„ãªã„ãŸã‚ã€å˜ç³»çµ±ã€Œã§ãªã„ã€ã¨ã„ã†åˆ¶ç´„ã‚’èª²ã™ã“ã¨ãŒã§ãã¾ã›ã‚“ã€‚ã—ã‹ã—ã€ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—è§£æçµæœã‹ã‚‰å¾—ã‚‰ã‚Œã‚‹å†…åˆ†æå‡ºç¾é »åº¦ã‚’è¦‹ã‚Œã°ã€ãã®è² ã®åˆ¶ç´„ä¸‹ã§æœ€ã‚‚å°¤åº¦ã®é«˜ã„æ¨¹å½¢ã‚’å«ã‚€æ­£ã®åˆ¶ç´„ï¼ç¬¬2ä½ã®ç³»çµ±ä»®èª¬ãŒæ¨å®šã§ãã¾ã™(å¿…ãšã“ã†ãªã‚‹ã‚ã‘ã§ã¯ã‚ã‚Šã¾ã›ã‚“ãŒ)ã®ã§ã€ãã‚Œã‚’èª²ã—ãŸæ¨¹å½¢æ¢ç´¢ã‚’è¡Œã†ã“ã¨ã§å¯¾å‡¦ã™ã‚‹ã“ã¨ãŒã§ãã¾ã™ã€‚

http://cse.naro.affrc.go.jp/minaka/cladist/NOTES/phylostat1.html
ç³»çµ±æ¨å®šã«é–¢ã‚ã‚‹çµ±è¨ˆçš„æ¤œå®šã«ã¤ã„ã¦ï¼ˆå°å…¥ï¼‰
PAUP*ã«ã¯ãƒ€ã‚¤ãƒ¬ã‚¯ãƒˆã«å´©å£ŠæŒ‡æ•°ã‚’è¨ˆç®—ã™ã‚‹ã‚³ãƒãƒ³ãƒ‰ã¯ãªã„ãŒï¼Œæ¨¹å½¢åˆ¶ç´„ï¼ˆã€ŒConstraintsã€ã‚³ãƒãƒ³ãƒ‰ï¼‰ã‚’ç”¨ã„ã‚‹ã“ã¨ã§çµæœçš„ã«å´©å£ŠæŒ‡æ•°ã‚’è¨ˆç®—ã™ã‚‹ã“ã¨ã¯å¯èƒ½ã§ã‚ã‚‹ï¼


http://www.fish-evol.org/Constraint.html
äº•ä¸Šæ½¤ï¼šconstraint ã®é¸ã³æ–¹
åˆ†å²å¹´ä»£æ¨å®šã«ç”¨ã„ã‚‹åˆ¶ç´„ã®é¸ã³æ–¹
2014 å¹´ 11 æœˆ 20 æ—¥ã€€æ”¹è¨‚
äº•ä¸Š æ½¤

----------
## partition
ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³

https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.pdf
2.1.2 åº§ä½é–“ã®ç½®æ›é€Ÿåº¦ä¸å‡è³ªæ€§
åº§ä½ (site) é–“ã«ãŠã‘ã‚‹ç½®æ›é€Ÿåº¦ã®ä¸å‡è³ªæ€§ (heterogeneity) ãŒã‚ã‚‹ã“ã¨ãŒçŸ¥ã‚‰ã‚Œã¦ãŠã‚Šã€ã“ã‚Œã‚’è¡¨ã™ãƒ¢ãƒ‡ãƒ«ãŒã„ãã¤ã‹
ææ¡ˆã•ã‚Œã¦ã„ã¾ã™ã€‚
ã¾ãŸã€ç½®æ›ã®èµ·ããªã„åº§ä½ (invariable site) ã¨ç½®æ›ãŒèµ·ãã‚‹åº§ä½ (variable site) ã® 2 ã¤ã«ã‚«ãƒ†ã‚´ãƒªåˆ†ã‘ã™ã‚‹ãƒ¢ãƒ‡ãƒ« ( + I
ã¨è¡¨è¨˜) ã‚„ã€ + G ã¨ + I ã‚’ä½µç”¨ã—ãŸãƒ¢ãƒ‡ãƒ«ã‚‚ã‚ã‚Šã¾ã™ã€‚ã“ã‚Œã‚‰ã¯ä¸€å®šã®æ³•å‰‡ã«å¾“ã£ã¦è‡ªå‹•çš„ã«è¡Œã‚ã‚Œã‚‹ã‚«ãƒ†ã‚´ãƒªåˆ†ã‘ã§
ã™ãŒã€è§£æè€…ãŒä»»æ„ã®ã‚«ãƒ†ã‚´ãƒªåˆ†ã‘ (partitioning) ã‚’æŒ‡å®šã™ã‚‹ã“ã¨ã‚‚ã§ãã¾ã™ ( + SS (site specific rate ã®æ„) ã¨è¡¨è¨˜)ã€‚
ç•°ãªã‚‹ã‚³ãƒ‰ãƒ³ä½ç½® (codon position) ã‚„éºä¼å­åº§ãªã©ã®ç½®æ›é€Ÿåº¦ã¯ç•°ãªã‚‹å¯èƒ½æ€§ãŒé«˜ã„ã®ã§ã€ã“ã‚Œã‚‰ãŒã—ã°ã—ã°ã‚«ãƒ†ã‚´
ãƒªã¨ã—ã¦æŒ‡å®šã•ã‚Œã¾ã™ã€‚

2.1.3 Mixed model
å‰ç¯€ã§ã¯åº§ä½ (site) é–“ã§ã®ç½®æ›é€Ÿåº¦ä¸å‡è³ªæ€§ã®ã¿ã‚’è€ƒæ…®ã—ã¦ã„ã¾ã—ãŸãŒã€å¡©åŸºç½®æ›é€Ÿåº¦è¡Œåˆ—ãŠã‚ˆã³ç½®æ›é€Ÿåº¦ä¸å‡è³ªæ€§ã®
ä¸å‡è³ªæ€§ã‚’è€ƒæ…®ã™ã‚‹ã“ã¨ã‚‚å¯èƒ½ã§ã™ã€‚ã¤ã¾ã‚Šã€ä»»æ„ã®åº§ä½ã®ã‚°ãƒ«ãƒ¼ãƒ—ï¼ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ (partition) ã”ã¨ã«ç•°ãªã‚‹å¡©åŸº
ç½®æ›é€Ÿåº¦è¡Œåˆ—ã€ç•°ãªã‚‹ ASRV ãƒ¢ãƒ‡ãƒ«ã‚’å½“ã¦ã¯ã‚ã¾ã™ã€‚ã“ã‚Œã¯ mixed model ã¨å‘¼ã°ã‚Œã¦ã„ã¾ã™ã€‚è«–æ–‡ã«ã‚ˆã£ã¦ã¯åŒºåˆ†
ãƒ¢ãƒ‡ãƒ« (partitioned model) ã¨å‘¼ã‚“ã§ã„ã‚‹ã“ã¨ã‚‚ã‚ã‚Šã¾ã™ã€‚ã“ã‚Œã«å¯¾ã—ã¦ã€ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³é–“ã«å…±é€šã®å¡©åŸºç½®æ›é€Ÿåº¦è¡Œ
åˆ—ã¨ ASRV ãƒ¢ãƒ‡ãƒ«ã‚’å½“ã¦ã¯ã‚ã‚‹ã‚‚ã®ã¯éåŒºåˆ†ãƒ¢ãƒ‡ãƒ« (nonpartitioned model) ã¨å‘¼ã°ã‚Œã¾ã™ã€‚

https://www.fifthdimension.jp/documents/molphytextbook/answers.pdf
åˆ†å­ç³»çµ±è§£æã«ãŠã‘ã‚‹
æ§˜ã€…ãªå•é¡Œã«ã¤ã„ã¦

ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã®åˆ‡ã‚Šæ–¹
â— Kakusan4ã¯ä»¥ä¸‹ã‚’æ¯”è¼ƒã—ã¦é¸æŠ
â€“ éºä¼å­åº§é–“ãƒ»ã‚³ãƒ‰ãƒ³ä½ç½®é–“å…¨éƒ¨åˆ‡ã‚‹
â€“ éºä¼å­åº§é–“å…¨éƒ¨åˆ‡ã‚‹ãƒ»ã‚³ãƒ‰ãƒ³ä½ç½®é–“å…¨éƒ¨åˆ‡ã‚‰ãªã„
â€“ éºä¼å­åº§é–“ãƒ»ã‚³ãƒ‰ãƒ³ä½ç½®é–“å…¨éƒ¨åˆ‡ã‚‰ãªã„
â— ã‚‚ã£ã¨æŸ”è»Ÿã«ãªåˆ‡ã‚Šæ–¹ãŒã‚ã‚‹ã®ã§ã¯?
â€“ PartitionFinderã§æ¢ç´¢å¯èƒ½

ç³»çµ±æ¨¹æ¨å®šã®å‹˜æ‰€
â— ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã®åˆ‡ã‚Šæ–¹
â— ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³é–“ãƒ¢ãƒ‡ãƒ«(ç­‰é€Ÿåº¦ãƒ»æ¯”ä¾‹ãƒ»åˆ†é›¢)
â— ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³å†…ãƒ¢ãƒ‡ãƒ«(JC69ï½GTR+G)

å†¨ç”°ç§€ä¸€éƒ è‘— Â· â€2018
https://www.jstage.jst.go.jp/article/konchubiotec/87/2/87_2_091/_pdf
èš•ç³¸ãƒ»æ˜†è™«ãƒã‚¤ã‚ªãƒ†ãƒƒã‚¯ 87(2)ã€91-94(2018) SANSHI-KONCHU BIOTEC
ç‰¹é›†ã€Œãƒã‚¹ãƒˆã‚²ãƒãƒ æ™‚ä»£ã®å½¢æ…‹é€²åŒ–ç ”ç©¶ã€ ã«ã‚ˆã›ã¦~é€²åŒ–ç ”ç©¶ã¨ç³»çµ±è§£æ~

åˆ†å­ç³»çµ±è§£æã«ã¤ã„ã¦ç§è¦‹ã‚’è¿°ã¹ã•ã›ã¦ã„ãŸã ãã€‚æœ€åˆã«ãŠæ–­ã‚Šã•ã›ã¦ã„ãŸã ããŒï¼Œç­†è€…ã¯ï¼Œç³»çµ±è§£æã®å°‚é–€å®¶ã§ã¯ãªãï¼Œã‚€ã—ã‚å˜ãªã‚‹ãƒ¦ãƒ¼ã‚¶ãƒ¼ã§ã‚ã‚‹ã®ã§ï¼Œæ•°ç†çµ±è¨ˆå­¦çš„ãªé¢ã§èª¤è§£ã‚’ã—ã¦ã„ã‚‹å¯èƒ½æ€§ã‚‚ã‚ã‚‹ã€‚

å›³ 1ï¼DNA å¡©åŸºé…åˆ—ãƒ‡ãƒ¼ã‚¿ã®å ´åˆã®åˆ†å­é€²åŒ–ãƒ¢ãƒ‡ãƒ«

è¤‡æ•°ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã¨ãªã‚‹å ´åˆã«ã¯ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã”ã¨ã«ç•°
ãªã‚‹ç½®æ›ç¢ºç‡è¡Œåˆ—ã¨ä¸å‡è³ªæ€§ãƒ¢ãƒ‡ãƒ«ã‚’å½“ã¦ã¯ã‚ã‚‹ã€‚

 ã•ã¦ï¼Œå®Ÿéš›ã®åˆ†å­ç³»çµ±è§£æã¯é€šå¸¸ä»¥ä¸‹ã®ã‚ˆã†ãªã‚¹ãƒ†ãƒƒ ãƒ—ã‚’è¸ã‚€ã“ã¨ã«ãªã‚‹ã€‚
  1)é…åˆ—æƒ…å ±ã‚’åé›†ã™ã‚‹ã€‚
  2)é›†ã‚ãŸé…åˆ—ã‚’ä¸¦ã¹ã¦ï¼Œãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã‚’è¡Œã†ã€‚
  3)åˆ†å­é€²åŒ–ãƒ¢ãƒ‡ãƒ«ã®é¸æŠã¨ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ‹ãƒ³ã‚°ã‚’è¡Œã†ã€‚
  4)ç³»çµ±æ¨å®šã‚’è¡Œã†ã€‚
  5)ç³»çµ±ä»®èª¬ã®æ¯”è¼ƒã‚„å¦¥å½“æ€§ã®æ¤œå®šã‚’è¡Œã†ã€‚

3)ã¨ 4)ã§ç§ã¯ ã«ã‚ã‹ã«é€”æ–¹ã«æš®ã‚Œã‚‹ã“ã¨ã«ãªã£ãŸã€‚ç³»çµ±æ¨å®šã‚’è¡Œã†ãƒ— ãƒ­ã‚°ãƒ©ãƒ ã¯ãƒ¢ãƒ‡ãƒ«é¸æŠã‚„ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã®åˆ‡ã‚Šæ–¹ã‚’æ•™ãˆ ã¦ãã‚Œãªã„ãŒï¼Œãã‚‚ãã‚‚ãã‚ŒãŒä½•ãªã®ã‹çŸ¥ã‚‰ãªã‹ã£ãŸã® ã§ã‚ã‚‹ã€‚

(å›³ 1)ã€‚ã“ã®åˆ† å­é€²åŒ–æ§˜å¼ã¯ï¼Œå³å¯†ã«ã„ãˆã°ã‚µã‚¤ãƒˆ(å¡©åŸºã®ãƒã‚¸ã‚·ãƒ§ãƒ³) ã”ã¨ã«ç•°ãªã£ã¦ã„ã‚‹ã¯ãšã§ã‚ã‚‹ã®ã§ï¼Œã‚µã‚¤ãƒˆã”ã¨ã«ç•°ãª ã‚‹ãƒ¢ãƒ‡ãƒ«ã‚’é©ç”¨ã™ã‚Œã°è‰¯ã„ã‚ˆã†ã«æ€ã†ã‹ã‚‚ã—ã‚Œãªã„ãŒï¼Œ ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰ãƒ¢ãƒ‡ãƒ«ã‚’å¼•ãå‡ºã—ã¦ã„ã‚‹ä»¥ä¸Šï¼Œãã‚Œã¯çµ±è¨ˆå­¦ ã§ã„ã†ã¨ã“ã‚ã®ã„ã‚ã‚†ã‚‹ãƒ•ãƒ«ãƒ¢ãƒ‡ãƒ«ã¨ãªã‚Šæ±ºã—ã¦è‰¯ã„ãƒ¢ ãƒ‡ãƒ«ã¨ã¯è¨€ãˆãªã„ã€‚ãã“ã§é€²åŒ–æ§˜å¼ãŒä¼¼ã¦ã„ãã†ãªã¨ã“ ã‚ã‚’ã¾ã¨ã‚ã‚‹ã®ãŒãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ‹ãƒ³ã‚°ã§ã‚ã‚‹ã€‚éºä¼å­ã” ã¨ï¼Œã‚³ãƒ‰ãƒ³ãƒã‚¸ã‚·ãƒ§ãƒ³ã”ã¨ãªã©ã®ã‚¹ãƒˆãƒ©ãƒ†ã‚¸ãƒ¼ãŒè€ƒãˆã‚‰ ã‚Œï¼Œãã‚Œã ã‘ã§ã‚‚è†¨å¤§ãªãƒ‘ã‚¿ãƒ¼ãƒ³ã«ãªã‚‹ã“ã¨ã‚‚å¤šã„ã€‚ã ã“ã§ï¼Œãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã®åˆ‡ã‚Šæ–¹ã‚„ãƒ¢ãƒ‡ãƒ«é¸æŠã‚’ AIC ã‚„ BIC ã®ã‚ˆã†ãªçµ±è¨ˆé‡åŸºæº–ã§è©•ä¾¡ã—ã¦ãã‚Œã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã‚’ åˆ©ç”¨ã™ã‚‹ã“ã¨ã«ãªã‚‹ãŒï¼Œæœ€çµ‚çš„ã«ãã®å¦¥å½“æ€§ã‚’åˆ¤æ–­ã™ã‚‹ ã®ã¯ç”Ÿç‰©å­¦è€…ã§ã‚ã‚‹ã€‚

2017å¹´3æœˆ22æ—¥
http://www.tezuru-mozuru.com/?p=9927
iqtreeã«ã‚ˆã‚‹æœ€å°¤æ³•ç³»çµ±æ¨¹æ¨å®š â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹
iq treeã¯ï¼Œãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³åˆ†ã‘ãŒå¯èƒ½ãªæœ€å°¤æ³•ç³»çµ±æ¨¹æ¨å®šã‚½ãƒ•ãƒˆã§ã™ï¼

2016 å¹´ 10 æœˆ 12æ—¥
http://www.fish-evol.org/RAxML.html
RAxML - äº•ä¸Š æ½¤
ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã®è¨­å®š

2012 å¹´ 7 æœˆ 16 æ—¥
http://www.fish-evol.org/DatabaseEnglish.html
äº•ä¸Š æ½¤: ç³»çµ±è§£æã®è‹±èªä¾‹æ–‡
Partition è§£æ

Last-modified: 2015-05-13 (æ°´) 16:39:20 (2042d)
http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH17%2Fç³»çµ±å­¦ç‰¹è«–%2Fæœ€å°¤æ³•
H17/ç³»çµ±å­¦ç‰¹è«–/PAUP*ä½¿ç”¨æ³•/æœ€å°¤æ³• â€ 
èª²é¡Œ2ï¼šModeltestã¨ç³»çµ±è§£æ â€ 
æœ¬æ¥ãªã‚‰ã°ã€Modeltestã®ã¨ãã‹ã‚‰ã‚³ãƒ‰ãƒ³ãƒã‚¸ã‚·ãƒ§ãƒ³ã§ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã‚’ä½œã£ã¦è§£æã‚’è¡Œã„ãŸã„ãŒã€ä»Šå›èª¬æ˜ã™ã‚‹æ‰‹é †ã§ãã‚Œã‚’ã‚„ã‚‹ã®ã¯é›£ã—ã„ã€‚ãã“ã§ã€ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ã‚’ä½œã‚‰ãšã«ãƒ¢ãƒ‡ãƒ«é¸æŠã—ãŸå¾Œã€Modeltestã§å¾—ã‚‰ã‚ŒãŸlsetã®siteratesã¨ratesã‚’ãƒ‘ãƒ¼ãƒ†ã‚£ã‚·ãƒ§ãƒ³ç”¨ã®è¨­å®šã«ç½®ãæ›ãˆã¦è§£æã™ã‚‹ã¨ã„ã†ã€ç°¡ä¾¿æ³•ã‚’ã¨ã‚‹ã€‚

2008
https://www.ism.ac.jp/editsec/toukei/pdf/56-1-145.pdf

è¤‡æ•°éºä¼å­ã®çµåˆãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ãåˆ†å­ç³»çµ±æ¨¹ã® æ¨æ¸¬
çœŸæ ¸ç”Ÿç‰©ã®å¤§ç³»çµ±ã®è§£æã‚’ä¾‹ã¨ã—ã¦
æ©‹æœ¬ å“²ç”·1,2 ãƒ»æœ‰æœ« ä¼¸å­3 ãƒ»å‚å£ ç¾äºœå­1 ãƒ»ç¨²å£ ç¥å¸1,2

æ–¹æ³•è«–ã®æ¦‚ç•¥ã‚’è¿°ã¹ï¼ŒçœŸæ ¸ç”Ÿç‰©ã®å¤§ç³»çµ±ã®å•é¡Œã«é–¢ã™ã‚‹ãƒ‡ãƒ¼ã‚¿è§£æã®å®Ÿä¾‹ã‚’ç¤ºã—ãŸï¼
çµåˆã®ãŸã‚ã®çµ±è¨ˆãƒ¢ãƒ‡ãƒ«ã¨ã—ã¦ï¼Œå˜ã«å€‹ã€…ã®éºä¼å­ï¼ˆã‚‚ã—ãã¯å…¨ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã‚’æ§‹æˆã™ã‚‹å€‹ã€…
ã® â€˜åŒºåˆ†â€™ï¼‰ã®é€£çµãƒ‡ãƒ¼ã‚¿ã«å¯¾ã—ã¦ 1 ã‚»ãƒƒãƒˆã®æé•·ã‚’æ¨å®šã™ã‚‹ã€Œé€£çµãƒ¢ãƒ‡ãƒ«ã€ï¼Œå€‹åˆ¥ã®éºä¼å­ï¼ˆåŒº
åˆ†ï¼‰ãã‚Œãã‚Œã«ã¤ã„ã¦ç‹¬ç«‹ã«æé•·ã®æ¨å®šã‚’è¡Œã†ã€Œåˆ†é›¢ãƒ¢ãƒ‡ãƒ«ã€ï¼Œæé•·ãŒéºä¼å­ï¼ˆåŒºåˆ†ï¼‰é–“ã§æ¯”ä¾‹ã—
ã¦ã„ã‚‹ã¨ã„ã†ä»®å®šã‚’ç½®ãã€Œæ¯”ä¾‹ãƒ¢ãƒ‡ãƒ«ã€ã® 3 ã¤ã®ãƒ¢ãƒ‡ãƒ«ã‚’å–ã‚Šä¸Šã’ï¼Œ
Three models of branch length estimation are considered assuming that all
genes (or partitions for the full data set)have the same branch length (concatenate model),
each gene (partition) has a separate set of branch lengths (separate model), and branch
lengths are proportional among genes (partitions) (proportional model).

----------
## branch length
æé•·

https://ja.wikipedia.org/wiki/ã‚¯ãƒ©ãƒ‰ã‚°ãƒ©ãƒ 
cladogram

http://www.h.chiba-u.jp/lab/florista/kokubun/biology/current/biology161005.pdf
ãƒ»æã®é•·ã•ã«æ„å‘³ãŒãªã„ç³»çµ±æ¨¹ï¼ˆã‚¯ãƒ©ãƒ‰ã‚°ãƒ©ãƒ ï¼‰ã¨æã®é•·ã•ãŒå¤‰åŒ–ã®åº¦åˆã„ã‚’ç¤ºã™ç³»çµ±æ¨¹ï¼ˆãƒ•ã‚¡ã‚¤ãƒ­ã‚°
ãƒ©ãƒ ï¼‰ãŒã‚ã‚‹ã€‚ãƒ•ã‚¡ã‚¤ãƒ­ã‚°ãƒ©ãƒ ã«ã¯å¿…ãšã‚¹ã‚±ãƒ¼ãƒ«ï¼ˆæ™‚é–“ã¾ãŸã¯å½¢è³ªã®å¤‰åŒ–ã®æ•°ï¼‰ãŒã¤ãã€‚

https://www.ebi.ac.uk/training/online/course/introduction-phylogenetics/what-phylogeny/aspects-phylogenies/branches
So if you see figures in the literature with branches longer than ~3 substitutions per site then you might want to worry about the confidence we have in those estimates!


https://github.com/haruosuz/bioinfo/blob/master/2019/CaseStudy.md#ws222
Akifumi Tanabe
https://www.fifthdimension.jp/documents/molphytextbook/
ç”°é‚‰æ™¶å²
https://www.fifthdimension.jp/documents/molphytextbook/answers.pdf
åˆ†å­ç³»çµ±è§£æã«ãŠã‘ã‚‹æ§˜ã€…ãªå•é¡Œã«ã¤ã„ã¦
ãã‚‚ãã‚‚ã©ã“ã®é…åˆ—ã‚’ä½¿ã†ã¹ã?
â— ç½®æ›ãŒæ—©ã™ããšé…ã™ããªã„(=å¤šã™ããšå°‘ãªã™ããªã„)
https://www.youtube.com/watch?time_continue=10844&v=vq9Fzd0Yqzc&feature=emb_title
åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ - ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§ @ åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨ ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ— - YouTube
2:57:40 / 3:22:38
ã€Œç½®æ›ãŒé€Ÿã™ãã‚‹ = æã®é•·ã•ï¼ˆ1ã‚µã‚¤ãƒˆã‚ãŸã‚Šã®ç½®æ›æ•°ã®å¹³å‡å€¤ï¼‰ãŒ1ã‚’è¶…ãˆã‚‹ã€‚å…¨ã‚µã‚¤ãƒˆã‚’é€šã—ã¦å¹³å‡ç½®æ›æ•°ãŒ1ã‚’è¶…ãˆã¦ã„ã‚‹ã€‚å¤šé‡ç½®æ›ãŒèµ·ãã¦ã„ã‚‹ã€

----------
## LBA
Long branch attraction (LBA)
é•·æèª˜å¼•

https://haruosuz.hatenadiary.org/entry/20080813

https://en.wikipedia.org/wiki/Long_branch_attraction

https://ja.wikipedia.org/wiki/é•·æèª˜å¼•
é•·æèª˜å¼•[1][2]ï¼ˆã¡ã‚‡ã†ã—ã‚†ã†ã„ã‚“ã€è‹±ï¼šLong Branch Attractionï¼‰ã¯ã€ç³»çµ±è§£æã«ãŠã„ã¦ã€ç³»çµ±çš„ã«è¿‘ç¸ã§ãªã„2ã¤ã®åˆ†é¡ç¾¤ã®é€²åŒ–é€Ÿåº¦ãŒæ¥µã‚ã¦é€Ÿã„å ´åˆã«ã€ç³»çµ±æ¨¹ã®æãŒç›¸å¯¾çš„ã«é•·ããªã‚Šã€ã“ã‚Œã‚‰ã®é•·ã„æã®å…ˆç«¯ã«ä»˜ãåˆ†é¡ç¾¤ãŒèª¤ã£ã¦è¿‘ç¸ã¨æ¨å®šã•ã‚Œã‚‹ç¾è±¡ã€‚ç³»çµ±æ¨å®šæ³•ã®ã†ã¡ç‰¹ã«æœ€å¤§ç¯€ç´„æ³•ã¸ã®å½±éŸ¿ãŒé¡•è‘—ã§ã‚ã‚‹ãŒã€æœ€å°¤æ³•ã‚„è¿‘éš£çµåˆæ³•ãªã©ã®ä»–ã®ç³»çµ±æ¨å®šæ³•ã§ã‚‚ä¸»è¦ãªèª²é¡Œã®ä¸€ã¤ã§ã‚ã‚‹[3]ã€‚

https://www.jstage.jst.go.jp/article/jsbibr/2/1/2_jsbibr.2021.7/_html/-char/ja
ç·èª¬
åˆ†å­ç³»çµ±è§£æã®æœ€å‰ç·š
æ¾äº• æ±‚ 
ç›¸å¯¾çš„ã«æ¥µç«¯ã«é•·ã„æãŒå­˜åœ¨ã™ã‚‹å ´åˆã€é•·æèª˜å¼•ã¨ã„ã†ç¾è±¡ãŒèµ·ãã‚‹ã“ã¨ãŒçŸ¥ã‚‰ã‚Œã¦ãŠã‚Šï¼»32ï¼½ã€ç‰¹ã«æœ€ç¯€ç´„æ³•ã¸ã®æ‚ªå½±éŸ¿ãŒå¤§ãã„ã€‚

https://pubmed.ncbi.nlm.nih.gov/33528562/
Syst Biol
. 2021 Jun 16;70(4):838-843. doi: 10.1093/sysbio/syab001.
Long Branch Attraction Biases in Phylogenetics
Edward Susko 1, Andrew J Roger 2
https://academic.oup.com/sysbio/article/70/4/838/6126433

2020 å¹´
https://www.jstage.jst.go.jp/article/mammalianscience/60/2/60_269/_article/-char/ja/
åˆ†å­æƒ…å ±ã«ã‚‚ã¨ã¥ã„ãŸçœŸç£é¡ã®ç³»çµ±ã¨é€²åŒ–
é•·è°·å· æ”¿ç¾
https://www.jstage.jst.go.jp/article/mammalianscience/60/2/60_269/_pdf
é…åˆ—ã®é•·ã•ã‹ï¼Ÿç¨®ã®æ•°ã‹ï¼Ÿ
é‡è¤‡éºä¼å­ã¯é™¤å¤–ã—ï¼Œ2,789 å€‹ã®ãŸã‚“ã±ãè³ªã‚’ã‚³ãƒ¼ãƒ‰ã—ã¦ã„ã‚‹éºä¼å­ã®åˆè¨ˆãŠã‚ˆã 100 ä¸‡å¡©åŸºåº§ä½ã‚’è§£æã«ç”¨ã„ãŸã®ã§ã‚ã‚‹ï¼
ã€€ã“ã® 100 ä¸‡å¡©åŸºã‚’ã¾ãšä¸€ã¤ãªãŒã‚Šã®é€£çµé…åˆ—ã¨ã—ã¦è§£æã™ã‚‹ã¨ï¼ˆã“ã‚Œã‚’é€£çµãƒ¢ãƒ‡ãƒ«ã¨ã„ã†ï¼‰ï¼Œã©ã®ã‚ˆã†ãªå¡©åŸºç½®æ›ãƒ¢ãƒ‡ãƒ«ã‚’ç”¨ã„ã¦ã‚‚ï¼ŒåŒ—æ–¹ç£é¡ãŒæœ€åˆã«ã»ã‹ã® 2 è€…ã‹ã‚‰åˆ†ã‹ã‚Œï¼Œãã®å¾Œã§ã‚¢ãƒ•ãƒªã‚«ç£é¡ã¨ç•°ç¯€é¡ãŒåˆ†å²ã—ãŸã¨ã„ã†ç³»çµ±æ¨¹ãŒ 100ï¼…ã®ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ã§æ”¯æŒã•ã‚ŒãŸï¼ã“ã‚Œã¯è¶…å¤§é™¸ã®åˆ†æ–­ã®é †ç•ªã¨åˆè‡´ã™ã‚‹ã‚‚ã®ã§ã‚ã‚‹ãŒï¼Œã“ã®çµæœã‚’å—ã‘å…¥ã‚Œã‚‹ã“ã¨ã¯ã§ããªã„ï¼
2,789 å€‹ã®éºä¼å­ãŒãã‚Œãã‚Œç‹¬è‡ªã®ãƒ‘ã‚¿ãƒ¼ãƒ³ã§é€²åŒ–ã™ã‚‹ã¨ã„ã†ãƒ¢ãƒ‡ãƒ«ï¼ˆã“ã‚Œã‚’ç‹¬ç«‹ãƒ¢ãƒ‡ãƒ«ã¨ã„ã†ï¼‰ã§è§£æã™ã‚‹ã¨ï¼Œé€£çµãƒ¢ãƒ‡ãƒ«ã§ã¯ 100ï¼…ã®ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ã§æ”¯æŒã•ã‚Œã¦ã„ãŸã‚¢ãƒ•ãƒªã‚«ç£é¡ / ç•°ç¯€é¡è¿‘ç¸ã§ã¯ãªãï¼ŒåŒ—æ–¹ç£é¡ / ç•°ç¯€é¡è¿‘ç¸ãŒæ”¯æŒã•ã‚Œã‚‹ã‚ˆã†ã«ãªã‚‹ï¼ã—ã‹ã—ãã®æ”¯æŒã¯ï¼Œã»ã‹ã®ç³»çµ±æ¨¹ã‚’æ£„å´ã§ãã‚‹ã»ã©å¼·ã„ã‚‚ã®ã§ã¯ãªã„ï¼
ã€€ã¤ã¾ã‚Šï¼Œç¾å®Ÿã®é€²åŒ–éç¨‹ã¨å¤§ããé£Ÿã„é•ã†ãƒ¢ãƒ‡ãƒ«ã‚’ç”¨ã„ãŸè§£æã§ã¯ï¼Œæ¨å®šã«åã‚ŠãŒç”Ÿã˜ã¦é–“é•ã£ãŸæ¨å®šã‚’ã—ã¦ã—ã¾ã†ã¨åŒæ™‚ã«ï¼Œé…åˆ—ãƒ‡ãƒ¼ã‚¿ãŒé•·ããªã‚‹ã¨ãã‚Œã«ã¤ã‚Œã¦ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°èª¤å·®ãŒå°ã•ããªã‚‹ãŸã‚ã«ï¼Œé–“é•ã£ãŸç³»çµ±æ¨¹ãŒéå¸¸ã«å¼·ãæ”¯æŒã•ã‚Œã‚‹ã¨ã„ã†ã“ã¨ãŒèµ·ã“ã‚‹ã®ã§ã‚ã‚‹ï¼å¾“ã£ã¦ï¼Œã‚²ãƒãƒ è¦æ¨¡ã®å¤§é‡ãƒ‡ãƒ¼ã‚¿ã«ã‚ˆã‚‹ç³»çµ±æ¨¹è§£æã®çµæœã‚’å—ã‘å…¥ã‚Œã‚‹éš›ã«ã¯æ³¨æ„ãŒå¿…è¦ã§ã‚ã‚‹ï¼
ã€€ä»¥ä¸Šã®ã‚ˆã†ã«é…åˆ—ãƒ‡ãƒ¼ã‚¿ãŒé•·ããªã£ã¦ã‚‚æ­£ã—ã„ç³»çµ±æ¨¹æ¨å®šã‚’è¡Œã†ã®ã¯ç°¡å˜ã§ã¯ãªã„ï¼ã“ã‚Œã‚’å…‹æœã™ã‚‹ã«ã¯ï¼Œ2ã¤ã®æ–¹å‘ãŒã‚ã‚‹ï¼1 ã¤ã¯è§£æã«ç”¨ã„ã‚‹ãƒ¢ãƒ‡ãƒ«ã‚’ã‚ˆã‚Šç¾å®Ÿçš„ãªã‚‚ã®ã«ã™ã‚‹ã“ã¨ã§ã‚ã‚‹ï¼ã‚‚ã† 1 ã¤ãŒç¨®ã®ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚’å¢—ã‚„ã™ã“ã¨ã§ã‚ã‚‹ï¼
ã€€ãƒ¢ãƒ‡ãƒ«ãŒé©åˆ‡ã§ãªã„ãŸã‚ã«ç³»çµ±æ¨¹æ¨å®šã‚’é–“é•ã†åŸå› ã¨ã—ã¦æœ€ã‚‚æœ‰åãªã‚‚ã®ãŒé•·æèª˜å¼•ã§ã‚ã‚‹ï¼ˆFelsenstein1978ï¼‰ï¼1991 å¹´ã« Graur ã‚‰ã¯ï¼Œ10 ç¨®é¡ã®ãŸã‚“ã±ãè³ªã®ã‚¢ãƒŸãƒé…¸é…åˆ—ãƒ‡ãƒ¼ã‚¿ã‚’æœ€ç¯€ç´„æ³•ã§è§£æã—ãŸçµæœï¼Œå›³ 3b ã§ç¤ºã—ãŸã‚ˆã†ãªãƒã‚ºãƒŸãŒãƒ¢ãƒ«ãƒ¢ãƒƒãƒˆã‚ˆã‚Šã‚‚ãƒ’ãƒˆã«è¿‘ã„ã¨ã„ã†ç³»çµ±æ¨¹ãŒå¼·ãæ”¯æŒã•ã‚ŒãŸï¼ˆGraur et al. 1991ï¼‰ï¼ã“ã®ç³»çµ±æ¨¹ãŒæ­£ã—ã„ã¨ã™ã‚‹ã¨ãƒã‚ºãƒŸã¨ãƒ¢ãƒ«ãƒ¢ãƒƒãƒˆã‚’å«ã‚€é½§æ­¯ç›®ã¯å˜ç³»çµ±ã§ãªã„ã“ã¨ã«ãªã‚‹ï¼ã¨ã“ã‚ãŒå½¼ã‚‰ã®è§£æã‚’å†æ¤œè¨ã—ã¦ã¿ã‚‹ã¨ï¼Œç”¨ã„ãŸãŸã‚“ã±ãè³ªã® 1 ã¤ã§ã‚ã‚‹ãƒªãƒãŸã‚“ã±ããƒªãƒ‘ãƒ¼ã‚¼ã®é€²åŒ–é€Ÿåº¦ãŒï¼Œãƒ¢ãƒ«ãƒ¢ãƒƒãƒˆã§éå¸¸ã«é«˜ããªã£ã¦ã„ã‚‹ã“ã¨ãŒåˆ†ã‹ã£ãŸã®ã§ã‚ã‚‹ï¼ˆHasegawa et al.1992ï¼‰ï¼æœ€ç¯€ç´„æ³•ã‚„æœ€å°¤æ³•ãªã©åˆ†å­ç³»çµ±æ¨¹æ¨å®šã«ç”¨ã„ã‚‰ã‚Œã‚‹æ–¹æ³•ã®ã»ã¨ã‚“ã©ã¯ï¼Œé€²åŒ–é€Ÿåº¦ã®ä¸€å®šæ€§ã‚’ä»®å®šã—ãªã„ãŒï¼Œç½®æ›ãŒå¤šãã¦é•·ã„æã»ã©å¤šé‡ç½®æ›ã®åŠ¹æœãŒéå°è©•ä¾¡ã«ãªã‚‹ï¼ãã‚‚ãã‚‚æœ€ç¯€ç´„æ³•ã§ã¯å¤šé‡ç½®æ›ã‚’ä¸€åˆ‡è€ƒãˆãªã„ã®ã§ã“ã®å½±éŸ¿ã¯å¤§ãã„ï¼ãã®ãŸã‚ã«å›³ 3a ãŒæ­£ã—ã„ç³»çµ±æ¨¹ã§ã‚ã£ã¦ã‚‚ï¼Œå›³ 3b ã®ã‚ˆã†ã«é•·ã„æåŒå£«ãŒé–“é•ã£ã¦çµ„ã‚“ã§ã—ã¾ã†ã“ã¨ã«ãªã‚‹ï¼ˆå¤–ç¾¤ã«è‡³ã‚‹æã¯é•·ã„ã“ã¨ã«æ³¨æ„ï¼‰ï¼ã“ã‚ŒãŒé•·æèª˜å¼•ã§ã‚ã‚‹ï¼ã“ã®å•é¡Œã¯æœ€ç¯€ç´„æ³•ã®å ´åˆã«ç‰¹ã«æ·±åˆ»ã§ã‚ã‚‹ãŒï¼Œæœ€å°¤æ³•ã§ã‚‚ãƒ¢ãƒ‡ãƒ«ãŒå˜ç´”éãã¦å¤šé‡ç½®æ›ã®åŠ¹æœãŒæ­£ã—ãè©•ä¾¡ã•ã‚Œãªã„å ´åˆã«ã¯å•é¡Œã«ãªã‚‹ï¼
ã€€ã“ã‚Œã‚’å…‹æœã™ã‚‹ã«ã¯ï¼Œç¨®ã®ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚’å¢—ã‚„ã™ã“ã¨ãŒæœ‰åŠ¹ã§ã‚ã‚‹ï¼æœ€å°¤æ³•ã§ã‚ã‚Œã°ï¼Œä¾‹ãˆã°å›³ 3a ã§ãƒ¢ãƒ«ãƒ¢ãƒƒãƒˆã«è‡³ã‚‹é•·ã„æã®é€”ä¸­ã‹ã‚‰åˆ†å²ã™ã‚‹ãƒãƒ¼ãƒ©ï¼Œã‚¢ã‚°ãƒ¼ãƒï¼Œãƒ¤ãƒã‚¢ãƒ©ã‚·ãªã©ã‚’åŠ ãˆãŸã‚Šï¼Œå¤–ç¾¤ã§ã‚ã‚‹ã‚«ãƒ³ã‚¬ãƒ«ãƒ¼ã«è‡³ã‚‹æã®é€”ä¸­ã‹ã‚‰åˆ†å²ã™ã‚‹ã‚¦ã‚·ï¼Œã‚¾ã‚¦ãªã©ã‚’åŠ ãˆã‚‹ã“ã¨ã«ã‚ˆã£ã¦å¤šé‡ç½®æ›ã®åŠ¹æœãŒã‚ˆã‚Šæ­£ã—ãè©•ä¾¡ã•ã‚Œã‚‹ã“ã¨ã«ãªã‚‹ï¼
ã€€é•·æèª˜å¼•ãªã©ï¼Œãƒ¢ãƒ‡ãƒ«ãŒç¾å®Ÿã¨é£Ÿã„é•ã£ã¦ã„ã‚‹ã“ã¨ã«ã‚ˆã‚‹ç³»çµ±æ¨¹æ¨å®šã®åã‚Šã‚’å…‹æœã™ã‚‹ãŸã‚ã«ã¯ï¼Œãƒ¢ãƒ‡ãƒ«ã‚’æ”¹å–„ã™ã‚‹ã¨ã¨ã‚‚ã«ï¼Œç¨®ã®ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚’å¢—ã‚„ã™ã“ã¨ãŒæœ‰åŠ¹ã§ã‚ã‚‹ã¨è¿°ã¹ãŸãŒï¼Œãã®ã“ã¨ã‚’ç¤ºã™ç ”ç©¶ä¾‹ã‚’ç´¹ä»‹ã—ã‚ˆã†ï¼

å›³ 3ï¼é•·æèª˜å¼• Long branch attraction ã«ã‚ˆã£ã¦ç³»çµ±æ¨¹æ¨å®šã‚’èª¤ã‚‹ä¾‹ï¼ˆé•·è°·å· 2018ï¼‰ï¼

### 2019-07-09
https://pubmed.ncbi.nlm.nih.gov/31289177/
mBio
. 2019 Jul 9;10(4):e00925-19. doi: 10.1128/mBio.00925-19.
A Robust Phylogenomic Time Tree for Biotechnologically and Medically Important Fungi in the Genera Aspergillus and Penicillium
Jacob L Steenwyk 1, Xing-Xing Shen 1, Abigail L Lind 2 3, Gustavo H Goldman 4, Antonis Rokas 5 2
https://journals.asm.org/doi/10.1128/mBio.00925-19
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6747717/

(iii) deeper bipartitions (I3, I4, I24, I35, I63, I74, and I78) with various levels of incongruence and typically associated with single-taxon long branches.

(iii) Deeper bipartitions often associated with single-taxon long branches.

Additionally, analytical factors such as model choice (51), taxon sampling (84, 85), hidden paralogy (86, 87), and alignment strategy (88) can lead to erroneous inference of gene histories. Perhaps the best-known instance of incongruence stemming from analytical factors is what is known as â€œlong branch attraction,â€ namely, the situation where highly divergent taxa, i.e., the ones with the longest branches in the phylogeny, will often artifactually group with other long branches (89). 

Finally, the remaining seven incongruent internodes (I3, I4, I24, I35, I63, I74, and I78) exhibit various levels of incongruence and are typically associated with single-taxon long branches (Fig. 1, 3, and 4), implicating taxon sampling as a likely driver of the observed incongruence. Given that inclusion of additional taxa robustly resolved the previously ambiguous placement of the long-branched Aspergillus section Nigri (see discussion above) as well as of other contentious branches of the fungal tree of life, such as the placement of the budding yeast family Ascoideaceae (59, 60), we predict that additional sampling of taxa that break up the long branches associated with these seven internodes will lead to their robust resolution.

### 2018-05-03
https://www.natureasia.com/ja-jp/nature/highlights/92171
Nature ãƒã‚¤ãƒ©ã‚¤ãƒˆï¼šãƒŸãƒˆã‚³ãƒ³ãƒ‰ãƒªã‚¢ã®ç¥–å…ˆã‚’å†æ¤œè¨ã™ã‚‹ | Nature | Nature Portfolio
2018å¹´5æœˆ3æ—¥ Nature 557, 7703
ã—ã‹ã—ã€æœ€åˆæœŸã®åŒ–çŸ³çœŸæ ¸ç”Ÿç‰©ã¯ç´„20å„„å¹´å‰ã®ã‚‚ã®ã§ã‚ã‚Šã€åˆ†å²ã‹ã‚‰ã¯éå¸¸ã«é•·ã„æ™‚é–“ãŒçµŒéã—ã¦ã„ã‚‹ãŸã‚ã€é•·æèª˜å¼•ï¼ˆlong-branch attractionï¼›LBAï¼‰ã«é–¢ã™ã‚‹å¤šãã®å•é¡ŒãŒã‚ã‚‹ã“ã¨ãŒç¤ºå”†ã•ã‚Œã‚‹ã€‚

BLASTæ¤œç´¢ãŒç³»çµ±å­¦çš„å†æ§‹æˆã®ä»£ç†ã¨ã—ã¦ä½¿ç”¨ã•ã‚Œã‚‹å ´åˆã€çŸ­ã„åˆ†å²ã¯ä½“ç³»çš„ãªã‚¢ãƒ¼ãƒ†ã‚£ãƒ•ã‚¡ã‚¯ãƒˆã«ã¤ãªãŒã‚Šã¾ã™ã€‚
https://www.ncbi.nlm.nih.gov/pubmed/27894995
Mol Phylogenet Evol. 2017 Feb;107:338-344. doi: 10.1016/j.ympev.2016.11.016. Epub 2016 Nov 26.
Short branches lead to systematic artifacts when BLAST searches are used as surrogate for phylogenetic reconstruction.
Dick AA1, Harlow TJ2, Gogarten JP3.
https://linkinghub.elsevier.com/retrieve/pii/S1055-7903(16)30377-3
Long Branch Attraction (LBA) is a well-known artifact in phylogenetic reconstruction when dealing with branch length heterogeneity. Here we show another phenomenon, Short Branch Attraction (SBA), which occurs when BLAST searches, a phenetic analysis, are used as a surrogate method for phylogenetic analysis.

http://sourui.org/publications/sorui/list/Sourui_PDF/Sourui-55-02-111.pdf
Jpn. J. Phycol. (Sorlli) 55: 111-116ï¼Œ JlI ly 10ï¼Œ2007
ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—A åˆ†å­ãƒ‡ãƒ¼ã‚¿è§£æã«ã‹ã‹ã‚ã‚‹ã‚¨ãƒˆã‚»ãƒˆãƒ©: åã‚Šã®ãªã„ç³»çµ±æ¨å®šã‚’ç›®æŒ‡ã—ã¦
ç¨²å£ç¥å¸ : ãƒ­ãƒ³ ã‚°ãƒ–ãƒ©ãƒ³ãƒã®èª˜æƒ‘ä¸€åˆ†å­ç³»çµ±è§£æã®ãƒ€ãƒ¼ã‚¯ã‚µã‚¤ãƒ‰
https://sites.google.com/site/memicrobes/home/members/yuji
ç¨²å£ç¥å¸ã€€åšå£«ï¼ˆç†å­¦ï¼‰ï¼Yuji Inagaki Ph.D


https://pubmed.ncbi.nlm.nih.gov/16209710/
BMC Evol Biol
. 2005 Oct 6;5:50. doi: 10.1186/1471-2148-5-50.
Heterotachy and long-branch attraction in phylogenetics
HervÃ© Philippe 1, Yan Zhou, Henner Brinkmann, Nicolas Rodrigue, FrÃ©dÃ©ric Delsuc
https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-5-50
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1274308/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1274308/figure/F1/
Illustration of the branch length heterogeneity conditions commonly referred as the Felsenstein zone (a) and the Farris zone (b). The Felsenstein zone [3] is characterised by two long branches that are not adjacent in the model topology, a situation where most phylogenetic methods fall into the long-branch attraction artefact [1]. Conversely, in the Farris zone [17], also called the inverse-Felsenstein zone [8], the two long branches are adjacent in the model topology. This last condition strongly favours MP over ML because of the intrinsic bias of parsimony towards interpreting multiple changes that occurred along the two long branches as false synapomorphies [8].


https://pubmed.ncbi.nlm.nih.gov/34892859/
Cladistics
. 2005 Apr;21(2):163-193. doi: 10.1111/j.1096-0031.2005.00059.x.
A review of long-branch attraction
Johannes Bergsten 1
https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.2005.00059.x
Methods suggested to avoid LBA-artifacts include excluding long-branch taxa, excluding faster evolving third codon positions, using inference methods less sensitive to LBA such as likelihood, the Aguinaldo et al. approach, sampling more taxa to break up long branches and sampling more characters especially of another kind, and the pros and cons of these are discussed.

Adding taxa to break up long branches
This leads to one of the most widely suggested remedies for LBA artifacts, i.e., to add more taxa to break up long branches (

https://pubmed.ncbi.nlm.nih.gov/15336677/
Mol Phylogenet Evol
. 2004 Nov;33(2):440-51. doi: 10.1016/j.ympev.2004.06.015.
Should we be worried about long-branch attraction in real data sets? Investigations using metazoan 18S rDNA
Frank E Anderson 1, David L Swofford
https://www.sciencedirect.com/science/article/abs/pii/S1055790304002052?via%3Dihub
Attempts to break up long branches for problematic subsamples through increased taxon sampling reduced, but did not eliminate, LBA problems. 


2004å¹´12æœˆ15æ—¥
https://www.brh.co.jp/research/formerlab/miyata/2004/post_000006.php
ã€ãƒŸã‚¯ã‚½ã‚¾ã‚¢ï¼šå¤šç´°èƒã‹ã‚‰å˜ç´°èƒã¸ã®é€€åŒ–ã€‘ | JTç”Ÿå‘½èªŒç ”ç©¶é¤¨
ã€€é€²åŒ–é€Ÿåº¦ã®ä¸Šæ˜‡ã¯åˆ†å­ã«åŸºã¥ãç³»çµ±æ¨¹ã®æ¨å®šã«å„ä»‹ãªå•é¡Œã‚’å¼•ãèµ·ã“ã™ã€‚åˆ†å­ç³»çµ±æ¨¹ã§ã¯é€šå¸¸ã€ã‚ã‚‹ç”Ÿç‰©ã®ç³»çµ±ã®æã®é•·ã•ã¯é€²åŒ–é€Ÿåº¦ã«æ¯”ä¾‹ã—ã¦å–ã‚‰ã‚Œã‚‹ã€‚å¾“ã£ã¦å¯„ç”Ÿæ€§ã®å‹•ç‰©ã§ã¯æã®é•·ã•ãŒé•·ã„ã€‚ã¨ã“ã‚ã§åˆ†å­ç³»çµ±æ¨¹ã®æ¨å®šã«éš›ã—ã€æ¥µç«¯ã«é•·ã„æåŒå£«ãŒçµã³ã¤ã‘ã¦ã—ã¾ã†èª¤ã‚Šã‚’ã—ã°ã—ã°çŠ¯ã™ã€‚ã“ã®ã“ã¨ã‚’Long Branch Attraction Artifact (LBA)ã¨å‘¼ã‚“ã§ã„ã‚‹ã€‚ã™ãªã‚ã¡ã€å¯„ç”Ÿæ€§ã®ç”Ÿç‰©ã§ã¯åˆ†å­ç³»çµ±æ¨¹ä¸Šã®ä½ç½®ã‚’æ­£ã—ãæ¨å®šã™ã‚‹ã“ã¨ãŒãã‚‚ãã‚‚é›£ã—ã„ã®ã§ã‚ã‚‹ã€‚
ã€€å‰ç½®ãã¯ã“ã®ãã‚‰ã„ã«ã—ã¦18Sãƒªãƒœã‚½ãƒ¼ãƒ RNï¼¡ã§æ¨å®šã•ã‚ŒãŸç³»çµ±æ¨¹ï¼ˆå›³1aï¼‰ä¸Šã§ã®ãƒŸã‚¯ã‚½ã‚¾ã‚¢ã®ä½ç½®ã‚’è¦‹ã¦ã¿ã‚ˆã†ã€‚ç¢ºã‹ã«ä¸»å¼µé€šã‚Šã€ãƒŸã‚¯ã‚½ã‚¾ã‚¢ã¯ä¸‰èƒšè‘‰æ€§ã®å‹•ç‰©ã¨è¿‘ç¸é–¢ä¿‚ã«ã‚ã‚‹ãŒã€ï¼’ã¤ã®æã¯ã„ãšã‚Œã‚‚é•·ã„ã€‚ã“ã‚Œã¯ä¸Šã§è¿°ã¹ãŸLBAã®å¯èƒ½æ€§ãŒã‚ã‚‹ã€‚ã¤ã¾ã‚Šã€ãƒŸã‚¯ã‚½ã‚¾ã‚¢ã®ç³»çµ±çš„ä½ç½®ã¯æ­£ã—ãæ¨å®šã•ã‚Œã¦ã„ãªã„ã®ã‹ã‚‚ã—ã‚Œãªã„ã€‚
ã€€ä¸€æ­©ä¸‹ãŒã£ã¦ã€å˜ç´°èƒã®ãƒŸã‚¯ã‚½ã‚¾ã‚¢ãŒå¤šç´°èƒå‹•ç‰©ç”±æ¥ã‹ã©ã†ã‹ã ã‘ã¯ç­”ãˆã‚‰ã‚Œã‚‹ã‹ã‚‚ã—ã‚Œãªã„ã€‚ãã‚Œã«ã¯ä¸€æ–¹ã®é•·ã„æã®ä¸‰èƒšè‘‰å‹•ç‰©ã‚’é™¤ã„ã¦æ¨å®šã—ç›´ã—ã¦ã¿ã‚‹ã¨ã‚ˆã„ã€‚ãã†ã—ã¦è§£æã‚’ã—ç›´ã—ã¦ã¿ã‚‹ã¨ã€ãƒŸã‚¯ã‚½ã‚¾ã‚¢ã¯è¾›ã†ã˜ã¦ã¯äºŒèƒšè‘‰æ€§ã®å‹•ç‰©ã«è¿‘ç¸ã®ã‚ˆã†ã§ã‚ã‚‹ï¼ˆå›³1bï¼‰ã€‚å¾“ã£ã¦ã€ã€ŒãƒŸã‚¯ã‚½ã‚¾ã‚¢ã¯å¤šç´°èƒå‹•ç‰©ç”±æ¥ã§ã€å¯„ç”Ÿã®çµæœå½¢æ…‹ã‚’è‘—ã—ãé€€åŒ–ã•ã›ãŸã€ã€ã¨ã„ã£ãŸç¨‹åº¦ã®ä¸»å¼µãªã‚‰æ­£ã—ã„ã‹ã‚‚ã—ã‚Œãªã„ã€‚


https://dl.ndl.go.jp/info:ndljp/pid/10940560?tocOpened=1
æœ€å°¤æ³•ã«ã‚ˆã‚‹ç”Ÿç‰©ã®ç³»çµ±é–¢ä¿‚æ¨å®šã«ã¤ã„ã¦(æ•°å­¦è€…ã®ãŸã‚ã®åˆ†å­ç”Ÿç‰©å­¦å…¥é–€,ç ”ç©¶ä¼šå ±å‘Š) - å›½ç«‹å›½ä¼šå›³æ›¸é¤¨ãƒ‡ã‚¸ã‚¿ãƒ«ã‚³ãƒ¬ã‚¯ã‚·ãƒ§ãƒ³
https://repository.kulib.kyoto-u.ac.jp/dspace/bitstream/2433/97613/1/KJ00004705532.pdf
Title æœ€å°¤æ³•ã«ã‚ˆã‚‹ç”Ÿç‰©ã®ç³»çµ±é–¢ä¿‚æ¨å®šã«ã¤ã„ã¦(æ•°å­¦è€…ã®ãŸã‚
ã®åˆ†å­ç”Ÿç‰©å­¦å…¥é–€,ç ”ç©¶ä¼šå ±å‘Š)
Author(s) åŠ è—¤, å’Œè²´; æ¡‘ç”°, å’Œæ­£
Citation ç‰©æ€§ç ”ç©¶ (2003), 81(1): 69-80
 | 
è¿‘éš£çµåˆæ³•
 | 
å›³ 5:LongBranchAttractionã®ä¾‹
 | 
ã“ã®æ–¹æ³•ã®å•é¡Œç‚¹ã¨ã—ã¦ï½¤é•·ã„æ (ç¨®ãŒåˆ†å²ã—ã¦ã‹ ã‚‰ã®ç½®æ›ãŒæ²¢å±±ç™ºç”Ÿã—ã¦ã„ã‚‹æ)ã«
å¯¾å¿œã™ã‚‹ç¨®åŒå£«ãŒè¿‘éš£ã®é–¢ä¿‚ã«ã‚ã‚‹ã¨æ¨å®šã•ã‚Œã‚„ã™ã„ã“ã¨ãŒæŒ™ã’ã‚‰ã‚Œã‚‹ï½¡ç½®æ›ãŒå…¨ã¦ç•°
ãªã‚‹ç®‡æ‰€ã§ç™ºç”Ÿã—ã¦ã„ã‚‹å ´åˆã¯ï½¤é…åˆ—ã®ç•°ãªã‚‹ç®‡æ‰€ãŒå³ã¡ç·ç½®æ›æ•°ã§ã‚ã‚‹ãŒï½¤åŒã˜ç®‡æ‰€ãŒ
å¹¾åº¦ã‹ç½®æ› ã—ã¦ã„ã‚‹å ´åˆã¯ï½¤é…åˆ—ã®ç•°ãªã‚‹ç®‡æ‰€ã¯ç·ç½®æ›æ•°ã‚ˆã‚Šå°ã•ããªã‚‹ï½¡å¾Œè€…ã®ç¾è±¡ã¯
ç½®æ› ã—ã¦ã„ã‚‹ç®‡æ‰€ãŒä½• ã‚‰ã‹ã®ç†ç”±ã§åã£ã¦ã„ã‚‹å ´åˆã‚„ï½¤ç·ç½®æ›æ•°ãŒå¤šã„å ´åˆã«ç™ºç”Ÿã™ã‚‹ï½¡
ã“ã®ã¨ãï½¤è·é›¢ã¯é…åˆ—ã® ï½¢è¿‘ã•ï½£ã§å®šã‚ã‚‹ãŸã‚ï½¤å®Ÿéš›ã®ç½®æ›æ•°ã‚ˆ ã‚Šè·é›¢ã‚’å°ã•ãè¦‹ç© ã‚‚ã‚‹
ã“ã¨ãŒã‚ã‚‹ï½¡å›³ 5ã«ãŠã„ã¦ï½¤æ­£ã—ãã¯å·¦ã®ç³»çµ±æ¨¹ã§ã‚ã£ãŸã‚‚ã®ãŒèª¤ã£ã¦å³ã®ç³»çµ±æ¨¹ã®ã‚ˆã†
ã«æ¨å®šã•ã‚Œã¦ ã—ã¾ã†ã®ã§ã‚ã‚‹ï½¡ã“ã‚Œã‚’longbranchattraction(LBA)ã¨å‘¼ã¶ï½¡
ã“ã®é–“é¡Œã«å¯¾ ã—ï½¤é…åˆ—ã®ç›¸é•ã‹ã‚‰ç·ç½®æ›æ•°ã‚’é©åˆ‡ã«æ¨å®šã™ã‚‹ãŸã‚ï½¤ä¿®æ­£ã‚’åŠ ãˆãŸæ–¹æ³•ãŒ
å¿…è¦ã«ãªã‚‹ï½¡ä¸€æ¡ˆ ã¨ã—ã¦ï½¤é…åˆ—ã®å„ç®‡æ‰€ã®ç½®æ›ã®ç™ºç”Ÿé »åº¦ (ç›¸å¯¾é€²åŒ–é€Ÿåº¦)ãŒ Fåˆ†å¸ƒ ã«å¾“
ã†ã¨ä»®å®š ã—ã¦ç·ç½®æ›æ•°ã‚’å‰² ã‚Šå‡ºã™æ–¹æ³•ãŒææ¡ˆã•ã‚Œã¦ã„ã‚‹ï½¡ã“ã‚Œã¯ MLæ³•ã¨NJæ³•ã®ã©ã¡ ã‚‰
ã§ã‚‚åˆ©ç”¨ã§ãã‚‹ï½¡

https://www.ism.ac.jp/editsec/toukei/pdf/50-1-045.pdf
çµ±è¨ˆæ•°ç†ï¼ˆ2002ï¼‰
åˆ†å­ç³»çµ±æ¨¹æ³•ã®å¿œç”¨ã¨ç¾çŠ¶ã®å•é¡Œç‚¹
çœŸæ ¸ç”Ÿç‰©ã®åˆæœŸé€²åŒ–ã®è§£æã‚’ä¾‹ã¨ã—ã¦ *
æ©‹æœ¬ å“²ç”·1,2 ãƒ»æœ‰æœ« ä¼¸å­2 ãƒ»é•·è°·å· æ”¿ç¾

åˆ†å­ç³»çµ±æ¨¹ã®æ¨è«–ã‚’èª¤ã‚‰ã›ã‚‹æœ€ã‚‚å¤§ããªè¦å› ã¨
ã—ã¦æœ€è¿‘æ³¨ç›®ã‚’é›†ã‚ã¦ã„ã‚‹ Long Branch Attraction ã‚¢ãƒ¼ãƒ†ãƒ•ã‚¡ã‚¯ãƒˆã«ã¤ã„ã¦å®Ÿä¾‹ã«å‰‡ã—ã¦è§£èª¬
ã—ãŸï¼ã•ã‚‰ã«ï¼Œãã‚Œã‚’å…‹æœã™ã‚‹ãŸã‚ã®æ‰‹æ³•ã¨ã—ã¦ï¼Œåº§ä½é–“ã®é€²åŒ–é€Ÿåº¦ã®ä¸å‡è³ªæ€§ã‚’ Î“ åˆ†å¸ƒã®å°
å…¥ã«ã‚ˆã‚Šè€ƒæ…®ã—ãŸè§£æã‚’å®Ÿä¾‹ã«å¯¾ã—ã¦è©¦ã¿ï¼Œã“ã®æ–¹æ³•ã®æœ‰åŠ¹æ€§ã‚’ç¤ºã—ãŸï¼

å›³ 5. Long Branch Attractionï¼ˆLBAï¼‰ã‚’ç¤ºã™æ¨¡å¼å›³ï¼


----------
## revisionterm

2017å¹´12æœˆ22æ—¥
http://jams.med.or.jp/dic/h29material_s5.pdf
å¹³æˆ29åº¦æ—¥æœ¬åŒ»å­¦ä¼šåˆ†ç§‘ä¼šç”¨èªå§”å“¡ä¼š
éºä¼å­¦ç”¨èªæ”¹è¨‚ã«é–¢ã™ã‚‹ãƒ¯ãƒ¼ã‚­ãƒ³ã‚°ã‚°ãƒ«ãƒ¼ãƒ—ã«ã¤ã„ã¦

ç¾è¡Œã®ç”¨èª æ—¥æœ¬äººé¡éºä¼å­¦ä¼š(2009)	æ—¥æœ¬éºä¼å­¦ä¼š(2017)

Variant å¤‰ç•°ä½“ å¤šæ§˜ä½“ï¼ˆãƒãƒªã‚¢ãƒ³ãƒˆï¼‰


https://www.jstage.jst.go.jp/article/mammalianscience/57/2/57_387/_article/-char/ja
ã€ŒVariationã€ã®è¨³èªã¨ã—ã¦ã€Œå¤‰ç•°ã€ãŒä½¿ãˆãªããªã‚‹ã‹ã‚‚ã—ã‚Œãªã„å•é¡Œã«ã¤ã„ã¦ï¼šæ—¥æœ¬éºä¼å­¦ä¼šã®æ–°ç”¨èªé›†ã«ãŠã‘ã‚‹å•é¡Œç‚¹
æµ…åŸ æ­£å’Œ
https://www.jstage.jst.go.jp/article/mammalianscience/57/2/57_387/_pdf/-char/ja
å“ºä¹³é¡ç§‘å­¦ã€€57(2)ï¼š387-390ï¼Œ2017
Â©æ—¥æœ¬å“ºä¹³é¡å­¦ä¼š
ä»Šå¾Œã‚‚ variation ã®è¨³èªã¨ã—ã¦ï¼Œä¼çµ±çš„ã«ç”¨ã„ã‚‰ã‚Œï¼Œç¾åœ¨ã‚‚
åºƒãä½¿ã‚ã‚Œã¦ã„ã‚‹ã€Œå¤‰ç•°ã€ã¨ã„ã†è¨³èªã‚’æ®‹ã™ã“ã¨ãŒæœ›ã¾ã—
ã„ã¨è€ƒãˆã‚‰ã‚Œã‚‹ï¼

https://gsj3.org/wordpress_v2/wp-content/themes/gsj3/assets/docs/pdf/revisionterm_20170911.pdf
éºä¼å­¦ç”¨èªæ”¹è¨‚ã«ã¤ã„ã¦
2017.9.11 æ—¥æœ¬éºä¼å­¦ä¼š
2017å¹´9æœˆã«ã€ã€Œéºä¼å˜ã€€ï½éºä¼å­¦ç”¨èªé›†ã€€å¯¾è¨³é›†ã¤ãï½ã€(ç”Ÿå‘½ã®ç§‘å­¦éºä¼åˆ¥
å†Šï¼šhttp://www.nts-book.co.jp/item/detail/summary/bio/20170929_182.html
)ãŒç™ºåˆŠã•ã‚Œã¾ã—ãŸã€‚ã“ã®
æ›¸ç±ã‚’é€šã˜ã¦éºä¼å­¦ç”¨èªã®æ”¹è¨‚ã‚’ææ¡ˆã—ã¦ã„ã¾ã™ã€‚æ”¹è¨‚ã•ã‚ŒãŸä¸»ãªç”¨èªã¯ä»¥ä¸‹ã®é€šã‚Šã§ã™ã€‚
```
èª¬æ˜ç•ªå· è‹±èª æ—§æ¥ã®è¨³èª æ–°ãŸã«æ”¹è¨‚ã•ã‚ŒãŸè¨³èª

4 mutation çªç„¶å¤‰ç•° [çªç„¶]å¤‰ç•°**
5 variation å¤‰ç•°ã€å½·å¾¨å¤‰ç•° (1)å¤šæ§˜æ€§(2)å¤‰å‹• ***
6 diversity å¤šæ§˜æ€§ (1)å¤šæ§˜æ€§(2)åˆ†å² ***
```

https://jshg.jp/about/notice-reference/
æ—¥æœ¬äººé¡éºä¼å­¦ä¼š
éºä¼å­¦ç”¨èªã®æ”¹è¨‚
ï¼ˆ2009å¹´ï¼ˆå¹³æˆ21å¹´ï¼‰9æœˆæ”¹è¨‚ï¼‰
https://jshg.jp/wp-content/uploads/2017/08/d5fdc84ae83d3a9a6627b7ac249e4db0.pdf
No è‹±èª æ—¥æœ¬èª ã“ã‚Œã¾ã§

4. variant å¤šæ§˜ä½“ï¼ˆãƒãƒªã‚¢ãƒ³ãƒˆï¼‰ å¤‰ç•°ä½“
5. mutant å¤‰ç•°ä½“ï¼ˆçªç„¶å¤‰ç•°ä½“ï¼‰ çªç„¶å¤‰ç•°ä½“

mutant ã¯ã€Œå¤‰ç•°ä½“ï¼ˆçªç„¶å¤‰ç•°ä½“ã‚‚å¯ï¼‰ã€ã€
variant ã¯ã€Œå¤šæ§˜ä½“ï¼ˆãƒãƒªã‚¢ãƒ³ãƒˆã‚‚å¯ï¼‰ã€ã®è¨³ã‚’å½“ã¦ã‚‹ã€‚


----------
## convergent
åæ–‚é€²åŒ–

https://ja.wikipedia.org/wiki/åæ–‚é€²åŒ–
convergent evolution

https://ja.wikipedia.org/wiki/å¹³è¡Œé€²åŒ–
parallel evolution
å¹³è¡Œé€²åŒ–ã®çµæœãŒåæ–‚ã§ã‚ã‚‹å ´åˆã‚‚ã‚ã‚‹ã€‚

5:17 AM Â· Feb 19, 2021
https://twitter.com/kwasiscientific/status/1362496383618600961
Kwasi Wrensford on Twitter: "Aight, I donâ€™t want to just complain about it, so hereâ€™s a simple explainer! Convergent evolution describes when unrelated populations of living things independently evolve similar traits in response to similar pressures in their respective environments. (1/9)" / Twitter

5:32 AM Â· Feb 15, 2021
https://twitter.com/vscooper/status/1361050785187049475
Vaughn Cooper on Twitter: "Chance, parallelism or convergence? A
inspired by today's preprint. https://t.co/13FVi6PSJJ" / Twitter


https://www.natureasia.com/ja-jp/nature/highlights/100765
é€²åŒ–éºä¼å­¦ï¼šå¹³è¡Œé€²åŒ–ã®çµŒè·¯
2019å¹´10æœˆ17æ—¥ Nature 574, 7778
ä¾‹ãˆã°ã€å¼·å¿ƒé…ç³–ä½“ã¨ã„ã†æ¯’ç´ ã¯ã€é€šå¸¸ã¯Na+/K+-ATPã‚¢ãƒ¼ã‚¼ã‚’æ¨™çš„ã¨ã™ã‚‹ãŒã€æ˜†è™«ç¶±ã®6ã¤ã®ç›®ã§ã¯ã€ã“ã®é…µç´ ã®ã‚¢ãƒŸãƒé…¸ç½®æ›ã‚’å¹³è¡Œé€²åŒ–ã•ã›ã‚‹ã“ã¨ã«ã‚ˆã‚Šã€ã“ã®ã‚ˆã†ãªæ¯’ç´ ã«å¯¾ã™ã‚‹è€æ€§ã‚’ç²å¾—ã—ã¦ã„ã‚‹ã€‚
åæ–‚å½¢è³ª

https://www.ncbi.nlm.nih.gov/pubmed/29650391
Trends Microbiol. 2018 Apr 10. pii: S0966-842X(18)30067-2. doi: 10.1016/j.tim.2018.03.004. [Epub ahead of print]
Convergent Evolution in Intracellular Elements: Plasmids as Model Endosymbionts.
Dietel AK1, Kaltenpoth M2, Kost C3.
REVIEW



6:21 AM Â· Apr 1, 2018
https://twitter.com/9w9w9w92/status/980193458995437569
ã€Œåæ–‚é€²åŒ–: ç•°ãªã‚‹ç³»çµ±ã§ã€ç•°ãªã‚‹ç™ºç”ŸçµŒè·¯ã‚’ä½¿ã£ã¦ã€é¡ä¼¼ã—ãŸè¡¨ç¾å‹ãŒé€²åŒ–ã™ã‚‹ã“ã¨ã€‚åˆ†å­ãƒ¬ãƒ™ãƒ«ã®å ´åˆã¯ã€ç•°ãªã‚‹ç³»çµ±ã§ã€é¡ä¼¼ã—ãŸå½¢è³ªãŒé€²åŒ–ã™ã‚‹ã“ã¨
å¹³è¡Œé€²åŒ–: ç•°ãªã‚‹ç³»çµ±ã§ã€åŒã˜ç™ºç”ŸçµŒè·¯ã‚’ä½¿ã£ã¦ã€é¡ä¼¼ã—ãŸè¡¨ç¾å‹ãŒé€²åŒ–ã™ã‚‹ã“ã¨ã€â‘¢232é 



http://kagakubar.com/evolution/07.html
ç¬¬7è©±
ãªãœå¤šæ§˜ãªç¨®ãŒé€²åŒ–ã—ãŸã‹ï¼Ÿ
æ–‡ã¨å†™çœŸã€€é•·è°·å·æ”¿ç¾
â—è‡ªç„¶é¸æŠ

â—åœ°çƒä¸Šã®ç”Ÿç‰©ç¨®æ•°
ãƒ€ãƒ¼ã‚¦ã‚£ãƒ³ä»¥æ¥ã€ç”Ÿç‰©å­¦è€…ã¯1ã¤ã²ã¨ã¤ã®ç¨®ã‚’â€œç”Ÿå‘½ã®æ¨¹â€ã®ãªã‹ã«ä½ç½®ã¥ã‘ã‚‹ä½œæ¥­ã‚’ã‚³ãƒ„ã‚³ãƒ„ã¨é€²ã‚ã¦ããŸãŒã€ãã®éš›ã®æ‰‹æ›ã‹ã‚Šã¯ãã‚Œãã‚Œã®ç¨®ãŒã‚‚ã¤å½¢æ…‹çš„ãªç‰¹å¾´ã‚„ç”Ÿç†çš„ãªç‰¹å¾´ã§ã‚ã£ãŸã€‚äº’ã„ã«ä¼¼ãŸã‚ˆã†ãªç‰¹å¾´ã‚’ã‚‚ã£ã¦ã„ã‚‹ç¨®ã¯ã€ç”Ÿå‘½ã®æ¨¹ã®ãªã‹ã§è¿‘ã„ä½ç½®ã‚’å ã‚ã‚‹ã¨è€ƒãˆã‚‰ã‚Œã‚‹ã®ã ã€‚ã—ã‹ã—ã€å•é¡Œã¯ç°¡å˜ã§ã¯ãªã„ã€‚åŒã˜ã‚ˆã†ãªç‰¹å¾´ãŒã€åˆ¥ã€…ã®ç³»çµ±ã§åæ–‚çš„ã«é€²åŒ–ã™ã‚‹ã“ã¨ãŒã‚ã‚‹ã‹ã‚‰ã ã€‚ãƒ€ãƒ¼ã‚¦ã‚£ãƒ³ã‚‚ãã®ã“ã¨ã«ã¯æ°—ãŒã¤ã„ã¦ã„ãŸã€‚

â—åæ–‚é€²åŒ–
ã“ã‚Œã¾ã§å–ã‚Šä¸Šã’ãŸã»ã‹ã®åæ–‚ã‚‚ã™ã¹ã¦ã€è©³ã—ãèª¿ã¹ã‚Œã°è¿‘ç¸ã ã‹ã‚‰ã§ã¯ãªãã¦ã€åæ–‚ã®çµæœã§ã‚ã‚‹ã“ã¨ãŒåˆ†ã‹ã‚‹ã€‚
ã—ã‹ã—ã€ã‚‚ã£ã¨è¿‘ç¸ãªçœŸç£é¡ã®ãªã‹ã®2ã¤ã®ç³»çµ±ã§åæ–‚é€²åŒ–ãŒèµ·ã£ãŸå ´åˆã«ã¯ã€ãã®ã‚ˆã†ãªå…±é€šã®ç‰¹å¾´ãŒç¥–å…ˆã‚’å…±æœ‰ã™ã‚‹ãŸã‚ãªã®ã‹ã€åæ–‚é€²åŒ–ã«ã‚ˆã‚‹ã‚‚ã®ãªã®ã‹ã‚’åˆ¤å®šã™ã‚‹ã“ã¨ãŒé›£ã—ããªã‚‹ã€‚

â—åˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬
ã“ã®ã‚ˆã†ã«åæ–‚é€²åŒ–ãŒèµ·ã‚‹ã®ã§ã€2ã¤ã®ç”Ÿç‰©ç¨®ã§ä¼¼ãŸã‚ˆã†ãªç‰¹å¾´ãŒè¦‹ã‚‰ã‚Œã¦ã‚‚ã€ãã‚ŒãŒå…±é€šç¥–å…ˆã®ç‰¹å¾´ã‚’å—ã‘ç¶™ã„ã ã‹ã‚‰ãªã®ã‹ã€ã‚ã‚‹ã„ã¯åˆ¥ã€…ã®ç³»çµ±ã§åŒã˜ã‚ˆã†ãªç‰¹å¾´ãŒåæ–‚çš„ã«é€²åŒ–ã—ãŸãŸã‚ãªã®ã‹ã‚’åˆ¤åˆ¥ã—ãªã‘ã‚Œã°ãªã‚‰ãªã„ã€‚
ã¨ã“ã‚ãŒã€ãƒãƒªãƒã‚ºãƒŸã¨ãƒãƒªãƒ†ãƒ³ãƒ¬ãƒƒã‚¯ã®ä¾‹ã§è¦‹ã‚‰ã‚Œã‚‹ã‚ˆã†ã«ã€å½¢æ…‹å½¢è³ªã ã‘ã«é ¼ã‚‹æ–¹æ³•ã§åæ–‚é€²åŒ–ã§ã‚ã‚‹ã¨åˆ¤å®šã™ã‚‹ã®ã¯é›£ã—ã„ã“ã¨ãŒå¤šã„ã€‚å½¢æ…‹çš„ãªç‰¹å¾´ã«ã¯ã€ç”Ÿãã¦ã„ãä¸Šã§æœ‰åˆ©ã«ãªã‚‹ã‚ˆã†ã«é©å¿œã—ãŸã‚‚ã®ãŒå¤šã„ã®ã§ã€ä¼¼ãŸç’°å¢ƒã§ä¼¼ãŸã‚ˆã†ãªç”Ÿæ´»ã‚’ã—ã¦ã„ã‚‹ç”Ÿç‰©ã®é–“ã«ã¯ä¼¼ãŸç‰¹å¾´ãŒåæ–‚çš„ã«é€²åŒ–ã—ã‚„ã™ã„ã®ã§ã‚ã‚‹ã€‚

â€2002 â€
https://www.ism.ac.jp/editsec/toukei/pdf/50-1-069.pdf
åˆ†å­ç³»çµ±æ¨¹æ¨å®šã«ãŠã‘ã‚‹ãƒ¢ãƒ‡ãƒ«ã®ãƒŸã‚¹ã‚¹ãƒšã‚·ãƒ•ã‚£ã‚±ãƒ¼ã‚·ãƒ§ãƒ³
ç³»çµ±æ¨¹ã‚’æ¨å®šã™ã‚‹ã“ã¨ã¯ï¼Œç”Ÿç‰©é€²åŒ–ã«é–¢ã‚ã‚‹ã»ã¨ã‚“ã©ã®å•é¡Œã‚’è§£ããŸã‚ã®å‡ºç™ºç‚¹ã§ã‚ã‚‹ï¼å¾“æ¥ã“ã®åˆ†é‡ã®ç ”ç©¶ã¯ï¼Œæ¯”è¼ƒå½¢æ…‹å­¦ã«ã‚ˆã‚‹ç³»çµ±æ¨¹æ¨å®šãŒä¸»æµã§ã‚ã£ãŸãŒï¼Œè¿‘å¹´ã¯ DNA ã®å¡©åŸºé…åˆ—ã‚„è›‹ç™½è³ªã®ã‚¢ãƒŸãƒé…¸é…åˆ—ã®ãƒ‡ãƒ¼ã‚¿ã‚’çµ±è¨ˆçš„ã«è§£æã™ã‚‹åˆ†å­ç³»çµ±å­¦ã«ã‚ˆã‚‹æ–¹æ³•ãŒç››ã‚“ã«ãªã£ã¦ããŸï¼æ¯”è¼ƒå½¢æ…‹å­¦ã§ã¯ç ”ç©¶è€…ã®ä¸»è¦³çš„ãªåˆ¤æ–­ãŒå…¥ã‚Šè¾¼ã‚€ä½™åœ°ãŒå¤šãï¼Œã‚µãƒ³ãƒ—ãƒ«ã§ãã‚‹å½¢æ…‹çš„å½¢è³ªã®æ•°ãŒé™ã‚‰ã‚Œã¦ã„ã‚‹ã®ã«å¯¾ã—ã¦ï¼Œåˆ†å­ç³»çµ±å­¦ã§ã¯ãƒã‚¯ãƒ†ãƒªã‚¢ã‹ã‚‰ãƒ’ãƒˆã«è‡³ã‚‹ã¾ã§ã®ã‚ã‚‰ã‚†ã‚‹ç”Ÿç‰©ã‚’å…±é€šã®å®¢è¦³çš„ãªå°ºåº¦ã§æ¯”è¼ƒã§ãï¼Œã—ã‹ã‚‚ç ”ç©¶è€…ãŒãã®æ°—ã«ãªã‚Šã•ãˆã™ã‚Œã°ï¼ˆç ”ç©¶è²»ãŒååˆ†ã«ã‚ã‚‹ã¨ã„ã†æ¡ä»¶ã¯å¿…è¦ã§ã‚ã‚‹ãŒï¼‰ï¼Œã»ã¨ã‚“ã©ã„ãã‚‰ã§ã‚‚å¤šãã®å½¢è³ªï¼ˆå®Ÿéš›ã«ã¯ã‚²ãƒãƒ å…¨ä½“ã¨ã„ã†åˆ¶é™ã¯ã‚ã‚‹ãŒï¼‰ã‚’ã‚µãƒ³ãƒ—ãƒ«ã™ã‚‹ã“ã¨ãŒå¯èƒ½ã§ã‚ã‚Šï¼Œã“ã‚Œã«ã‚ˆã£ã¦åœ°çƒä¸Šã§å®Ÿéš›ã«èµ·ã£ãŸé€²åŒ–ã®ç³»çµ±æ¨¹ã‚’å†æ§‹ç¯‰ã™ã‚‹ã“ã¨ãŒã§ãã‚‹ã¨è€ƒãˆã‚‰ã‚Œã¦ã„ã‚‹ï¼

----------
## character trait
å½¢è³ª

![](https://upload.wikimedia.org/wikipedia/commons/7/76/Autapomorphy.jpg)

https://ja.wikipedia.org/wiki/å…±æœ‰åŸå§‹å½¢è³ª
åŸå§‹å½¢è³ª/ç¥–å…ˆå½¢è³ª plesiomorphy å…±æœ‰åŸå§‹å½¢è³ª/å…±æœ‰ç¥–å…ˆå½¢è³ª symplesiomorphy or symplesiomorphic character
åŸå§‹å½¢è³ªã®æ¦‚å¿µã¯æ´¾ç”Ÿçš„ãªå½¢è³ªçŠ¶æ…‹ã¨ç¥–å…ˆçš„ãªå½¢è³ªçŠ¶æ…‹ã‚’åŒºåˆ¥ã›ãšã«ã€å˜ã«å½¢æ…‹å­¦çš„ã‚ã‚‹ã„ã¯éºä¼å­¦çš„é¡ä¼¼ã«åŸºã¥ã„ã¦ã€ç¨®ã‚’ã‚°ãƒ«ãƒ¼ãƒ”ãƒ³ã‚°ã™ã‚‹å±é™ºæ€§ã‚’ç¤ºã—ã¦ã„ã‚‹ã€‚å…±é€šç¥–å…ˆã‹ã‚‰å—ã‘ç¶™ãŒã‚ŒãŸç¥–å…ˆå½¢è³ªã¯ç³»çµ±æ¨¹ã®ã„ã‹ãªã‚‹éƒ¨åˆ†ã«ã‚‚å­˜åœ¨ã—ã†ã‚‹ã‚‚ã®ã§ã€ãã®å­˜åœ¨ã‚’ã‚‚ã£ã¦è¿‘ç¸é–¢ä¿‚ã‚’è¡¨ã™ã“ã¨ã¯ã§ããªã„[3]ã€‚

https://ja.wikipedia.org/wiki/å›ºæœ‰æ´¾ç”Ÿå½¢è³ª
autapomorphy

https://ja.wikipedia.org/wiki/å…±æœ‰æ´¾ç”Ÿå½¢è³ª
synapomorphy

https://sci-tech.ksc.kwansei.ac.jp/~tohhiro/systematics19/systematics3-19.pdf
- homology ç›¸åŒï¼šã‚ã‚‹å½¢è³ªã‚„éºä¼å­ãŒå…±é€šã®ç¥–å…ˆã«ç”±æ¥ã™ã‚‹ã“ã¨. 
- homoplasy åŒå½¢å½¢è³ªï¼šå…±é€šç¥–å…ˆã«ç”±æ¥ã—ãªã„é¡ä¼¼å½¢è³ª. å¹³è¡Œé€²åŒ– (parallelism)ã€é€²åŒ–çš„é€†è»¢(evolutionary reversal)ã€. åæ–‚ (convergence)ãŒã‚ã‚‹

----------
1:28 PM - 6 Feb 2018
https://twitter.com/kfuku0502/status/960943421211815936
Kenji Fukushima on Twitter: "ã‚¯ãƒãƒ ã‚·æ¯”è¼ƒã‚²ãƒãƒ è«–æ–‡ã‚’è¦‹è¿”ã—ã¦ã¦æ°—ã¥ã„ãŸã‘ã©ã€éºä¼å­ãƒ•ã‚¡ãƒŸãƒªãƒ¼ã‚’å…±æœ‰æ´¾ç”Ÿå½¢è³ªã«è¦‹ç«‹ã¦ã¦ç³»çµ±é€²åŒ–ã‚’è­°è«–ã—ã¦ã‚‹ã¨ã“ã‚é¢ç™½ã„ãªã€‚ Comparative genomics of the tardigrades Hypsibius dujardini and Ramazzottius varieornatus https://t.co/IOeYd4tmm6"

----------
## homology

https://en.wikipedia.org/wiki/Sequence_homology
Sequence homology is the biological homology between DNA, RNA, or protein sequences, defined in terms of shared ancestry in the evolutionary history of life.

https://ja.wikipedia.org/wiki/ç›¸åŒ#éºä¼å­ã®ç›¸åŒæ€§
éºä¼å­¦ã§ã¯ã€ç›¸åŒæ€§ã¨ã„ã†è¨€è‘‰ã¯ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚„éºä¼å­ã®å¡©åŸºé…åˆ—ãŒå…±é€šã®ç¥–å…ˆã‚’ã‚‚ã¤ã¨ãã«ç”¨ã„ã‚‰ã‚Œã‚‹ã€‚
ç›¸åŒæ€§ã¯ã‚ã‚‹ã‹ãªã„ã‹ã®ã©ã¡ã‚‰ã‹ã§ã‚ã£ã¦ã€ã€Œç›¸åŒæ€§ãŒé«˜ã„ã€ã€Œãƒ›ãƒ¢ãƒ­ã‚¸ãƒ¼ãŒä½ã„ã€ã¨ã„ã£ãŸè¡¨ç¾ã¯èª¤ã‚Šã§ã‚ã‚‹ã€‚ã“ã‚Œã¯ãŸã„ã¦ã„ã®å ´åˆã€Œé…åˆ—é¡ä¼¼æ€§ãŒé«˜ã„ã€ã€Œã‚·ãƒŸãƒ©ãƒªãƒ†ã‚£ãƒ¼ãŒä½ã„ã€ã¨è¨€ã„æ›ãˆã‚‹ã“ã¨ã§é©åˆ‡ãªè¡¨ç¾ã«ãªã‚‹[2]ã€‚DNAã‚„ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®é…åˆ—ã®é¡ä¼¼æ€§ã‚’é«˜é€Ÿã«èª¿ã¹ã‚‹ãŸã‚ã®ãƒ„ãƒ¼ãƒ«ã¨ã—ã¦ã€BLASTã€FASTAãªã©ãŒã‚ã‚‹ã€‚

https://www.fifthdimension.jp/documents/molphytextbook/datapreparation_lecture.pdf
åˆ†å­ç³»çµ±æ¨¹æ¨å®šã«é©ã—ãŸé…åˆ—ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆ
ç”°è¾ºæ™¶å²
- ç›¸åŒ = åŒä¸€ã®ç¥–å…ˆå½¢è³ªã«ç”±æ¥ã™ã‚‹
- å¤šé‡é…åˆ—æ•´åˆ— = ç›¸åŒå½¢è³ªã®åŒå®š

https://github.com/davidemms/OrthoFinder
OrthoFinder: phylogenetic orthology inference for comparative genomics

https://pubmed.ncbi.nlm.nih.gov/32657391/
Bioinformatics. 2020 Jul 1;36(Suppl_1):i219-i226. doi: 10.1093/bioinformatics/btaa468.
The ortholog conjecture revisited: the value of orthologs and paralogs in function prediction
Moses Stamboulian 1, Rafael F Guerrero 1 2, Matthew W Hahn 1 3, Predrag Radivojac 4
https://academic.oup.com/bioinformatics/article/36/Supplement_1/i219/5870499
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7355290/

10:01 PM Â· May 20, 2019
https://twitter.com/marc_rr/status/1130458465300828160
Marc RobinsonRechavi on Twitter: "10 years ago @RomainStuder and I published "How confident can we be that orthologs are similar, but paralogs differ?". We were proposing that an assumption which was taken for granted should in fact be an hypothesis to test. https://t.co/qDkBKRzs3N A thread on what happened since https://t.co/2uZ39HRh4f" / Twitter
https://pubmed.ncbi.nlm.nih.gov/19368988/
Trends Genet
. 2009 May;25(5):210-6. doi: 10.1016/j.tig.2009.03.004. Epub 2009 Apr 14.
How confident can we be that orthologs are similar, but paralogs differ?
Romain A Studer 1, Marc Robinson-Rechavi

https://pubmed.ncbi.nlm.nih.gov/26966373/
Review Genomics Insights
. 2016 Feb 25;9:17-28. doi: 10.4137/GEI.S37925. eCollection 2016.
Inferring Orthologs: Open Questions and Perspectives
Fredj Tekaia 1
https://journals.sagepub.com/doi/10.4137/GEI.S37925
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4778853/
Abstract
With the increasing number of sequenced genomes and their comparisons, the detection of orthologs is crucial for reliable functional annotation and evolutionary analyses of genes and species. Yet, the dynamic remodeling of genome content through gain, loss, transfer of genes, and segmental and whole-genome duplication hinders reliable orthology detection. Moreover, the lack of direct functional evidence and the questionable quality of some available genome sequences and annotations present additional difficulties to assess orthology. 

Xenologous genes are genes that appear falsely as orthologs because at least one of the pair is acquired via HGT from another species. Usually it is difficult to identify xenologs in pairwise genome comparisons. Xenologs and true orthologs might be distinguished in multiple genome comparisons if the origin of each gene can be identified.16


https://pubmed.ncbi.nlm.nih.gov/23552219/
Review Nat Rev Genet. 2013 May;14(5):360-6. doi: 10.1038/nrg3456. Epub 2013 Apr 4.
Functional and evolutionary implications of gene orthology
Toni GabaldÃ³n 1, Eugene V Koonin
https://www.nature.com/articles/nrg3456
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5877793/
Abstract
Orthologues and paralogues are types of homologous genes that are related by speciation or duplication, respectively. Orthologous genes are generally assumed to retain equivalent functions in different organisms and to share other key properties.

Glossary

Paralogues	Homologous genes related by duplication.
Xenologues	Homologous genes that originate from horizontal gene transfer.


https://pubmed.ncbi.nlm.nih.gov/25646426/
Proc Natl Acad Sci U S A
. 2015 Feb 17;112(7):2058-63. doi: 10.1073/pnas.1412770112. Epub 2015 Feb 2.
Phylogenomics with paralogs
Marc Hellmuth 1, Nicolas Wieseke 2, Marcus Lechner 3, Hans-Peter Lenhof 4, Martin Middendorf 2, Peter F Stadler 5
https://www.pnas.org/doi/10.1073/pnas.1412770112
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4343152/
SIGNIFICANCE
We demonstrate that the distribution of paralogs in large gene families contains in itself sufficient phylogenetic signal to infer fully resolved species phylogenies. This source of phylogenetic information is independent of information contained in orthologous sequences and is resilient against horizontal gene transfer. An important consequence is that phylogenomics data sets need not be restricted to 1:1 orthologs.

To evaluate the robustness of the species trees in response to noise in the input data, we used simulated gene families with different noise models and levels: (i) insertion and deletion of edges in the orthology graph (homologous noise), (ii) insertion of edges (orthologous noise), (iii) deletion of edges (paralogous noise), and (iv) modification of gene/species assignments (xenologous noise). 

https://pubmed.ncbi.nlm.nih.gov/16285863/
Review Annu Rev Genet
. 2005;39:309-38. doi: 10.1146/annurev.genet.39.073003.114725.
Orthologs, paralogs, and evolutionary genomics
Eugene V Koonin 1
https://www.annualreviews.org/doi/10.1146/annurev.genet.39.073003.114725
Keywords
homolog, ortholog, paralog, pseudoortholog, pseudoparalog, xenolog

2022/07/26
https://ultrabem-branch3.com/informatics/bioinformatics/homolog.html
ãƒ›ãƒ¢ãƒ­ã‚°ï¼Œã‚ªãƒ¼ã‚½ãƒ­ã‚°ï¼Œãƒ‘ãƒ©ãƒ­ã‚°ã®å®šç¾©ã¨é•ã„

https://github.com/haruosuz/ksbn/blob/master/2018/README.2018.4.md#5-7
é€²åŒ–ç³»çµ±æ¨¹ã«ã‚ˆã‚‹ãƒ›ãƒ¢ãƒ­ã‚°ãƒ»ãƒ‘ãƒ©ãƒ­ã‚°ãƒ»ã‚ªãƒ¼ã‚½ãƒ­ã‚°ã®è§£æ

2013-06-25 17:29:07
https://ameblo.jp/sakaguchikengo/entry-11560351406.html
é…µç´ ã‚„ã‚¿ãƒ³ãƒ‘ã‚¯è³ªå› å­ã®å¤šæ§˜æ€§åˆ†é¡ | ãã†ãŸã‚‰èƒ½ç„¡ã—æ•™æˆã®æ—¥è¨˜ï¼ˆå‚å£è¬™å¾ï¼‰
ã€€æ›´ã«ã“ã®å¤šæ§˜åŒ–ã‚’ã¤ã¶ã•ã«è¦³å¯Ÿã™ã‚‹ã¨ã€ã‚‚ã†ä¸€ã¤ã®ç‰¹å¾´ãŒæµ®ãå‡ºã‚‹ã€‚æ™®é€šã«è€ƒãˆã‚‹ã¨ãƒ¡ã‚«ãŒåŒã˜ãªã‚‰ã€ãã®ãƒ¡ã‚«ã‚’å¸ã‚‹é…µç´ ã‚„å› å­ã‚‚é€²åŒ–ã®éç¨‹ã§åŒã˜éºä¼å­ãŒé€²åŒ–ç™ºå±•ã—ãŸã¨è€ƒãˆãŒã¡ã§ã‚ã‚‹ã€‚ã“ã†ã„ã†åŒã˜éºä¼å­ã‹ã‚‰ç™ºå±•ã—ãŸé¡ç¸é–¢ä¿‚ã®ã‚ã‚‹éºä¼å­ç¾¤ï¼ˆé¡ç¸éºä¼å­ï¼‰ã®ã“ã¨ã‚’â€œãƒ›ãƒ¢ãƒ­ã‚°â€ï¼ˆhomologï¼‰ã¨å‘¼ã‚“ã§ã„ã‚‹ã€‚ã—ã‹ã—ã€å¿…ãšã—ã‚‚â€œãƒ›ãƒ¢ãƒ­ã‚°â€ã§ã¯ãªã„ã€‚SSB/RPAã‚’è¦‹ã‚‹ã¨ã€ç¢ºã‹ã«åŸæ ¸ç”Ÿç‰©ã®SSBã¨çœŸæ ¸ç”Ÿç‰©ã®RPAã«ã¯éºä¼å­ã®ç™ºå±•ã¨ã—ã¦æ–­çµ¶ãŒã‚ã‚‹ã€‚ã“ã®ã‚ˆã†ã«æ©Ÿèƒ½æ´»æ€§ã¯åŒã˜ã ãŒã€éºä¼å­ã®èµ·æºãŒç•°ãªã‚‹éºä¼å­åŒå£«ã¯â€œã‚¢ãƒŠãƒ­ã‚°â€ï¼ˆanalogï¼‰ã¨å‘¼ã‚“ã§ã„ã‚‹ã€‚ã“ã®ã‚ˆã†ãªè¦³ç‚¹ã‹ã‚‰é…µç´ ã‚„ã‚¿ãƒ³ãƒ‘ã‚¯è³ªå› å­ã‚’è¦‹ã‚‹ã¨ã€ã‹ãªã‚Šç´°ã‹ãã„ã‚ã„ã‚ãªã‚¿ã‚¤ãƒ—ã«åˆ†ã‹ã‚Œã‚‹ã€‚ã“ã‚Œã‚’æ­£ç¢ºã«åˆ†é¡ã™ã‚‹ãŸã‚ã«5ã¤ã«åˆ†ã‘ãŸå°‚é–€ç”¨èªãŒä½œã‚‰ã‚ŒãŸã€‚

ã€€ï¼Šâ€œãƒ›ãƒ¢ãƒ­ã‚°â€ï¼ˆhomologï¼‰ï¼›ã“ã‚Œã¯â€œãƒ‘ãƒ©ãƒ­ã‚°â€ï¼ˆparalogï¼‰ã‚„â€œã‚ªãƒ«ã‚½ãƒ­ã‚°â€ï¼ˆorthologï¼‰ã‚’å«ã‚€ç·ç§°ã§ã€å…±é€šã®ç¥–å…ˆï¼ˆèµ·æºï¼‰ã‚’æŒã¤é¡ç¸éºä¼å­ã§ã‚ã‚‹ã€‚ç›¸åŒæ€§ã®é«˜ã„éºä¼å­é…åˆ—ã‚’æŒã¤ã€‚

ã€€ï¼Šâ€œã‚¢ãƒŠãƒ­ã‚°â€ï¼ˆanalogï¼‰ï¼›é•ã†èµ·æºã‚’æŒã¤éºä¼å­ã ãŒã€æ©Ÿèƒ½ã‚„æ´»æ€§ãŒåŒã˜ã«ãªã£ãŸéºä¼å­åŒå£«ã€‚

2011-03-24
https://togetter.com/li/307635
ç›¸åŒæ€§æ¤œç´¢ã˜ã‚ƒãªãã¦ã€é…åˆ—é¡ä¼¼æ€§æ¤œç´¢ - Togetter

2009
https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/
å¹³æˆ21å¹´åº¦ã‚²ãƒãƒ ãƒªãƒ†ãƒ©ã‚·ãƒ¼è¬›åº§
åˆ†å­ç³»çµ±è§£æï¼ˆè¬›ç¾©ï¼“ï¼‰
è¬›ç¾©è³‡æ–™3 (PDF;1569 KB)
https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/h21_4_3b.pdf
ãƒ›ãƒ¢ãƒ­ã‚¯ã‚™ã¯ã€ã‚ªãƒ¼ã‚½ãƒ­ã‚¯ã‚™ã¨ãƒã‚šãƒ©ãƒ­ã‚¯ã‚™ã®2ç¨®é¡ã«åˆ†ã‘ã‚‰ã‚Œã‚‹

2007å¹´07æœˆ25æ—¥(æ°´)
http://seesaawiki.jp/w/psyberformula/d/%B0%E4%C5%C1%BB%D2%A5%DB%A5%E2%A5%ED%A5%B0
éºä¼å­ãƒ›ãƒ¢ãƒ­ã‚° - PsyberStyle - Seesaa Wikiï¼ˆã‚¦ã‚£ã‚­ï¼‰

----------
## homoplasy

http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
ãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼ï¼ˆåŒå½¢å½¢è³ªï¼‰homoplasy ï¼šå…±æœ‰ç¥–å…ˆã‹ã‚‰ã®ç”±æ¥ã«åŸºã¥ã‹ãªã„é¡ä¼¼æ€§ã€‚å¹³è¡Œç¾è±¡ã€é€†è»¢ãŒã‚ã‚‹

http://www5b.biglobe.ne.jp/~hilihili/keitou/cnvergence.html
çŸ›ç›¾ï¼šç¥–å…ˆã‹ã‚‰å—ã‘ç¶™ãŒã‚ŒãŸã®ã§ã¯ãªã„ç‰¹å¾´ãŒæ··ã˜ã£ã¦ã„ã‚‹å ´åˆ

ãã‚Œã¯åæ–‚

ã‚ã‚‰ã—ï¼šåŒå½¢å½¢è³ªï¼ˆhomoplasy ï¼šãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼ï¼‰ã£ã¦ã‚„ã¤ã ãªã€‚

https://www.keisoshobo.co.jp/book/b61590.html
ã€€1.4ã€€å½¢è³ªä¸æ•´åˆã¨ãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼ã®å•é¡Œ

http://seikilo.s1008.xrea.com/sober.html
ã€éå»ã‚’å¾©å…ƒã™ã‚‹ : æœ€ç¯€ç´„åŸç†,é€²åŒ–è«–,æ¨è«–ã€ç”¨èªé›†(æ›´æ–°5.12)

1.3å¯¾è±¡ã¨å±æ€§

ç¥–å…ˆçš„(ancestral)ã‚ã‚‹ã„ã¯åŸå§‹çš„(plesiomorphic)ãªå½¢è³ªçŠ¶æ…‹ã‚’0ã€
å­å­«çš„(derived)æ´¾ç”Ÿçš„(apomorphic)ãªå½¢è³ªå½¢æ…‹ã‚’1ã¨ç½®ãã€‚

1.4å½¢è³ªä¸æ•´åˆã¨ãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼ã®å•é¡Œ

ãƒ›ãƒ¢ãƒ­ã‚¸ãƒ¼(ç›¸åŒ:homology):ï¼’ç¨®ã®å…±æœ‰ã™ã‚‹ã‚ã‚‹ç‰¹æ€§ãŒã€ä¸€æ–¹ã®ç¨®ã‹ã‚‰ä»–æ–¹ã«å—ã‘ç¶™ãŒã‚ŒãŸã‚‚ã®ã‹ã¾ãŸã¯ãã®ç‰¹æ€§ã‚’ã‚‚ã¤å…±é€šç¥–å…ˆã‹ã‚‰å¤‰åŒ–ã™ã‚‹ã“ã¨ãªãéºä¼ã•ã‚ŒãŸã‚‚ã®ãªã‚‰ã°ã€ãã®ç¨®ã®ç‰¹æ€§ã¯ä»–ç¨®ã®ç‰¹æ€§ã¨ãƒ›ãƒ¢ãƒ­ã‚¸ãƒ¼ã®é–¢ä¿‚ã«ã‚ã‚‹ã¨å‘¼ã°ã‚Œã‚‹ã€‚

ãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼(éç›¸åŒ:homoplasy):é¡ä¼¼ç‰¹æ€§ãŒåˆ¥ã€…ã®èµ·æºã«ç”±æ¥ã™ã‚‹ã‚‚ã®ãªã‚‰ã°ã€ãã®å…±é€šç‰¹æ€§ã¯ãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼ã¨ã•ã‚Œã‚‹ã€‚

http://www.kochi-u.ac.jp/w3museum/Fish_Labo/Member/Endoh/animal_taxonomy/2013animaltaxonomyPDF/2013animaltaxonomy03.pdf
ãƒ›ãƒ¢ãƒ—ãƒ©ã‚¸ãƒ¼ãŒç”Ÿã˜ã‚‹åŸå› ã¯ï¼Œåã‚Œã‚“é€²åŒ–ï¼Œå¹³è¡Œé€²åŒ–ï¼Œå½¢è³ªã®é€†è»¢ã§ã‚ã‚‹ï¼

https://neurobiotaxis.livejournal.com/886.html
Homology, homoplasy, analogy: The concept of â€œsamenessâ€ in evolutionary biology - The Principles of Neurobiotaxis

Homology and homoplasy

The opposite of historical homology is homoplasy, defined as the structural similarity between two traits in two species without phyletic continuity â€“ which is equivalent to saying that, even though the traits are similar, the common ancestor of species A and B did not present the trait. There are three different types of homoplasy: convergence, parallelism, and reversal. 

Analogy

The wing of a bird and the wing of a bat are homoplastic as wings, but share the same function â€“ flying. 

https://wikidiff.com/analogy/homoplasy
Homoplasy vs Analogy - What's the difference? | WikiDiff

----------
## overall similarity
**å…¨ä½“çš„é¡ä¼¼åº¦**

http://www.columbia.edu/itc/envsci/hahn/w4601/class_notes/week_2.html
Overall similarity may or may not be best estimate of relationship.

http://leeswijzer.org/files/SystematicThinking.html
ã€€ã€€ã€€ã€”3ã€•å…¨ä½“çš„é¡ä¼¼åº¦ã¨ã‚¯ãƒ©ã‚¹ã‚¿ãƒªãƒ³ã‚° â€” ã„ãã¤ã‹ã®è¨ˆç®—ä¾‹ 122

http://cse.naro.affrc.go.jp/minaka/R/R-cluster2.html
è·é›¢å°ºåº¦
æ•°é‡è¡¨å½¢å­¦ã§ã¯ï¼ŒOTUé–“ã®é¡ä¼¼æ€§ã‚’è¡¨å½¢çš„ï¼ˆpheneticï¼‰ãªå…¨ä½“çš„é¡ä¼¼åº¦ï¼ˆoverall similarityï¼‰ã«ã‚ˆã£ã¦æ•°å€¤åŒ–ã™ã‚‹ã¨ã„ã†åŸºæœ¬çš„ãªå§¿å‹¢ãŒã‚ã‚‹ï¼ãã®å“²å­¦çš„ãªå‹•æ©Ÿã¥ã‘ã¯åˆ¥ã¨ã—ã¦ï¼ŒOTUã”ã¨ã«æ•°å€¤åŒ–ã•ã‚ŒãŸå½¢è³ªãƒ‡ãƒ¼ã‚¿ãŒã‚ã‚‹ã¨ãï¼ŒOTUé–“ã®é¡ä¼¼åº¦ã‚’ã©ã®ã‚ˆã†ãªå°ºåº¦ã«ã‚ˆã£ã¦æ•°å€¤åŒ–ã™ã‚Œã°ã„ã„ã®ã‹ã¨ã„ã†å•é¡Œã¯ï¼Œã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼åˆ†æã ã‘ã§ã¯ãªãï¼Œè·é›¢æ³•ã«åŸºã¥ãç³»çµ±æ¨¹æ¨å®šæ³•ã®è«–è­°ã«ã‚‚é–¢ã‚ã£ã¦ãã‚‹ï¼
ï¼ˆ8 September 2003ï¼‰

http://cse.naro.affrc.go.jp/minaka/R/R-cluster.html
ã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼åˆ†æã®å…‰ã¨é—‡
ã€€æ•°é‡è¡¨å½¢å­¦ã¯ï¼Œæ•°å€¤åŒ–ã•ã‚ŒãŸå¤šå¤‰é‡ãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ã„ã¦åˆ†é¡å¯¾è±¡ï¼ˆOTU: operational taxonomic unitï¼‰ã®é–“ã®è¿‘ã•ã‚’è·é›¢ï¼ˆå…¨ä½“çš„é¡ä¼¼åº¦overall similarityï¼‰ã¨ã—ã¦è¨ˆç®—ã—ï¼Œè·é›¢ã®è¿‘ã„ã‚‚ã®ã‚’ç¾¤ï¼ˆã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼ï¼‰ã«ã¾ã¨ã‚ã¦ã„ãã¨ã„ã†ã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼åˆ†æã®æ‰‹æ³•ã‚’ç”Ÿç‰©åˆ†é¡ä½“ç³»ã®æ§‹ç¯‰ã«é©ç”¨ã™ã‚‹ï¼æ•°é‡è¡¨å½¢å­¦è€…ã¯ï¼Œç³»çµ±ã¨ã„ã†å®Ÿè¨¼ä¸å¯èƒ½ãªæ¦‚å¿µã‚’å«ã‚€åˆ†é¡ä½“ç³»ã ã‚ã ã¨æ‰¹åˆ¤ã—ï¼Œç³»çµ±ã«ä»£ã‚ã‚‹ç”Ÿç‰©é–“ã®é–¢ä¿‚ã‚’è¡¨ç¾ã™ã‚‹å°ºåº¦ã¨ã—ã¦ï¼Œå¤šæ•°ã®å½¢è³ªã«åŸºã¥ãå…¨ä½“çš„é¡ä¼¼åº¦ã‚’ç”¨ã„ã‚ˆã†ã¨ã—ãŸï¼è‡ªç„¶åˆ†é¡ã¨ã¯ã§ãã‚‹ã ã‘å¤šãã®å½¢è³ªã‚’å…±æœ‰ã™ã‚‹åˆ†é¡ç¾¤ã‹ã‚‰æˆã‚‹ã¹ãã§ã‚ã‚Šï¼Œãã†ã„ã†åˆ†é¡ä½“ç³»ã¯ã‚ˆã‚Šå¤šãã®äºˆæ¸¬ã‚’å¯èƒ½ã«ã™ã‚‹ä¸€èˆ¬çš„ãªåˆ†é¡ä½“ç³»ï¼ˆgeneral purpose classificationï¼‰ã§ã‚ã‚‹ã¨ã„ã†ä¿¡å¿µã«é¼“èˆã•ã‚ŒãŸæ•°é‡è¡¨å½¢å­¦è€…ãŸã¡ã¯ï¼Œä¼çµ±çš„ãªé€²åŒ–åˆ†é¡å­¦ã¸ã®æ”»å‹¢ã‚’å¼·ã‚ãŸï¼ 
ï¼ˆ3 September 2003ï¼‰ 

http://seikilo.s1008.xrea.com/sober.html
ã€éå»ã‚’å¾©å…ƒã™ã‚‹ : æœ€ç¯€ç´„åŸç†,é€²åŒ–è«–,æ¨è«–ã€ç”¨èªé›†(æ›´æ–°5.12)

å…¨ä½“çš„é¡ä¼¼åº¦æ³•ã«åŸºã¥ã„ã¦ç³»çµ±æ¨è¨ˆã‚’ã™ã‚‹ãªã‚‰ã°A(BC)ã¨ã„ã†ä»®èª¬ãŒæœ€ã‚‚å¼·ãæ”¯æŒã•ã‚Œã‚‹ã€‚

å…¨ä½“çš„é¡ä¼¼åº¦æ³•ã¯å…±æœ‰æ´¾ç”Ÿå½¢è³ªã¨å…±æœ‰åŸå§‹å½¢è³ªã«åŒç­‰ã®é‡ã¿ã¥ã‘ã‚’ã™ã‚‹ã€‚
åˆ†å²å­¦çš„æœ€ç¯€ç´„æ³•ã¯å…±æœ‰æ´¾ç”Ÿå½¢è³ªã ã‘ã‚’ç³»çµ±é–¢ä¿‚ã®è¨¼æ‹ ã¨ã¿ãªã™ã€‚

å…¨ä½“çš„é¡ä¼¼åº¦æ³•ã¯ã€å…±æœ‰æ´¾ç”Ÿå½¢è³ªã¨å…±æœ‰åŸå§‹å½¢è³ªã®ã©ã¡ã‚‰ã«ã‚‚åŒã˜é‡ã¿ã‚’ä¸ãˆã‚‹ã®ã§ã€ã‚ã‚‹å½¢è³ªã®ã©ã®çŠ¶æ…‹ãŒåŸå§‹çš„ã§ã‚ã‚Šã€ã©ã‚ŒãŒæ´¾ç”Ÿçš„ã§ã‚ã‚‹ã‹ã‚’æ±ºã‚ã‚‹å¿…è¦ãŒãªã„ã€‚ä¸€æ–¹ã€åˆ†å²å­¦çš„æœ€ç¯€ç´„æ³•ã¯ã€ã“ã®ï¼’ç¨®é¡ã®å…±æœ‰åˆ¶ã«ã¤ã„ã¦ã®è¨¼æ‹ ã¨ã—ã¦ã®ä¾¡å€¤ã¯æ ¹æœ¬çš„ã«ç•°ãªã‚‹ã¨ã„ã†è¦‹è§£ã«ç«‹ã£ã¦ã„ã‚‹ã€‚ã—ãŸãŒã£ã¦ã€åˆ†å²å­¦çš„æœ€ç¯€ç´„æ³•ã‚’ç”¨ã„ã‚‹ã¨ãã«ã¯ã€å„å½¢è³ªã«ã¤ã„ã¦ã©ã®å½¢è³ªçŠ¶æ…‹ãŒåŸå§‹çš„ã§ã‚ã‚‹ã®ã‹ã‚’ç¢ºèªã—ãªã‘ã‚Œã°ãªã‚‰ãªã„ã€‚ç¨®ã‚¼ãƒ­ã®å½¢è³ªçŠ¶æ…‹ã¯ã€ç›´æ¥è¦³å¯Ÿã™ã‚Œã°ã‚ã‹ã‚‹ã‚ã‘ã§ã¯ãªãã€æ¨å®šã•ã‚Œã‚‹ã‚‚ã®ã§ã‚ã‚‹ã€‚ã“ã®æ¨å®šã€ã™ãªã‚ã¡å½¢è³ªã®ã€Œæ–¹å‘æ€§ã€ã®æ±ºå®šã‚’ã©ã®ã‚ˆã†ã«ã—ã¦è¡Œã†ã‹ã«ã¤ã„ã¦ã¯ã€ç¬¬å…­ç« ã§ãƒ»ãƒ»ãƒ»ã€‚

ï¼»ç¬¬ä¸‰ç« ï¼šå…±é€šåŸå› ã®åŸç†ï¼½

3.1 è¡¨å½¢å­¦ã®ï¼’ã¤ã®é™¥ç©½

å…¨ä½“çš„é¡ä¼¼åº¦(overall similarity)ã‹ã‚‰çœŸã®ç³»çµ±é–¢ä¿‚ãŒæ¨å®šã§ããªã„ç†ç”±
ï¼‘ï¼ãƒ›ãƒ¢ãƒ—ãƒ©ã‚·ãƒ¼
ï¼’ï¼å›ºæœ‰æ´¾ç”Ÿå½¢è³ª(autapomorphy)
å…¨ä½“çš„é¡ä¼¼åº¦ã¯æ´¾ç”Ÿçš„é¡ä¼¼æ€§ã ã‘ã§ãªãåŸå§‹çš„é¡ä¼¼æ€§ã‚’ã‚‚å«ã‚“ã§ã„ã‚‹ã€‚
ãã®ãŸã‚ã«ã“ã®äºŒã¤ã®é™¥ç©½ã«ãŠã¡ã„ã‚‹å±é™ºæ€§ãŒã‚ã‚‹ã€‚

----------
## RELL
Shimodaira-Hasegawaæ¤œå®š(SH test)

http://www.microbesonline.org/fasttree
To quickly estimate the reliability of each split in the tree, FastTree computes local support values with the Shimodaira-Hasegawa test (these are the same as PhyML 3's "SH-like local supports").
http://www.microbesonline.org/fasttree/#Support
Local support values
To quickly estimate the reliability of each split in the tree, FastTree uses the Shimodaira-Hasegawa test on the three alternate topologies (NNIs) around that split. 

2015/10/20
https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§
ç”°è¾ºæ™¶å²
7.2.1 KHãƒ»SHãƒ»AUæ¤œå®š
ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ãƒªã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã«ã‚ˆã£ã¦ã€è¤‡æ•°ã®ç³»çµ±æ¨¹é–“ã§å°¤åº¦ã®å·®ãŒæœ‰æ„ã¨è¨€ãˆã‚‹ã®ã‹å¦ã‹ã‚’èª¿ã¹ã‚‹æ–¹æ³•ãŒKishino-Hasegawaæ¤œå®š(KH test)ã§ã™(Kishino and Hasegawa, 1989)ã€‚ã—ã‹ã—ã€ã“ã®æ–¹æ³•ã§ã¯3ã¤ä»¥ä¸Šã®ç³»çµ±æ¨¹ã‚’æ¯”è¼ƒã™ã‚‹å ´åˆã«å¤šé‡æ¤œå®šã¨ãªã£ã¦ã—ã¾ã„ç¬¬1ç¨®ã®éèª¤(æœ‰æ„ãªå·®ã¯ç„¡ã„ã®ã«èª¤æ¤œå‡ºã™ã‚‹)ãŒå¢—å¤§ã—ã¦ã—ã¾ã†ãŸã‚ã€ãã®æŠ‘åˆ¶ã‚’è¡Œã†è£œæ­£ã‚’åŠ ãˆãŸã®ãŒShimodaira-Hasegawaæ¤œå®š(SH test)ã§ã™(Shimodaira and Hasegawa, 1999)ã€‚ãŸã ã—ã€ã“ã®æ–¹æ³•ã§ã¯é€†ã«ç¬¬2ç¨®ã®éèª¤(æœ‰æ„ãªå·®ãŒã‚ã‚‹ã®ã«æ¤œå‡ºã§ããªã„)ãŒå¢—å¤§ã—ã¦ã—ã¾ã„ã¾ã™ã€‚ãã“ã§ã€è¿‘ä¼¼çš„ã«ä¸åãªæ¤œå®š(approximately unbiased (AU) test)ã¯ã€ãƒãƒ«ãƒã‚¹ã‚±ãƒ¼ãƒ«ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—æ³•ã‚’ç”¨ã„ã¦ã•ã‚‰ã«é«˜åº¦ãªè£œæ­£ã‚’è¡Œã†ã“ã¨ã§ã“ã‚Œã‚’ã‚ã‚‹ç¨‹åº¦è§£æ±ºã—ã¦ã„ã¾ã™(Shimodaira, 2002)ã€‚

Dec 12, 2014
https://www.slideshare.net/astanabe/hypothesistesting-lecture
ç³»çµ±æ¨¹ã¨ç³»çµ±ä»®èª¬ã®é–¢ä¿‚ã¨ç³»çµ±ä»®èª¬é–“ã®çµ±è¨ˆçš„æ¯”è¼ƒæ–¹æ³•ã«é–¢ã™ã‚‹èª¬æ˜ã€‚
https://www.fifthdimension.jp/documents/molphytextbook/2011/hypothesistesting.pdf
https://www.fifthdimension.jp/documents/molphytextbook/hypothesistesting_lecture.pdf
ç³»çµ±æ¨¹ãƒ»ç³»çµ±ä»®èª¬ã®å¯è¦–åŒ–ã¨ç³»çµ±ä»®èª¬é–“ã®çµ±è¨ˆçš„æ¯”è¼ƒ
RELL æ³•ã‚’å¿œç”¨ã—ãŸæ¤œå®š
â— Kishino-Hasegawa æ¤œå®š
â— 2 ã¤ã®æ¨¹å½¢ã®å¯¾æ•°å°¤åº¦æ¯”ã®ãƒãƒ©ãƒ„ã‚­ã‚’ RELL æ³•ã«ã‚ˆã‚Šæ¨å®šã—ï¼Œ 0
ã‚ˆã‚Šã‚‚æœ‰æ„ã«å¤§ãã„ãªã‚‰å¸°ç„¡ä»®èª¬ ( å°¤åº¦ãŒç­‰ã—ã„ ) ã‚’æ£„å´ã™ã‚‹
â— Shimodaira-Hasegawa æ¤œå®š
â— 3 ã¤ä»¥ä¸Šã®æ¨¹å½¢ã‚’æ¯”è¼ƒã™ã‚‹ã¨ãã« FWER ã‚’çµ±åˆ¶ã—ã¦ç¬¬ 1 ç¨®ã®é
èª¤ã‚’æŠ‘åˆ¶ã™ã‚‹ ( ãã®ã‹ã‚ã‚Šï¼Œç¬¬ 2 ç¨®ã®éèª¤ãŒå¢—å¤§ )
â— è¿‘ä¼¼çš„ã«ä¸åãª (Approximately Unbiased) æ¤œå®š
â— ãƒãƒ«ãƒã‚¹ã‚±ãƒ¼ãƒ«ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ã«ã‚ˆã‚Šç¬¬ 1 ç¨®ã®éèª¤ã¨ç¬¬ 2 ç¨®ã®
éèª¤ã®ä¸¡æ–¹ã‚’æŠ‘åˆ¶ã™ã‚‹

2015å¹´10æœˆ30æ—¥(é‡‘)
https://twilog.org/leeswijzer/date-151030
#196ws Approximately Unbiased æ¤œå®šï¼šãƒãƒ«ãƒã‚¹ã‚±ãƒ¼ãƒ«ãƒ–ãƒ¼ãƒ„ã‚¹ãƒˆãƒ©ãƒƒãƒ—ã‚’ç”¨ã„ã¦ Type-I ã¨ Type-II éèª¤ã‚’ã¨ã‚‚ã«æŠ‘ãˆã‚‹æ¤œå®šæ³•ï¼
#196ws Shimodaira-Hasegawa æ¤œå®šï¼šKHæ¤œå®šã®å¤šé‡è£œæ­£ï¼
#196ws Kishino-Hasegawa æ¤œå®šï¼šRELL æ³•ã‚’ç”¨ã„ã¦å„ã‚µã‚¤ãƒˆã®å°¤åº¦ã®ãƒªã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚’ç”¨ã„ã‚Œã°ï¼Œæ¨¹å½¢ã”ã¨ã®å°¤åº¦ã®å·®ã‚’æ¤œå®šã™ã‚‹ã“ã¨ãŒã§ãã‚‹ï¼

2012-11-02
https://twilog.org/leeswijzer/date-121102
#166ws ä¸‹å¹³è‹±å¯¿ã®ãƒãƒ«ãƒã‚¹ã‚±ãƒ¼ãƒ«ãƒ–ãƒ¼ãƒ„ã‚¹ãƒˆãƒ©ãƒƒãƒ—ã‚’ç”¨ã„ãŸ Approximately Unbiased test ã¯ Type-I, II ã®éèª¤ã‚’åŒæ™‚ã«æœ€å°åŒ–ã™ã‚‹ï¼
#166ws ï¼ˆæ‰¿å‰ï¼‰å¤šé‡æ¯”è¼ƒã®è£œæ­£ã‚’ã—ãŸã‚‚ã®ãŒï¼ŒShimodaira-Hasegawa æ¤œå®šï¼
#166ws å…ƒãƒ‡ãƒ¼ã‚¿ã®ãƒªã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚ˆã‚Šã‚‚ï¼Œã‚µã‚¤ãƒˆã”ã¨ã®å°¤åº¦ã‚’ãƒªã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã™ã‚Œã°å°¤åº¦ã®åˆ†æ•£ãŒãƒ©ã‚¯ã«èª¿ã¹ã‚‰ã‚Œã‚‹ï¼ˆKishino-Hasegawa ã®RELLæ¤œå®šï¼‰ï¼

March 2013
http://sesj.kenkyuukai.jp/images/sys%5Cinformation%5C20130321180221-8212BF2A71DFD6F3C0771B1CD38A4B573F10E1A1DFB4257C3EA694B142996870.pdf
Shimodaira-Hasegawaæ¤œå®šã‚’é–‹ç™ºã—ãŸã€‚ã“ã‚Œã¯ã€Kishino-Hasegawaæ¤œå®šã§æ¬ ã‘ã¦ã„ãŸ
å¤šé‡æ¯”è¼ƒã‚’ç³»çµ±æ¨¹é¸æŠã®å•é¡Œã«æŒã¡è¾¼ã‚“ã ã‚‚ã®ã§ã‚ã‚‹ï¼ˆH. Shimodaira and M. Hasegawaï¼ˆ1999ï¼‰Mol. Biol.
Evol., 16: 1114-1116ï¼‰ã€‚

ï¼’ï¼ï¼ï¼”å¹´ï¼—æœˆ
http://stat.sys.i.kyoto-u.ac.jp/titech/multiboot-j.html
ãƒãƒ«ãƒã‚¹ã‚±ãƒ¼ãƒ«ãƒ»ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—æ³•
å›³ï¼’ï¼ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ï¼
ã€€ã¾ãšé€šå¸¸ã®ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—æ³•ã‚’é©ç”¨ã—ã¦ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ã‚’è¨ˆç®—ã—ãŸçµæœãŒå›³ï¼’ã®å„ç³»çµ±æ¨¹ã®ä¸‹ã«æ›¸ã„ã¦ã‚ã‚Šã¾ã™ï¼ã“ã‚ŒãŒã‚ã‚‹é–¾å€¤ä»¥ä¸Šã®ç³»çµ±æ¨¹ã¯çœŸå®Ÿã§ã‚ã‚‹å¯èƒ½æ€§ãŒã‚ã‚‹ã¨åˆ¤æ–­ã—ï¼Œé€†ã«ãã‚Œã‚ˆã‚Šå°ã•ã„ç³»çµ±æ¨¹ã¯çœŸå®Ÿã§ã¯ãªã„ã¨åˆ¤æ–­ã—ã¾ã™ï¼é–¾å€¤ã¨ã—ã¦ä¸€èˆ¬ã«è‰¯ãç”¨ã„ã‚‰ã‚Œã‚‹0.05ã‚’ä½¿ã†ã¨ï¼Œèµ¤ç·šã§å›²ã‚“ã ï¼’å€‹ã®ç³»çµ±æ¨¹ãŒé¸ã°ã‚Œã¾ã™ï¼æ®‹ã‚Šã®ï¼‘ï¼“å€‹ã¯çœŸå®Ÿã§ãªã„ã¨åˆ¤æ–­ã•ã‚Œã¾ã™ï¼
å›³ï¼“ï¼ä¸‹å¹³ãƒ»é•·è°·å·æ¤œå®šã«ã‚ˆã‚‹ç¢ºç‡å€¤ï¼
ã€€ä¸€èˆ¬ã«ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ã¯ç²¾åº¦ãŒä½ãï¼Œç³»çµ±æ¨¹æ¨å®šã«å¿œç”¨ã™ã‚‹ã¨ï¼Œã‚ã‚„ã¾ã£ãŸç™ºè¦‹ã«çµã³ã¤ãå¯èƒ½æ€§ãŒé«˜ã„ã“ã¨ãŒåˆ†ã‹ã£ã¦ã„ã¾ã™ (false positiveã®ç¢ºç‡ãŒå¤§ãã„ï¼‰ï¼ãã“ã§ï¼Œã“ã®èª¤ã‚Šç¢ºç‡ã‚’å°ã•ãã™ã‚‹æ‰‹æ³•ã‚’é–‹ç™ºã—ã¦ï¼ˆä¸‹å¹³ 1993, Shimodaira 1998)ï¼Œç³»çµ±æ¨¹æ¨å®šã«å¿œç”¨ã—ã¾ã—ãŸ (Shimodaira and Hasegawa 1999ï¼Œè¢«å¼•ç”¨å›æ•°)ï¼ã“ã®æ–¹æ³•ã¯çµ±è¨ˆå­¦ã§å¤ãã‹ã‚‰çŸ¥ã‚‰ã‚Œã¦ã„ã‚‹å¤šé‡æ¯”è¼ƒæ³•ã¨ã„ã†è€ƒãˆæ–¹ã‚’ãƒ¢ãƒ‡ãƒ«é¸æŠã«å¿œç”¨ã—ãŸã‚‚ã®ã§ã‚ã‚Šï¼Œç¾åœ¨ã§ã¯ä¸‹å¹³ãƒ»é•·è°·å·æ¤œå®šï¼ˆShimodaira-Hasegawa test)ã¨å‘¼ã°ã‚Œï¼Œåˆ†å­é€²åŒ–å­¦ã®æ¨™æº–æ‰‹æ³•ã®ä¸€ã¤ã¨ã—ã¦åºƒãåˆ©ç”¨ã•ã‚Œã¦ã„ã¾ã™ï¼
ã€€è¨ˆç®—ã—ãŸç¢ºç‡å€¤ã‚’å›³ï¼“ã«ç¤ºã—ã¾ã™ï¼ç¢ºç‡å€¤ãŒé–¾å€¤0.05ä»¥ä¸Šã«ãªã‚‹ç³»çµ±æ¨¹ã¯ç·‘ç·šã§å›²ã‚“ã ï¼˜å€‹ã‚ã‚Šï¼Œã‚¦ã‚µã‚®ã¨ãƒã‚¦ã‚¹ãŒè¿‘ç¸ã§ã‚ã‚‹ã¨ã„ã†å¤å…¸çš„ãªä»®èª¬ã‚‚çœŸå®Ÿã®å¯èƒ½æ€§ãŒã‚ã‚‹ã¨åˆ¤æ–­ã•ã‚Œã¾ã™ï¼ã¤ã¾ã‚Šç”Ÿç‰©å­¦çš„ãªç™ºè¦‹ã‚’ã—ãŸã¨ã¯ã¿ãªã•ã‚Œãªã„ã“ã¨ã«ãªã‚Šã¾ã™ï¼
ã€€ä¸‹å¹³ãƒ»é•·è°·å·æ¤œå®šã¯å®‰å…¨ã§ã‚ã‚Šï¼Œã‚ã‚„ã¾ã£ãŸç™ºè¦‹ã‚’ã™ã‚‹ç¢ºç‡ã¯å°ã•ã„ã®ã§ã™ãŒï¼Œä¸€æ–¹ã§æœ¬å½“ã®ç™ºè¦‹ã‚’è¦‹é€ƒã—ã¦ã—ã¾ã†å¯èƒ½æ€§ãŒé«˜ã„ã“ã¨ãŒåˆ†ã‹ã£ã¦ã„ã¾ã™ ï¼ˆfalse negativeã®ç¢ºç‡ãŒå¤§ãã„ï¼‰ï¼ã¤ã¾ã‚Šãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—æ³•ã¨ä¸‹å¹³ãƒ»é•·è°·å·æ¤œå®šã¯ã©ã¡ã‚‰ã‚‚æ¤œå®šã®ãƒã‚¤ã‚¢ã‚¹ãŒã‚ã‚Šï¼Œãã®æ–¹å‘ãŒç•°ãªã£ã¦ã„ã‚‹ã®ã§ã™ï¼

ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—æ³•ã«ã‚ˆã‚‹ã‚¯ãƒ©ã‚¹ã‚¿åˆ†æã®ãƒãƒ©ãƒ„ã‚­è©•ä¾¡ (ç‰¹é›† DNAé…åˆ—ã®çµ±è¨ˆè§£æ)
ä¸‹å¹³ è‹±å¯¿
çµ±è¨ˆæ•°ç† 50(1), 33-44, 2002
https://www.ism.ac.jp/editsec/toukei/pdf/50-1-033.pdf
åˆ†å­ç³»çµ±æ¨¹ã®è§£æã§ã¯å¯¾æ•°å°¤åº¦ã‚’æœ€å¤§ã«ã™ã‚‹æ¨¹çŠ¶å›³ã‚’é¸æŠã—ã¦ã„ã‚‹ã®ã§ï¼Œã“ã‚Œã¯çµ±è¨ˆçš„ãƒ¢ãƒ‡
ãƒ«é¸æŠã®ä¸€ä¾‹ã«ãªã‚‹ï¼ä¸‹å¹³ï¼ˆ1993, 1999ï¼‰ï¼ŒShimodairaï¼ˆ1998ï¼‰ã¯ãƒªã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚’åˆ©ç”¨ã—ã¦å¯¾
æ•°å°¤åº¦ã®å¤šé‡æ¯”è¼ƒã‚’è¡Œã„ãƒ¢ãƒ‡ãƒ«é¸æŠã®ä¿¡é ¼æ€§ã‚’ç¢ºç‡å€¤ã¨ã—ã¦è©•ä¾¡ã™ã‚‹æ–¹æ³•ã‚’ææ¡ˆã—ãŸï¼ã“ã‚Œã¯
Shimodaira-Hasegawaï¼ˆSHï¼‰testï¼ˆShimodaira and Hasegawaï¼ˆ1999ï¼‰, Goldman et al.ï¼ˆ2000ï¼‰ï¼‰ã¨
ã—ã¦åˆ†å­ç³»çµ±æ¨¹ã®åˆ†é‡ã§åˆ©ç”¨ã•ã‚Œã¯ã˜ã‚ã¦ã„ã‚‹ï¼ã“ã®åˆ†é‡ã§ã¯ Kishino-Hasegawaï¼ˆKHï¼‰test
ï¼ˆKishino and Hasegawaï¼ˆ1989ï¼‰ï¼‰ãŒãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ã«ä¸¦ã¶æ¨™æº–æ‰‹æ³•ã¨ã—ã¦åºƒãåˆ©ç”¨ã•ã‚Œ
ã¦ã„ã‚‹ãŒï¼ŒSH test ã¯ KH test ã§è¦‹è½ã¨ã•ã‚Œã¦ã„ãŸã€Œé¸æŠãƒã‚¤ã‚¢ã‚¹ã€ã‚’å¤šé‡æ¯”è¼ƒæ³•ã§è£œæ­£ã—ãŸã‚‚
ã®ã§ã‚ã‚‹ï¼

----------

## selective inference

https://twitter.com/search?q=Selective%20inference%20lang%3Aja

2018
https://www.jst.go.jp/kisoken/crest/evaluation/nenpou/h29/JST_1111081_15656320_2017_PYR.pdf
Selective Inference ã¨ã¯ï¼Œãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ã„ã¦é¸æŠã•ã‚ŒãŸä»®èª¬ã‚’åŒä¸€ã®ãƒ‡ãƒ¼ã‚¿ã§è©•
ä¾¡ã™ã‚‹éš›ã«ï¼Œé¸æŠãƒã‚¤ã‚¢ã‚¹ã‚’é™¤å»ã™ã‚‹ãŸã‚ï¼Œä»®èª¬ã‚’é¸æŠã™ã‚‹ã‚¤ãƒ™ãƒ³ãƒˆã®æ¡ä»¶ä»˜åˆ†å¸ƒã«ã‚ˆã‚Šçµ±è¨ˆçš„ä»®
èª¬æ¤œå®šã‚’è¡Œã†ã‚‚ã®ã§ã‚ã‚‹ï¼

2016-02-28
http://ryamada22.hatenablog.jp/entry/20160228/1456616097
ã±ã‚‰ã±ã‚‰ã‚ãã‚‹selective inferenceè«–æ–‡ - ryamadaã®éºä¼å­¦ãƒ»éºä¼çµ±è¨ˆå­¦ãƒ¡ãƒ¢
Selective inferenceã¯ã€Œãƒ‡ãƒ¼ã‚¿ã‚’çœºã‚ã‚‹ã“ã¨ã§ã€èª¬æ˜å¤‰æ•°ã‚’çµã‚Šã“ã¿(selectionã—)ã€ãã®ã†ãˆã§çµã‚Šè¾¼ã‚“ã å¤‰æ•°ã«ã¤ã„ã¦æ¨å®š(inference)ã™ã‚‹ã“ã¨ã«ã™ã‚‹ãŒã€ãã®ã¨ãselectionãŒinferenceã«å½±éŸ¿ã™ã‚‹ã®ã§ã€ã©ã†ã™ã‚‹ã®ãŒã‚ˆã„ã®ã‹ã‚’è€ƒãˆã‚ˆã†ã€ã¨è¨€ã£ã¦ã‚ˆã„ã§ã—ã‚‡ã†ã€‚

----------

## dNdS

2018/10/25
https://ultrabem.com/other_topics/genetics/synonymous_substitutions.html
åŒç¾©ç½®æ›ã¨éåŒç¾©ç½®æ›: è¨ˆç®—æ–¹æ³•ã€æ„å‘³ã€è«–æ–‡ã§ã®ç¤ºã—æ–¹

http://mikuhatsune.hatenadiary.com/entry/20130621/1371803603
Ka/Ks (dN/dS) ã®è¨ˆç®— - é©šç•°ã®ã‚¢ãƒ‹ãƒ²ã‚¿ç¤¾ä¼šå¾©å¸°ã®äºˆå‚™


http://www.fish-evol.org/dNdS_ji.html
dN/dS æ¤œå®š
2018 å¹´ 4 æœˆ 12 æ—¥ã€€äº•ä¸Š æ½¤ï¼Œç±³æ¾¤ éš†å¼˜
ã“ã®ã‚µã‚¤ãƒˆã§ã¯ dN/dS å€¤ã‚’æ¨å®šã™ã‚‹ã“ã¨ã§ï¼Œã‚¿ãƒ³ãƒ‘ã‚¯è³ªã‚³ãƒ¼ãƒ‡ã‚£ãƒ³ã‚°éºä¼å­ (DNA é…åˆ—) ã«åƒã„ãŸæ­£ã®è‡ªç„¶é¸æŠã‚’æ¤œå‡ºã™ã‚‹è§£æ (dN/dS æ¤œå®š) ã‚’ç´¹ä»‹ã—ã¾ã™ï¼PAML ã«å«ã¾ã‚Œã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ  codeml ã‚’ä½¿ã„ã¾ã™ï¼ãƒãƒ¼ã‚¸ãƒ§ãƒ³ã¯ paml4.7a ã‚’ä½¿ã£ã¦ã„ã¾ã™ï¼

----------
## arms race
https://ja.wikipedia.org/wiki/é€²åŒ–çš„è»æ‹¡ç«¶èµ°
é€²åŒ–çš„è»æ‹¡ç«¶èµ°ï¼ˆã¾ãŸã¯ã€-äº‰ã€evolutionary arms raceï¼‰ã¨ã¯ç”Ÿç‰©ã®é€²åŒ–ã«ãŠã„ã¦ã€ã‚ã‚‹é©å¿œã¨ãã‚Œã«å¯¾ã™ã‚‹å¯¾æŠ—é©å¿œãŒç«¶ã†ã‚ˆã†ã«ç™ºé”ã™ã‚‹ï¼ˆã‹ã®ã‚ˆã†ã«ã¿ãˆã‚‹ï¼‰å…±é€²åŒ–ãƒ—ãƒ­ã‚»ã‚¹ã®ä¸€ç¨®ã€‚

å…±é€²åŒ–ã¯å¿…ãšã—ã‚‚é€²åŒ–çš„è»æ‹¡ã‚’ä¿ƒã™ã‚ã‘ã§ã¯ãªã„ã€‚ä¾‹ãˆã°ç›¸åˆ©å…±ç”Ÿã¯å”èª¿çš„ãªé©å¿œã‚’äºŒã¤ã®ç¨®ã®é–“ã«å¼•ãèµ·ã“ã™ã‹ã‚‚çŸ¥ã‚Œãªã„ã€‚ã‚ã‚‹ç¨®ã®èŠ±ã¯ç´«å¤–ç·šè‰²ã®æ¨¡æ§˜ã§ãƒŸãƒ„ãƒãƒã‚’èŠ±ã®ä¸­å¿ƒã«èª˜å°ã—ã€å—ç²‰ã‚’ä¿ƒã™ã€‚ã¾ãŸå…±é€²åŒ–ã¯ä¸€èˆ¬çš„ãªå®šç¾©ã§ã¯ç•°ç¨®é–“ã«èµ·ãã‚‹ã‚‚ã®ã‚’æŒ‡ã™ã€‚é›Œé›„é–“ã®å¯¾ç«‹ã®ã‚ˆã†ãªåŒä¸€ç¨®å†…ã®è»æ‹¡ç«¶èµ°ã¯é™¤å¤–ã•ã‚Œã‚‹ã€‚èº«è¿‘ãªé€²åŒ–çš„è»æ‹¡ã¯äººé–“ã¨å¾®ç”Ÿç‰©ã®é–“ã§è¡Œã‚ã‚Œã‚‹ã€‚æŠ—ç”Ÿç‰©è³ªã¯å¾®ç”Ÿç‰©ã«é¸æŠåœ§ã‚’åŠ ãˆã€å¾®ç”Ÿç‰©ã¯è–¬å‰¤è€æ€§ã‚’é€²åŒ–ã•ã›ã‚‹ã€‚

----------
## adaptation
- https://ja.wikipedia.org/wiki/é©å¿œ
- https://kotobank.jp/word/é©å¿œ-100764

é©å¿œä¸»ç¾©
æ‰¹åˆ¤çš„ãªæ„å‘³ã§ã¯ã€ååˆ†ãªè¨¼æ‹ ç„¡ã—ã«ç”Ÿç‰©ã®ç‰¹å¾´ã‚’é©å¿œã§ã‚ã‚‹ã¨çµè«–ã™ã‚‹ç ”ç©¶ã‚„ç ”ç©¶è€…ã®ã“ã¨ã‚’æŒ‡ã™ã“ã¨ãŒå¤šã„ã€‚

é©å¿œã¨ç›®çš„è«–
é€šå¸¸ã€ç”Ÿç‰©å­¦è€…ã¯é©å¿œçš„ãªå½¢è³ªã‚’ã€Œâ—‹â—‹ã®ãŸã‚ã®å½¢è³ªã€ã¨å‘¼ã¶ã€‚ã“ã®ãŸã‚ã«ã€ã—ã°ã—ã°é€²åŒ–ã«ã¯æ„å›³ã‚„æ–¹å‘æ€§ãŒã‚ã‚‹ã€ã¾ãŸã¯ç›®çš„è«–ã‚’å«æ„ã—ã¦ã„ã‚‹ã¨èª¤è§£ã•ã‚Œã‚‹ã€‚ã“ã®ã‚ˆã†ãªè¡¨ç¾ã¯ã€Œè‡ªç„¶é¸æŠã«ã‚ˆã£ã¦ãã®å½¢è³ªã«å½±éŸ¿ã‚’ä¸ãˆã‚‹ä¸€é€£ã®éºä¼çš„å¤‰ç•°ãŒè“„ç©ã•ã‚Œã€ãã®å½¢è³ªãŒå½¢æˆã•ã‚ŒãŸã€ã¨è¨€ã†è¡¨ç¾ã®çŸ­ç¸®å½¢ã§ã‚ã‚‹ã€‚
ç¾åœ¨ã§ã¯é©å¿œã¯è‡ªç„¶é¸æŠã®çµæœã§ã€å—å‹•çš„ãªã‚‚ã®ã§ã‚ã‚Šã€ç”Ÿç‰©ãŒä¸»ä½“çš„ã«é©å¿œã—ã‚ˆã†ã¨ã—ã¦èµ·ãã‚‹ã‚‚ã®ã§ã¯ãªã„ã¨è€ƒãˆã‚‰ã‚Œã¦ã„ã‚‹ã€‚

https://ja.wikipedia.org/wiki/å‰é©å¿œ
preadaptation

2:15 PM Â· Oct 9, 2022
https://twitter.com/KawataMasakado/status/1578977453623545856
Masakado Kawata on Twitter: "é€²åŒ–å­¦ã«ãŠã‘ã‚‹ã€Œé©å¿œã€ã¤ã„ã¦æ”¹ã‚ã¦è§£èª¬ã—ã¾ã—ãŸã€‚å®šç¾©ã‚’æ˜ç¢ºã«ã™ã‚‹ã¨ã„ã†ã‚ˆã‚Šã‚‚ã€ã€Œé©å¿œã€ã‚’å®šç¾©ã™ã‚‹ã“ã¨ã®é›£ã—ã•ã«ã¤ã„ã¦è€ƒå¯Ÿã—ã¾ã—ãŸã€‚ é€²åŒ–ã«ãŠã‘ã‚‹ã€Œé©å¿œã€ã¨ã„ã†è¨€è‘‰ã‚’ã‚ãã£ã¦ï½œæ²³ç”° é›…åœ­ @KawataMasakado #note #æœ€è¿‘ã®å­¦ã³ https://t.co/decOO7lLRS" / Twitter
https://note.com/masakadokawata/n/n2dfc4217831e

8:06 AM Â· May 11, 2020
https://twitter.com/mutselbalance/status/1259620836740263937
ã‚‰ã‚€ on X: "ä¸­å­¦ç”Ÿã‹ã‚‰å§‹ã‚ã‚‹é©å¿œé€²åŒ–ç†è«– - Life is Beautiful @academist_cf ã¨ã®å…±å‚¬ #æ•°ç†ã§èª­ã¿è§£ãç§‘å­¦ã®ä¸–ç•Œ ã®ãƒ•ã‚©ãƒ­ãƒ¼ã‚¢ãƒƒãƒ—ç¬¬ä¸€å¼¾ã§ã™ã€‚ https://t.co/rELMhF4T2c" / X

https://lambtani.hatenablog.jp/entry/2020/05/10/200459
ä¸­å­¦ç”Ÿã‹ã‚‰å§‹ã‚ã‚‹é©å¿œé€²åŒ–ç†è«– - Life is Beautiful
1. ç”Ÿç‰©ã®é©å¿œé€²åŒ–ã®è€ƒãˆæ–¹
1.1 éºä¼ã™ã‚‹æ€§è³ªãŒé€²åŒ–ã™ã‚‹åŸç†
éºä¼å½¢è³ª
è‡ªç„¶æ·˜æ±°ã«ã‚ˆã‚‹é€²åŒ–
ï¼ˆé©å¿œé€²åŒ–
1.2 éºä¼å­ã¯ã©ã“ã‹ã‚‰ãã‚‹ã®ã‹ï¼šçªç„¶å¤‰ç•°
1.3 é©å¿œã¨ã„ã†è€ƒãˆæ–¹ãŒã€Œç”Ÿç‰©ã‚’ç†è§£ã™ã‚‹å–¶ã¿ã€ã‚’å­¦å•ãŸã‚‰ã—ã‚ãŸ
1.4 ã„ã‚ã„ã‚ãªã€Œãªãœã€ãŒã‚ã£ã¦ã„ã„
Nothing in Biology Makes Sense Except in the Light of Evolution -- ç›´è¨³ã™ã‚‹ã¨ã€ã€Œç”Ÿç‰©å­¦ã«ãŠã„ã¦ã¯ã€é€²åŒ–ã‚’è€ƒæ…®ã—ã¦ã“ãã€ãã®æ„ç¾©ãŒã‚ã‚‹ã€

https://lambtani.hatenablog.jp/entry/2020/05/19/233210
ä¸­å­¦ç”Ÿã‹ã‚‰å§‹ã‚ã‚‹é©å¿œé€²åŒ–ç†è«–2ï¼šã‚ˆãã‚ã‚‹èª¤è§£ - Life is Beautiful
2. è‡ªç„¶æ·˜æ±°ã«ã‚ˆã‚‹é©å¿œé€²åŒ–ã«ã¾ã¤ã‚ã‚‹ ã‚ˆãã‚ã‚‹è³ªå•ãƒ»èª¤è§£
2.1ã€Œé †å¿œã€ã¯äººç”Ÿã®ä¸€éƒ¨ã€‚ã ãŒè¨€è‘‰ã¯å¤‰åŒ–ï¼ˆé€²åŒ–ï¼‰ã™ã‚‹
2.2 ç¨®ã®ãŸã‚ã®é€²åŒ–ã¯èµ·ã“ã‚Šã¾ã›ã‚“ã€‚
2.3 é›†å›£ãƒ¬ãƒ™ãƒ«ã§è‡ªç„¶æ·˜æ±°ãŒä½œç”¨ã™ã‚‹ã“ã¨ã¯ã‚ã‚Šã¾ã™ã€‚
2.4 é©å¿œé€²åŒ–ä»¥å¤–ã«ã‚‚ã€é€²åŒ–ã¯èµ·ã“ã‚Šã†ã‚‹
ã‚ˆã‚Šä¸€èˆ¬ã«ã€é€²åŒ–ã‚’èµ·ã“ã™è¦å› ã«ã¯
è‡ªç„¶æ·˜æ±°
çªç„¶å¤‰ç•°
éºä¼çš„ãªæµ®å‹•ï¼ˆå€‹ä½“æ•°ãŒå°ã•ã„ã“ã¨ã§ã€ç¢ºç‡çš„ãªå¤‰åŒ–ã®åŠ¹æœãŒã€ç›¸å¯¾çš„ã«å¤§ãããªã‚‹ï¼‰
ç§»å‹•åˆ†æ•£ã€€ã€€ï¼ˆé›†å›£é–“ã§ã€å€‹ä½“ã®è¡Œãæ¥ãŒã‚ã‚‹ã“ã¨ã§ã€é›†å›£ã®å¹³å‡çš„ãªæ€§è³ªã®è¦³æ¸¬å€¤ãŒå¤‰åŒ–ã™ã‚‹ï¼‰
ãŒã‚ã‚Šã€ã©ã‚Œã‚‚é©å¿œé€²åŒ–ã¨ä¸­ç«‹é€²åŒ–ã«ã¨ã£ã¦é‡è¦ãªè¦ç´ ã§ã™ã€‚
2.6 é€€åŒ–ã‚‚é€²åŒ–ã®ä¸€éƒ¨
2.7 å¤§äº‹ãªã“ã¨ï¼šã€Œè‰¯ã„ã€ã€ã€Œå„ªã‚Œã¦ã„ã‚‹ã€ã€ã¯è‡ªç„¶ã«ã¯å­˜åœ¨ã—ãªã„ã€‚

https://ja.wikipedia.org/wiki/ã‚¤ãƒ³ãƒ†ãƒªã‚¸ã‚§ãƒ³ãƒˆãƒ»ãƒ‡ã‚¶ã‚¤ãƒ³
ã€å®‡å®™ãƒ»è‡ªç„¶ç•Œã«èµ·ã“ã£ã¦ã„ã‚‹ã“ã¨ã¯æ©Ÿæ¢°çš„ãƒ»éäººç§°çš„ãªè‡ªç„¶çš„è¦å› ã ã‘ã§ã¯ã™ã¹ã¦ã®èª¬æ˜ã¯ã§ããšã€ãã“ã«ã¯ã€Œãƒ‡ã‚¶ã‚¤ãƒ³ã€ã™ãªã‚ã¡æ§‹æƒ³ã€æ„å›³ã€æ„å¿—ã€ç›®çš„ã¨ã„ã£ãŸã‚‚ã®ãŒåƒã„ã¦ã„ã‚‹ã“ã¨ã‚’ç§‘å­¦ã¨ã—ã¦èªã‚ã‚ˆã†ã€ã¨ã„ã†ç†è«–ãƒ»é‹å‹•ã§ã‚ã‚‹ã€‚è¿‘å¹´ã®ã‚¢ãƒ¡ãƒªã‚«åˆè¡†å›½ã§å§‹ã¾ã£ãŸã‚‚ã®ã§ã‚ã‚Šã€1990å¹´ä»£ã«ã‚¢ãƒ¡ãƒªã‚«ã®åé€²åŒ–è«–å›£ä½“ã€ä¸€éƒ¨ã®ç§‘å­¦è€…ãªã©ãŒæå”±ã—å§‹ã‚ãŸã‚‚ã®ã§ã‚ã‚‹ã€‚

----------
## 
ç”Ÿç‰©ã®åˆç›®çš„æ€§

10æœˆ 24, 2014
https://junotkja.wordpress.com/2014/10/
10æœˆ | 2014 | å¤§å¡šæ·³ã®ã‚µã‚¤ãƒˆè·¡åœ°
ç¾ä»£ã®æˆ‘ã€…ã¯ã€ãã†ã—ãŸåˆç›®çš„æ€§ãŒã€è‡ªç„¶é¸æŠã‚’é€šã˜ã¦å®Ÿéš›ã®é€²åŒ–ã®åŸå› ã«ãªã‚Šã†ã‚‹ã€ã¨ã„ã†ã“ã¨ã‚’çŸ¥ã£ã¦ã„ã¾ã™ã€‚ã—ã‹ã—ãã‚Œã¯ä¸€ã¤ã®å¯èƒ½æ€§ãƒ»ä»®èª¬ã«éãã¾ã›ã‚“ã€‚å®Ÿã®ã¨ã“ã‚ãã†ã—ãŸåˆç›®çš„æ€§ã¯ãŸã¾ãŸã¾ç”Ÿã˜ãŸå‰¯ç”£ç‰©ã§ã‚ã£ã¦ã€å®Ÿéš›ã®é€²åŒ–ã®åŸå› ã§ã¯ãªã‹ã£ãŸã‹ã‚‚ã—ã‚Œãªã„ã€‚


2013å¹´09æœˆ25æ—¥ 22:30
https://mixi.jp/view_bbs.pl?comm_id=82248&id=74922095
[mixi]åˆç›®çš„æ€§ã¸ã®æŒ‘æˆ¦ - é€²åŒ–ç”Ÿç‰©å­¦ | mixiã‚³ãƒŸãƒ¥ãƒ‹ãƒ†ã‚£

https://kagakubar.com/evolution/05.html
é€²åŒ–ã®æ­´å²ï½œç§‘å­¦ãƒãƒ¼
ç¬¬5è©±
å¶ç„¶æ€§ã®é‡è¦–
æ–‡ã¨å†™çœŸã€€é•·è°·å·æ”¿ç¾
â—ãƒ€ãƒ¼ã‚¦ã‚£ãƒ³ã®ã€Œè‡ªç„¶é¸æŠèª¬ã€
ãƒ€ãƒ¼ã‚¦ã‚£ãƒ³ã®ã€Œè‡ªç„¶é¸æŠèª¬ã€ã¯ã€ã“ã®ã‚ˆã†ãªãƒ‡ã‚¶ã‚¤ãƒ³è«–ã‚„ç›®çš„è«–ã‚’å¦å®šã™ã‚‹ã€‚å½¼ã¯ã€é›†å›£ä¸­ã«å¶ç„¶ã«ç”Ÿãšã‚‹ãŸãã•ã‚“ã®å¤‰ç•°ã®ãªã‹ã‹ã‚‰ã•ã¾ã–ã¾ãªç’°å¢ƒã«ã‚ˆã‚Šã‚ˆãé©å¿œã—ãŸå€‹ä½“ãŒç”Ÿãæ®‹ã‚‹ã“ã¨ã«ã‚ˆã£ã¦ã€é€²åŒ–ãŒèµ·ã‚‹ã¨ã—ãŸã€‚å¤šæ§˜ãªç”Ÿç‰©ãŒé€²åŒ–ã™ã‚‹ä»•çµ„ã¿ã¨ã—ã¦è‡ªç„¶é¸æŠèª¬ã‚’è€ƒãˆãŸã®ã§ã‚ã£ãŸã€‚ç›®çš„è«–ã«é ¼ã‚‰ãªãã¦ã‚‚ã€ãƒ©ãƒ³ãƒ€ãƒ ã«èµ·ãã‚‹å¤‰ç•°ã®ãªã‹ã‹ã‚‰ã‚ˆã‚Šé©å¿œã—ãŸå€‹ä½“ãŒé¸æŠã•ã‚Œã‚‹ã“ã¨ã«ã‚ˆã£ã¦ã€çµæœçš„ã«é€²åŒ–ã«æ–¹å‘æ€§ã‚’ä¸ãˆã‚‰ã‚Œã‚‹ã®ã ã€‚ç„¡ç›®çš„ãªå¶ç„¶çš„ãªç¾è±¡ã«ã‚ˆã£ã¦ã€ç”Ÿç‰©ã®åˆç›®çš„æ€§ãŒã‚‚ãŸã‚‰ã•ã‚Œã‚‹ã¨ã„ã†ã“ã®è€ƒãˆã¯ã€ç”»æœŸçš„ãªã‚‚ã®ã§ã‚ã£ãŸã€‚

http://fs1.law.keio.ac.jp/~popper/v3n1yamane.html
é€²åŒ–è«–çš„èªè­˜è«–ã¨è¡Œå‹•çš„ç„¡æ„å‘³
ï¼ã€€ãƒªãƒ¼ãƒ‰ãƒ«ã®äººé–“ç†è§£ã€€ï¼
å±±æ ¹ã€€æ­£æ°—
ãã®ã‚ˆã†ã«ãµã‚‹ã¾ã†ç”Ÿç‰©ã®ã‚·ã‚¹ãƒ†ãƒ ã‚’ã€æ“¬åˆç†çš„è£…ç½®ã¨ã‚ˆã³ã€ãã®ç”Ÿç‰©ãŒéå»ã«é¸åˆ¥ã•ã‚ŒãŸæ¡ä»¶ãŒã“ã‚“ã«ã¡ã‚‚ç¶šã„ã¦ã„ã‚‹ã¨ã™ã‚Œã°ã€ãã‚Œã¯ã¾ã•ã«åˆç›®çš„çš„ã¨ã¿ã‚‰ã‚Œã‚‹ã‚ã‘ã§ã‚ã‚‹ã€‚ç”Ÿç‰©ã®ã‚‚ã¤åˆç›®çš„æ€§ã‚’ã“ã®ã‚ˆã†ã«ã¨ã‚‰ãˆã‚‹ç«‹å ´ã«ãŸã„ã—ã¦ã€è‘—è€…ã¯ç§‘å­¦çš„ç›®çš„è«–ï¼ˆãƒ†ãƒ¬ã‚ªãƒãƒŸãƒ¼ï¼‰ã¨ã„ã†è¡¨ç¾ã‚’ã‚‚ã¡ã„ã¦ã„ã‚‹ã€‚ã•ã¦ã€å•é¡Œã¯ã“ã®å…ˆã§ã‚ã‚‹ã€‚
ç”Ÿç‰©å€‹ä½“ãŒããªãˆã¦ã„ã‚‹åˆç›®çš„æ€§ã‚’ç–‘ã†è€…ã¯ã„ã¾ã„ã€‚
ã¨ã“ã‚ãŒã€ã“ã®æœ¬ã®è‘—è€…ã¯ã€åˆç›®çš„æ€§ã®è«–ç†ã‚’ã„ã¨ã‚‚ç„¡æ‰¹åˆ¤ã«ç¤¾ä¼šã‚„ç¨®ã«ã¾ã§æ‹¡å¼µã™ã‚‹ã€‚
åˆç›®çš„æ€§ã®åŸå› ã‚’ç¨®ã®ç¶­æŒã«ã‚‚ã¨ã‚ã‚‹ã“ã¨ãŒã€ã„ã‹ã«èª¤ã£ãŸçµè«–ã‚’å°ãã‹ã‚’ã¿ã¦ã¿ã‚ˆã†ã€‚

----------
## neutral theory
Neutral theory of molecular evolution

- https://ja.wikipedia.org/wiki/ä¸­ç«‹é€²åŒ–èª¬
- https://kotobank.jp/word/ä¸­ç«‹èª¬-185536

https://www.brh.co.jp/research/formerlab/miyata/2005/post_000003.html
ãƒ‘ãƒ©ãƒ€ã‚¤ãƒ ã‚·ãƒ•ãƒˆï¼šåˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬- å®®ç”°ã€€éš†ã®é€²åŒ–ã®è©± - JTç”Ÿå‘½èªŒç ”ç©¶é¤¨
![](https://www.brh.co.jp/_old/imgs/katari/shinka/09_zu03.gif)

2003
https://www.nig.ac.jp/museum/evolution/01_c.html
åˆ†å­é€²åŒ–ã®ä¸­ç«‹èª¬ã€€ï½æœ¨æ‘è³‡ç”Ÿã¨ä¸­ç«‹èª¬

----------
## 2018

## 2016
Mar 11, 2016
https://www.slideshare.net/takahironishimu/ss-59418690
ãƒã‚±ãƒ¢ãƒ³ç³»çµ±æ¨¹ã‹ã„ã¦ã¿ãŸ

## 2010
2010-02-24
https://www3.atwiki.jp/cloud9science/pages/175.html
é€²åŒ–ã®ç³»çµ±æ¨¹ - cloud9science @Wiki - ã‚¢ãƒƒãƒˆã‚¦ã‚£ã‚­

----------
## abduction
ã‚¢ãƒ–ãƒ€ã‚¯ã‚·ãƒ§ãƒ³
çœŸå®Ÿ

https://www.yodosha.co.jp/smart-lab-life/statics_pitfalls/statics_pitfalls03.html
çµ±è¨ˆã®è½ã¨ã—ç©´ã¨èœ˜è››ã®ç³¸
è‘—ï¼ä¸‰ä¸­ä¿¡å®
ç¬¬3å› ãƒ‡ãƒ¼ã‚¿ã®ãµã‚‹ã¾ã„ã‚’ãƒ¢ãƒ‡ãƒ«åŒ–ã™ã‚‹ ï¼ˆ2017/11/24å…¬é–‹ï¼‰
ã‚¢ãƒ–ãƒ€ã‚¯ã‚·ãƒ§ãƒ³ã®è¦ç‚¹ã¯ï¼Œé¸ã³å‡ºã•ã‚ŒãŸâ€œãƒ™ã‚¹ãƒˆâ€ã®ä»®èª¬ãŒå¿…ãšã—ã‚‚æœ€çµ‚çš„ãªâ€œçœŸå®Ÿâ€ã§ã‚ã‚‹å¿…è¦ã¯ãªã„ã“ã¨ã§ã™ï¼
ãƒ¢ãƒ‡ãƒ«ã¨æœ¬è³ªï¼šæ—¢çŸ¥ã‹ã‚‰æœªçŸ¥ã¸ã®ã‚¢ãƒ–ãƒ€ã‚¯ã‚·ãƒ§ãƒ³
ã—ã‹ã—ï¼Œãƒ‡ãƒ¼ã‚¿è§£æã®ç¾å ´ã§ã¯å¤‰é‡é–“ã®é–¢ä¿‚ã‚’æ”¯é…ã™ã‚‹â€œçœŸâ€ã®å¼ã¯ã„ã¤ã¾ã§ã‚‚æœªçŸ¥ã®ã¾ã¾ç¾è±¡ã®èƒŒå¾Œã«éš ã‚Œã¦ã„ã¾ã™ï¼
ã‚¢ãƒ–ãƒ€ã‚¯ã‚·ãƒ§ãƒ³ã¯ã€ŒçœŸå®Ÿã€ã‚’è¨€ã„å½“ã¦ã‚‹äºˆè¨€ã‚’è¡Œãªã†ã®ã§ã¯ãªãï¼Œè¦³å¯Ÿãƒ‡ãƒ¼ã‚¿ã‚’èª¬æ˜ã™ã‚‹ã«ã¯é¸æŠè‚¢ä¸­ã®ã©ã®ãƒ¢ãƒ‡ãƒ«ãŒã€Œã‚ˆã‚Šã‚ˆã„ã€ã‹ã‚’æ¯”è¼ƒæ¤œè¨ã™ã‚‹æ¨è«–ä½œæ¥­ã§ã™ï¼ã“ã“ã§ã„ã†ã€Œã‚ˆã‚Šã‚ˆã„ãƒ¢ãƒ‡ãƒ«ã€ã¨ã¯ã€Œã‚ˆã‚ŠçœŸå®Ÿã«è¿‘ã„ãƒ¢ãƒ‡ãƒ«ã€ã¨ã¯ã‹ãã‚Šã¾ã›ã‚“ï¼

https://twitter.com/leeswijzer/status/536990181241671680
#TodaiStat ã€å›ç­”ã€‘ã—ã‹ã—ï¼Œçµ±è¨ˆå­¦ã§ã®ã‚¢ãƒ–ãƒ€ã‚¯ã‚·ãƒ§ãƒ³ã¯ï¼Œãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ãå¸°ç„¡ä»®èª¬ã¨å¯¾ç«‹ä»®èª¬ã®çµ¶å¯¾çš„ãƒ©ãƒ³ã‚­ãƒ³ã‚°ï¼ˆçœŸå½ï¼‰ã§ã¯ãªãï¼Œå¯¾ç«‹ä»®èª¬é–“ã®ç›¸å¯¾çš„ãƒ©ãƒ³ã‚­ãƒ³ã‚°ï¼ˆæ”¯æŒã•ã‚Œã‚‹ã‹ã—ãªã„ã‹ï¼‰ã«éãã¾ã›ã‚“ï¼
4:10 PM - 24 Nov 2014

https://sakstyle.hatenadiary.jp/entry/20140419/p1
2014-04-19
ã‚¨ãƒªã‚ªãƒƒãƒˆãƒ»ã‚½ãƒ¼ãƒãƒ¼ã€éå»ã‚’å¾©å…ƒã™ã‚‹ã€ä¸‰ä¸­ä¿¡å®è¨³
ãã“ã§ã€å…¨ä½“çš„é¡ä¼¼æ€§ã‹ã‚‰ã™ã‚‹ã¨ã€ï¼¡ï¼ˆï¼¢ï¼£ï¼‰ãŒçœŸã®ç³»çµ±ä»®èª¬ã¨ã„ã†ã“ã¨ã«ãªã‚‹ï¼ˆï¼¢ã¨ï¼£ãŒã‚ˆã‚Šè¿‘ã„åˆ†é¡ç¾¤ï¼‰ã€‚

http://www.jppa.or.jp/shiryokan/pdf/63_03_66.pdf
åˆ†å­ç³»çµ±å­¦ï¼šæœ€è¿‘ã®é€²æ­©ã¨ä»Šå¾Œã®å±•æœ›
ã—ã‹ã—ï¼Œç³»çµ±æ¨å®šã¨ã¯ãã‚‚ãã‚‚ã€ŒçœŸå®Ÿã€ã®ç³»çµ±æ¨¹ã‚’ç™ºè¦‹ã™ã‚‹ã“ã¨ã‚’ç›®çš„ã¨ã¯ã—ã¦ã„ãªã„ã€‚æˆ‘ã€…ã¯ï¼Œãƒ‡ãƒ¼ã‚¿ã‹ã‚‰çµè«–ã«ã„ãŸã‚‹è«–è¨¼æ§˜å¼ã¨ã„ãˆã°ï¼Œæ¼”ç¹¹ï¼ˆdeductionï¼‰ã‹ã‚ã‚‹ã„ã¯å¸°ç´ï¼ˆinductionï¼‰ã—ã‹æ€ã„æµ®ã‹ã°ãªã„ã“ã¨ãŒå¤šã„ã€‚ã„ãšã‚Œã‚‚ï¼Œç‰¹å®šã®ä»®èª¬ã®çœŸå½ã®è¨¼æ˜ã‚’ç›®æŒ‡ã™è«–è¨¼æ§˜å¼ã§ã‚ã‚‹ã€‚ã—ã‹ã—ï¼Œç³»çµ±æ¨å®šã«ãŠã‘ã‚‹è«–è¨¼ã¯æ¼”ç¹¹ã§ã‚‚å¸°ç´ã§ã‚‚ãªã„ã€‚
ã‚¢ãƒ–ãƒ€ã‚¯ã‚·ãƒ§ãƒ³ã«ã‚ˆã‚‹æ¨è«–ã§ã¯ï¼Œé¸ã°ã‚ŒãŸä»®èª¬ã®çœŸå½ã¯å•é¡Œã§ã¯ãªã„ã€‚

http://cse.naro.affrc.go.jp/minaka/files/seibutsu-keitou.html
ç”Ÿç‰©ç³»çµ±å­¦
ä¸‰ä¸­ä¿¡å®
ã€€ã€€ï¼”ï¼ï¼–ï¼ï¼’ï¼šæœ€ç¯€ç´„æ³•ã¯çœŸå®Ÿã‚’å°ãã‹ï¼Ÿï¼ä¸€èˆ¬åŒ–æœ€ç¯€ç´„æ³•ã‚’ç›®æŒ‡ã—ã¦ 284

https://ja.wikipedia.org/wiki/Wikipedia:æ¤œè¨¼å¯èƒ½æ€§#ã€ŒçœŸå®Ÿã‹ã©ã†ã‹ã€ã§ã¯ãªãã€Œæ¤œè¨¼å¯èƒ½ã‹ã©ã†ã‹ã€

----------
## amelioration
æ”¹å–„
åŒåŒ–

2017-09-08
https://www.ncbi.nlm.nih.gov/pubmed/28887484
Sci Rep. 2017 Sep 8;7(1):10985. doi: 10.1038/s41598-017-11226-9.
Differential degeneration of the ACTAGT sequence among Salmonella: a reflection of distinct nucleotide amelioration patterns during bacterial divergence.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5591236/
Discussion
To gain a snapshot on the general patterns of genomic nucleotide composition optimization or amelioration in the process in which bacteria diverge into different niches such as distinct hosts (e.g., humans vs livestock or wild animals) or different tissues (causing systemic vs local infections), we analyzed a subset of the highly conserved short sequence CTAG among representative Salmonella pathogens by profiling the SpeI cleavage sequence ACTAGT in comparison with E. coli and other enteric bacteria. 


January 9th, 2013 by eric
https://www.reric.org/wordpress/archives/1034
G+C content amelioration of horizontally transferred genes

2013-03-12
https://pubmed.ncbi.nlm.nih.gov/23487592/
Front Microbiol
. 2013 Mar 12:4:41. doi: 10.3389/fmicb.2013.00041. eCollection 2013.
Lateral transfer and GC content of bacterial resistant genes
Nabil Hayek 1
https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2013.00041/full
https://pmc.ncbi.nlm.nih.gov/articles/PMC3594838/
Moreover, changing the environmental factors during the experiment could allow researchers to prospectively study the influence of the various factors on the evolution/amelioration of the GC content of the transferred resistant determinant and the bacterial genome. 


2008-02
https://pubmed.ncbi.nlm.nih.gov/18356951/
Genome
. 2008 Feb;51(2):164-8. doi: 10.1139/g07-105.
Gene amelioration demonstrated: the journey of nascent genes in bacteria
Pradeep Reddy Marri 1, G Brian Golding
Abstract
Gene amelioration is the hypothesis that genes acquired via lateral gene transfer will, over time, acquire the molecular characteristics of the host genome. Species for which multiple strains have been sequenced permit a demonstration that this hypothesis is correct. We use 7 sequenced genomes of Streptococcus pyogenes and 6 sequenced genomes of Staphylococcus aureus to illustrate the action of amelioration on these genomes.
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_pubmed_primary_18356951
EBSCOhost Academic Search Premier


2004-10
https://www.ncbi.nlm.nih.gov/pubmed/15240837
Mol Biol Evol. 2004 Oct;21(10):1884-94. Epub 2004 Jul 7.
Successful lateral transfer requires codon usage compatibility between foreign genes and recipient genomes.
Medrano-Soto A1, Moreno-Hagelsieb G, Vinuesa P, Christen JA, Collado-Vides J.
https://academic.oup.com/mbe/article/21/10/1884/1025177
Codon Usage Amelioration Is Unnecessary
We need to re-evaluate the notion that a foreign gene or fragment of DNA (assumed as atypical in sequence characteristics) becomes compositionally more similar to the host genome with increasing residence time. This process has been called â€œamelioration,â€ after reasoning that it makes a gene â€œbetterâ€ (Lawrence and Ochman 1997), implying better translatability. This concept is a natural consequence of methods assuming that most foreign genes display mainly a poor-CU profile. However, as our results indicate, most foreign genes with poor CU are counter-selected for successful integration, suggesting that CU amelioration might occur in a small fraction of genes. 

2005-02-23
https://pubmed.ncbi.nlm.nih.gov/15728743/
Nucleic Acids Res
. 2005 Feb 23;33(4):1141-53. doi: 10.1093/nar/gki242. Print 2005.
Variation in the strength of selected codon usage bias among bacteria
Paul M Sharp 1, Elizabeth Bailes, Russell J Grocock, John F Peden, R Elizabeth Sockett
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC549432/
But even this may have little relevance: in the same way that it is thought that the codon usage of horizontally transferred genes may take many millions of years to ameliorate to that of a new host genome (58), strongly selectively biased codon usage may take a very long time to decay after a reduction in effective population size, i.e. the codon usage bias currently observed may still be due in some part to evolutionary processes that occurred millions of years ago.

2001-10
https://pubmed.ncbi.nlm.nih.gov/11719972/
Review Curr Issues Mol Biol
. 2001 Oct;3(4):91-7.
Synonymous codon usage in bacteria
M D Ermolaeva 1
https://www.mdpi.com/1467-3045/3/4/13
cimb-03-00013.pdf

https://www.caister.com/cimb/v/v3/91.pdf

At the time of transfer the introduced genes have codon usage that is typical for the donor genome. With time, however, the codon usage evolves to match the codon usage of the host genome. This process of codon usage adjustment is called â€œameliorationâ€ (Lawrence and Ochman, 1997; Lawrence and Ochman, 1998) and it is sometimes possible to estimate its rate and predict the age of lateral gene transfer events. 

1999-08-03
https://pubmed.ncbi.nlm.nih.gov/10430917/
Comparative Study Proc Natl Acad Sci U S A
. 1999 Aug 3;96(16):9184-9. doi: 10.1073/pnas.96.16.9184.
Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA
A Campbell 1, J MrÃ¡zek, S Karlin
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC17754/
- This similarity might mean either that relatively close genome signatures promote plasmid establishment or that the plasmids have acquired their hostsâ€™ signatures during long-term residence. Experiments on conjugation can address issues such as specificity vs. wide host range and relevance of size and signature for plasmid compatibility. We interpret the similarities in signature between plasmids and their bacterial hosts as implying that they share much replication and repair machinery, perhaps because the prokaryotic cell is not compartmentalized to the degree that the eukaryotic cell is.
- (ii) During the plasmidâ€™s residence in its current host, the same pressures that homogenize the signature throughout the chromosome will also drive the plasmidâ€™s signature towards that of the host. Such amelioration has been postulated for the G+C content of laterally transferred DNA (22). We suspect that the signature should ameliorate even more rapidly, for both plasmids and laterally transferred chromosomal segments. 

2000-05-18
https://pubmed.ncbi.nlm.nih.gov/10830951/
Review Nature
. 2000 May 18;405(6784):299-304. doi: 10.1038/35012500.
Lateral gene transfer and the nature of bacterial innovation
H Ochman 1, J G Lawrence, E A Groisman
https://www.nature.com/articles/35012500
35012500.pdf
The rate of DNA acquisition by the E. coli chromosome was
measured indirectly by examining the amelioration of atypical
sequence characteristics (for example, nucleotide composition)
towards the equilibrium values displayed by this genome1,9. 

1998-08-04
https://www.ncbi.nlm.nih.gov/pubmed/9689094
Proc Natl Acad Sci U S A. 1998 Aug 4;95(16):9413-7.
Molecular archaeology of the Escherichia coli genome.
Lawrence JG1, Ochman H.
https://www.pnas.org/doi/10.1073/pnas.95.16.9413
https://pmc.ncbi.nlm.nih.gov/articles/PMC21352/
Amelioration of Horizontally Transferred Genes.

https://www.ncbi.nlm.nih.gov/pubmed/9089078
J Mol Evol. 1997 Apr;44(4):383-97.
Amelioration of bacterial genomes: rates of change and exchange.
Lawrence JG1, Ochman H.
https://link.springer.com/article/10.1007/PL00006158

At the time of introgression, horizontally transferred genes reflect the base composition of the donor genome; but, over time, these sequences will ameliorate to reflect the DNA composition of the new genome because the introgressed genes are subject to the same mutational processes affecting all genes in the recipient genome. This process of amelioration is evident in a large group of genes involved in host-cell invasion by enteric bacteria and can be modeled to predict the amount of time required after transfer for foreign DNA to resemble native DNA. 

1. Base composition varies widely among bacterial species. Base compositions range from 25% GC in Mycoplasma to 75% GC in Micrococcus, which is much
larger than the range in overall GC contents observed
among animals or plants. The differences in base
composition among bacterial species are largely due
to biases in the mutation rates at each of the four
basesâ€”termed â€˜â€˜directional mutation pressureâ€™â€™ by
Sueoka (1961, 1962, 1988, 1992, 1993, Sueoka et al.
1959)â€”which vary between species.

Because the process of amelioration is caused principally by mutational biases, the effects of this process are
most obvious at sites having little or no selective constraints.




----------
## allele
https://ja.wikipedia.org/wiki/å¯¾ç«‹éºä¼å­

dominant 
recessive 

https://ja.wikipedia.org/wiki/å„ªæ€§

https://www.m3.com/open/clinical/news/article/559530/
ã€Œå„ªæ€§ã€ã€ŒåŠ£æ€§ã€è¨€ã„æ›ãˆå ±é“ã€å­¦ä¼šãŒä¸€éƒ¨å¦å®š
æ—¥æœ¬äººé¡éºä¼å­¦ä¼šã€ã€Œæ—¥æœ¬éºä¼å­¦ä¼šã¨å”è­°ã®äº‹å®Ÿãªã„ã€ã¨ã‚³ãƒ¡ãƒ³ãƒˆ
2017å¹´9æœˆ27æ—¥

2017/9/15
https://www.nikkei.com/article/DGXLASDG15H7R_V10C17A9CR8000/
ã€Œå„ªæ€§ã€ã€ŒåŠ£æ€§ã€ç”¨èªä½¿ã‚ãšã€€æ—¥æœ¬éºä¼å­¦ä¼šãŒè¨€ã„æ›ãˆã€€ã€€:æ—¥æœ¬çµŒæ¸ˆæ–°è

https://mainichi.jp/articles/20170913/k00/00m/040/036000c
éºä¼å­¦ä¼šï¼šå„ªæ€§ãƒ»åŠ£æ€§ã®ç”¨èªè¦‹ç›´ã—ã€€æ–‡ç§‘çœã«ã‚‚æ”¹è¨‚è¦è«‹ã¸ - æ¯æ—¥æ–°è
2017å¹´9æœˆ12æ—¥

http://gsj3.jp/revisionterm.html
éºä¼å­¦ç”¨èªæ”¹è¨‚ã«ã¤ã„ã¦
2017.9.11 æ—¥æœ¬éºä¼å­¦ä¼š

----------
## ancestral reconstruction
**ç¥–å…ˆæ¨å®š**

https://github.com/haruosuz/r4bioinfo/tree/master/R_tree#ancestral-reconstruction

https://en.wikipedia.org/wiki/Ancestral_reconstruction

### Akanuma

https://www.waseda.jp/top/news/70339
æ–°é…µç´ å‰µå‡ºã€€ç’°å¢ƒè² è·ã®ä½æ¸›ã«æœŸå¾… â€“ æ—©ç¨²ç”°å¤§å­¦
æœ€ã‚‚å¤ã„ç¥–å…ˆã‚ˆã‚Šã‚‚ç¾ä»£ã«è¿‘ã„å¹´ä»£ã®é…µç´ ã‚’å¾©å…ƒã—ãŸã¨ã“ã‚ã€é«˜ã„è€ç†±æ€§ã¨å¸¸æ¸©ã§ã®é«˜ã„é…µç´ æ´»æ€§ã‚’ä½µã›æŒã¤ã€ç¾å­˜ã®å¤©ç„¶é…µç´ ã«ã¯ã‚ã¾ã‚Šè¦‹ã‚‰ã‚Œãªã„ç‰¹æ€§ã‚’ç¤ºã—ãŸ
æ—©ç¨²ç”°å¤§å­¦äººé–“ç§‘å­¦å­¦è¡“é™¢ã®èµ¤æ²¼å“²å²ï¼ˆã‚ã‹ã¬ã¾ã€€ã•ã¨ã—ï¼‰æ•™æˆã€å¤å·é¾å¤ªéƒï¼ˆãµã‚‹ã‹ã‚ã€€ã‚Šã‚…ã†ãŸã‚ã†ï¼‰åŠ©æ•™ã‚‰ã®ç ”ç©¶ãƒãƒ¼ãƒ ã¯ã€ç¾åœ¨ã®è‡ªç„¶ç•Œã«ã¯å­˜åœ¨ã—ãªã„ãŒéå»ã«ã¯å­˜åœ¨ã—ãŸã¨æ¨å®šã•ã‚Œã‚‹é…µç´ ã®ç¥–å…ˆã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’äºˆæ¸¬ã—ã€å®Ÿéš›ã«å¾©å…ƒã—ã¾ã—ãŸã€‚å¾—ã‚‰ã‚ŒãŸç¥–å…ˆå¾©å…ƒå‹é…µç´ ã®è€ç†±æ€§ã¨é…µç´ æ´»æ€§ã‚’èª¿ã¹ãŸçµæœã€å¸¸æ¸©ã§ã®é«˜ã„é…µç´ æ´»æ€§ã¨è€ç†±æ€§ã‚’ä½µã›æŒã¤ã€ç¾å­˜ã®å¤©ç„¶é…µç´ ã«ã¯ã‚ã¾ã‚Šè¦‹ã‚‰ã‚Œãªã„ç‰¹æ€§ã‚’æŒã¤é…µç´ ã§ã‚ã‚‹ã“ã¨ãŒåˆ†ã‹ã‚Šã¾ã—ãŸã€‚
ãƒ»æ²è¼‰èªŒï¼šScientific Reports
ãƒ»è«–æ–‡åï¼šAncestral sequence reconstruction produces thermally stable enzymes with mesophilic enzymeâ€‘like catalytic properties
https://www.nature.com/articles/s41598-020-72418-4
Ryutaro Furukawa, Wakako Toma, Koji Yamazaki & Satoshi Akanuma 

2016
https://www.jstage.jst.go.jp/article/chikyukagaku/50/3/50_199/_article/-char/ja/
ã‚²ãƒãƒ é…åˆ—ã®æ¯”è¼ƒã‹ã‚‰æ˜ã‚‰ã‹ã«ãªã£ãŸåˆæœŸç”Ÿå‘½ã®å¥½ç†±æ€§
èµ¤æ²¼ å“²å²
4.1 ã‚³ãƒ³ãƒ’ã‚šãƒ¥ãƒ¼ã‚¿è§£æã«ã‚ˆã‚‹ç¥–å…ˆé…åˆ—ã®æ¨å®š

2013?
http://www.f.waseda.jp/akanuma/research/currenttopics/ancestor.html
Ancestral design
ç¥–å…ˆå‹è¨­è¨ˆæ³•ã«ã‚ˆã‚‹è€ç†±æ€§ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®å‰µå‡º
ç”£æ¥­åˆ©ç”¨ã«æœ‰åˆ©ãªã‚¿ãƒ³ãƒ‘ã‚¯è³ªã‚’é–‹ç™ºã™ã‚‹ã†ãˆã§ã€ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®è€ç†±åŒ–è¨­è¨ˆã¯é‡è¦ãªèª²é¡Œã§ã‚ã‚‹ã€‚
ã€€ä»Šå¾Œã¯ã€ã“ã®ç¥–å…ˆå‹è¨­è¨ˆæ³•ã‚’ç”¨ã„ã¦ã€ç”£æ¥­ã«åˆ©ç”¨å¯èƒ½ãªå¤šãã®ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®è€ç†±åŒ–è¨­è¨ˆã«å–ã‚Šçµ„ã‚€äºˆå®šã§ã‚ã‚Šã€ã™ã§ã«ã€ãƒªã‚° ãƒ‹ãƒ³åˆ†è§£é…µç´ ç­‰ã‚’æ¨™çš„ã¨ã—ã¦é¸ã‚“ã§ã„ã‚‹ã€‚
èµ¤æ²¼å“²å²ã€å±±å²¸æ˜å½¦

### PastML
https://pastml.pasteur.fr/
http://iwasakilab.k.u-tokyo.ac.jp/iwasaki/publications.html
é€²åŒ–æƒ…å ±å­¦ã«ãŠã„ã¦ã€Œç¥–å…ˆçŠ¶æ…‹å¾©å…ƒã€ã¯ç”Ÿç‰©é€²åŒ–ã®ãƒ—ãƒ­ã‚»ã‚¹ã‚’çŸ¥ã‚‹ãŸã‚ã®é‡è¦ãªæ‰‹æ®µã§ã™ã€‚ãã®å…·ä½“çš„ãªæ‰‹æ³•ã¨ã—ã¦ã¯å‘¨è¾ºäº‹å¾Œç¢ºç‡ã‚’æ¨å®šã™ã‚‹ã‚‚ã®ã¨åŒæ™‚ç¢ºç‡ã‚’æ¨å®šã™ã‚‹ã‚‚ã®ãŒã‚ã‚Šã¾ã™ãŒã€ã„ãšã‚Œã‚‚ã€çµæœã‚’ãã®ã¾ã¾å¯è¦–åŒ–ã—ã¦è§£é‡ˆã‚’è¡Œã†ã“ã¨ã¯å›°é›£ã§ã—ãŸã€‚æœ¬ç ”ç©¶ã§ã¯æ„æ€æ±ºå®šç†è«–ã§ç”¨ã„ã‚‰ã‚Œã‚‹Brier scoreã¨å¯è¦–åŒ–ä¸Šã®å·¥å¤«ã«ã‚ˆã‚Šã€è§£é‡ˆãŒå®¹æ˜“ãªç¥–å…ˆçŠ¶æ…‹å¾©å…ƒã‚½ãƒ•ãƒˆã‚¦ã‚§ã‚¢PastMLã¨ã—ã¦å®Ÿè£…ã—ã€ã‚¦ã‚§ãƒ–ã‚µãƒ¼ãƒãƒ¼ã‚’å…¬é–‹ã—ã¾ã—ãŸã€‚è«–æ–‡ã§ã¯ã•ã‚‰ã«å®Ÿéš›ã«ãƒ‡ãƒ³ã‚°ã‚¦ã‚¤ãƒ«ã‚¹ã‚„HIVã‚¦ã‚¤ãƒ«ã‚¹ã®ç¥–å…ˆçŠ¶æ…‹å¾©å…ƒã‚’è¡Œã„ã€ãã®æœ‰åŠ¹æ€§ã‚’æ¤œè¨¼ã—ã¦ã„ã¾ã™ã€‚

2019-06-17
https://pubmed.ncbi.nlm.nih.gov/31127303/
Mol Biol Evol
. 2019 Sep 1;36(9):2069-2085. doi: 10.1093/molbev/msz131.
A Fast Likelihood Method to Reconstruct and Visualize Ancestral Scenarios
Sohta A Ishikawa 1 2 3, Anna Zhukova 1, Wataru Iwasaki 2, Olivier Gascuel 1
https://academic.oup.com/mbe/article/36/9/2069/5498561
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6735705/

https://www.ccs.tsukuba.ac.jp/wp-content/uploads/sites/14/2018/05/17a52.pdf
ç­‘æ³¢å¤§å­¦è¨ˆç®—ç§‘å­¦ç ”ç©¶ã‚»ãƒ³ã‚¿ãƒ¼ å¹³æˆï¼’ï¼™å¹´åº¦å­¦éš›å…±åŒåˆ©ç”¨ å ±å‘Šæ›¸
å›³ 2ï¼šPASTML ã«ã‚ˆã‚‹ HIV-1C 3,036 æ ªã®é€²åŒ–ç–«å­¦è§£æ
ãƒ–ãƒ©ã‚¸ãƒ«ãŠã‚ˆã³ã‚¤ãƒ³ãƒ‰ã¸ã® HIV-1C æ ªã®æµå…¥å¾Œã€ç‰¹å®šã®è–¬å‰¤è€æ€§å¤‰ç•°
ï¼ˆRT:D67N, RT:K70R, RT:M184Vï¼‰ã®ç²å¾—ã¨æ‹¡æ•£ãŒèµ·ã“ã£ãŸã“ã¨ãŒç¤ºã•ã‚ŒãŸã€‚

### 2019-01-08
https://www.ncbi.nlm.nih.gov/pubmed/30371900
Nucleic Acids Res. 2019 Jan 8;47(D1):D271-D279. doi: 10.1093/nar/gky1009.
Ancestral Genomes: a resource for reconstructed ancestral genes and genomes across the tree of life.
Huang X1,2, Albou LP2, Mushayahama T2, Muruganujan A2, Tang H2, Thomas PD2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323951/
Ancestral Genomes (http://ancestralgenomes.org) is a resource for comprehensive reconstructions of these â€˜fossil genomesâ€™.
<img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323951/bin/gky1009fig2.jpg" width=50%>
http://ancestralgenomes.org/
Ancestral Genomes Resource
|
Comparative analysis of protein sequences from diverse extant species can infer the evolutionary history of protein coding genes and reconstruct the repertoire of protein coding genes in extinct ancestral species.
|
NEW!Ancestral Genomes is now updated with latest release of PANTHER library (version 14.1)
http://www.pantherdb.org/

http://grasp.scmb.uq.edu.au/
GRASP Ancestral Sequence Reconstruction
Version beta 2019.12.08

https://www.ncbi.nlm.nih.gov/pubmed/30458842
BMC Syst Biol. 2018 Nov 20;12(Suppl 5):100. doi: 10.1186/s12918-018-0618-2.
On the reconstruction of the ancestral bacterial genomes in genus Mycobacterium and Brucella.
Guyeux C1, Al-Nuaimi B2,3, AlKindy B4, Couchot JF2, Salomon M2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6245693/
It was not the case of AlignSeqs, available in the R module called decipher [30]. 

https://www.ncbi.nlm.nih.gov/pubmed/28993971
Bull Math Biol. 2017 Dec;79(12):2865-2886. doi: 10.1007/s11538-017-0354-6. Epub 2017 Oct 5.
Ancestral Sequence Reconstruction with Maximum Parsimony.
Herbst L1, Fischer M2.

https://www.ncbi.nlm.nih.gov/pubmed/27404731
PLoS Comput Biol. 2016 Jul 12;12(7):e1004763. doi: 10.1371/journal.pcbi.1004763. eCollection 2016 Jul.
Ancestral Reconstruction.
Joy JB1,2, Liang RH1, McCloskey RM1, Nguyen T1, Poon AF1,2.

https://www.ncbi.nlm.nih.gov/pubmed/26351909
Biol Chem. 2016 Jan;397(1):1-21. doi: 10.1515/hsz-2015-0158.
Ancestral protein reconstruction: techniques and applications.
Merkl R, Sterner R.

http://kmooog.hatenablog.com/entry/2015/07/31/133246
ç¥–å…ˆé…åˆ—å†æ§‹ç¯‰-marginal reconstruction ã¨ joint reconstruction

![](http://cdn-ak.f.st-hatena.com/images/fotolife/k/kmooog/20150731/20150731132600.png)
ã¤ã¾ã‚Šã€â˜†å°ã®é…åˆ—ã‚’æ¨å®šã™ã‚‹éš›ã€marginal reconstructionã®å ´åˆã¯ABCDã®é…åˆ—ã®å½±éŸ¿ã—ã‹å—ã‘ãªã„ã‘ã©ã€joint reconstructionã§ã¯Eã®é…åˆ—ã«ã‚‚å½±éŸ¿ã‚’å—ã‘ã‚‹ã€‚

2015-05-13
http://nesseiken.info/Chiba_lab/index.php?æˆæ¥­%2FH19%2Fç³»çµ±è§£æè«–%2FMesquite
è¬›ç¾©ãƒ¡ãƒ¢ï¼šMESQUITEã®ã”ãç°¡å˜ãªä½¿ã„æ–¹
ç¥–å…ˆå½¢è³ªçŠ¶æ…‹ã®æ¨å®š

http://d.hatena.ne.jp/leeswijzer/20071128/1196279645
Ancestral Sequence Reconstruction
ç¥–å…ˆå¡©åŸºé…åˆ—ã‚„ç¥–å…ˆã‚¢ãƒŸãƒé…¸é…åˆ—ã®æ¨å®šã¨å¾©å…ƒã«é–¢ã™ã‚‹è«–æ–‡é›†ï¼ç¥–å…ˆå¾©å…ƒå•é¡Œã¨æ¨¹å½¢æ¨å®šå•é¡Œã‚ˆã‚Šã‚‚â€œè§£ãã‚„ã™ã„â€å•é¡Œã§ã¯ã‚ã‚‹ãŒï¼Œå¿œç”¨çš„ã«ã‚‚å®Ÿç”¨çš„ã«ã‚‚é‡è¦ãªåˆ†é‡ã ã¨æ€ã†ï¼ç³»çµ±æ¨¹ã‚’è¸ã¾ãˆãŸæ¨è«–ã‚„è€ƒå¯Ÿã‚’ã™ã‚‹å ´åˆï¼Œæ¨¹å½¢ã ã‘ã§ãªãï¼Œä»®æƒ³ç¥–å…ˆã®å½¢è³ªçŠ¶æ…‹ãŒæ±ºã¾ã£ã¦ã„ãªã„ã¨å…ˆã«é€²ã‚ãªã„è«–è­°ã¯å°‘ãªããªã„ï¼ã‚€ã—ã‚ï¼Œæ¨¹å½¢ã¨ç¥–å…ˆçŠ¶æ…‹ã¯ã¤ã­ã«ãƒšã‚¢ã§æ¨å®šã—ã¦ãŠãæ–¹ãŒã®ã¡ã®ã¡ã®ãŸã‚ã«ã¯ä¾¿åˆ©ã ã¨æ€ã†ï¼

1997-05-15
https://researchmap.jp/leeswijzer/
https://ci.nii.ac.jp/naid/110002936265
åˆ†å²å­¦ã«ãŠã‘ã‚‹ç¥–å…ˆæ¨å®šï¼šã€€çµ„åˆã›æœ€é©åŒ–å•é¡Œã¨ã—ã¦ã®ç³»çµ±æ¨å®š
ä¸‰ä¸­ ä¿¡å® 

1995
http://www.naro.affrc.go.jp/archive/niaes/sinfo/result/result11/result11_27.html
ç³»çµ±æ¨¹ã«ãŠã‘ã‚‹ç¥–å…ˆå½¢è³ªçŠ¶æ…‹ã®æœ€ç¯€ç´„çš„å¾©å…ƒæ³•
[è¦ç´„]
ç¨®ã®å½¢è³ªãƒ‡ãƒ¼ã‚¿ã¨ç³»çµ±æ¨¹ã®æ¨¹å½¢ã‚’ä¸ãˆãŸã¨ãã€ç³»çµ±æ¨¹ä¸Šã®ä»®æƒ³çš„å…±é€šç¥–å…ˆã®å½¢è³ªçŠ¶æ…‹ã‚’æœ€ç¯€ç´„çš„ã«å¾©å…ƒã™ã‚‹ãŸã‚ã®åŸºç¤ç†è«–ã‚’é›¢æ•£æ•°å­¦ã‚’ç”¨ã„ã¦å±•é–‹ã—ã€å‹•çš„è¨ˆç”»æ³•ã«åŸºã¥ãè¨ˆç®—ã‚¢ãƒ«ã‚´ãƒªã‚ºãƒ ã‚’é–‹ç™ºã—ãŸã€‚
[ãã®ä»–]
ç ”ç©¶èª²é¡Œåï¼šç”Ÿç‰©ç³»çµ±åˆ†é¡ã®ãŸã‚ã®æ•°é‡çš„æ–¹æ³•ã®é–‹ç™º

Published on Mar 3, 2012
https://www.slideshare.net/tmr_kohei/kashiwar-2
è¨€èªç³»çµ±æ¨¹ã¨é›‘ç…®ã®ç¥–å…ˆå½¢è³ªå¾©å…ƒï¼ˆKashiwa.R #2ï¼‰
tmr_kohei
ç”°æ‘å…‰å¹³

----------
## concerted evolution

https://www.weblio.jp/content/å”èª¿é€²åŒ–
concerted evolution
éºä¼å­å¤‰æ›ãªã©ã«ã‚ˆã‚Šã€é‡è¤‡éºä¼å­ãŒåŒã˜æ§‹é€ ã‚’ä¿ã¤ã‚ˆã†ã«é€²åŒ–ã™ã‚‹ã“ã¨ã€‚

https://en.wikipedia.org/wiki/Concerted_evolution
Concerted evolution is a process that may explain the observation that paralogous genes within one species are more closely related to each other than to members of the same gene family in another species, even though the gene duplication event preceded the speciation event. The high sequence similarity between paralogs is maintained by homologous recombination events that lead to gene conversion, effectively copying some sequence from one and overwriting the homologous region in the other.

https://ja.wikipedia.org/wiki/éºä¼å­ãƒ•ã‚¡ãƒŸãƒªãƒ¼
å”èª¿é€²åŒ–ã¸ã®å¯„ä¸
å¤šé‡éºä¼å­æ—ã®å¤šãã§ã¯éºä¼å­ãŒå®Œå…¨ã«åŒã˜ã‹ã»ã¼åŒã˜é…åˆ—ã‚’æœ‰ã—ã€éå¸¸ã«åŒä¸€æ€§ãŒé«˜ã„ã€‚ã“ã®åŒä¸€æ€§ã®ç¶­æŒã®éç¨‹ã¯å”èª¿é€²åŒ–ã§ã‚ã‚‹ã€‚å”èª¿é€²åŒ–ã¯ã€ä¸ç­‰äº¤å·®ãŒç¹°ã‚Šè¿”ã•ã‚ŒãŸã‚Šéºä¼å­ã®è»¢ç§»ã‚„ä¿å­˜ãŒç¹°ã‚Šè¿”ã•ã‚ŒãŸã‚Šã—ã¦èµ·ã“ã‚‹ã€‚

https://www.primate.or.jp/forum/ç¬¬42å›é›†å›£éºä¼å­¦è¬›åº§/
ã“ã®ç¾è±¡ã‚’å”èª¿é€²åŒ–concerted evolutionã¨ã‚ˆã¶ãŒã€é‡è¤‡ã—ãŸéºä¼å­ç¾¤ã«ã‚ˆãã¿ã‚‰ã‚Œã‚‹ã€‚

ã§ã¯å”èª¿é€²åŒ–ã¯ã©ã®ã‚ˆã†ã«ã—ã¦ç”Ÿã˜ãŸã®ã§ã‚ã‚ã†ã‹ã€‚ä»£è¡¨çš„ãªæ©Ÿæ§‹ã¨ã—ã¦è€ƒãˆã‚‰ã‚Œã‚‹ã®ã¯æŸ“è‰²ä½“ã®ä¸ç­‰äº¤å‰unequal crossing-overã¨éºä¼å­å¤‰æ›gene conversionã®äºŒã¤ã§ã‚ã‚‹ã€‚

----------
## concordance
ä¸€è‡´

incongruence
ä¸èª¿å’Œ

Last update: May 8, 2020, Contributors: M Bui
http://www.iqtree.org/doc/Concordance-Factor
Concordance Factor

https://www.fifthdimension.jp/documents/molphytextbook/
åˆ†å­ç³»çµ±å­¦æ¼”ç¿’ - ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã‹ã‚‰ä»®èª¬æ¤œå®šã¾ã§

ç³»çµ±æ¨¹ãƒ»ç³»çµ±ä»®èª¬ã®å¯è¦–åŒ–ã¨ç³»çµ±ä»®èª¬é–“ã®çµ±è¨ˆçš„æ¯”è¼ƒï¼šè¬›ç¾©ç·¨
https://www.fifthdimension.jp/documents/molphytextbook/hypothesistesting_lecture.pdf
RELL æ³•ã‚’å¿œç”¨ã—ãŸæ¤œå®š
â— Kishino-Hasegawa æ¤œå®š
â€“ 2 ã¤ã®æ¨¹å½¢ã®å¯¾æ•°å°¤åº¦æ¯”ã®ãƒãƒ©ãƒ„ã‚­ã‚’ RELL æ³•ã«ã‚ˆã‚Šæ¨å®šã—ï¼Œ 0
ã‚ˆã‚Šã‚‚æœ‰æ„ã«å¤§ãã„ãªã‚‰å¸°ç„¡ä»®èª¬ ( å°¤åº¦ãŒç­‰ã—ã„ ) ã‚’æ£„å´ã™ã‚‹
â— Shimodaira-Hasegawa æ¤œå®š
â€“ 3 ã¤ä»¥ä¸Šã®æ¨¹å½¢ã‚’æ¯”è¼ƒã™ã‚‹ã¨ãã« FWER ã‚’çµ±åˆ¶ã—ã¦ç¬¬ 1 ç¨®ã®éèª¤
ã‚’æŠ‘åˆ¶ã™ã‚‹ ( ãã®ã‹ã‚ã‚Šï¼Œç¬¬ 2 ç¨®ã®éèª¤ã®å±é™ºãŒå¢—å¤§ )
â— è¿‘ä¼¼çš„ã«ä¸åãª (Approximately Unbiased) æ¤œå®š
â€“ ãƒãƒ«ãƒã‚¹ã‚±ãƒ¼ãƒ«ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ã«ã‚ˆã‚Šç¬¬ 1 ç¨®ã®éèª¤ã¨ç¬¬ 2 ç¨®ã®
éèª¤ã®ä¸¡æ–¹ã‚’æŠ‘åˆ¶ã™ã‚‹

ã„ã¤ã‚„ã‚‹ã‹
â— æœ€å°¤ç³»çµ±æ¨¹ã®ä¸­ã§ç‰¹ã«é‡è¦ãªç³»çµ±ä»®èª¬ã‚’æ¤œè¨¼ã™ã‚‹
â— æ—¢çŸ¥ã®å¯¾ç«‹ã™ã‚‹ç³»çµ±ä»®èª¬ã‚’æ¯”è¼ƒã™ã‚‹

åˆ†å­ç³»çµ±è§£æã«ãŠã‘ã‚‹æ§˜ã€…ãªå•é¡Œã«ã¤ã„ã¦
https://www.fifthdimension.jp/documents/molphytextbook/answers.pdf
å¤šéºä¼å­åº§é€£çµè§£æã®å•é¡Œ
â— ãƒ‘ãƒ©ãƒ­ã‚°æ··å…¥ã‚„æµ¸é€äº¤é›‘ã€æ°´å¹³ä¼æ’­ã€incomplete lineage
sortingã§ã€éºä¼å­åº§é–“ã§æ”¯æŒã™ã‚‹ç³»çµ±æ¨¹ãŒç•°ãªã‚‹(ä¸èª¿å’Œ)
â€“ é€£çµè§£æã®ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—å€¤æ‚ªåŒ–ã‚„ã‚¢ãƒ¼ãƒ†ã‚£ãƒ•ã‚¡ã‚¯ãƒˆã®åŸå› 
â—
Internode Certainty, ICAll, TreeC, TCAå€¤ã§ä¸èª¿å’Œã‚’è©•ä¾¡
â€“ IC, ICAã¯ç³»çµ±ä»®èª¬ã”ã¨ã«å‡ºã‚‹ãŒTC, TCAã¯ç³»çµ±æ¨¹å…¨ä½“ã§1ã¤
â€“ ICã®ç¯„å›²ã¯1ï½0ã§ã€ICAã¯1ï½ãƒã‚¤ãƒŠã‚¹?ã€å°ã•ã„ã»ã©ä¸èª¿å’Œ
â€“ TC, TCAã¯IC, ICAã®ç·å’Œï¼OTUæ•°-3ã§å‰²ã£ã¦ãƒ‡ãƒ¼ã‚¿é–“æ¯”è¼ƒ
â— ä½¿ç”¨ã™ã‚‹éºä¼å­åº§ã‚’é¸åˆ¥ã™ã‚‹
â€“ Clusterflock, Concaterpiller, Conclustador
â— species tree methodã‚’ä½¿ã†
â€“ STEM, BUCKy, ASTRAL, *BEAST, BEST(MrBayes)

2015-04-15
http://hashiyuki.hatenablog.com/entry/2015/04/15/195458
RNA-Seqãƒ‡ãƒ¼ã‚¿ã‚’ç”¨ã„ãŸç³»çµ±è§£æ (1): è§£æã®æ–¹é‡ - NGSãƒ‡ãƒ¼ã‚¿è§£æã¾ã¨ã‚
(7) Bayesian concordance analysisã«ã‚ˆã‚‹concordance treeã®ä½œæˆã€ç¨®ç³»çµ±æ¨¹ã¨ç•°ãªã‚‹åˆ†å²ã‚’ç¤ºã™éºä¼å­ã®åŒå®šï¼ˆBUCKyãªã©ã‚’ä½¿ç”¨ï¼‰


2015å¹´9æœˆ18æ—¥
http://www.nibb.ac.jp/plantdic/blog/?p=778
Phylogeny of land plants é™¸ä¸Šæ¤ç‰©ã®ç³»çµ±

é€£çµéºä¼å­ã‚’ç”¨ã„ãŸè§£æã§ã¯ã€ãƒ„ãƒã‚´ã‚±é¡ãŒé™¸ä¸Šæ¤ç‰©ã®æœ€åŸºéƒ¨ã€æ¬¡ã„ã§ã€ã‚¿ã‚¤é¡ã¨ã‚»ãƒ³é¡ã‹ã‚‰ãªã‚‹å˜ç³»çµ±ç¾¤ãŒåˆ†å²ã—ãŸã€‚ã‚³ã‚¢ãƒ¬ãƒƒã‚»ãƒ³ã‚¹è§£æã§ã¯ã€ã‚³ã‚±æ¤ç‰©ã®ï¼“ç¾¤ãŒå˜ç³»çµ±ç¾¤ã¨ãªã£ãŸã€‚ã‚³ã‚±æ¤ç‰©ãŒå˜ç³»çµ±ã¨ã„ã†æ¨å®šã¯ã€è‘‰ç·‘ä½“ã‚²ãƒãƒ ä¸Šã®éºä¼å­ã‚’ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’ç”¨ã„ã¦è§£æã—ãŸçµæœã¨ä¸€è‡´ã—ã¦ã„ã‚‹(Nishiyama et al. 2004ï¼‰ã€‚

é€£çµéºä¼å­ã‚’ç”¨ã„ãŸè§£æã¨ã‚³ã‚¢ãƒ¬ãƒƒã‚»ãƒ³ã‚¹è§£æã¨ã§ã¯ã©ã¡ã‚‰ãŒå¦¥å½“ãªã®ã ã‚ã†ã‹ã€‚ã¾ãšã€ã‚³ã‚¢ãƒ¬ãƒƒã‚»ãƒ³ã‚¹è§£æã«ã¤ã„ã¦ç°¡å˜ã«èª¬æ˜ã—ã‚ˆã†ã€‚ç¥–å…ˆé›†å›£ã§éºä¼å­å¤šå‹ãŒã‚ã£ãŸå ´åˆã€ãã®éºä¼å­ã¯ç¨®ãŒåˆ†å²ã™ã‚‹ã‚ˆã‚Šã‚‚å‰ã«åˆ†å²ã—ã¦ã„ãŸã‚Šã€ç¨®åˆ†åŒ–ã®é€”ä¸­ã§å¤šå‹ã‚¢ãƒªãƒ«ã®ã©ã¡ã‚‰ã‹ãŒå¤±ã‚ã‚ŒãŸã‚Šã™ã‚‹ã€‚ã“ã‚Œã¯ã€ä¸å®Œå…¨éºä¼å­ç³»çµ±ä»•åˆ†ã‘ï¼ˆincomplete lineage sorting)ã¨å‘¼ã°ã‚Œã€éºä¼å­ç³»çµ±æ¨¹ã¨ç¨®ç³»çµ±æ¨¹ãŒä¸€è‡´ã—ãªã„ç†ç”±ã®ä¸€ã¤ã§ã‚ã‚‹ã€‚ãã®ãŸã‚ã€ã‚ˆã‚Šå¤šãã®éºä¼å­ç³»çµ±æ¨¹ã‚’æ¯”è¼ƒã—ã¦ã€ãã®ã†ã¡ã§ã‚‚ã£ã¨ã‚‚ç¢ºç‡ã®é«˜ã„ã‚‚ã®ã‚’ç¨®ç³»çµ±æ¨¹ã¨ã—ã¦æ¨å®šã™ã‚‹ã‚³ã‚¢ãƒ¬ãƒƒã‚»ãƒ³ã‚¹è§£æãŒè€ƒæ¡ˆã•ã‚ŒãŸ (Mirarab et al. 2014)ã€‚

ä¸€æ–¹ã€é€£çµéºä¼å­ã‚’ç”¨ã„ãŸç³»çµ±è§£æã¯ã€ã‚³ã‚¢ãƒ¬ãƒƒã‚»ãƒ³ã‚¹è§£æã‚ˆã‚Šã‚‚è§£æãŒæ¥½ã§ã‚ã‚‹ãŒã€ç•°ãªã£ãŸé€²åŒ–ãƒ¢ãƒ‡ãƒ«ï¼ˆé€²åŒ–é€Ÿåº¦ãªã©ï¼‰ã¨æ­´å²ã‚’æŒã¤éºä¼å­ã‚’ã¾ã¨ã‚ã¦ä¸€ã¤ã«ã—ã¦ã„ã‚‹ãŸã‚ã€é–“é•ã£ãŸç³»çµ±æ¨¹ãŒé«˜ã„çµ±è¨ˆçš„æ”¯æŒã‚’å—ã‘ã‚‹å ´åˆãŒã‚ã‚‹ã“ã¨ãŒã‚ã‹ã£ã¦ã„ã‚‹ï¼ˆMirarab et al. 2014ï¼‰ã€‚


http://www.fish-evol.org/phyMarker.html
ç³»çµ±æ¨å®š
19 Sep. 2019. Jun Inoue
ã€€é…µæ¯ 23 ç¨®ã‹ã‚‰å¾—ã‚‰ã‚ŒãŸ 1070 éºä¼å­ (ã‚·ãƒ³ãƒ†ãƒ‹ãƒ¼ã‹ã‚‰ã‚ªãƒ¼ã‚½ãƒ­ã‚°ã¨ç¢ºèªã•ã‚Œã¦ã„ã‚‹) ã‚’ç³»çµ±è§£æã—ãŸç ”ç©¶ã§ã¯ï¼Œã©ã®éºä¼å­ã‹ã‚‰å¾—ã‚‰ã‚ŒãŸæ¨¹å½¢ã‚‚ concatenate ã—ã¦å¾—ã‚‰ã‚ŒãŸæ¨¹å½¢ã¨ã¯ç•°ãªã£ã¦ã„ãŸï¼
concatenate ã¯æœ¬å½“ã«è‰¯ã„ã‹ç–‘å•ã‚’æŠ•ã’ã‹ã‘ã‚‹ï¼å…¨é…åˆ—ã‚’ä½¿ã†ã“ã¨ã‚ˆã‚Šã‚‚è§£æã«æœ‰ç”¨ãªéºä¼å­é…åˆ—ã ã‘ã‚’ä½¿ã†ã“ã¨ï¼ŒãŠã‚ˆã³æ¥µç«¯ãªä¸ä¸€è‡´ãŒè¦‹ã‚‰ã‚ŒãŸåˆ†å²ã‚’è¦‹æ¥µã‚ã‚‹ã“ã¨
ã€€ ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—è§£æã¯å˜ä¸€éºä¼å­ãªã©å°‘æ•°ã®ãƒ‡ãƒ¼ã‚¿è§£æã§ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã‚¨ãƒ©ãƒ¼ã‚’æ¤œå‡ºã™ã‚‹ãŸã‚ã«é–‹ç™ºã•ã‚ŒãŸã¨ã™ã‚‹ï¼ã“ã®ãŸã‚ï¼Œgenome wide ãªãƒ‡ãƒ¼ã‚¿ã‚’ concatenate ã—ãŸè§£æã«ã¯ä¸å‘ãã¨æŒ‡æ‘˜ï¼ãŸã¨ãˆæ¥µç«¯ã«ç•°ãªã‚‹å¯¾ç«‹ä»®èª¬ãŒã‚ã£ã¦ã‚‚ 100% ã®æ”¯æŒç¢ºç‡ã‚’å‡ºã—ã‚„ã™ã„ã¨ã™ã‚‹ (Perspective æœ€åˆã®æ®µè½)ï¼

https://www.ncbi.nlm.nih.gov/pubmed/23657258
Nature. 2013 May 16;497(7449):327-31. doi: 10.1038/nature12130. Epub 2013 May 8.
Inferring ancient divergences requires genes with strong phylogenetic signals.
Salichos L1, Rokas A.
https://www.nature.com/articles/nature12130
http://vu-wp0.s3.amazonaws.com/wp-content/uploads/sites/191/pdfs/2013_Salichos_Rokas_Nature.pdf
Phylogenomic analysis of 1,070 orthologues from 23 yeast genomes identified 1,070 distinct gene trees, which were all incongruent with the phylogeny inferred from concatenation. 

Figure 1: The yeast species phylogeny recovered from the concatenation analysis of 1,070â€‰genes disagrees with every gene tree, despite absolute bootstrap support.
a, The yeast species phylogeny recovered from concatenation analysis of 1,070â€‰genes using maximum likelihood. Asterisks denote internodes that received 100% bootstrap support by the concatenation analysis. Values near internodes correspond to gene-support frequency and internode certainty, respectively. The scale bar is in units of amino-acid substitutions per site. b, The distribution of the agreement between the bipartitions present in the 1,070â€‰individual gene trees and the concatenation phylogeny, as well as the distribution of the agreement between the bipartitions present in 1,000â€‰randomly generated trees of equal taxon number and the concatenation phylogeny, measured using the normalized Robinsonâ€“Foulds tree distance. 

![https://www.nature.com/articles/nature12130/figures/1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnature12130/MediaObjects/41586_2013_Article_BFnature12130_Fig1_HTML.jpg?as=webp)

First, we should abandon using bootstrap support on concatenation analyses of large data sets. Bootstrapping was developed long before the discovery of high-throughput sequencing, and it is an extremely useful measure of sampling errorâ€”that is, the robustness of inference when data are limited39â€”such as when a single gene is analysed. Given the availability and ease of generating genome-scale data40, relying on bootstrap to analyse phylogenomic data sets is misleading, not only because sampling error is minimal but also because its application will, even in the presence of notable conflict9 or systematic error6,16, almost always result in 100% values9,19,41.

https://tandy.cs.illinois.edu/yang-zhang.pdf
Presented by Yang Zhang
4/16/2015
Bootstrap in concatenation is misleading



https://www.ncbi.nlm.nih.gov/pubmed/22666370
PLoS One. 2012;7(5):e37607. doi: 10.1371/journal.pone.0037607. Epub 2012 May 30.
Gene repertoire evolution of Streptococcus pyogenes inferred from phylogenomic analysis with Streptococcus canis and Streptococcus dysgalactiae.
LefÃ©bure T1, Richards VP, Lang P, Pavinski-Bitar P, Stanhope MJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3364286/
Level of concordance between core gene trees was also assessed using Bucky [39] 
Figure 2
The seven topologies and the number of genes mapped to them, found by the Bayesian gene tree concordance analysis.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3364286/figure/pone-0037607-g002/

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3364286/bin/pone.0037607.g002.jpg)

http://www.stat.wisc.edu/~ane/bucky/
BUCKy: A program for Bayesian Concordance Analysis

![](http://www.stat.wisc.edu/~ane/bucky/images/buckywebfig2.gif)

https://www.ncbi.nlm.nih.gov/pubmed/22272658
BMC Genomics. 2012 Jan 24;13:38. doi: 10.1186/1471-2164-13-38.
Comparative genomic analysis of the genus Staphylococcus including Staphylococcus aureus and its newly described sister species Staphylococcus simiae.
Suzuki H1, LefÃ©bure T, Bitar PP, Stanhope MJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3317825/

çµ±è¨ˆæ•°ç†(2008)
https://www.ism.ac.jp/editsec/toukei/abstract/56-1j.html
è¤‡æ•°éºä¼å­ã®çµåˆãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ãåˆ†å­ç³»çµ±æ¨¹ã®æ¨æ¸¬
â€”çœŸæ ¸ç”Ÿç‰©ã®å¤§ç³»çµ±ã®è§£æã‚’ä¾‹ã¨ã—ã¦â€”
http://www.ism.ac.jp/editsec/toukei/pdf/56-1-145.pdf
ä¸€æ–¹ï¼Œåˆ†å­ç³»çµ±å­¦ç ”ç©¶ã®ç¾å ´ã§ã¯ï¼Œç³»çµ±ãƒãƒ¼ã‚«ãƒ¼ã¨ã—ã¦è¤‡æ•°ã®éºä¼å­ã‚’ç”¨ã„ã‚‹ã¨ï¼Œéºä¼å­é–“
ã§çŸ›ç›¾ã—ãŸè§£æçµæœãŒå¾—ã‚‰ã‚Œã‚‹ã¨ã„ã†çŠ¶æ³ã®å¤šã„ã“ã¨ãŒæ˜ã‚‰ã‹ã¨ãªã£ãŸï¼ãã®çŸ›ç›¾ãŒçµ±è¨ˆçš„èª¤
å·®ã®ç¯„å›²ã‚’è¶…ãˆã¦ã„ã‚‹ã¨ã„ã†å ´åˆã‚‚å°‘ãªã‹ã‚‰ãšå­˜åœ¨ã™ã‚‹ã“ã¨ã‚‚åˆ¤æ˜ã—ãŸï¼ã—ã‹ã—ã“ã®ã‚ˆã†ãªçŠ¶
æ³ãŒç”Ÿãšã‚‹ã®ã¯å½“ç„¶ã®å¸°çµã¨ã‚‚è€ƒãˆã‚‰ã‚ŒãŸï¼å€‹ã€…ã®éºä¼å­ã«ã¯ãã‚Œãã‚Œã®æ­´å²ãŒã‚ã‚Šï¼Œå›ºæœ‰ã®
é€²åŒ–ãƒ‘ã‚¿ãƒ¼ãƒ³ãŒã‚ã‚‹ã®ã«ã‚‚ã‹ã‹ã‚ã‚‰ãšï¼Œãã‚Œã‚‰ã‚’åŒä¸€ã®ï¼Œã—ã‹ã‚‚æ¥µã‚ã¦ä¸ååˆ†ãªé€²åŒ–ãƒ¢ãƒ‡ãƒ«ã®
ã‚‚ã¨ã§æ¯”è¼ƒã—ã¦ã„ã‚‹ã¨ã„ã†çŠ¶æ³ã‚’é‘‘ã¿ã‚Œã°ï¼Œèª¤å·®è‡ªä½“ãŒéå°‘æ¨å®šã§ã‚ã‚‹ã“ã¨ã¯ç–‘ã†ä½™åœ°ã¯ãªã„ï¼
ã¾ãŸï¼Œéºä¼å­é–“ã§çµ±è¨ˆçš„èª¤å·®ã®ç¯„å›²å†…ã®çŸ›ç›¾ãŒç”Ÿãšã‚‹ã“ã¨ã¯é »ç¹ã§ã‚ã‚‹ãŸã‚ï¼Œä¸€èˆ¬ã« 1 ã¤ã®éº
ä¼å­ã«å«ã¾ã‚Œã¦ã„ã‚‹ï¼Œç³»çµ±ã«é–¢ã™ã‚‹æƒ…å ±ã®ã†ã¡çœŸã®ç³»çµ±ã‚’åæ˜ ã™ã‚‹ã‚·ã‚°ãƒŠãƒ«ã®ç·å’ŒãŒãã‚Œã»ã©
å¤§ãããªã„ã“ã¨ã‚‚æ˜ã‚‰ã‹ã§ã‚ã‚‹ï¼ã“ã†ã—ãŸçŠ¶æ³ä¸‹ï¼Œã‚·ã‚°ãƒŠãƒ«ã®å¢—å¼·ã®ãŸã‚ã«è¤‡æ•°ã®éºä¼å­ã®æƒ…
å ±ã‚’çµåˆã—ã¦ç³»çµ±ã®æ¨æ¸¬ã‚’è¡Œã†ã¨ã„ã†è©¦ã¿ãŒãªã•ã‚Œã‚‹ã‚ˆã†ã«ãªã£ã¦ããŸï¼ãã—ã¦ç¾åœ¨ï¼Œå¤§è¦æ¨¡
ãƒ‡ãƒ¼ã‚¿ã®è“„ç©ã‚„è¨ˆç®—ç’°å¢ƒã®é£›èºçš„æ”¹å–„ã¨ç›¸ä¿Ÿã£ã¦ï¼Œè¤‡æ•°éºä¼å­ã®çµåˆãƒ‡ãƒ¼ã‚¿è§£æã¯æ—¥å¸¸çš„ãªã‚¢
ãƒ—ãƒ­ãƒ¼ãƒã¨ã—ã¦é »ç¹ã«è¡Œã‚ã‚Œã¦ã„ã‚‹ï¼

2005
http://jppa.or.jp/archive/pdf/59_03_01.pdf
ä¸‰ã¤ä»¥ä¸Šã®éºä¼å­ã§ç‹¬ç«‹ã«ç³»çµ±æ¨¹ã‚’ä½œæˆã—,æ¬¡ã«ãã‚Œã‚’é‡ã­åˆã‚ã›ã‚‹ã€‚
ãã—ã¦æ¨¹å½¢ãŒ concordance ã‹ã‚‰ conflict ã«è»¢æ›ã™ã‚‹ç‚¹ã‚’
ç¨®ã®å¢ƒç•Œã¨èªè­˜ã™ã‚‹ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/14663083
Microbiology. 2003 Dec;149(Pt 12):3507-17.
Extracting phylogenetic information from whole-genome sequencing projects: the lactic acid bacteria as a test case.
Coenye T1, Vandamme P.
http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.26515-0#tab2
Analysis of concordance between techniques

----------
## treedist

https://github.com/haruosuz/r4bioinfo/blob/master/R_tree/README.md#tree-distance

https://en.wikipedia.org/wiki/Robinsonâ€“Foulds_metric
Robinsonâ€“Foulds metric

http://evolution.genetics.washington.edu/phylip/doc/treedist.html
Treedist -- distances between trees

http://www.fish-evol.org/TREEDIST_JGI.html
äº•ä¸Š æ½¤ï¼šTREEDIST
2009 å¹´ 11 æœˆ 14 æ—¥ã€€æ”¹è¨‚
TREEDIST ã¯ç³»çµ±æ¨¹é–“ã®é•ã„ã‚’è·é›¢ã¨ã—ã¦è¡¨ã™ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã§ã™ï¼PHYLIP ãƒ‘ãƒƒã‚±ãƒ¼ã‚¸ã«å«ã¾ã‚Œã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ ãªã®ã§ï¼ŒPHYLIP ã®ãƒšãƒ¼ã‚¸ã‹ã‚‰ä»–ã®ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã¨ä¸€ç·’ã«ãƒ€ã‚¦ãƒ³ãƒ­ãƒ¼ãƒ‰ã—ã¦ãã ã•ã„ï¼

https://sites.google.com/site/kfuku52/mp/disttopol
æ¨¹å½¢é–“è·é›¢ - Kenji Fukushima's website
```
Unweighted Robinson-Foulds distance 
Weighted Robinson-Foulds distance
Nearest-neighbor interchange distance (NNI distance)
Subtree pruning and regrafting distance (SPR distance)
Tree bisection and reconnection distance (TBR distance) 
Quartet metric
Triples metric
Nodal distance
Split nodal distance
Branch score
Transposition distance
TreeKO
Average standard deviation of split frequency (ASDSF)
ãã®ä»–ã®æ–¹æ³•
```

----------
## divergence time
**åˆ†å²æ™‚é–“**

http://evolgen.biol.se.tmu.ac.jp/labo/tamura/161026.pdf
å®Ÿç¿’1: MEGA7ã®ã‚¿ã‚™ã‚¦ãƒ³ãƒ­ãƒ¼ãƒˆã‚™ã¨ã‚¤ãƒ³ã‚¹ãƒˆãƒ¼ãƒ«
4.RelTime ã«ã‚ˆã‚‹åˆ†å²æ™‚é–“æ¨å®š
(1)ã“ã“ã¦ã‚™ã¯ç´°èƒéª¨æ ¼å‹ã®ã‚¢ã‚¯ãƒãƒ³éºä¼å­ã®ã¿ã‚’ç”¨ã„ã€ç¨®é–“ã®åˆ†å²å¹´ä»£æ¨å®šã‚’è¡Œã£ã¦ã¿ã‚‹ã€‚è„Šæ¤å‹•ç‰©ã«ã¯ paralog ã‹ã‚™è¤‡æ•°ã‚ã‚‹ã®ã¦ã‚™ã€ä»¥ä¸‹ã®ã‚ˆã†ã« Sequence Data Explorer ã‚’ä½¿ã£ã¦ beta actin ã®ã¿ã‚’ãƒã‚§ãƒƒã‚¯ã™ã‚‹ã€‚
(6)å·¦å´ã®ãƒ•ãƒ¬ãƒ¼ãƒ ã¦ã‚™ Min ãŠã‚ˆã²ã‚™ Max Divergence Time ã« 1216(Time Tree of Life ã‚ˆã‚Š)ã‚’å…¥åŠ›ã—ã€[Finished]ã‚’ã‚¯ãƒªãƒƒ ã‚¯ã™ã‚‹ã€‚

åˆ†å²å¹´ä»£ã¯ 678.5(287-1070)MYA ã¨æ¨å®šã•ã‚Œã‚‹(Timetree of life ã«ã‚ˆã‚Œã¯ã‚™å¹³å‡ 692 MYA)ã€‚

https://ja.wikipedia.org/wiki/å¹´ä»£æ¸¬å®š
åˆ†å­æ™‚è¨ˆ
éºä¼å­ã‚„ãŸã‚“ã±ãè³ªã®ç¨®ã«ã‚ˆã‚‹é•ã„ã‹ã‚‰ã€ç¨®ãŒåˆ†å²ã—ãŸå¹´ä»£ã‚’è¨ˆç®—ã™ã‚‹ã€‚

2017 å¹´ 9 æœˆ 25 æ—¥ã€€æ”¹è¨‚
http://www.fish-evol.com/mcmctreeExampleVert6/text1.html
äº•ä¸Š æ½¤ï¼šMCMCTREEãƒ»ç°¡å˜ãªä¾‹é¡Œ
MCMCTREE ã¯ãƒ™ã‚¤ã‚ºæ³•ã«åŸºã¥ã„ã¦åˆ†å²å¹´ä»£æ¨å®šã‚’è¡Œã†ã‚½ãƒ•ãƒˆã‚¦ã‚§ã‚¢ã§ã™ï¼

2012 å¹´ 7 æœˆ 16 æ—¥
http://www.geocities.jp/ancientfishtree/DatabaseEnglish.html
äº•ä¸Š æ½¤: ç³»çµ±è§£æã®è‹±èªä¾‹æ–‡
åˆ†å²å¹´ä»£æ¨å®š

http://www.geocities.jp/ancientfishtree/Cauchy.html
äº•ä¸Šæ½¤ï¼šåŒ–çŸ³åˆ¶ç´„ãŒãƒ™ã‚¤ã‚ºå¹´ä»£æ¨å®šã«åŠã¼ã™å½±éŸ¿

http://www.geocities.jp/ancientfishtree/AncientDivTime2.html
äº•ä¸Šæ½¤ï¼šå¤ä»£é­šã®åˆ†å²å¹´ä»£æ¨å®š

https://kimuraseminar.wordpress.com/2017å¹´8æœˆ3æ—¥-ç³»çµ±å­¦å®Ÿç¿’/
åˆ†å²å¹´ä»£æ¨å®š
ã“ã“ã§ã¯PAMLãƒ‘ãƒƒã‚±ãƒ¼ã‚¸ã®MCMCTREEãƒ—ãƒ­ã‚°ãƒ©ãƒ ã‚’ç”¨ã„ã¾ã™ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/28673048
Syst Biol. 2017 Jun 29. doi: 10.1093/sysbio/syx060. [Epub ahead of print]
Taming the BEAST - A community teaching material resource for BEAST 2.

https://www.ncbi.nlm.nih.gov/pubmed/17996036
BMC Evol Biol. 2007 Nov 8;7:214.
BEAST: Bayesian evolutionary analysis by sampling trees.

https://sites.google.com/site/kaysakuma/japanese/ima-migrate-beast/beastno-shii-fang
BEASTã®ä½¿ã„æ–¹ - Kay's website
    ï¼ˆ20170204è¿½è¨˜)

2014
https://www.jstage.jst.go.jp/article/sjst/54/3/54_13037/_article/-char/ja/
ã‚ªãƒ³ãƒ©ã‚¤ãƒ³ãƒ„ãƒ¼ãƒ«â€œTimeTree: the timescale of lifeâ€ã‚’ç”¨ã„ãŸç”Ÿç‰©å¤šæ§˜æ€§ç§‘å­¦ã®æˆæ¥­é–‹ç™ºã¨ãã®è©•ä¾¡

http://yagays.github.io/blog/2013/07/10/timetree/
TimeTreeã§ç”Ÿç‰©ç¨®é–“ã®åˆ†å²å¹´ä»£ã‚’èª¿ã¹ã‚‹ - Wolfeyes Bioinformatics beta

https://www.fifthdimension.jp/wiki.cgi
ç”°è¾ºæ™¶å², 2010, "ãƒ‡ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã®ä½œæˆã¨ä»®èª¬æ¤œå®šã€åˆ†å²å¹´ä»£æ¨å®šæ³•æ¦‚è«–", æ—¥æœ¬é€²åŒ–å­¦ä¼šç¬¬12å›æ±äº¬å¤§ä¼šé€²åŒ–å­¦å¤ã®å­¦æ ¡ã€æ–°ã—ã„åˆ†å­ç³»çµ±è§£æè«–ï¼šãƒ‡ãƒ¼ã‚¿ä½œæˆã‹ã‚‰ç¥–å…ˆå½¢è³ªå¾©å…ƒã¾ã§ã€. é€²åŒ–å­¦å¤ã®å­¦æ ¡2010ï¼šæ–°ã—ã„åˆ†å­ç³»çµ±è§£æè«–ï¼šãƒ‡ãƒ¼ã‚¿ä½œæˆã‹ã‚‰ç¥–å…ˆå½¢è³ªå¾©å…ƒã¾ã§å‚ç…§.

http://www.ism.ac.jp/editsec/toukei/abstract/56-1j.html
ç¬¬56å·»ç¬¬1å·37ï¼54ï¼ˆ2008ï¼‰ã€€ã€€ç‰¹é›†ã€Œåˆ†å­é€²åŒ–ã¨çµ±è¨ˆç§‘å­¦ã€ã€€ã€€[ç ”ç©¶è©³è§£]
ã‚³ãƒ‰ãƒ³ãƒ¢ãƒ‡ãƒ«ã‚’ç”¨ã„ãŸåˆ†å²å¹´ä»£ã®ãƒ™ã‚¤ã‚ºæ¨å®š
ftp://statgen.ncsu.edu/pub/thorne/mypapers/seokishinothorne2008.pdf

----------
## effective population size

https://en.wikipedia.org/wiki/Effective_population_size

https://ja.wikipedia.org/wiki/æœ€å°å­˜ç¶šå¯èƒ½å€‹ä½“æ•°
Minimum Viable Populationã€MVPï¼‰
é¡ä¼¼ã—ãŸç”¨èªã«æœ‰åŠ¹å€‹ä½“æ•°ï¼ˆEffective population sizeï¼‰ãŒã‚ã‚‹ã€‚

https://www.primate.or.jp/old/PF/yasuda/10.html
é›†å›£ã®æœ‰åŠ¹ãªå¤§ãã•effective size
https://www.primate.or.jp/old/PF/yasuda/26.html
10.é›†å›£ã®æœ‰åŠ¹ãªå¤§ãã•
é›†å›£ã®æœ‰åŠ¹ãªå¤§ãã•effective size (number) of populationã‚’æ±‚ã‚ã‚‹å·¥å¤«ãŒWrigt(1931)ã«ã‚ˆã‚Šæœ€åˆã«å°å…¥ã•ã‚ŒãŸã€‚

æ°´ç”£è‚²ç¨®48ï¼ˆ2018ï¼‰
https://researchmap.jp/?action=cv_download_main&upload_id=224033
æœ€
ä½å¿…è¦å€‹ä½“æ•°ã‚’ç®—å‡ºã™ã‚‹ãŸã‚ã®å€¤ã€ä¾‹ãˆã°é›†å›£ã®æœ‰åŠ¹
ãªå¤§ãã•ï¼ˆeffective population size, Neï¼‰ã¨

1991
https://www.jstage.jst.go.jp/article/psj1985/7/1/7_1_23/_pdf
é›† å›£ã®æœ‰åŠ¹ ãªå¤§ ãã• (effective population size
:N) ã¨ã¯å„è¦ªå€‹ä½“ ãŒæ¬¡ä¸–ä»£ ã«æ®‹ã™å­ã®æ•°ã§ ã‚¦ã‚¨
ã‚¤ ãƒˆã‚’ã¤ã‘ãŸç¹æ®– å€‹ä½“æ•° ã¨å®šç¾© ã•ã‚Œ

----------
## phylogenetic diversity

https://en.wikipedia.org/wiki/Phylogenetic_diversity

https://methodsblog.wordpress.com/tag/faiths-phylogenetic-diversity/
Faithâ€™s Phylogenetic Diversity | methods.blog

https://sites.google.com/site/noteofpaediatricsurgery/in-silico/meta16s/figtree/keitouteki
ç³»çµ±çš„å¤šæ§˜æ€§ - Draft of Pediatric Surgery

2007.3
http://www.naro.affrc.go.jp/archive/niaes/magazine/083/mgzn08304.html
è«–æ–‡ã®ç´¹ä»‹ï¼š ç”Ÿç‰©å¤šæ§˜æ€§ã‚’é€²åŒ–ç³»çµ±å­¦çš„ãªå°ºåº¦ã§æ¸¬ã‚‹ (æƒ…å ±ï¼šè¾²æ¥­ã¨ç’°å¢ƒ No.83 2007.3)
ç”Ÿç‰©å¤šæ§˜åº¦ã®å°ºåº¦ã¨ã—ã¦ã€ç³»çµ±æ¨¹ï¼ˆphylogenetic treeï¼‰ã«åŸºã¥ãã€Œç³»çµ±å­¦çš„å¤šæ§˜æ€§ï¼ˆPD: phylogenetic diversityï¼‰ã€ã‚’ææ¡ˆ
æå”±è€…ã§ã‚ã‚‹ Daniel P. Faith ã®å®šç¾©ã§ã¯ ï¼ˆ1ï¼‰ã€ã‚ã‚‹ç”Ÿç‰©ç¾¤ã®ç³»çµ±å­¦çš„å¤šæ§˜åº¦ã¨ã¯ã€ãã®ç”Ÿç‰©ç¾¤ã‚’ç³»çµ±æ¨¹ä¸Šã§çµã¶æã®é•·ã•ã®ç·å’Œã¨ã—ã¦æ±‚ã‚ã‚‰ã‚Œã‚‹ã€‚



https://www.ncbi.nlm.nih.gov/pubmed/16535456
Appl Environ Microbiol. 1996 Nov;62(11):4299-301.
Problems in measuring bacterial diversity and a possible solution.
Watve MG, Gangal RM.

http://www.ism.ac.jp/editsec/toukei/abstract/60-2j.html#263
ã€Œçµ±è¨ˆæ•°ç†ã€60å·»ç¬¬2å·è¦æ—¨
ç¬¬60å·»ç¬¬2å·263ï¼278ï¼ˆ2012ï¼‰ã€€ã€€ç‰¹é›†ã€Œå¤šæ§˜æ€§ã¨é€²åŒ–ã®çµ±è¨ˆè§£æã€ã€€ã€€[ç ”ç©¶è©³è§£]
Î±å¤šæ§˜æ€§ã®æ¸¬å®šã¨ç¢ºç‡æ–‡å­—åˆ—ã®ç†è«–

Shannon æŒ‡æ•°ã‚„ Simpson æŒ‡æ•°ã¦ã‚™ã¯,ç¨®ã‹ã‚™æ˜ç¢ºã«å®šç¾©ã•ã‚Œã¦ã„ã‚‹ã“ã¨ã‚„å„å€‹ä½“ã‚’ã‚ã„ã¾ã„ ã•ãªãåŒå®šã¦ã‚™ãã‚‹ã“ã¨ã‹ã‚™å‰æã«ãªã£ã¦ãŠã‚Š,ã“ã‚Œã‚‰ã®æŒ‡æ¨™ã®å¾®ç”Ÿç‰©ç¾¤é›†ã¸ã®é©ç”¨ã«ã¯å•é¡Œã‹ã‚™ã‚ ã‚‹ã“ã¨ã‹ã‚™å½“åˆã‹ã‚‰æŒ‡æ‘˜ã•ã‚Œã¦ã„ãŸ(Staley, 1980; Torsvik et al., 1990). ã¾ãŸ,å›³ 1 ã¨ 2 ã«ãŠã„ã¦ ç¤ºã•ã‚Œã¦ã„ã‚‹ã‚ˆã†ã«,å¾®ç”Ÿç‰©ç¾¤é›†ã‹ã‚‰ã® 16S ãƒªãƒ›ã‚™ã‚½ãƒ¼ãƒ  RNA éºä¼å­é…åˆ—ã®ãƒ†ã‚™ãƒ¼ã‚¿ã® 1 ã¤ã®ç‰¹ å¾´ã¨ã—ã¦å¼·ã„ã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼æ€§ã‹ã‚™ã‚ã‚‹ã‹ã‚™,ç¨®æ•°ã«å¯¾å¿œã™ã‚‹ç•°ãªã‚‹é…åˆ—æ•°ã‚’æ•°ãˆã‚‹æŒ‡æ¨™ã‚„,å¡©åŸºå¤šæ§˜ åº¦ã®ã‚ˆã†ãªç•°ãªã‚‹ã‚µã‚¤ãƒˆæ•°ã«åŸºã¤ã‚™ãæŒ‡æ¨™ã¦ã‚™ã¯,ã“ã®ã‚ˆã†ãªã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼æ§‹é€ ã‚’åæ˜ ã¦ã‚™ããªã„ã¨ã„ ã†å•é¡Œã‹ã‚™ã‚ã‚‹.ä»£æ›¿çš„ãªæ–¹æ³•ã®ç ”ç©¶ã¨ã—ã¦,Watve and Gangal(1996), Hughes et al.(2001), Hong et al.(2006)ãªã¨ã‚™ã‹ã‚™ã‚ã‚‹ã‹ã‚™,ç†è«–çš„ãªåŸºç¤ã‚’æŒã¤ä½“ç³»çš„ãªæ–¹æ³•ã¨ã¯è¨€ãˆãªã‹ã£ãŸ.

----------
## recombination
çµ„æ›ãˆ

http://hp.brs.nihon-u.ac.jp/~inasweb/narai/narai/xiang_tong_zu_huanetoha.html
ç›¸åŒçµ„æ›ãˆã¨ã¯
éºä¼çš„çµ„æ›ãˆã¯ï¼Œã•ã‚‰ã«ä»¥ä¸‹ã®5ç¨®é¡ã«åˆ†ã‘ã‚‰ã‚Œã¾ã™ã€‚

https://pubmed.ncbi.nlm.nih.gov/33238876/
BMC Genomics
. 2020 Nov 25;21(1):829. doi: 10.1186/s12864-020-07262-x.
Impact of homologous recombination on core genome phylogenies
Caroline M Stott 1, Louis-Marie Bobay 2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7691112/


https://pubmed.ncbi.nlm.nih.gov/30670614/
mBio
. 2019 Jan 22;10(1):e02494-18. doi: 10.1128/mBio.02494-18.
Impact of Homologous Recombination on the Evolution of Prokaryotic Core Genomes
Pedro GonzÃ¡lez-Torres 1 2 3, Francisco RodrÃ­guez-Mateos 4, Josefa AntÃ³n 1 5, Toni GabaldÃ³n 6 3 7
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6343036/


----------
## duplication

https://ja.wikipedia.org/wiki/éºä¼å­é‡è¤‡

https://www.jstage.jst.go.jp/article/nl2008jsce/45/166/45_26/_article/-char/ja
ã‚²ãƒãƒ æƒ…å ±å…¨ç››ã®ã„ã¾ä½¿ãˆã‚‹åˆ†å­ç³»çµ±è§£æã®ã‚¨ãƒƒã‚»ãƒ³ã‚¹
å±±å£ å’Œæ™ƒ, å·¥æ¨‚ æ¨¹æ´‹
2019 å¹´ 45 å·» 166 å· p. 26-31
https://www.jstage.jst.go.jp/article/nl2008jsce/45/166/45_26/_pdf/-char/ja
MBD4 ã¨ MeCP2 ã¯ã¨ã‚‚ã«ãƒ¡ãƒãƒ«åŒ– CpG çµåˆãƒ‰ãƒ¡ã‚¤ãƒ³ã‚’æŒã£ã¦ãŠã‚Šã€ç„¡è„Šæ¤å‹•ç‰©ã‚’å¤–ç¾¤ã¨ã—ã¦å«ã‚ãŸåˆ†å­ç³»çµ±æ¨¹ã«ã‚ˆã£ã¦ã€ã“ã‚Œã‚‰ãŒè„Šæ¤å‹•ç‰©ã®é€²åŒ–ã®åˆæœŸã«éºä¼å­é‡è¤‡ã«ã‚ˆã£ã¦åˆ†ã‹ã‚ŒãŸã“ã¨ãŒç¤ºã•ã‚ŒãŸï¼ˆå›³4Dï¼‰ã€‚

https://www.brh.co.jp/research/formerlab/miyata/2005/post_000008.html
ç”Ÿç‰©æœ€å¤ã®æåˆ†ã‹ã‚Œï¼šå•é¡Œç‚¹ã¨é‡è¤‡éºä¼å­ã«ã‚ˆã‚‹è§£æ±º- å®®ç”°ã€€éš†ã®é€²åŒ–ã®è©± - JTç”Ÿå‘½èªŒç ”ç©¶é¤¨

ç”Ÿç‰©æœ€åˆã®åˆ†å²ã«é–¢ã™ã‚‹è«–äº‰ï¼šä½•ãŒå•é¡Œã ã£ãŸã®ã‹

å•é¡Œã®è§£æ±ºï¼šå¤ç´°èŒã¯çœŸæ ¸ç”Ÿç‰©ã«è¿‘ç¸

ã€€ã™ã¹ã¦ã®ç”Ÿç‰©ã¯ä¸€å¯¾ã®é‡è¤‡éºä¼å­EF-1a/Tuã¨ EF-2/Gã‚’æŒã£ã¦ã„ã‚‹ã€‚ã™ã¹ã¦ã®ç”Ÿç‰©ãŒã“ã®ä¸€å¯¾ã®é…µç´ ã‚’æŒã£ã¦ã„ã‚‹ã¨ã„ã†ã“ã¨ã¯ã€ã“ã®ä¸€å¯¾ã®éºä¼å­ã‚’ä½œã£ãŸéºä¼å­é‡è¤‡ã¯3ã¤ã®è¶…ç”Ÿç‰©ç•ŒãŒæåˆ†ã‹ã‚Œã™ã‚‹å‰ã«èµ·ããŸã“ã¨ã«ãªã‚‹ã€‚ãŸã—ã‹ã«GTPçµåˆã‚¿ãƒ³ãƒ‘ã‚¯è³ªæ—ã®ç³»çµ±æ¨¹ã¯ãã®ã“ã¨ã‚’å†ç¾ã—ã¦ã„ã‚‹ã€‚ã“ã®ã“ã¨ã‚’åˆ©ç”¨ã™ã‚‹ã¨è¶…ç”Ÿç‰©ç•Œã®æœ‰æ ¹ç³»çµ±æ¨¹ãŒä½œã‚Œã‚‹ã€‚3ã¤ã®è¶…ç”Ÿç‰©ç•Œãã‚Œãã‚Œã‹ã‚‰EF-1a/Tuã¨ EFï½°2/Gã‚’å–ã‚Šå‡ºã—ã€ã“ã‚Œã‚‰ã®ã‚¢ãƒŸãƒé…¸é…åˆ—ã®æ¯”è¼ƒã‹ã‚‰ç„¡æ ¹ç³»çµ±æ¨¹ã‚’ã¾ãšä½œã‚‹ã€‚ãã—ã¦EFï½°1Î±/Tuã¨EFï½°2/GãŒéºä¼å­é‡è¤‡ã«ã‚ˆã£ã¦æåˆ†ã‹ã‚Œã—ãŸæ™‚æœŸãŒã€3ã¤ã®è¶…ç”Ÿç‰©ç•ŒãŒæåˆ†ã‹ã‚Œã™ã‚‹ä»¥å‰ã«ãªã‚‹ã‚ˆã†ã«ç³»çµ±æ¨¹ã®æ ¹ã‚’æ±ºã‚ã‚‹ã€‚

ã“ã®è¤‡åˆç³»çµ±æ¨¹ã¯ã€EFï½°1Î±/Tuã¨EFï½°2/Gã®ã„ãšã‚Œã«ãŠã„ã¦ã‚‚ã€å¤ç´°èŒã¯çœŸæ ¸ç”Ÿç‰©ã«è¿‘ç¸ãªé–¢ä¿‚ã«ã‚ã‚Šã€çœŸæ­£ç´°èŒã¨ã¯é ç¸ã«ãªã‚‹ã“ã¨ã‚’ç¤ºã—ã¦ã„ã‚‹ï¼ˆå›³3ï¼‰ã€‚ã“ã†ã—ã¦ç”Ÿç‰©ã®æœ€ã‚‚åˆæœŸã®é€²åŒ–ã§èµ·ããŸåˆ†å²ã®é †åºãŒæ±ºå®šã§ããŸã€‚

ç”Ÿç‰©æœ€å¤ã®æ™‚ä»£ã«é »ç¹ã«èµ·ããŸéºä¼å­æ°´å¹³ç§»å‹•

https://www.jstage.jst.go.jp/article/jjb/19/Special_Issue/19_Special_Issue_S5/_article/-char/ja/
éºä¼å­ç³»çµ±æ¨¹ã‹ã‚‰æˆ‘ã€…ã¯ãªã«ã‚’çµã‚Šã ã›ã‚‹ã‹
æ–è—¤ æˆä¹Ÿ
1998 å¹´ 19 å·» Special_Issue å· p. S5-S13

éºä¼å­é‡è¤‡ã®æ™‚ç‚¹ã‚’ç¨®åˆ†åŒ–ã®æ™‚ç‚¹ã¨èª¤ã£ã¦æ¨å®šã™ã‚‹ã“ã¨ãŒã‚ã‚Šãˆã‚‹(å›³3 a)ã€‚ã“ã®å ´åˆ,ç¨®åˆ†å²ã®å¹´ä»£ã‚’éå¤§ã«è¦‹ç©ã‚‚ã‚‹ã“ã¨ã«ãªã‚‹ã€‚ã‚‚ã¡ã‚ã‚“,ç¨®åˆ†åŒ–ã®ã‚ã¨ã«,ã. ã‚Œãã‚Œã®ç¨®ã§ç‹¬ç«‹ã«éºä¼å­é‡è¤‡ãŒç”Ÿã˜ã‚‹ã“ã¨ã‚‚ã‚ã‚‹(å›³3b)ã€‚



----------
## Newick

https://www.hobochuritsu.com/entry/2019/02/12/215303
Newickå½¢å¼ã§ç³»çµ±æ¨¹ã®æ¨¹å½¢ãŒåŒã˜ã‹ç¢ºã‹ã‚ã‚‹ - ã»ã¼ä¸­ç«‹ãƒ–ãƒ­ã‚°
apeã®read.treeé–¢æ•°ã§Newickå½¢å¼ã‚’èª­ã¿è¾¼ã‚“ã§ã€all.equal.phyloé–¢æ•°ã§æ¨¹å½¢ãŒåŒã˜ã‹ã‚’åˆ¤å®šã—ã¾ã™ã€‚

https://sci-tech.ksc.kwansei.ac.jp/~tohhiro/bioinfo18/bioinfo2018-5.pdf

2017.12.29
https://stats.biopapyrus.jp/r/graph/phylogenetic-tree.html
ç³»çµ±æ¨¹ ape ade4 | R ã‚’åˆ©ç”¨ã—ãŸç³»çµ±æ¨¹ã®æãæ–¹
ç³»çµ±æ¨¹ã®ãƒ‡ãƒ¼ã‚¿ã¯ã€newick ãƒ•ã‚©ãƒ¼ãƒãƒƒãƒˆã‚ã‚‹ã„ã¯ nexus ãƒ•ã‚©ãƒ¼ãƒãƒƒãƒˆã«ã‚ˆã£ã¦è¨˜è¼‰ã•ã‚Œã‚‹ã€‚R ã§ã¯ã€ape ã‚„ ade4 ãªã©ã®ãƒ‘ãƒƒã‚±ãƒ¼ã‚¸ã‚’åˆ©ç”¨ã™ã‚‹ã“ã¨ã§ã€ã“ã‚Œã‚‰ã®ãƒ•ã‚©ãƒ¼ãƒãƒƒãƒˆã®ãƒ•ã‚¡ã‚¤ãƒ«ã‚’æ‰±ã†ã“ã¨ãŒã§ãã‚‹ã€‚

http://kiliwave.hatenablog.com/entry/2016/11/16/205345
Newickå½¢å¼ã®ãƒ•ã‚¡ã‚¤ãƒ«ã‚’ä¿®æ­£ã—ã¦å¤šåˆ†å²ã®ç³»çµ±æ¨¹ã‚’ä½œæˆã™ã‚‹

http://cse.naro.affrc.go.jp/minaka/cladist/tamagawa2011-1.html
2011å¹´åº¦ãƒ»ç‰å·å¤§å­¦ã€Œåˆ†å­ç³»çµ±é€²åŒ–å­¦ã€ãƒ»å®Ÿç¿’1
ä»Šå›ã®å®Ÿç¿’ã§ã¯ï¼Œç³»çµ±æ¨¹ã«å«ã¾ã‚Œã‚‹ã€Œå˜ç³»çµ±ç¾¤ï¼ˆmonophyletic groupï¼‰ã€ã®æƒ…å ±ãŒã©ã®ã‚ˆã†ã«å›³å¼è¡¨ç¾ã•ã‚Œã¦ã„ã‚‹ã®ã‹ã‚’ç†è§£ã™ã‚‹ãŸã‚ã«ï¼Œç”Ÿç‰©ç³»çµ±å­¦ã§åºƒãç”¨ã„ã‚‰ã‚Œã¦ã„ã‚‹ã€ŒNewickå½¢å¼ã€ï¼ˆâ†’è§£èª¬è¨˜äº‹ï¼šã€Œç³»çµ±æ¨å®šã®åŸºæœ¬ç”¨èªã€ï¼ã€ŒThe Newick tree formatã€ï¼‰ã®ã‚³ãƒ¼ãƒ‰åŒ–ã®æ–¹æ³•ã‚’å­¦ã³ï¼Œã€Œã“ã¨ã°ã€ã¨ã—ã¦ã®ç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬æ¦‚å¿µã‚’å­¦ç¿’ã™ã‚‹ï¼

http://ryamada22.hatenablog.jp/entry/20050513/1115948852
Newickæ›¸å¼ã‹ã‚‰ç³»çµ±æ¨¹ã‚’æã - ryamadaã®éºä¼å­¦ãƒ»éºä¼çµ±è¨ˆå­¦ãƒ¡ãƒ¢

https://www1.doshisha.ac.jp/~mjin/R/Chap_43/43.html
Rã¨ç³»çµ±æ¨¹(2)
ã€€ ç³»çµ±æ¨¹ã‚’ãƒ†ã‚­ã‚¹ãƒˆå½¢å¼ã§è¨˜éŒ²ã—ãŸãƒ‡ãƒ¼ã‚¿ã¨ã—ã¦ Newick ãƒ•ã‚©ãƒ¼ãƒãƒƒãƒˆãŒã‚ã‚‹ã€‚Newick ã¯ ç³»çµ±æ¨¹ã‚’æ¬¡ã®ã‚ˆã†ã«ã€éšå±¤çš„ã«ä¸¸æ‹¬å¼§ã§åŒºåˆ‡ã£ãŸãƒ†ã‚­ã‚¹ãƒˆå½¢å¼ã®ãƒ‡ãƒ¼ã‚¿ã§ã‚ã‚‹ã€‚æã®é•·ã•ã¯ã‚³ãƒ­ãƒ³ã®å³ã«æ•°å€¤ã§ç¤ºã™ã€‚

https://sites.google.com/site/kawashima38/home/writingtable/xi-tong-shu-ji-shunofomatto
ç³»çµ±æ¨¹è¨˜è¿°ã®ãƒ•ã‚©ãƒ¼ãƒãƒƒãƒˆ - kawashima38

http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
ãƒ‹ãƒ¥ãƒ¼ã‚¤ãƒƒã‚¯ãƒ•ã‚©ãƒ¼ãƒãƒƒãƒˆ(Newick format)ã€€ç³»çµ±æ¨¹ã®åˆ†å²é–¢ä¿‚ã‚’ï¼ˆï¼‰ã‚’ä½¿ã£ã¦è¡¨ç¾ã™ã‚‹æ–¹æ³•ã€‚æé•·ã‚„ãƒ–ãƒ¼ãƒ„ã‚¹ãƒˆãƒ©ãƒƒãƒ—ç¢ºç‡ã‚‚å«ã‚ã‚‹ã“ã¨ãŒã§ãã‚‹ã€‚

https://www.jst.go.jp/nbdc/bird/jinzai/literacy/streaming/h21_4_3b.pdf
Newick format: ãƒ†ã‚­ã‚¹ãƒˆå½¢å¼ã§ã®ç³»çµ±æ¨¹è¡¨ç¾
åŒã˜ç³»çµ±æ¨¹ã‚’è¡¨ã™Newick formatã¯è¤‡æ•°

----------
## species


2018-08-02
http://gendai.ismedia.jp/articles/-/56608
ã€ç¨®ã®èµ·æºã€ã‹ã‚‰ç´„160å¹´â€¦â€¦ã¾ã ã€Œç¨®ã€ãŒå®šç¾©ã§ããªã„ã£ã¦ãƒã‚¸ï¼Ÿï¼ˆå±±ç”° ä¿Šå¼˜ï¼‰ | ãƒ–ãƒ«ãƒ¼ãƒãƒƒã‚¯ã‚¹ | è¬›è«‡ç¤¾ï¼ˆ1/3ï¼‰

January 16, 2017
http://schaechter.asmblog.org/schaechter/2017/01/do-bacterial-species-really-exist-and-why-should-we-care.html
Small Things Considered: Do bacterial species really exist and why should we care?

2016å¹´8æœˆ
http://www.keisoshobo.co.jp/book/b239975.html
ç”Ÿç‰©å­¦ã®å“²å­¦å…¥é–€ - æ ªå¼ä¼šç¤¾ã€€å‹è‰æ›¸æˆ¿

ï¼»ç¬¬6ç« ï¼½ã€€ç¨®
6.4ã€€å¾®ç”Ÿç‰©ã¨æœ¬è³ªä¸»ç¾©

http://d.hatena.ne.jp/shorebird/20161222
æ›¸è©•ã€€ã€Œç”Ÿç‰©å­¦ã®å“²å­¦å…¥é–€ã€ - shorebirdã€€é€²åŒ–å¿ƒç†å­¦ä¸­å¿ƒã®æ›¸è©•ãªã©

æœ€å¾Œã«è‘—è€…ãŸã¡ã¯è©±ã‚’å¾®ç”Ÿç‰©ã«åºƒã’ã¦æ¤œè¨ã‚’è¡Œã„ï¼Œæ’å¸¸çš„æ€§è³ªã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼èª¬ã¯æ‰€è©®å·¨ç”Ÿç‰©ï¼ˆæœ¬æ›¸ã§ã¯éå¾®ç”Ÿç‰©ã‚’ã“ã†å‘¼ã¶ï¼‰ä¸­å¿ƒä¸»ç¾©ã§ã‚ã‚‹ã¨æ–­ç½ªã—ï¼Œç¨®ã‚«ãƒ†ã‚´ãƒªãƒ¼åå®Ÿåœ¨ï¼Œç¨®æ¦‚å¿µã®æ’é™¤ãªã©ã®è­°è«–ã‚’å¥½æ„çš„ã«ç´¹ä»‹ã—ã¦ã„ã‚‹ï¼

http://d.hatena.ne.jp/shorebird/20100920
æ›¸è©•ã€€ã€Œé€²åŒ–è«–ã¯ãªãœå“²å­¦ã®å•é¡Œã«ãªã‚‹ã®ã‹ã€ - shorebirdã€€é€²åŒ–å¿ƒç†å­¦ä¸­å¿ƒã®æ›¸è©•ãªã©

http://leeswijzer.hatenablog.com/entry/2016/09/10/102132
ã€Œç¨®å•é¡Œã€ã¯ã¯ã¦ã—ãªãç¶šã - archief voor stambomen
http://www.nikkei-science.com/page/magazine/0809/200809_060.html
ç”Ÿç‰©ã®ç¨®ã¨ã¯ä½•ã‹ | æ—¥çµŒã‚µã‚¤ã‚¨ãƒ³ã‚¹
ã€Œç³»çµ±å­¦çš„ç¨®æ¦‚å¿µã€
ã€€ã—ã‹ã—ï¼Œç³»çµ±å­¦çš„ç¨®æ¦‚å¿µã®æ¡ç”¨ã«ã‚ˆã‚Šï¼Œç¨®ã®ç´°åˆ†åŒ–ãŒä»Šå¾Œã©ã‚“ã©ã‚“é€²ã‚“ã§ã—ã¾ã†ã®ã§ã¯ãªã„ã‹ã¨è­¦æˆ’ã™ã‚‹æ„è¦‹ã‚‚ã‚ã‚‹ã€‚ãƒ­ãƒ³ãƒ‰ãƒ³å¤§å­¦ã‚¤ãƒ³ãƒšãƒªã‚¢ãƒ«ã‚«ãƒ¬ãƒƒã‚¸ã®ãƒ¡ã‚¤ã‚¹ï¼ˆGeorgina Maceï¼‰ã¯ï¼Œã€Œç³»çµ±å­¦çš„ç¨®æ¦‚å¿µã®å•é¡Œç‚¹ã¯ï¼Œã©ã“ã¾ã§åˆ†ã‘ã‚Œã°ã„ã„ã®ã‹ãŒã‚ã‹ã‚‰ãªã„ã¨ã„ã†ç‚¹ã«ã‚ã‚‹ã€ã¨ã„ã†ã€‚å°‘ãªãã¨ã‚‚åŸç†çš„ã«ã¯ï¼Œçªç„¶å¤‰ç•°ãŒ1ã¤ã§ã‚‚ã‚ã‚Œã°ï¼Œãã‚Œã‚’å…±æœ‰ã™ã‚‹å°ã•ãªå‹•ç‰©ç¾¤ã«å¯¾ã—ã¦ç¨®åã‚’ä¸ãˆã‚‹æ ¹æ‹ ã¨ãªã‚‹ã ã‚ã†ã€‚ã€Œã—ã‹ã—ï¼Œãã“ã¾ã§ç´°ã‹ãåˆ†ã‘ã‚‹ã®ã¯å°‘ã€…ã°ã‹ã’ã¦ã„ã‚‹ã€ã¨å½¼å¥³ã¯è¨€ã†ã€‚ãƒ¡ã‚¤ã‚¹ã¯ï¼Œã‚ã‚‹å€‹ä½“ç¾¤ã‚’æ–°ç¨®ã¨ã—ã¦ç‹¬ç«‹ã•ã›ã‚‹ãŸã‚ã«ã¯ï¼Œãã‚ŒãŒï¼Œåœ°ç†çš„åˆ†å¸ƒã‚„æ°—å€™æ¡ä»¶ã‚ã‚‹ã„ã¯æ•é£Ÿè€…è¢«é£Ÿè€…ã®é–¢ä¿‚ã®ç‚¹ã§ç”Ÿæ…‹çš„ã«ç•°ãªã£ã¦ã„ã‚‹ã“ã¨ã‚’ç¤ºã™ã¹ãã ã¨è¨€ã†ã€‚
ã€€ã—ã‹ã—ï¼Œç¨®ã‚’åˆ†ã‘ã™ãã‚‹ã“ã¨ã‚’å¿ƒé…ã™ã‚‹ã®ã§ã¯ãªãï¼Œã‚ãã¾ã§ãƒ‡ãƒ¼ã‚¿ã«åŸºã¥ã„ã¦åˆ¤æ–­ã™ã¹ãã ã¨ã„ã†ç ”ç©¶è€…ã‚‚ã„ã‚‹ã€‚ãƒ‹ãƒ¥ãƒ¼ãƒ¨ãƒ¼ã‚¯å·ç«‹å¤§å­¦ã‚¹ãƒˆãƒ¼ãƒ‹ãƒ¼ãƒ–ãƒ«ãƒƒã‚¯æ ¡ã®ã‚¦ã‚£ãƒ¼ãƒ³ã‚ºï¼ˆJohn Wiensï¼‰ã¯ï¼Œã€Œç¨®ã®ç´°åˆ†åŒ–ã‚’æ‡¸å¿µã™ã‚‹ã®ã¯æœ¬æœ«è»¢å€’ã ã‚ã†ã€ã¨åè«–ã™ã‚‹ã€‚ã€Œæœ€åˆã‹ã‚‰ã€ç¨®æ•°ã®ä¸Šé™ã¯ã“ã“ã¾ã§ã€ã¨åˆ¶é™ã™ã‚‹ã®ã¯ã¨ã¦ã‚‚ç§‘å­¦çš„ã¨ã¯ã„ãˆãªã„ã€ã¨å½¼ã¯ä¸»å¼µã™ã‚‹ã€‚
ã€€ç¨®ã®å®šç¾©ã‚’ã‚ãã‚‹å•é¡Œã‚’ã•ã‚‰ã«å„ä»‹ã«ã—ã¦ã„ã‚‹ã®ãŒå¾®ç”Ÿç‰©ã ã€‚ç”Ÿç‰©å¤šæ§˜æ€§ã®90ï¼…ä»¥ä¸Šã‚’å ã‚ã‚‹å¾®ç”Ÿç‰©ã®ç¨®ã¯ï¼Œã©ã†å®šç¾©ã™ã‚Œã°ã‚ˆã„ã®ã ã‚ã†ï¼Ÿã€€å‹•æ¤ç‰©ã«ã‚‚ï¼Œå¾®ç”Ÿç‰©ã«ã‚‚é©ç”¨ã§ãã‚‹ç¨®ã®å®šç¾©ã¯å¯èƒ½ãªã®ã ã‚ã†ã‹ï¼Ÿã€€


----------
## taxon

http://nesseiken.info/Chiba_lab/index.php?cmd=read&page=æˆæ¥­%2FH24%2Fé€²åŒ–ç”Ÿç‰©å­¦I%2Fç³»çµ±æ¨¹ã«é–¢ã™ã‚‹åŸºæœ¬ç”¨èª
- ã€Œç¯€ã€ã¯ã€ã€Œç¨®ã€ï¼ˆã‚ã‚‹ã„ã¯ã‚ˆã‚Šä¸Šä½ã®åˆ†é¡ç¾¤ï¼‰ã€ã€Œé›†å›£ã€ã€ã€Œå€‹ä½“ã€ãªã©ã€ã‚ã‚‹ç”Ÿç‰©ã®ã¾ã¨ã¾ã‚Šã‚„ã€ã€Œéºä¼å­ã€ã‚’ä»£è¡¨ã—ã¦ç¤ºã—ã¦ã„ã‚‹ã€‚
- åˆ†é¡ç¾¤ (taxon, è¤‡æ•°å½¢ã¯taxa)ã€€åç§°ã®ä¸ãˆã‚‰ã‚ŒãŸã€ç”Ÿç‰©ç¾¤ã€‚

https://ja.wikipedia.org/wiki/ã‚¿ã‚¯ã‚½ãƒ³
ï¼ˆtaxonã€è¤‡ï¼šã‚¿ã‚¯ã‚µã€taxaï¼‰ã¨ã¯ã€ç”Ÿç‰©ã®åˆ†é¡ã«ãŠã„ã¦ã€ã‚ã‚‹åˆ†é¡éšç´šã«ä½ç½®ã¥ã‘ã‚‰ã‚Œã‚‹ç”Ÿç‰©ã®é›†åˆã®ã“ã¨ã€‚è¨³èªã¨ã—ã¦ã¯åˆ†é¡ç¾¤ï¼ˆã¶ã‚“ã‚‹ã„ãã‚“ï¼‰ã¨ã„ã†ç”¨èªãŒä¸€èˆ¬çš„ã§ã‚ã‚‹ã€‚

https://ja.wikipedia.org/wiki/éšç´š_(ç”Ÿç‰©å­¦)
rank
ã¯ã€é–€ãƒ»ç¶±ãƒ»ç›®ãƒ»ç§‘ãƒ»å±ãƒ»ç¨®ãªã©ã®ã€åˆ†é¡ã®éšå±¤ã®ã“ã¨ã€‚

https://ja.wiktionary.org/wiki/åˆ†é¡ç¾¤
ç•Œ(kingdom)â†’é–€(phylum)â†’ç¶²(class)â†’ç›®(order)â†’ç§‘(family)â†’å±(genus)â†’ç¨®(species)ã¨åˆ†é¡ã•ã‚ŒãŸé›†å›£ã€‚

----------













