
Last Update: 2020-07

----------

# evolve.tools
é€²åŒ–ç³»çµ±è§£æãƒ„ãƒ¼ãƒ«

## Table of Contents
- [unclassified](#unclassified)
- [updates](#updates)
[2023](#2023)

[2019](#2019)
[2018](#2018)
[2017](#2017)
- [ortholog](#ortholog)
  - [JustOrthologs](JustOrthologs)
  - [OrthoFinder](#orthofinder)
- [alignment](#alignment) ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã€ç·¨é›†
  - [clustal-omega](#clustal-omega)
  - [macse](#macse)
  - [mafft](#mafft)
  - [gap](#gap)
  - [Gblocks](#gblocks)
  - [trimAl](#trimal)
- [tree](#tree) ç³»çµ±æ¨¹
  - [root_digger](#root_digger)
  - [IQ-Tree](#iqtree)
  - [FastTree](#fasttree)
  - [RAxML](#raxml)
  - [phyml](#phyml)
  - [bcgTree](#bcgTree)
  - [phylophlan](#phylophlan)
  - [etetoolkit](#etetoolkit)
- [viewer](#viewer) å¯è¦–åŒ–ãƒ„ãƒ¼ãƒ«
Dendroscope3
Tree viewer
IcyTree
  - [figtree](#figtree)
  - [iTOL](#itol)
  - [SeaView](#seaview)
- [gene loss](#gene-loss) éºä¼å­ãƒ­ã‚¹
  - [Notung](#notung)
  - [count](#count)
  - [DupliPHY](#dupliPHY)
- [](#)
- [MEGA](#mega)
- [physpetools](#physpetools)
- [HyPhy](#hyphy): Hypothesis testing using Phylogenies
- [TreeBASE](#treebase)
- [timetree](#timetree)
- [jlsteenwyk](#jlsteenwyk)

----------

----------
## jlsteenwyk

https://x.com/jlsteenwyk

https://jlsteenwyk.com/tutorials/phylogenomics_made_easy.html
Five-step phylogenomics, from proteomes to species tree

----------
### ClipKIT

https://github.com/JLSteenwyk/ClipKIT
https://jlsteenwyk.com/ClipKIT/

2020-12-02
https://pubmed.ncbi.nlm.nih.gov/33264284/
PLoS Biol
. 2020 Dec 2;18(12):e3001007. doi: 10.1371/journal.pbio.3001007. eCollection 2020 Dec.
ClipKIT: A multiple sequence alignment trimming software for accurate phylogenomic inference
Jacob L Steenwyk 1, Thomas J Buida 3rd 2, Yuanning Li 1, Xing-Xing Shen 3, Antonis Rokas 1
https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001007
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7735675/

https://jlsteenwyk.com/ClipKIT/advanced/index.html#codon
Trims codon-based alignments. If one position in a codon should be trimmed, the whole codon will be trimmed. To conduct codon-based trimming, use the -co/\-\-codon argument.

----------
### BioKIT

https://github.com/JLSteenwyk/BioKIT
online documentation.
https://jlsteenwyk.com/BioKIT/
Quick Start â€” biokit documentation

2022-07-04
https://pubmed.ncbi.nlm.nih.gov/35536198/
Genetics
. 2022 Jul 4;221(3):iyac079. doi: 10.1093/genetics/iyac079.
BioKIT: a versatile toolkit for processing and analyzing diverse types of sequence data
Jacob L Steenwyk 1 2, Thomas J Buida 3, Carla GonÃ§alves 1 2 4 5, Dayna C Goltz 6, Grace Morales 7, Matthew E Mead 1 2, Abigail L LaBella 1 2, Christina M Chavez 1 2, Jonathan E Schmitz 7, Maria Hadjifrangiskou 2 7, Yuanning Li 1, Antonis Rokas 1 2
https://academic.oup.com/genetics/article-abstract/221/3/iyac079/6583183

----------
### PhyKIT

https://github.com/JLSteenwyk/PhyKIT
online documentation.
https://jlsteenwyk.com/PhyKIT/
https://jlsteenwyk.com/PhyKIT/tutorials/index.html

https://www.preprints.org/manuscript/202404.1514/v1
PhyKIT: A Multitool for Phylogenomics

2021-08-25
https://pubmed.ncbi.nlm.nih.gov/33560364/
Bioinformatics
. 2021 Aug 25;37(16):2325-2331. doi: 10.1093/bioinformatics/btab096.
PhyKIT: a broadly applicable UNIX shell toolkit for processing and analyzing phylogenomic data
Jacob L Steenwyk 1, Thomas J Buida 2, Abigail L Labella 1, Yuanning Li 1, Xing-Xing Shen 3, Antonis Rokas 1
https://academic.oup.com/bioinformatics/article/37/16/2325/6131675
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8388027/

----------
## phyx

https://github.com/FePhyFoFum/phyx

https://github.com/FePhyFoFum/phyx/releases
Oct 22, 2024

2017-06-15
https://pubmed.ncbi.nlm.nih.gov/28174903/
Bioinformatics
. 2017 Jun 15;33(12):1886-1888. doi: 10.1093/bioinformatics/btx063.
Phyx: phylogenetic tools for unix
Joseph W Brown 1, Joseph F Walker 1, Stephen A Smith 1
https://academic.oup.com/bioinformatics/article/33/12/1886/2975328?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870855/

----------
## newick_utils

https://anaconda.org/bioconda/newick_utils
The Newick Utilities are a suite of Unix shell tools for processing phylogenetic trees. We distribute the package under the BSD License. Functions include re-rooting, extracting subtrees, trimming, pruning, condensing, drawing (ASCII graphics or SVG).

9 years ago
https://github.com/tjunier/newick_utils

https://gensoft.pasteur.fr/docs/newick-utils/1.6/nwutils_tutorial.pdf
Version 1.6.0 â€“ October 27, 2011

2010-07-01
https://pubmed.ncbi.nlm.nih.gov/20472542/
Bioinformatics
. 2010 Jul 1;26(13):1669-70. doi: 10.1093/bioinformatics/btq243. Epub 2010 May 13.
The Newick utilities: high-throughput phylogenetic tree processing in the UNIX shell
Thomas Junier 1, Evgeny M Zdobnov
https://academic.oup.com/bioinformatics/article/26/13/1669/200713
https://pmc.ncbi.nlm.nih.gov/articles/PMC2887050/


----------
## HyPhy

New
https://github.com/haruosuz/evolve/tree/master/tools/hyphy

Old
https://github.com/haruosuz/mgsa/tree/master/hyphy

----------
### TreeBASE
https://treebase.org/
TreeBASE is a repository of phylogenetic information, specifically user-submitted phylogenetic trees and the data used to generate them. TreeBASE accepts all kinds of phylogenetic data (e.g., trees of species, trees of populations, trees of genes) representing all biotic taxa.

https://treebase.org/treebase-web/urlAPI.html
Programmatic Data Access
In addition to the web interface, TreeBASE can be accessed programmatically through a stateless web service interface and URL architecture. This interface can deliver data in several different formats, including NEWICK, NEXUS, JSON, NeXML.

https://ja.wikipedia.org/wiki/TreeBASE
TreeBASE (ãƒˆã‚¥ãƒªãƒ¼ãƒ™ã‚¤ã‚¹) [1]ã¯ã€ã€ŒScientific journalï¼ˆè‹±èªç‰ˆï¼‰ ã€èªŒã«æ²è¼‰ã•ã‚ŒãŸç³»çµ±å­¦ã®ãƒ‡ãƒ¼ã‚¿ã®ãƒªãƒã‚¸ãƒˆãƒªã§ã‚ã‚‹ã€‚TreeBASEä½¿ç”¨ä¸Šã®ç³»çµ±å­¦ã®ç ”ç©¶ã«ãŠã‘ã‚‹ç ”ç©¶ãƒ‡ãƒ¼ã‚¿ã¨ã¯ã€åˆ†é¡ç¾¤ã®ä¸€çµ„ã‚„ãã®ä¸€çµ„ã®åˆ†é¡ç¾¤ã«é–¢ã™ã‚‹ãƒ¡ã‚¿ãƒ‡ãƒ¼ã‚¿ãŠã‚ˆã³åˆ†é¡ç¾¤é–“ã§ã®é€²åŒ–çš„é–¢ä¿‚ã‚’æœ€ã‚‚ã‚ˆãèª¬æ˜ã™ã‚‹ç‚ºã«æ¨è«–ã•ã‚Œã¦ã„ã‚‹ç³»çµ±æ¨¹ã«ã¤ã„ã¦ã€ï¼ˆä¾‹ãˆã°ã€å½¢è³ªé€²åŒ–ã®å¤‰åŒ–ã®å›æ•°ãŒæœ€å°ã«ãªã‚‹ã‚ˆã†ã«ã€ç³»çµ±æ¨å®šã‚’è¡Œã†æ–¹æ³•ã‚„å¤šé‡æ•´åˆ—ãªã©ã‚’ï¼‰æ¯”è¼ƒã—ã¦ã„ã‚‹çµæœã‚„å ±å‘Šãªã©ã‚’åé›†ã¾ãŸã¯ç”Ÿæˆã•ã‚Œã¦å¾—ã‚‰ã‚Œã‚‹ã€‚

2011 å¹´ 1 æœˆ 31 æ—¥ã€€æ”¹è¨‚
äº•ä¸Š æ½¤
https://fish-evol.org/TreeBase_JI.html
TreeBASE ã¯ç³»çµ±è§£æã®è«–æ–‡ã§ç”¨ã„ã‚‰ã‚ŒãŸã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆãƒ‡ãƒ¼ã‚¿ç­‰ã‚’ä¿å­˜ã—ã¦ã„ã‚‹ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã§ã™ï¼

https://pubmed.ncbi.nlm.nih.gov/19426482/
BMC Evol Biol
. 2009 May 8:9:93. doi: 10.1186/1471-2148-9-93.
Improved data retrieval from TreeBASE via taxonomic and linguistic data enrichment
Nadia Anwar 1, Ela Hunt
https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-9-93
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2685121/

https://pubmed.ncbi.nlm.nih.gov/17511869/
BMC Bioinformatics
. 2007 May 18:8:158. doi: 10.1186/1471-2105-8-158.
TBMap: a taxonomic perspective on the phylogenetic database TreeBASE
Roderic D M Page 1
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-158
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1885449/

----------
## timetree
http://www.timetree.org/

http://yagays.github.io/blog/2013/07/10/timetree/
TimeTreeã§ç”Ÿç‰©ç¨®é–“ã®åˆ†å²å¹´ä»£ã‚’èª¿ã¹ã‚‹ - Wolfeyes Bioinformatics beta

2017å¹´1æœˆ6æ—¥
http://trhujgitudjf.blogspot.com/2017/01/imetree-timescale-of-life.html
åœŸã¨ç”Ÿãç‰©: TimeTree: the timescale of life

http://molbiol.hatenablog.com/entry/2016/02/27/203751
TimeTreeã‚¢ãƒ—ãƒªã¯å­¦è¡“çš„ãªã‚¢ãƒ—ãƒªã§ã‚ã‚‹ - ç”Ÿç‰©å­¦ç ”ç©¶è€…ã®è¨€ã„ãŸã„æ”¾é¡Œãƒ–ãƒ­ã‚°

https://www.jstage.jst.go.jp/article/sjst/54/3/54_13037/_article/-char/ja/
ã‚ªãƒ³ãƒ©ã‚¤ãƒ³ãƒ„ãƒ¼ãƒ«â€œTimeTree: the timescale of lifeâ€ã‚’ç”¨ã„ãŸç”Ÿç‰©å¤šæ§˜æ€§ç§‘å­¦ã®æˆæ¥­é–‹ç™ºã¨ãã®è©•ä¾¡
å¡©é‡ è²´ä¹‹, çœŸæ „åŸ äº®, æ¥ æœ¬ èå¤ªéƒ, ä¹…ä¿ç”° åº·è£•
2014 å¹´ 54 å·» 3 å· p. 427-437
https://www.jstage.jst.go.jp/article/sjst/54/3/54_13037/_pdf

----------
### OrthoFinder

http://www.stevekellylab.com/software/orthofinder

https://github.com/davidemms/OrthoFinder

https://github.com/davidemms/OrthoFinder/blob/master/OrthoFinder-manual.pdf

https://davidemms.github.io/
OrthoFinder | OrthoFinder Tutorials

https://www.ncbi.nlm.nih.gov/pubmed/31727128
Genome Biol. 2019 Nov 14;20(1):238. doi: 10.1186/s13059-019-1832-y.
OrthoFinder: phylogenetic orthology inference for comparative genomics.
Emms DM1, Kelly S2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857279/
Fig. 1
Right-hand side: Reciprocal best hits (RBH) based on gene similarity scores that are monotonic with branch length and the orthology relationships inferred from these scores using standard heuristics (orthologs inferred using RBHs and co-orthology identified from within species hits better than closest RBH [8, 16]). 

43. Emms D, Kelly S: Supplemental dataset for: OrthoFinder2: fast and accurate phylogenomic orthology analysis from gene sequences. https://doi.org/10.5281/zenodo.1481147 2019.

å‹•ç”»
Aug 3, 2020
https://www.youtube.com/watch?v=L6eXJAE5J7g
VirtualPhyloComp2020 day 2 David Emms - YouTube
https://sites.google.com/view/phylocompgenomics2020/schedule
OrthoFinder: Building a Fully Phylogenetic Orthology Analysis

3:51 AM Â· Jul 16, 2020
https://twitter.com/David__Emms/status/1283474268701982720
David Emms on Twitter: "There's a new version of OrthoFinder out! It uses the gene trees it already infers to infer a set of orthogroups at each phylogenetic level within the species tree. This increases accuracy substantially &amp; also lets you add outgroups to your analysis: https://t.co/ogbupnnbxx https://t.co/Th3uZOCHqW" / Twitter

Mar 9
https://github.com/davidemms/OrthoFinder/issues/355
No support value in SpeciesTree_rooted.txt Â· Issue #355 Â· davidemms/OrthoFinder


2018 å¹´ 11 æœˆ 9 æ—¥ã€€æ”¹è¨‚
äº•ä¸Š æ½¤
http://www.fish-evol.org/orthofinder_ji.html
Orthofinder (githab) ã¯ã‚ªãƒ¼ã‚½ãƒ­ã‚°ã‚’æ¨å®šã™ã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã§ã™ï¼é…åˆ—é¡ä¼¼æ€§ all-in-all æ¯”è¼ƒã®çµæœã‚’ç”¨ã„ãŸ MCL è§£æã«ã‚ˆã‚‹ã‚°ãƒ«ãƒ¼ãƒ”ãƒ³ã‚°ã‹ã‚‰ï¼Œã‚ªãƒ¼ã‚½ãƒ­ã‚°ã‚’ã‚ªãƒ¼ã‚½ã‚°ãƒ«ãƒ¼ãƒ— (orthogroup) ã¨ã—ã¦æ¨å®šã—ã¾ã™ï¼
ã‚ªãƒ¼ã‚½ãƒ­ã‚¸ãƒ¼ã¯éºä¼å­ç³»çµ±æ¨¹ã«ã‚ˆã£ã¦æ¨å®šã™ã‚‹ã‚‚ã®ã§ï¼Œé…åˆ—é¡ä¼¼æ€§ã¯äºˆæƒ³ã«ã™ããªã„ï¼Œ

https://www.hobochuritsu.com/entry/2019/04/20/223105
å…¨è‡ªå‹•ã§ã‚·ãƒ³ã‚°ãƒ«ã‚³ãƒ”ãƒ¼éºä¼å­ã®é€£çµç³»çµ±æ¨¹æ¨å®š(OrthoFinder, MAFFT, trimAL, IQ-TREE) - ã»ã¼ä¸­ç«‹ãƒ–ãƒ­ã‚°

https://www.hobochuritsu.com/entry/2018/10/24/204801
OrthoFinderã§FastTreeã‚’ä½¿ãŠã†ã¨ã—ãŸã‚‰ã‚¨ãƒ©ãƒ¼ãŒå‡ºãŸ - ã»ã¼ä¸­ç«‹ãƒ–ãƒ­ã‚°
ã‚ªãƒ¼ã‚½ãƒ­ã‚°æ¨å®šãƒ—ãƒ­ã‚°ãƒ©ãƒ ã€ŒOrthoFinderã€ã¯ã€æ¨å®šã—ãŸã‚ªãƒ¼ã‚½ãƒ­ã‚°ã‚°ãƒ«ãƒ¼ãƒ—é…åˆ—ã‚’ä½¿ç”¨ã—ã¦ãã®ã¾ã¾ç³»çµ±æ¨¹ä½œæˆã¾ã§è¡Œã†ã“ã¨ãŒã§ãã¾ã™ã€‚

http://kazumaxneo.hatenablog.com/entry/2017/08/01/201224
OrthoFinderã§ã‚ªãƒ«ã‚½ãƒ­ã‚°ã‚°ãƒ«ãƒ¼ãƒ—ã‚’æ¢ç´¢ã™ã‚‹ - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

2017/2/22
https://qiita.com/okuman/items/c7ef9588e990a670d3ee
OrthoFinderã‚’ç”¨ã„ãŸOrthologousè§£æ - Qiita
OrthoFinderã§ã¯ã€MCL (markov cluster algorithm)ã‚’ç”¨ã„ã¦ã‚ªãƒ¼ã‚½ãƒ­ã‚°ã‚’æ¨å®šã—ã¾ã™ã€‚
ã“ã®OrthoGroupã«ã¯ã‚ªãƒ¼ã‚½ãƒ­ã‚°ã ã‘ã§ãªãã€ãƒ‘ãƒ©ãƒ­ã‚°ã‚‚å«ã¾ã‚Œã¦ã—ã¾ã†

2016å¹´03æœˆ12æ—¥
https://qiita.com/NariseT/items/e2c90d0235316f8878d8
OrthoFinder ã®ä½¿ã„æ–¹ - Qiita


----------
## gene loss
### Notung

http://www.cs.cmu.edu/~durand/Notung/
Notung-2.9 is a gene tree-species tree reconciliation software package that supports duplication-loss (DL) and duplication-transfer-loss (DTL) event models with a parsimony-based optimization criterion.

https://www.ncbi.nlm.nih.gov/pubmed/27998934
Bioinformatics. 2017 Mar 1;33(5):640-649. doi: 10.1093/bioinformatics/btw686.
Xenolog classification.
Darby CA1, Stolzer M1, Ropp PJ1, Barker D2, Durand D1,3.
horizontal transfer

http://www.fish-evol.org/Notung_JI.html
Notung - äº•ä¸Š æ½¤
201ï¼— å¹´ 12 æœˆ 10 æ—¥ã€€æ”¹è¨‚
Notung ã¯ species tree (åºƒãèªã‚ã‚‰ã‚Œã¦ã„ã‚‹ç³»çµ±æ¨¹) ã¨ gene tree (ä¸€ã¤ã®ç¨®ã‹ã‚‰è¤‡æ•°ã®ãƒ›ãƒ¢ãƒ­ã‚°ã‚’å«ã‚€) ã‚’æ¯”è¼ƒã—ã¦ï¼Œéºä¼å­é‡è¤‡ã¨æ¬ å¤±ãŒç”Ÿã˜ãŸãƒãƒ¼ãƒ‰ã‚’å†ç¯€ç´„æ¨å®šã™ã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã§ã™ï¼
å¾—ã‚‰ã‚ŒãŸ gene tree ã‚’ species tree ã¨ç…§ã‚‰ã—åˆã‚ã›ã¦ï¼Œéºä¼å­é‡è¤‡ãŒç”Ÿã˜ãŸ node ã‚’æ¨å®šã—ã¾ã™ï¼ã“ã¡ã‚‰ã«ã©ã®ã‚ˆã†ãªè§£æãŒè¡Œã‚ã‚Œã‚‹ã‹å›³ã§è¡¨ã•ã‚Œã¦ã„ã¾ã™ï¼
http://goby.compbio.cs.cmu.edu/Notung/dupTimes.html

----------
## physpetools

https://github.com/yangfangs/physpetools
PhySpeTree: an automated pipeline for reconstructing phylogenetic species trees

12:00 PM Â· Dec 15, 2019
https://twitter.com/fiddler_K/status/1206046415211728896
(1) Kawai_Yusuke on Twitter: "ï¼»ãƒ¡ãƒ¢ï¼½ã‚ã¨ã§ã¡ã‚‡ã£ã¨ä½¿ã£ã¦ã¿ã‚‹ã€‚ç¨®åã‚’å…¥ã‚ŒãŸã‚‰è‡ªå‹•ã§ç³»çµ±æ¨¹ã‚’æ¨å®šã™ã‚‹ã‚½ãƒ•ãƒˆ/PhySpeTree: an automated pipeline for reconstructing phylogenetic species trees https://t.co/Nnk2ysKCSV" / Twitter

https://pubmed.ncbi.nlm.nih.gov/31791235/
BMC Evol Biol
. 2019 Dec 2;19(1):219. doi: 10.1186/s12862-019-1541-x.
PhySpeTree: an automated pipeline for reconstructing phylogenetic species trees
Yang Fang 1, Chengcheng Liu 2, Jiangyi Lin 3, Xufeng Li 1, Kambiz N Alavian 4 5, Yi Yang 6, Yulong Niu 7
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6889546/
https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-019-1541-x
Sequence alignment and tree reconstruction
PhySpeTree integrates various tools for multiple sequence alignment and tree reconstruction.
MUSCLE [21], MAFFT [22], and Clustal [22]
RAxML
IQ-TREE [26] and FastTree [27]




----------
## command-line

ETE tools


### BuddySuite
https://pubmed.ncbi.nlm.nih.gov/28333216/
Mol Biol Evol
. 2017 Jun 1;34(6):1543-1546. doi: 10.1093/molbev/msx089.
BuddySuite: Command-Line Toolkits for Manipulating Sequences, Alignments, and Phylogenetic Trees
Stephen R Bond 1, Karl E Keat 1, Sofia N Barreira 1, Andreas D Baxevanis 1
https://academic.oup.com/mbe/article/34/6/1543/3049544
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850830/
Documentation for each BuddySuite tool, including usage examples, is available at http://tiny.cc/buddysuite_wiki. All software is open source and freely available through http://research.nhgri.nih.gov/software/BuddySuite.

### phyx

https://github.com/FePhyFoFum/phyx

https://pubmed.ncbi.nlm.nih.gov/28174903/
Bioinformatics
. 2017 Jun 15;33(12):1886-1888. doi: 10.1093/bioinformatics/btx063.
Phyx: phylogenetic tools for unix
Joseph W Brown 1, Joseph F Walker 1, Stephen A Smith 1
https://academic.oup.com/bioinformatics/article/33/12/1886/2975328
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870855/


### phylommand

https://github.com/RybergGroup/phylommand
Phylommand (Phylogenetics on the command line) is a software package for creating, manipulating, and/or getting statistics from trees or working with pairwise alignments. 

22 Dec 2016
https://doi.org/10.12688/f1000research.10446.1
Phylommand - a command line software package for phylogenetics [version 1; peer review: 2 approved with reservations]


----------
## unclassified

http://yukke.hateblo.jp/entry/2015/10/05/120924
ãƒ©ãƒ³ãƒ€ãƒ ãªå¡©åŸºé…åˆ—ã‚’ã¤ãã‚‹ - yukke::note

https://git.scicore.unibas.ch/TBRU/Treemmer
Treemmer 
Python tool to reduce size and redundancy of phylogenetic datasets

3:55 PM Â· Sep 17, 2019
https://twitter.com/level3defless/status/1173852867712565248
Koichi Higashi on Twitter: "ã™ã”ãä¹…ã€…ã«ç³»çµ±è§£æã‚„ã£ã¦ã‚‹ã‘ã©ã€ModelTest-NGãŒã™ã”ãä½¿ã„ã‚„ã™ã„ã—é€Ÿã„ã€‚ProtTestã®æ”¹è‰¯ç‰ˆã€‚DNAã«é–¢ã—ã¦ã¯GTR+I+Gã«ã—ã¨ã‘ã°ã„ã„ãƒ¢ãƒ‡ãƒ«é¸æŠã„ã‚‰ãªã„ã€ã¿ãŸã„ãªè©±ãŒæœ€è¿‘ã‚ã£ãŸã‘ã©ã€ä¸€å¿œDNAã«ã‚‚å¯¾å¿œã—ã¦ã‚‹ã€‚ https://t.co/bggA8Is0Kt" / Twitter


http://www.fish-evol.org/
äº•ä¸Š æ½¤ã®ã‚¦ã‚§ãƒ–ã‚µã‚¤ãƒˆ

https://sites.google.com/view/enter-the-fungi/phylogenetic-analysis
ç³»çµ±æ¨å®šã«ãŠã‘ã‚‹çµæœã¯ã„ã¤ã‚‚ä»®èª¬

http://www.tezuru-mozuru.com/?cat=200
ç³»çµ±è§£æ â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹

- å¤šé‡é…åˆ—ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã¨ç³»çµ±æ¨¹ [Multiple Alignment and Phylogenetic trees](https://github.com/haruosuz/r4bioinfo/blob/master/R_Avril_Coghlan/README.md#multiple-alignment-and-phylogenetic-trees)
- https://github.com/haruosuz/DS4GD/blob/master/2018giga/CaseStudy.md

http://d.hatena.ne.jp/haruosuz/20080813
ç³»çµ±è§£æ - Haruo Suzuki / Bioinformatics

https://ja.wikipedia.org/wiki/éåŠ é‡çµåˆæ³•
éåŠ é‡çµåˆæ³•ï¼ˆUnweighted Pair Group Method with Arithmetic meanã€UPGMAã¨ç•¥ã™)


----------

## updates

### 2023

https://pubmed.ncbi.nlm.nih.gov/36799928/
Genome Biol Evol
. 2023 Mar 3;15(3):evad026. doi: 10.1093/gbe/evad026.
OrthoPhy: A Program to Construct Ortholog Data Sets Using Taxonomic Information
Tomoaki Watanabe 1, Akinori Kure 2, Tokumasa Horiike 3
https://academic.oup.com/gbe/article/15/3/evad026/7044703
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9991595/
Homologs are genes derived from a common ancestor, and orthologs and paralogs are homologs derived from speciation and gene duplication, respectively. Homologs derived from horizontal gene transfer are called xenologs. In addition to speciation, paralogs and xenologs go through gene duplication or horizontal gene transfer; thus, their phylogenetic relationships do not reflect those of the species (Koonin 2005).


https://pubmed.ncbi.nlm.nih.gov/32886787/
Mol Biol Evol
. 2021 Jan 23;38(2):727-734. doi: 10.1093/molbev/msaa224.
CoreCruncher: Fast and Robust Construction of Core Genomes in Large Prokaryotic Data Sets
Connor D Harris 1, Ellis L Torrance 1, Kasie Raymann 1, Louis-Marie Bobay 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7826169/
CoreCruncher does not compute all pairwise genome comparisons and uses a heuristic based on the distributions of identity scores to classify sequences as orthologs or paralogs/xenologs. Although it is much faster than current methods, our results indicate that our approach is more conservative than other tools and less sensitive to the presence of paralogs and xenologs.


----------

### 2021


### 2021-05-17

https://kazumaxneo.hatenablog.com/entry/2020/10/27/204402
ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã‚’ã‚³ãƒ¼ãƒ‰ã™ã‚‹éºä¼å­é…åˆ—ã®çµ„æ›ãˆã‚¤ãƒ™ãƒ³ãƒˆã‚„æ­£ã®é¸æŠä¸‹ã«ã‚ã‚‹éƒ¨ä½ã‚’è¦‹ã¤ã‘ã‚‹ PoSeiDon - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
https://pubmed.ncbi.nlm.nih.gov/32735310/
Bioinformatics
. 2021 May 17;37(7):1018-1020. doi: 10.1093/bioinformatics/btaa695.
PoSeiDon: a Nextflow pipeline for the detection of evolutionary recombination events and positive selection
Martin HÃ¶lzer 1 2, Manja Marz 1 2 3
https://academic.oup.com/bioinformatics/article/37/7/1018/5879277
HyPhy GARD CODEML PAML

### 2021-02

1:01 AM Â· Feb 18, 2021
https://twitter.com/richardneher/status/1362069636381032448
Richard Neher on Twitter: "Do you find yourself in need of aligning 500k very similar sequences on a laptop? We (mostly @ivan_aksamentov) developed a reference aligner 'nextalign' that aligns 500k sequences in 30min using 8 cores and just a few GB of RAM. https://t.co/ElItsr2sWr https://t.co/GCBCbSyIkq" / Twitter

https://twitter.com/AineToole
4:29 AM Â· Feb 18, 2021
https://twitter.com/AineToole/status/1362122104796282880
Ãine O'Toole on Twitter: "pangolin PSA ğŸ“¢ We've just tagged a new release of pangolin (v2.3) that comes with an optional flag `--alignment`, so you can now easily output an alignment of your sequences along with the lineage report. https://t.co/0Cso0z8DPk" / Twitter

08 February 2021
https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab093/6130791
FASTRAL: Improving scalability of phylogenomic analysis | Bioinformatics | Oxford Academic
ASTRAL is the current leading method for species tree estimation from phylogenomic datasets (i.e., hundreds to thousands of genes) that addresses gene tree discord resulting from incomplete lineage sorting (ILS). ASTRAL is statistically consistent under the multi-locus coalescent model (MSC), runs in polynomial time, and is able to run on large datasets.
ASTRALã¯ã€æ•°ç™¾ã‹ã‚‰æ•°åƒã®éºä¼å­ã‚’å«ã‚€ç³»çµ±å›³ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰ç¨®ã®æœ¨ã‚’æ¨å®šã™ã‚‹ãŸã‚ã®ç¾åœ¨ã®ä¸»è¦ãªæ‰‹æ³•ã§ã™ã€‚
incomplete lineage sorting (ILS)ã«èµ·å› ã™ã‚‹éºä¼å­ãƒ„ãƒªãƒ¼ã®ä¸ä¸€è‡´ã«å¯¾å‡¦ã™ã‚‹ã€
ASTRALã¯ã€multi-locus coalescent model (MSC)ã®ä¸‹ã§çµ±è¨ˆçš„ã«ä¸€è²«ã—ã¦ãŠã‚Šã€
https://github.com/PayamDiba/FASTRAL

### 2020

### 2019


2019å¹´12æœˆ2æ—¥æœˆæ›œæ—¥
https://evotools.blogspot.com/2019/12/beast2.html
ç³»çµ±è§£æå‚™å¿˜éŒ²: BEAST2ã«ã‚ˆã‚‹åˆ†å²å¹´ä»£æ¨å®šâ‘ 

12:10 PM Â· Oct 28, 2019
https://twitter.com/art_poon/status/1188654334436872192
Art Poon on Twitter: "For my #BioInformatics class, I wrote a #d3js interactive #dataviz to teach my students the difference between an unrooted and rooted phylogeny. This displays a simplified version of the @sabeti_lab's Ebola virus phylogeny - root-to-tip regression is next! https://t.co/COTdboVyfb" / Twitter

2019å¹´10æœˆ21æ—¥ã«æ›´æ–°
https://qiita.com/kaizen_nagoya/items/150646f72c55a36f8c39
éºä¼å­è§£æã€éºä¼å­æ©Ÿèƒ½è§£æ - Qiita

2019/10/19
https://kimbio.info/mafftâ†’raxmlâ†’figtreeã§éºä¼å­ã®ç³»çµ±æ¨¹ã‚’ä½œæˆã™ã‚‹-ubuntu19-04
MAFFTâ†’RAxMLâ†’FigTreeã§éºä¼å­ã®ç³»çµ±æ¨¹ã‚’ä½œæˆã™ã‚‹ - Ubuntu19.04 - Kim Biology & Informatics

2019/03/06
https://qiita.com/MaedaTaro_Umiushi/items/d4004d3fb219a4f991e8
orthofinder + prequal+ mafft + iq-treeã§ã‚²ãƒãƒ è¦æ¨¡ã®ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰ç³»çµ±æ¨¹ã‚’ä½œã‚‹ - Qiita

### 2018

2018å¹´07æœˆ27æ—¥
https://qiita.com/MaedaTaro_Umiushi/items/7c8cee435347eeee1cf5
orthofinder + mafft + trimal + iq-treeã§ã‚²ãƒãƒ è¦æ¨¡ã®ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰ç³»çµ±æ¨¹ã‚’ä½œã‚‹ - Qiita

https://doi.org/10.7875/togotv.2018.093
2018-04-03 MAFFTãƒ»RAxMLãƒ»FigTreeã‚’çµ„ã¿åˆã‚ã›ã¦åˆ†å­ç³»çµ±è§£æã‚’è¡Œã†

### 2017

https://www.ncbi.nlm.nih.gov/pubmed/28174903
Bioinformatics. 2017 Jun 15;33(12):1886-1888. doi: 10.1093/bioinformatics/btx063.
Phyx: phylogenetic tools for unix.

### 2015

2015/08/31
http://www.kenkyuu2.net/cgi-biotech2012/biotechforum.cgi?mode=viewer;Code=4381
BioTechnicalãƒ•ã‚©ãƒ¼ãƒ©ãƒ  [16SrRNAã«ã‚ˆã‚‹ç³»çµ±è§£æ]

August 6, 2013
http://yuifu.github.io/pplacer-tutorial/
pplacerã®ä½¿ã„æ–¹ â€“ Haruka Ozaki, Ph.D. â€“ Computational biolology & Bioinformatics
pplacerã¯ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ä¸­ã«ã©ã®ã‚ˆã†ãªç¨®ãŒå¤šã„ã‹ã‚’å¯è¦–åŒ–ãƒ»æ¯”è¼ƒã™ã‚‹ãƒ„ãƒ¼ãƒ«ã§ã™ï¼

2008å¹´11æœˆ 4æ—¥ï¼Œ11æ—¥
http://lbm.ab.a.u-tokyo.ac.jp/~omori/phylogeny/txt/phylogeny_zikken.html
2008å¹´åº¦ç”Ÿç‰©æ¸¬å®šå­¦å¿œç”¨å®Ÿé¨“
åˆ†å­ç³»çµ±æ¨¹ã®ä½œæˆ
æ±äº¬å¤§å­¦å¤§å­¦é™¢è¾²å­¦ç”Ÿå‘½ç§‘å­¦ç ”ç©¶ç§‘ã€€å¤§æ£®å®

----------
## ortholog

----------
### SwiftOrtho

https://pubmed.ncbi.nlm.nih.gov/31648300/
Gigascience
. 2019 Oct 1;8(10):giz118. doi: 10.1093/gigascience/giz118.
SwiftOrtho: A fast, memory-efficient, multiple genome orthology classifier
Xiao Hu 1, Iddo Friedberg 1
https://academic.oup.com/gigascience/article/8/10/giz118/5606727
While the proper inference of homology type involves tracing gene history using phylogenetic trees [2], several proxy methods have been developed over the years. The most common method to infer orthologs by proxy is reciprocal best hits (RBH) [3, 4]. Briefly, RBH states the following: when 2 proteins that are encoded by 2 genes, each in a different genome, find each other as the best-scoring match among all homologs, they are considered to be orthologs [3, 4].

http://kazumaxneo.hatenablog.com/entry/2019/04/13/073000
ã‚ªãƒ«ã‚½ãƒ­ã‚°ã¨ãƒ‘ãƒ©ãƒ­ã‚°ã‚’å°ãƒ¡ãƒ¢ãƒªä½¿ç”¨é‡ã§é«˜é€Ÿæ¢ç´¢ã™ã‚‹ SwiftOrtho - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
ã‚ªãƒ«ã‚½ãƒ­ã‚°ã‚’æ¨è«–ã™ã‚‹æœ€ã‚‚ä¸€èˆ¬çš„ãªæ–¹æ³•ã¯ã€Reciprocal Best Hit ï¼ˆç›¸äº’ãƒ™ã‚¹ãƒˆãƒ’ãƒƒãƒˆï¼‰ã¾ãŸã¯RBHã§ã‚ã‚‹[ref.2ã€3]ã€‚ç°¡å˜ã«è¨€ã†ã¨ã€RBHã¯æ¬¡ã®ã‚ˆã†ã«è¿°ã¹ã‚‰ã‚Œã¦ã„ã‚‹ã€‚ç•°ãªã‚‹ï¼’ã‚²ãƒãƒ ã®ï¼’ã¤ã®éºä¼å­ã«ã‚ˆã£ã¦ã‚³ãƒ¼ãƒ‰ã•ã‚Œã¦ã„ã‚‹2ã¤ã®ã‚¿ãƒ³ãƒ‘ã‚¯è³ªãŒäº’ã„ã«æœ€è‰¯ã®ã‚¹ã‚³ã‚¢ãƒªãƒ³ã‚°ãƒãƒƒãƒã¨ã—ã¦è¦‹ã¤ã‹ã‚‹ã¨ã€ãã‚Œã‚‰ã¯ã‚ªãƒ«ã‚½ãƒ­ã‚°ã¨è¦‹ãªã•ã‚Œã‚‹[ref.2ã€3]ã€‚

----------
### JustOrthologs
https://github.com/ridgelab/JustOrthologs/

https://www.ncbi.nlm.nih.gov/pubmed/30084941
Bioinformatics. 2019 Feb 15;35(4):546-552. doi: 10.1093/bioinformatics/bty669.
JustOrthologs: a fast, accurate and user-friendly ortholog identification algorithm.
Miller JB1, Pickett BD1, Ridge PG1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6378933/

http://kazumaxneo.hatenablog.com/entry/2019/03/29/073000
é«˜é€Ÿãªã‚ªãƒ«ã‚½ãƒ­ã‚°æ¨è«–ãƒ„ãƒ¼ãƒ« JustOrthologs - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
### ORTHOSCOPE

å‹•ç”»
2019-12-07
https://doi.org/10.7875/togotv.2019.194
åˆ†å²å¹´ä»£ã®ãƒ™ã‚¤ã‚ºæ¨å®š @ åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨ ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—
0:00 - 15:00 / 1:04:11
ã‚ªãƒ¼ã‚½ã‚°ãƒ«ãƒ¼ãƒ—

2019 å¹´ 1 æœˆ 10 æ—¥ã€€æ”¹è¨‚
http://www.fish-evol.org/orthoscope_ji.html
ORTHOSCOPE ã¯éºä¼å­ç³»çµ±æ¨¹ã‚’æ¨å®šã—ã¦ï¼Œã‚ªãƒ¼ã‚½ãƒ­ã‚°ã‚’åˆ¤å®šã™ã‚‹ã‚¦ã‚§ãƒ–ãƒ„ãƒ¼ãƒ«ã§ã™ï¼ä»¥ä¸‹ã®è§£æãŒå¯èƒ½ã§ã™ï¼š

1. æœªçŸ¥ (ã‚ã‚‹ã„ã¯æ—¢çŸ¥) ã®éºä¼å­é…åˆ—ã®æ©Ÿèƒ½ã‚’æ¨å®šã™ã‚‹ï¼
2. ã‚ªãƒ¼ã‚½ãƒ­ã‚°ã®é…åˆ—ã‚’ï¼Œä»–ã®ç¨®ã‹ã‚‰å¾—ã‚‹ï¼
3. ã‚ªãƒ¼ã‚½ãƒ­ã‚°ãŒä»–ã®ç¨®ã«å­˜åœ¨ã™ã‚‹ç¢ºã‹ã‚ã‚‹ï¼

è§£æã«ã¯ï¼Œè„Šæ¤å‹•ç‰©ã‚’ä¸­å¿ƒã¨ã—ãŸå·¦å³ç›¸ç§°å‹•ç‰© 350 ç¨®ä»¥ä¸Šã®ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ (ã‚¢ãƒŸãƒé…¸é…åˆ—ã¨ DNA é…åˆ—) ã‚’ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã¨ã—ã¦è‡ªç”±ã«é¸ã¹ã¾ã™ï¼

----------
## alignment
ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ



http://kazumaxneo.hatenablog.com/archive/category/multiple%20sequence%20alignment%20%28MSA%29

https://pubmed.ncbi.nlm.nih.gov/32797207/
Syst Biol
. 2021 Apr 15;70(3):440-462. doi: 10.1093/sysbio/syaa064.
Do Alignment and Trimming Methods Matter for Phylogenomic (UCE) Analyses?
Daniel M Portik 1 2, John J Wiens 1
https://academic.oup.com/sysbio/article-abstract/70/3/440/5892776?redirectedFrom=fulltext

https://pubmed.ncbi.nlm.nih.gov/26031838/
Syst Biol
. 2015 Sep;64(5):778-91. doi: 10.1093/sysbio/syv033. Epub 2015 Jun 1.
Current Methods for Automated Filtering of Multiple Sequence Alignments Frequently Worsen Single-Gene Phylogenetic Inference
Ge Tan 1, Matthieu Muffato 2, Christian Ledergerber 3, Javier Herrero 4, Nick Goldman 2, Manuel Gil 5, Christophe Dessimoz 6
https://academic.oup.com/sysbio/article/64/5/778/1685763
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4538881/


https://pubmed.ncbi.nlm.nih.gov/17081313/
BMC Bioinformatics
. 2006 Nov 3;7:484. doi: 10.1186/1471-2105-7-484.
A statistical score for assessing the quality of multiple sequence alignments
Virpi Ahola 1, Tero Aittokallio, Mauno Vihinen, Esa Uusipaikka
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-484
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1687212/
https://bio.tools/multidisp
MultiDisp
http://structure.bmc.lu.se/MultiDisp/index.html



https://molevol.mbl.edu/index.php/Alignment_tutorial
Alignment tutorial - MolEvol
This page was last edited on 2 August 2019, at 14:13.


http://www.gen-info.osaka-u.ac.jp/~uhmin/study/history.html
History of uhmin tools
2013å¹´
4æœˆ5æ—¥
ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ Multiple sequence alignment
ãƒ»ã‚¢ãƒŸãƒé…¸ã®èª­ã¿æ ã‚’è€ƒæ…®ã—ãŸã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆãŒå¯èƒ½ã«ãªã‚Šã¾ã—ãŸã€‚
tranalignã‚’é¸æŠã—ã¦ã‚³ãƒ¼ãƒ‡ã‚£ãƒ³ã‚°ãƒ•ãƒ¬ãƒ¼ãƒ ã‚’å…¥åŠ›ã™ã‚‹ã¨ã€ ã‚¢ãƒŸãƒé…¸ã«ç¿»è¨³ã—ãŸå¾Œã« mafft ã®è‡ªå‹•ãƒ¢ãƒ¼ãƒ‰ã§ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã‚’è¡Œã„ã€ ã‚¢ãƒŸãƒé…¸ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã®çµæœã‚’å…ƒã«å¡©åŸºé…åˆ—ã‚’ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã—ã¾ã™ã€‚
ã‚¿ãƒ³ãƒ‘ã‚¯ã‚³ãƒ¼ãƒ‰é ˜åŸŸå¡©åŸºé…åˆ—ã®ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã®ã‚µã‚¤ãƒˆã‚’å‚è€ƒã«ã•ã›ã¦ã‚‚ã‚‰ã„ã¾ã—ãŸã€‚æ„Ÿè¬ï¼


æœ€çµ‚æ›´æ–°æ™‚é–“ï¼š2007å¹´03æœˆ17æ—¥ 05æ™‚57åˆ†52ç§’
https://www.fifthdimension.jp/wiki.cgi?page=%A5%BF%A5%F3%A5%D1%A5%AF%A5%B3%A1%BC%A5%C9%CE%CE%B0%E8%B1%F6%B4%F0%C7%DB%CE%F3%A4%CE%A5%DE%A5%EB%A5%C1%A5%D7%A5%EB%A5%A2%A5%E9%A5%A4%A5%E1%A5%F3%A5%C8
ã‚¿ãƒ³ãƒ‘ã‚¯ã‚³ãƒ¼ãƒ‰é ˜åŸŸå¡©åŸºé…åˆ—ã®ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ - Life is fifthdimension.

ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’å‚ç…§ã—ã¦ã®å¡©åŸºé…åˆ—ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ
ã€€EMBOSSã®tranalignã‚³ãƒãƒ³ãƒ‰ã‚’ç”¨ã„ã¦ã€ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆæ¸ˆã¿ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’ãƒªãƒ•ã‚¡ãƒ¬ãƒ³ã‚¹ã¨ã—ã¦å¡©åŸºé…åˆ—ã‚’ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã—ã¾ã™ã€‚ã“ã‚Œã§ç¿»è¨³å¾Œã®ã‚¢ãƒŸãƒé…¸é…åˆ—ã®å¤‰åŒ–ã‚’æœ€å°åŒ–ã™ã‚‹ã‚ˆã†ãªå¡©åŸºé…åˆ—ã®ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆãŒå®Œäº†ã—ã¾ã™ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/22936717
Mol Biol Evol. 2013 Jan;30(1):154-66. doi: 10.1093/molbev/mss210. Epub 2012 Aug 30.
Inferring the evolutionary history of IncP-1 plasmids despite incongruence among backbone gene trees.
Sen D1, Brown CJ, Top EM, Sullivan J.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3525142/
Nucleotide Sequence Alignments and Model Selection
The amino acid sequences of each gene were aligned with ClustalX (Thompson et al. 2002). Tranalign (Rice et al. 2000) was used to align the nucleotide sequences of each gene guided by the aligned amino acid sequences.

----------
### clustal-omega

https://kazumaxneo.hatenablog.com/entry/2020/07/30/073000
ãƒãƒ«ãƒãƒ—ãƒ«ã‚·ãƒ¼ã‚±ãƒ³ã‚¹ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã‚’è¡Œã† Clustal Omega - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

2017.04.09
https://bi.biopapyrus.jp/seq/alignment/software/clustal-omega.html
Clustal Omega

2015.03.16
https://doi.org/10.7875/togotv.2015.019
Clustal Omega ã‚’ä½¿ã£ã¦ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã‚’è¡Œã† | TogoTV

----------
### macse
MACSE: Multiple Alignment of Coding SEquences

https://bioweb.supagro.inra.fr/macse/

https://bioweb.supagro.inra.fr/macse/index.php?menu=downloadTuto
Download documents related to MACSE v2.03:
Download the tutorial file: doc_MACSE_v2.03.pdf
https://bioweb.supagro.inra.fr/macse/doc/doc_MACSE_v2.03.pdf
In the output alignment produced by MACSE, frameshifts are indicated using '!'. 

http://mbb.univ-montp2.fr/macse
http://mbb.univ-montp2.fr/MBB/

http://kazumaxneo.hatenablog.com/entry/2017/09/09/014350
ã‚³ãƒ¼ãƒ‰é ˜åŸŸã®ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’è€ƒãˆã¦ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã‚’è¡Œã†MUCSE - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
2016å¹´ã«version2ã«ã‚¢ãƒƒãƒ—ãƒ‡ãƒ¼ãƒˆã•ã‚Œã€2018å¹´å¤ã«v2ã®è«–æ–‡ãŒå‡ºã¾ã—ãŸã€‚

https://pubmed.ncbi.nlm.nih.gov/30165589/
Mol Biol Evol
. 2018 Oct 1;35(10):2582-2584. doi: 10.1093/molbev/msy159.
MACSE v2: Toolkit for the Alignment of Coding Sequences Accounting for Frameshifts and Stop Codons
Vincent Ranwez 1, Emmanuel J P Douzery 2, CÃ©dric Cambon 1 2, Nathalie Chantret 1, FrÃ©dÃ©ric Delsuc 2
https://academic.oup.com/mbe/article/35/10/2582/5079334
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6188553/

https://pubmed.ncbi.nlm.nih.gov/21949676/
PLoS One
. 2011;6(9):e22594. doi: 10.1371/journal.pone.0022594. Epub 2011 Sep 16.
MACSE: Multiple Alignment of Coding SEquences accounting for frameshifts and stop codons
Vincent Ranwez 1, SÃ©bastien Harispe, FrÃ©dÃ©ric Delsuc, Emmanuel J P Douzery
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3174933/
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0022594

----------
### translatorx
http://translatorx.co.uk

https://pubmed.ncbi.nlm.nih.gov/20435676/
Nucleic Acids Res
. 2010 Jul;38(Web Server issue):W7-13. doi: 10.1093/nar/gkq291. Epub 2010 Apr 30.
TranslatorX: multiple alignment of nucleotide sequences guided by amino acid translations
Federico Abascal 1, Rafael Zardoya, Maximilian J Telford
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896173/
https://academic.oup.com/nar/article/38/suppl_2/W7/1094709
Figure 1.
Example illustrating the different performance of the direct and back-translated nucleotide alignments (multiple alignments were built with Muscle with default parameters).

----------
### mafft

https://mafft.cbrc.jp/alignment/software/
MAFFT - a multiple sequence alignment program

Fasta format.  example1 (LSU rRNA), example2 (protein)

Manual (v6.240)
https://mafft.cbrc.jp/alignment/software/manual/manual.html
This manual does not cover all the features of the latest version.
See also tips and changelog. (Feb. 2011)

Tips (not yet included in the manual) for large alignment, ncRNA alignment, profile alignment, etc.
https://mafft.cbrc.jp/alignment/software/tips0.html
- How to get a guide tree or a rough clustering
- How to give a user-defined guide tree
- 
- Codon-based alignment or DNA alignment based on translated protein sequences

https://mafft.cbrc.jp/alignment/software/anysymbol.html
Ambiguous nucleotides (r, y, w, s, k, m, d, v, h, b; IUPAC-IUB codes) can be used and are scored as:
```
score(r,a) = ( score(a,a) + score(g,a) ) / 2
```

https://mafft.cbrc.jp/alignment/software/algorithms/algorithms.html
MAFFT ver.7 - a multiple sequence alignment program
```
Algorithms and parameters (unfinished)
(a) FFT-NS-1, FFT-NS-2 â€” Progressive methods
(b) FFT-NS-i, NW-NS-i â€” Iterative refinement method
(c) L-INS-i, E-INS-i, G-INS-i â€” Iterative refinement methods using WSP and consistency scores
```
For pairwise alignment, three different types of algorithms are implemented, global alignment (Needleman-Wunsch), local alignment (Smith-Waterman) with affine gap costs (Gotoh) and local alignment with generalized affine gap costs (Altschul). The differences in the accuracy values among these methods are small for the currently available benchmarks, as shown here. However, each of them has different characteristics, according to the algorithm in the pairwise alignment stage:


https://mafft.cbrc.jp/alignment/software/manual/manual.html
Accuracy-oriented methods:

*L-INS-i (probably most accurate; recommended for <200 sequences; iterative refinement method incorporating local pairwise alignment information):
```
mafft --localpair --maxiterate 1000 input [> output]
linsi input [> output]
```

*G-INS-i (suitable for sequences of similar lengths; recommended for <200 sequences; iterative refinement method incorporating global pairwise alignment information):
```
mafft --globalpair --maxiterate 1000 input [> output]
ginsi input [> output]
```

*E-INS-i (suitable for sequences containing large unalignable regions; recommended for <200 sequences):
```
mafft --ep 0 --genafpair --maxiterate 1000 input [> output]
einsi input [> output]
```
For E-INS-i, the --ep 0 option is recommended to allow large gaps.


--genafpair
All pairwise alignments are computed with a local algorithm with the generalized affine gap cost (Altschul 1998). More accurate but slower than --6merpair. Suitable when large internal gaps are expected. Applicable to up to ~200 sequences. A combination with --maxiterate 1000 is recommended (E-INS-i). Default: off (6mer distance is used)


2006 å¹´ 44 å·» 2 å· p. 102-108
https://www.jstage.jst.go.jp/article/kagakutoseibutsu1962/44/2/44_2_102/_article/-char/ja/
å®Ÿè·µçš„ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆ
åŠ è—¤ å’Œè²´, éšˆ å•“ä¸€
https://www.jstage.jst.go.jp/article/kagakutoseibutsu1962/44/2/44_2_102/_pdf/-char/ja
--globalpair ã¨--localpair ã¯é€šå¸¸ã®affine gap
cost(15)ã‚’ ç”¨ ã„ ã‚‹ ãŒ, --genafpair ã¯ generalized affine
gap cost(16)ã¨ ã„ã†, ã‚ˆ ã‚Šè¤‡é›‘ãªã‚® ãƒ£ãƒƒãƒ—ã‚³ã‚¹ ãƒˆã‚’ç”¨ã„ ã‚‹. generalized affine gap cost ã¯, ã‚¢ ãƒ© ã‚¤ ãƒ³ ãƒ¡ ãƒ³ ãƒˆã‚’
ä½œæˆã™ã‚‹ã“ã¨ãŒç„¡æ„å‘³ã§ã‚ã‚‹ã»ã©é¡ä¼¼åº¦ã®ä½ã„é ˜åŸŸã‚’å« ã‚“ã§ã„ã‚‹ã¨ãã«æœ‰åŠ¹ã§ã‚ã‚‹ã¨è€ƒãˆã¦å°å…¥ ã—ãŸãŒ, æœ‰åŠ¹æ€§ ã‚’æ˜ç¢º ã«ç¤ºã™ã“ã¨ã¯ã§ãã¦ã„ãªã„. 


https://github.com/DomBennett/om..mafft
Run mafft through outsider in R
```
#> There are 10 ambiguous characters.
#>     1 / 13
#> done.
```

3:17 PM Â· Oct 21, 2019
https://twitter.com/chasewnelson/status/1186164467622498305
Chase W. Nelson on Twitter: "@MinionLab @AprilWei001 The #MAFFT output for the same data was (1) faster and (2) makes actual sense. Thanks again! https://t.co/8bR2nJt1t1" / Twitter

12:36 PM Â· Sep 28, 2019
https://twitter.com/sighex/status/1177789066097487872
å·¥æ¨‚æ¨¹æ´‹
"MAFFTé–‹ç™ºç§˜è©±ã€‚ç§ãŒ4å›ç”Ÿã§å®®ç”°ç ”ã«åŠ ã‚ã£ãŸæ™‚ã€ã¾ãšæ‰‹å‹•ã§å¤šé‡ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã™ã‚‹è¨“ç·´ã‹ã‚‰å§‹ã¾ã‚Šã¾ã—ãŸã€‚ç ”ç©¶å®¤å†…éƒ¨ã§ä½¿ã‚ã‚Œã¦ã„ãŸãã®ã‚¨ãƒ‡ã‚£ã‚¿ã«ã€å¾Œã«MAFFTã®ä¸»è»¸ã¨ãªã‚‹ã‚¢ãƒ«ã‚´ãƒªã‚ºãƒ ãŒé–“ã‚‚ãªãå…¥ã‚Šã¾ã—ãŸã€‚Xcedã¨å‘¼ã°ã‚ŒãŸã€ãƒ¡ãƒ³ãƒãƒ¼ã—ã‹çŸ¥ã‚‰ãªã„ã‚¨ãƒ‡ã‚£ã‚¿ã§ã™ã€‚

http://kazumaxneo.hatenablog.com/entry/2017/10/18/000312
ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã‚’è¡Œã†MAFFT - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

http://kazumaxneo.hatenablog.com/entry/2018/07/20/212444
webä¸Šã§ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã‚’å®Ÿè¡Œã—åˆ†å­ç³»çµ±æ¨¹ã‚’å‡ºåŠ›ã™ã‚‹ MAFFT online service - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹


----------
### gap

https://sci-tech.ksc.kwansei.ac.jp/~tohhiro/bioinfo18/bioinfo2018-2.pdf
è—¤ åšå¹¸
ã‚¢ãƒ•ã‚£ãƒ³ãƒ» ã‚®ãƒ£ãƒƒãƒ—ãƒ»ãƒšãƒŠãƒ«ãƒ†ã‚£

2010/05/21
æŸæœ¨ æ˜åš
https://sites.google.com/site/kashiwagiakihiro/Home/local-alignment-and-affine-gap-penalty
ãƒ­ãƒ¼ã‚«ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã¨ã‚¢ãƒ•ã‚£ãƒ³ã‚®ãƒ£ãƒƒãƒ—ãƒšãƒŠãƒ«ãƒ†ã‚£
 ã¾ãŸã€Gap ãŒé€£ç¶šã—ãŸã¨ãã€é€£ç¶šã—ãŸ Gap ã‚‚åŒã˜å€¤ã‚’ä½¿ã£ãŸç·šå½¢ Gap Penalty å€¤ã‚’ä½¿ç”¨ã—ã¦ã„ã¾ã—ãŸãŒã€
ä»Šå›ã¯ã€Gap ãŒé€£ç¶šã—ãŸæ™‚ã«ã¯ã€2 ã¤ç›®ä»¥é™ã® Gap ã«ã¯é•ã†å€¤ã€é€šå¸¸æœ€åˆã® Gap ã‚ˆã‚Šå°ã•ãªå€¤ã‚’ä½¿ç”¨ã™ã‚‹
Affine Gap Penalty ã«å¯¾å¿œã—ã¾ã™ã€‚


https://doi.org/10.7875/togotv.2019.193
2019-12-06 MEGA X ã‚’ç”¨ã„ãŸåˆ†å­ç³»çµ±è§£æ @ åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨ ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ—
41:40 / 52:32
GapãŒå…¥ã‚‹ã¨ã“ã‚ã¯æ¨ã¦ã‚‹ã€‚
GapãŒå…¥ã‚‰ãªã„ã¨ã“ã‚ã ã‘ä½¿ã†ã€‚

http://www.tezuru-mozuru.com/?tag=mega
- Gaps/Missing Data Treatmentï¼šã‚®ãƒ£ãƒƒãƒ—ã®æ‰±ã„ã®è¨­å®šã§ã™ï¼Complete deletionã¨ã™ã‚‹ã¨ï¼Œä¸€ã¤ã§ã‚‚ã‚®ãƒ£ãƒƒãƒ—ãŒã‚ã‚‹ã‚µã‚¤ãƒˆï¼ˆåˆ—ï¼‰ã‚’è§£æã‹ã‚‰ã®ãã‘ã¾ã™ï¼
- Gaps/Missing Data Treatmentï¼šã‚®ãƒ£ãƒƒãƒ—ã‚„ãƒŸãƒƒã‚·ãƒ³ã‚°ãƒ‡ãƒ¼ã‚¿ã®å–ã‚Šæ‰±ã„ãŒé¸ã¹ã¾ã™ï¼Complete deletionã§ï¼Œã‚®ãƒ£ãƒƒãƒ—ãŒä¸€ã¤ã§ã‚‚ã‚ã‚‹åˆ—ã¯è§£ææ®»é™¤ãã¾ã™ï¼Partial Deletionã§ï¼Œæ¯”è¼ƒã™ã‚‹é…åˆ—ã”ã¨ã«ã‚®ãƒ£ãƒƒãƒ—ã®æ‰±ã„ã‚’æ±ºã‚ã¾ã™ï¼Use all siteã§ã‚®ãƒ£ãƒƒãƒ—ã‚‚ãƒŸãƒƒã‚·ãƒ³ã‚°ãƒ‡ãƒ¼ã‚¿ã‚‚å…¨ã¦æ‰±ã„ã¾ã™ï¼


June 10, 2011
https://a-little-book-of-r-for-bioinformatics.readthedocs.io/en/latest/index.html
https://github.com/haruosuz/r4bioinfo/blob/master/R_Avril_Coghlan/README.md#discarding-very-poorly-conserved-regions-from-an-alignment
ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã‹ã‚‰ä¿å­˜åº¦ã®ä½ã„é ˜åŸŸã‚’ç ´æ£„ã™ã‚‹
Trimming a multiple sequence alignment by discarding columns with too many gaps.
å¤šé‡é…åˆ—ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆã‹ã‚‰ã‚®ãƒ£ãƒƒãƒ—ã®å¤šã„åˆ—ã‚’ç ´æ£„ã™ã‚‹

----------
### Gblocks

http://sy41211.hatenablog.com/entry/2015/12/04/195438
é…åˆ—ä¿å­˜é ˜åŸŸã®æŠ½å‡ºãƒ—ãƒ­ã‚°ãƒ©ãƒ Gblocksã®ä½¿ã„æ–¹ - ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹åˆå¿ƒè€…ã®æ—¥å¸¸

http://molevol.cmima.csic.es/castresana/Gblocks.html
http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html
t. specifies the type of sequences in the current alignment. It can be Protein, DNA or Codons. In protein alignments, the 20 amino acid letters are used to calculate the degree of conservation of positions, and it is possible to invoke the use of a similarity matrix. In DNA and codon alignments, only A, C, G, T and U letters are considered (other symbols are allowed but do not count in the calculations). In codon alignments, selected blocks are made to contain only complete codons (if the alignment is really based on codons).

-t=	Type Of Sequence
(Protein, DNA, Codons)	p, d, c

https://www.ncbi.nlm.nih.gov/pubmed/17654362
Syst Biol. 2007 Aug;56(4):564-77.
Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments.
Talavera G1, Castresana J.

https://www.ncbi.nlm.nih.gov/pubmed/10742046
Mol Biol Evol. 2000 Apr;17(4):540-52.
Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis.
Castresana J1.

----------
### trimAl

http://trimal.cgenomics.org

http://trimal.cgenomics.org/getting_started_with_trimal_v1.2
```
   trimal -in example1 -out output1 -htmlout output1.html -gt 1
```
will remove all columns with any gap (equivalent to -nogaps option)


https://pubmed.ncbi.nlm.nih.gov/19505945/
Bioinformatics
. 2009 Aug 1;25(15):1972-3. doi: 10.1093/bioinformatics/btp348. Epub 2009 Jun 8.
trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses
Salvador Capella-GutiÃ©rrez 1, JosÃ© M Silla-MartÃ­nez, Toni GabaldÃ³n
https://academic.oup.com/bioinformatics/article/25/15/1972/213148
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2712344/
In most scenarios (90%), trimAl outperformed Gblocks v0.91b with default parameters. 

http://www.fish-evol.org/trimAl_ji.html
trimAl - JI
2016 å¹´ 3 æœˆ 31æ—¥ã€€æ”¹è¨‚
äº•ä¸Š æ½¤
ä»¥ä¸‹ã®ã‚ˆã†ã«ï¼Œè‡ªå‹•çš„ã«å‰Šé™¤ã™ã‚‹ã‚µã‚¤ãƒˆã‚’é¸ã‚“ã§ãã‚Œã‚‹ã‚ªãƒ—ã‚·ãƒ§ãƒ³ (-gappyout) ã‚‚ã‚ã‚Šã¾ã™ï¼ã‚ˆã‚Šå³ã—ãå‰Šã‚‹å ´åˆã¯ -strict ã‹ -strictplus ã‚’é¸ã³ã¾ã™ï¼

https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.html
1.5.3 æ•´åˆ—ã®ä¿¡é ¼ã§ããªã„åº§ä½
å½éºä¼å­ã‚„éºä¼å­é–“é ˜åŸŸã€ã‚¤ãƒ³ãƒˆãƒ­ãƒ³ã€rRNA/tRNAã®loopé ˜åŸŸãªã©ã®æ¬ å¤±ã‚„æŒ¿å…¥ã®å¤šã„é…åˆ—ã§ã¯ã€æ•´åˆ—ã®ä¿¡é ¼æ€§ãŒä½ããªã£ã¦ã—ã¾ã„ã¾ã™ã€‚èª¤ã£ã¦æ•´åˆ—ã•ã‚ŒãŸåº§ä½ã¯ã€ç³»çµ±æ¨¹æ¨å®šã®éš›ã®ãƒã‚¤ã‚ºã¨ãªã£ã¦ã—ã¾ã†ãŸã‚ã€é™¤å»ã—ãŸæ–¹ãŒã‚ˆã„ã¨è¨€ã‚ã‚Œã¦ã„ã¾ã™(Talavera and Castresana, 2007)ã€‚ã“ã‚Œã¾ã§ã®ã¨ã“ã‚ã€ãã®ã‚ˆã†ãªå‡¦ç†ãŒç ”ç©¶è€…ã®çµŒé¨“ã¨å‹˜ã§ãªã•ã‚Œã‚‹ã“ã¨ãŒå¤šã‹ã£ãŸã®ã§ã™ãŒã€è¿‘å¹´ã«ãªã£ã¦è‡ªå‹•çš„ã«è¡Œã£ã¦ãã‚Œã‚‹ã‚½ãƒ•ãƒˆã‚¦ã‚§ã‚¢ãŒç™»å ´ã—ã¦ãã¾ã—ãŸã€‚ãã‚ŒãŒGblocks (Castresana, 2000)ãƒ»trimAl (Capella-GutiÃ©rrez et al., 2009)ãƒ»Aliscore (Misof and Misof, 2009)ãƒ»BMGE (Criscuolo and Gribaldo, 2010)ã§ã™ã€‚ã“ã“ã§ã¯trimAlã‚’ç”¨ã„ã¦æ•´åˆ—ã®ä¿¡é ¼ã§ããªã„åº§ä½ã‚’ãƒˆãƒªãƒŸãƒ³ã‚°ã™ã‚‹æ‰‹é †ã‚’èª¬æ˜ã—ã¾ã™ã€‚

https://www.fifthdimension.jp/wiki.cgi?page=%CA%AC%BB%D2%B0%E4%C5%C1%B3%D8%B4%D8%CF%A2
åˆ†å­éºä¼å­¦é–¢é€£ - Life is fifthdimension.
 Gblocks
ã€€multiple sequence alignmentã‹ã‚‰alignmentã®ä¿¡é ¼æ€§ã®ä½ã„åº§ä½ã‚’æ¢ã—å‡ºã—ã¦é™¤å»ã™ã‚‹ãŸã‚ã®ã‚½ãƒ•ãƒˆã€‚
 trimAl
ã€€Gblocksã‚’ç™ºå±•ã•ã›ã¦ãƒ‘ãƒ©ãƒ¡ãƒ¼ã‚¿è‡ªå‹•è¨­å®šæ©Ÿèƒ½ã‚’å……å®Ÿã•ã›ãŸã‚ˆã†ãªã‚‚ã®ã€‚


----------
## tree

21 Nov 2017
https://twitter.com/RokasLab/status/933007398398251009
RokasLab on Twitter: "Interested on how fast ML phylogenetic programs (RAxML/ExaML, PhyML, IQ-TREE, &FastTree) perform on state-of-the-art empirical phylogenomicâ€¦ https://t.co/tRfWRSZawe"
https://www.ncbi.nlm.nih.gov/pubmed/29177474
Mol Biol Evol. 2018 Feb 1;35(2):486-503. doi: 10.1093/molbev/msx302.
Evaluating Fast Maximum Likelihood-Based Phylogenetic Programs Using Empirical Phylogenomic Data Sets.
Zhou X1,2, Shen XX3, Hittinger CT4, Rokas A3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850867/

### ezTree
https://github.com/yuwwu/ezTree

https://www.ncbi.nlm.nih.gov/pubmed/29363425
BMC Genomics. 2018 Jan 19;19(Suppl 1):921. doi: 10.1186/s12864-017-4327-9.
ezTree: an automated pipeline for identifying phylogenetic marker genes and inferring evolutionary relationships among uncultivated prokaryotic draft genomes.
Wu YW1.

http://kazumaxneo.hatenablog.com/entry/2018/09/12/221645
phylogenetic marker genesã‚’æ¤œå‡ºã—ã€marker geneså…¨ã¦ã‚’ä½¿ã£ã¦ç³»çµ±æ¯”è¼ƒã™ã‚‹è‡ªå‹•åŒ–ã•ã‚ŒãŸãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³ ezTree - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

### bcgTree

https://www.ncbi.nlm.nih.gov/pubmed/27603265
Genome. 2016 Oct;59(10):783-791. Epub 2016 May 11.
bcgTree: automatized phylogenetic tree building from bacterial core genomes.
Ankenbrand MJ1,1, Keller A1,1.
http://www.nrcresearchpress.com/doi/full/10.1139/gen-2015-0175?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed
The pipeline automatically extracts 107 essential single-copy core genes, found in a majority of bacteria, using hidden Markov models and performs a partitioned maximum-likelihood analysis. 

https://github.com/molbiodiv/bcgTree
Automatically calculate phylogenetic trees from bacterial core genes

See this file for instructions on how to reproduce results from our article.
https://github.com/molbiodiv/bcgTree/blob/master/reproduce_results.org

### PhyloPhlAn

https://huttenhower.sph.harvard.edu/phylophlan

https://www.ncbi.nlm.nih.gov/pubmed/23942190
Nat Commun. 2013;4:2304. doi: 10.1038/ncomms3304.
PhyloPhlAn is a new method for improved phylogenetic and taxonomic placement of microbes.
Segata N1, BÃ¶rnigen D, Morgan XC, Huttenhower C.

https://github.com/alvaralmstedt/Tutorials/wiki/Creating-bacterial-phylogenetic-trees-with-PhyloPhlAn
Creating bacterial phylogenetic trees with PhyloPhlAn
alvaralmstedt edited this page on Aug 13, 2015

https://jpn.bioconus.com/phylophlan-is-new-method-795397
ãƒ•ã‚£ãƒ­ãƒ•ãƒ©ãƒ³ã¯ã€å¾®ç”Ÿç‰©ã®ç³»çµ±ç™ºç”Ÿå­¦çš„ãŠã‚ˆã³åˆ†é¡å­¦çš„é…ç½®ã‚’æ”¹å–„ã™ã‚‹ãŸã‚ã®æ–°ã—ã„æ–¹æ³•ã§ã™ã€‚ è‡ªç„¶é€šä¿¡ - ã‚³ãƒŸãƒ¥ãƒ‹ã‚±ãƒ¼ã‚·ãƒ§ãƒ³ - 2019

http://kazumaxneo.hatenablog.com/entry/2018/09/12/221645
phylogenetic marker genesã‚’æ¤œå‡ºã—ã€marker geneså…¨ã¦ã‚’ä½¿ã£ã¦ç³»çµ±æ¨¹ã‚’ä½œæˆã™ã‚‹è‡ªå‹•åŒ–ãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³ ezTree - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
PhyloPhlAn (Segata et al,. 2014) ã‚‚ç°¡å˜ã«ç´¹ä»‹ã—ã¦ã„ã¾ã™ã€‚

http://kazumaxneo.hatenablog.com/entry/2017/05/22/164017
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ« - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
ä½¿ã£ã¦ã¿ã¦ä¾¿åˆ©ã ã£ãŸãƒ„ãƒ¼ãƒ«ã‚’ç´¹ä»‹ã™ã‚‹ã€‚
PhyloPhlAn Segata et al. (2014)
ãƒ©ãƒ³ã«ã¯FasttreeãŒå¿…è¦ã€‚

### etetoolkit
http://etetoolkit.org/
ETE Toolkit - Analysis and Visualization of (phylogenetic) trees


http://etetoolkit.org/download/

http://etetoolkit.org/cookbook/
ETE Cookbook

2019å¹´12æœˆ11æ—¥ã«æ›´æ–°
ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹ Advent Calendar 2019 11æ—¥ç›®
https://qiita.com/aical/items/2231198d3d75e72bee39
èª°ã«ã‚‚æ•™ãˆãŸããªã„è¶…ã‚«ãƒ³ã‚¿ãƒ³åˆ†å­ç³»çµ±æ¨¹ä½œæˆè¡“ - Qiita
ETE Toolkitã‚’ä½¿ã£ãŸè¶…ç°¡ä¾¿ãªåˆ†å­ç³»çµ±æ¨¹ä½œæˆæ³•ã‚’ç´¹ä»‹ã—ã¾ã™ã€‚

https://www.hobochuritsu.com/entry/2018/11/08/223813
ETE Toolkitã§ãƒãƒ¼ãƒ‰ã«é€šã—ç•ªå·ã‚’ä»˜ã‘ã‚‹ - ã»ã¼ä¸­ç«‹ãƒ–ãƒ­ã‚°

2017å¹´05æœˆ26æ—¥
https://qiita.com/okuman/items/3dd1cd1292940442fcd0
ETE Toolkitã‚’ä½¿ã£ã¦Pythonã§è‰²ã€…ãªå½¢ã®ç³»çµ±æ¨¹ã‚’æ›¸ã - Qiita

### root_digger

https://github.com/computations/root_digger
RootDigger is a program that will, when given a MSA and an unrooted tree with branch lengths place a root on the given tree. For the foreseeable future, RootDigger will only support DNA data, as the method RootDigger uses is ineffective when using AA data.
RootDiggerã¯ã€MSAã¨ã€æã®é•·ã•ãŒæŒ‡å®šã•ã‚ŒãŸæ ¹ã®ãªã„æœ¨ãŒä¸ãˆã‚‰ã‚Œã‚‹ã¨ã€ä¸ãˆã‚‰ã‚ŒãŸæœ¨ã«æ ¹ã‚’å¼µã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã§ã™ã€‚RootDiggerãŒä½¿ç”¨ã™ã‚‹æ–¹æ³•ã¯AAãƒ‡ãƒ¼ã‚¿ã‚’ä½¿ç”¨ã™ã‚‹éš›ã«ã¯åŠ¹æœãŒãªã„ãŸã‚ã€å½“é¢ã®é–“ã€RootDiggerã¯DNAãƒ‡ãƒ¼ã‚¿ã®ã¿ã‚’ã‚µãƒãƒ¼ãƒˆã™ã‚‹ã€‚

2021-05-01
https://pubmed.ncbi.nlm.nih.gov/33932975/
BMC Bioinformatics
. 2021 May 1;22(1):225. doi: 10.1186/s12859-021-03956-5.
Root Digger: a root placement program for phylogenetic trees
Ben Bettisworth 1, Alexandros Stamatakis 2 3
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03956-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8088003/

### iqtree

- http://www.iqtree.org/
IQ-TREE: Efficient phylogenomic software by maximum likelihood
- http://www.iqtree.org/doc/Quickstart
Getting Started
- http://www.iqtree.org/doc/Tutorial
Beginner's Tutorial
- http://www.iqtree.org/doc/Advanced-Tutorial
Advanced Tutorial
- http://www.iqtree.org/doc/Dating
Phylogenetic Dating
- http://www.iqtree.org/doc/Frequently-Asked-Questions
Frequently Asked Questions
- http://www.iqtree.org/doc/Substitution-Models
Substitution Models

http://www.iqtree.org/doc/
 IQ-TREE Tutorials and Manual in one PDF
http://www.iqtree.org/doc/iqtree-doc.pdf
IQ-TREE version 2.1.2: Tutorials and Manual
Phylogenomic software by maximum likelihood
http://www.iqtree.org
Bui Quang Minh, Rob Lanfear, Nhan-Trong Ly
Jana Trifinopoulos, Dominik Schrempf, Heiko A. Schmidt
November 1, 2021


https://pubmed.ncbi.nlm.nih.gov/32011700/
Mol Biol Evol
. 2020 May 1;37(5):1530-1534. doi: 10.1093/molbev/msaa015.
IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era
Bui Quang Minh 1 2, Heiko A Schmidt 3, Olga Chernomor 3, Dominik Schrempf 3 4, Michael D Woodhams 5, Arndt von Haeseler 3 6, Robert Lanfear 2
https://academic.oup.com/mbe/article/37/5/1530/5721363
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182206/


6:19 PM Â· May 6, 2020
https://twitter.com/RobLanfear/status/1257963198474403840
Rob Social Distancing Lanfear on Twitter: "Here's what I've been doing all day. If you're looking to estimate a single tree topology for SARS-CoV-2 sequences, it's very hard to beat Maximum Parsimony. rapidnj is also very good (not significantly different from MP) and *unbelievably* fast. https://t.co/QNKdnxORwC" / Twitter
Note that if you use IQ-TREE with the -fast option (or any option, actually), you will get a tree that's at least as good as the MP tree. That's because IQ-TREE always uses at least one MP tree as a starting tree and tries to improve it.

http://www.iqtree.org/doc/Tutorial#choosing-the-right-substitution-model

#### iqtree_ModelFinder

http://www.iqtree.org/ModelFinder/

https://pubmed.ncbi.nlm.nih.gov/28481363/
Nat Methods
. 2017 Jun;14(6):587-589. doi: 10.1038/nmeth.4285. Epub 2017 May 8.
ModelFinder: fast model selection for accurate phylogenetic estimates
Subha Kalyaanamoorthy 1 2, Bui Quang Minh 3, Thomas K F Wong 1 4, Arndt von Haeseler 3 5, Lars S Jermiin 1 4
https://www.nature.com/articles/nmeth.4285
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5453245/

#### iqtree_Rootstrap

Last update: Jun 7, 2024, Contributors: Minh Bui, Suha Naser
http://www.iqtree.org/doc/Rootstrap
Rooting phylogenies
http://www.iqtree.org/doc/Rootstrap#inferring-rooted-trees-without-outgroup

2022-06-16
https://pubmed.ncbi.nlm.nih.gov/34387349/
Syst Biol
. 2022 Jun 16;71(4):959-972. doi: 10.1093/sysbio/syab067.
Assessing Confidence in Root Placement on Phylogenies: An Empirical Study Using Nonreversible Models for Mammals
Suha Naser-Khdour 1, Bui Quang Minh 1 2, Robert Lanfear 1
https://academic.oup.com/sysbio/article/71/4/959/6350503
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9260635/

#### iqtree_Assessing-Phylogenetic-Assumptions

Last update: Mar 15, 2021, Contributors: Minh Bui
http://www.iqtree.org/doc/Assessing-Phylogenetic-Assumptions
Assessing Phylogenetic Assumptions

Tests of symmetry
IQ-TREE provides three matched-pairs tests of symmetry (Naser-Khdour et al., 2019) to test the two assumptions of stationarity and homogeneity. A simple analysis:

2019-12-01
https://pubmed.ncbi.nlm.nih.gov/31536115/
Genome Biol Evol
. 2019 Dec 1;11(12):3341-3352. doi: 10.1093/gbe/evz193.
The Prevalence and Impact of Model Violations in Phylogenetic Analysis
Suha Naser-Khdour 1, Bui Quang Minh 1 2, Wenqi Zhang 1, Eric A Stone 1, Robert Lanfear 1
https://academic.oup.com/gbe/article/11/12/3341/5571717
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6893154/
In phylogenetic inference, we commonly use models of substitution which assume that sequence evolution is stationary, reversible, and homogeneous (SRH).

#### iqtree_Concordance-Factor

Last update: May 27, 2024, Contributors: Minh Bui, Rob Lanfear, Trongnhan Uit
http://www.iqtree.org/doc/Concordance-Factor
Concordance Factor

HINT: See very nice tips on how to use and interpret concordance factors written by Rob Lanfear.
14 December 2018
https://www.robertlanfear.com/blog/files/concordance_factors.html
Calculating and interpreting gene- and site-concordance factors in phylogenomics | None | The Lanfear Lab @ANU

2020-09
https://pubmed.ncbi.nlm.nih.gov/32365179/
Mol Biol Evol
. 2020 Sep 1;37(9):2727-2733. doi: 10.1093/molbev/msaa106.
New Methods to Calculate Concordance Factors for Phylogenomic Datasets
Bui Quang Minh 1 2, Matthew W Hahn 3 4, Robert Lanfear 2
https://academic.oup.com/mbe/article/37/9/2727/5828940
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7475031/

#### iqtree_UFBoot

https://pubmed.ncbi.nlm.nih.gov/29077904/
Mol Biol Evol
. 2018 Feb 1;35(2):518-522. doi: 10.1093/molbev/msx281.
UFBoot2: Improving the Ultrafast Bootstrap Approximation
Diep Thi Hoang 1, Olga Chernomor 2, Arndt von Haeseler 2 3, Bui Quang Minh 2, Le Sy Vinh 1
https://academic.oup.com/mbe/article/35/2/518/4565479
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850222/


#### iqtree_lang_ja

https://twitter.com/search?q=iqtree%20lang%3Aja&f=live

2023å¹´08æœˆ07æ—¥
https://qiita.com/YF_bio/items/d952f54c8d570a6ea0cb
IQ-TREEã§ç³»çµ±æ¨¹ã®æœ€å°¤æ¨å®š #bioinformatics - Qiita

http://kazumaxneo.hatenablog.com/entry/2020/06/07/235834
IQ-TREE ã‚’webã§ä½¿ãˆã‚‹ W-IQ-TREE - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
æ—¢ã«IQ-TREE2ã‚‚ç™ºè¡¨ï¼ˆpubmedï¼‰ã•ã‚Œã¦ãŠã‚Šã€condaã‚’ä½¿ã£ã¦å°å…¥ã§ãã‚‹ãƒãƒ¼ã‚¸ãƒ§ãƒ³ã‚‚ï¼’ã«ãªã£ã¦ã„ã¾ã™ãŒã€2020å¹´6æœˆç¾åœ¨ã€webç‰ˆã¯stable versionã®1.6ã§ã™ã€‚æ³¨æ„ã—ã¦ä¸‹ã•ã„ã€‚

https://www.hobochuritsu.com/entry/2018/10/21/125017
IQ-Treeã®ä½¿ã„æ–¹ã¾ã¨ã‚ - ã»ã¼ä¸­ç«‹ãƒ–ãƒ­ã‚°

2017å¹´3æœˆ22æ—¥
http://www.tezuru-mozuru.com/?p=9927
iqtreeã«ã‚ˆã‚‹æœ€å°¤æ³•ç³»çµ±æ¨¹æ¨å®š â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹

----------
### QMaker

https://pubmed.ncbi.nlm.nih.gov/33616668/
Syst Biol
. 2021 Feb 22;syab010. doi: 10.1093/sysbio/syab010. Online ahead of print.
QMaker: Fast and accurate method to estimate empirical models of protein evolution
Bui Quang Minh 1 2, Cuong Cao Dang 3, Le Sy Vinh 3, Robert Lanfear 2
https://doi.org/10.1093/sysbio/syab010


Software implementation
We provided an implementation of QMaker as part of the IQ-TREE software version
2.0-rc1. The entire training stage for the Pfam dataset can be accomplished with just two
command lines. The first one is
iqtree -S ALN_DIR -nt 48

All data are available from the online supplementary material
(https://doi.org/10.6084/m9.figshare.9768101).


7:28 AM Â· Feb 23, 2021
https://twitter.com/RobLanfear/status/1363979095390912513
Rob Social Distancing Lanfear on Twitter: "Pleased to announce the birth of QMaker, which allows you to easily estimate amino-acid replacement matrices from your own data. Doing this work with @bq_minh, Cuong Cao Dang, and Le Sy Vinh threw up a few surprises, so here's a little  https://t.co/wWlFi5pHAf" / Twitter


https://www.biorxiv.org/content/10.1101/2020.02.20.958819v1
QMaker: Fast and accurate method to estimate empirical models of protein evolution | bioRxiv
https://www.biorxiv.org/content/10.1101/2020.02.20.958819v1.full


8:20 PM Â· Oct 8, 2020
https://twitter.com/kfuku0502/status/1314163771741556738
iqtreeã§çµŒé¨“ãƒ¢ãƒ‡ãƒ«ä½œã‚Œã‚‹æ–¹æ³•ã®ãƒ—ãƒ¬ãƒ—ãƒªãƒ³ãƒˆå‡ºã¦ãŸæ°—ãŒã™ã‚‹ã‘ã©ãªã‚“ã ã£ãŸã‘ï¼Ÿã‹ã‚‰30åˆ†ãã‚‰ã„ã‹ã‘ã¦ã‚„ã£ã¨è¾¿ã‚Šç€ã„ãŸã®ã§ãƒ¡ãƒ¢ã€‚QMakerã ã£ãŸã€‚
ã§ã‚‚ã“ã‚Œã‚¢ãƒŸãƒé…¸é…åˆ—é™å®šã‹ã€‚ã‚³ãƒ‰ãƒ³ã®ã‚’ä½œã‚ŠãŸã„ã‚“ã ã‚ˆãªã€‚ãƒŸãƒˆã‚³ãƒ³ã¨ã‹ãƒ—ãƒ©ã‚¹ãƒãƒ‰ç”¨ã®çµŒé¨“çš„ã‚³ãƒ‰ãƒ³ãƒ¢ãƒ‡ãƒ«ã£ã¦ã©ã“ã‹ã«ãªã„ã§ã—ã‚‡ã†ã‹ï¼Ÿã‚‚ã—æ±ç”¨çš„ãªã®ãŒã‚ã‚Œã°è‡ªå‰ã§ä½œã‚‹å¿…è¦ãªã„ã‚“ã§ã™ãŒã€‚

6:15 AM Â· Mar 15, 2020
https://twitter.com/kfuku0502/status/1238936843028254720
IQ-TREEã§ã‚µãƒãƒ¼ãƒˆã•ã‚Œã¦ã„ã‚‹ã‚‰ã—ã„æ–°ã—ã„ã‚¢ãƒŸãƒé…¸ç½®æ›ãƒ¢ãƒ‡ãƒ«ã®ãƒ—ãƒ¬ãƒ—ãƒªãƒ³ãƒˆã€‚å¤§ã¾ã‹ãªç³»çµ±ã”ã¨ã«çµŒé¨“çš„ç½®æ›ãƒ¢ãƒ‡ãƒ«ã‚’ä½œã£ã¦ã„ã‚‹ã€‚Fig. 2ã‚’è¦‹ã‚‹ã¨ã€ãªãœã‹æ¤ç‰©ã§ä»–ã®ç³»çµ±ã‚ˆã‚Šã‚¹ã‚³ã‚¢ãŒã„ã„ã€‚





----------

### VeryFastTree
https://github.com/citiususc/veryfasttree.


7:50 AM Â· Mar 11, 2021
https://twitter.com/RobLanfear/status/1369782808558772226
VeryFastTree takes twice as long to do SPR moves as FastTree. Go figure! (Supposed to be the same codebase...)


Dec 1, 2020
https://twitter.com/kfuku0502/status/1333425581963292672
ç³»çµ±æ¨å®šç•Œã®ã‚¹ãƒ”ãƒ¼ãƒ‰ç‹‚ã€FastTree2ãŒæ›´ãªã‚‹ä¸¦åˆ—åŒ–ã‚’å«ã‚€ã‚«ãƒªãƒƒã‚«ãƒªã®ãƒãƒ¥ãƒ¼ãƒ‹ãƒ³ã‚°ã‚’çµŒã¦VeryFastTreeã¸é€²åŒ–ã€‚331kå€‹ã®16S rRNAé…åˆ—ã®ç³»çµ±æ¨å®šãŒ4.5 hã§çµ‚ã‚ã‚‹ã€‚
https://pubmed.ncbi.nlm.nih.gov/32573652/
Bioinformatics
. 2020 Nov 1;36(17):4658-4659. doi: 10.1093/bioinformatics/btaa582.
Very Fast Tree: speeding up the estimation of phylogenies for large alignments through parallelization and vectorization strategies
CÃ©sar PiÃ±eiro 1, JosÃ© M AbuÃ­n 1, Juan C Pichel 1

### FastTree

http://www.microbesonline.org/fasttree/
FastTree 2.1: Approximately-Maximum-Likelihood Trees for Large Alignments

FastTree computes local support values with the Shimodaira-Hasegawa test
http://www.microbesonline.org/fasttree/#Support
If you want to use the traditional bootstrap instead, 

2018-10-26
https://www.hobochuritsu.com/entry/2018/10/26/223658
FastTreeã®ä½¿ã„æ–¹ã¾ã¨ã‚ - ã»ã¼ä¸­ç«‹ãƒ–ãƒ­ã‚°

RAxMLã¨FastTreeã®æ¯”è¼ƒã«é–¢ã™ã‚‹ãƒ„ã‚¤ãƒ¼ãƒˆ
https://twitter.com/pathogenomenick/status/453823811805544448
https://www.ncbi.nlm.nih.gov/pubmed/22132132
PLoS One. 2011;6(11):e27731. doi: 10.1371/journal.pone.0027731. Epub 2011 Nov 21.
RAxML and FastTree: comparing two methods for large-scale maximum likelihood phylogeny estimation.
Liu K1, Linder CR, Warnow T.

### RAxML

https://sco.h-its.org/exelixis/web/software/raxml/index.html
RAxML - Randomized Axelerated Maximum Likelihood

https://sco.h-its.org/exelixis/web/software/raxml/hands_on.html
RAxML hands-on session

2020å¹´01æœˆ09æ—¥ã«æ›´æ–°
https://qiita.com/YF_bio/items/40344f77e13f5da0ad07
RAxML-NGã®ä½¿ã„æ–¹ - Qiita

2017-05-25
https://qiita.com/MaedaTaro_Umiushi/items/3bc287672429ba1a511f
Phylogenomicè§£æã‚’Biopythonï¼‹RaxMLï¼‹Conselã§ã‚„ã£ã¦ã¿ã‚‹ ãã®2-AUæ¤œå®š - Qiita

2017å¹´3æœˆ5æ—¥
http://www.tezuru-mozuru.com/?p=9808
RAxMLã«ã‚ˆã‚‹æœ€å°¤æ³•ç³»çµ±æ¨¹æ¨å®š â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹

2016-12-02
http://www.fish-evol.org/raxmlExample.html
RAxMLãƒ»ç°¡å˜ãªä¾‹é¡Œ - äº•ä¸Šæ½¤

2016-10-12
http://www.fish-evol.org/RAxML.html
RAxML - äº•ä¸Š æ½¤

https://www.fifthdimension.jp/documents/molphytextbook/
Life is fifthdimension.::Documents::åˆ†å­ç³»çµ±å­¦æ¼”ç¿’
https://www.fifthdimension.jp/documents/molphytextbook/molphytextbook.ja.pdf
2015å¹´åº¦ç‰ˆPDF(2015å¹´10æœˆ20æ—¥æ›´æ–°)

https://sites.google.com/site/ouminishiswebsite/8-shi-yan-fang-fa-deta-jie-xi/raxmlde-xi-tong-jie-xi
ç³»çµ±è§£æ: RAxML - Oumi Nishi's Website

----------
### phyml

http://www.atgc-montpellier.fr/phyml/usersguide.php
PhyML 3.0: new algorithms, methods and utilities

http://www.atgc-montpellier.fr/models/index.php?model=lg
LG: An Improved, General Amino-Acid Replacement Matrix

http://www.atgc-montpellier.fr/phyml/alrt/
A fast implementation of aLRT in PhyML.
"Approximate likelihood ratio test for branchs: A fast, accurate and powerful alternative."

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3548315/
Evolution of Conjugation and Type IV Secretion Systems
We performed the phylogenetic inference as mentioned earlier and additionally with PhyML 3.0 (Gascuel et al. 2010) under the LG model and with the bioNJ starting tree to get aLRT support values.

----------

## viewer

http://dendroscope.org/
Dendroscope3
An interactive viewer for rooted phylogenetic trees and networks

http://etetoolkit.org/treeview/
Tree viewer - Online visualization of phylogenetic trees (newick) and alignments.

http://kazumaxneo.hatenablog.com/entry/2018/10/09/111845
webã§å‹•ä½œã™ã‚‹é«˜é€Ÿã§è»½é‡ãªåˆ†å­ç³»çµ±æ¨¹å¯è¦–åŒ–ãƒ„ãƒ¼ãƒ« IcyTree - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
PhyMLï¼ˆGuindon and Gascuelã€2003ï¼‰ã‚„BEASTï¼ˆBouckaert et alã€2014ï¼‰ã®ã‚ˆã†ãªä»–ã®ãƒ‘ãƒƒã‚±ãƒ¼ã‚¸ã¯ã€å›³å½¢å‡ºåŠ›ã‚’ç”Ÿæˆã›ãšã€ä»£ã‚ã‚Šã«FigTreeï¼ˆhttp://tree.bio.ed.ac.jp  linkï¼‰ã‚„Dendroscopeï¼ˆHuson and Scornavaccaã€2012 linkï¼‰ãªã©ã®ç‰¹æ®Šãªç³»çµ±ç™ºç”Ÿå¯è¦–åŒ–ãƒ„ãƒ¼ãƒ«ã«ä¾å­˜ã—ã¦ã„ã‚‹ã€‚

----------
### FigTree
http://tree.bio.ed.ac.uk/software/figtree/

http://www.fish-evol.org/FigTree.html
FigTree
2018 å¹´ 5 æœˆ 9 æ—¥ã€€æ”¹è¨‚
äº•ä¸Š æ½¤
FigTree ã¯ newick å½¢å¼ã§ä½œæˆã•ã‚ŒãŸç³»çµ±æ¨¹ã‚’ graphical ã«è¡¨ç¤ºã™ã‚‹ãƒ—ãƒ­ã‚°ãƒ©ãƒ ã§ã™ï¼

https://sites.google.com/view/enter-the-fungi/phylogenetic-analysis/figtree-ã®ä½¿ã„æ–¹
FigTree ã®ä½¿ã„æ–¹

----------
### iTOL

https://itol.embl.de/
iTOL: Interactive Tree Of Life

https://pubmed.ncbi.nlm.nih.gov/38613393/
Nucleic Acids Res
. 2024 Jul 5;52(W1):W78-W82. doi: 10.1093/nar/gkae268.
Interactive Tree of Life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool
Ivica Letunic 1, Peer Bork 2 3
https://pmc.ncbi.nlm.nih.gov/articles/PMC11223838/

https://pubmed.ncbi.nlm.nih.gov/27095192/
Nucleic Acids Res
. 2016 Jul 8;44(W1):W242-5. doi: 10.1093/nar/gkw290. Epub 2016 Apr 19.
Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees
Ivica Letunic 1, Peer Bork 2

https://kazumaxneo.hatenablog.com/entry/2020/07/02/100000
ã‚¤ãƒ³ã‚¿ãƒ©ã‚¯ãƒ†ã‚£ãƒ–ãªã‚ªãƒ³ãƒ©ã‚¤ãƒ³ã®ç³»çµ±æ¨¹ãƒ„ãƒ¼ãƒ« Interactive Tree Of Life (iTOL) v4 - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
å¼•ç”¨

5:40 AM Â· Jul 16, 2020
https://twitter.com/kmoooooog/status/1283501696417554432
itolæœ‰æ–™åŒ–ã‹ã€‚å…¨ç„¶æ‰•ã£ã¦ã‚‚ã„ã„ã€‚
https://itol.embl.de/pricing.cgi

https://besshomanabulumi.wixsite.com/manabuhome/research-tips
iTOL: ç³»çµ±æ¨¹ã‚’ã‹ã£ã“ã‚ˆããƒ¡ã‚¤ã‚¯ã‚¢ãƒƒãƒ—ã§ãã‚‹ãƒ„ãƒ¼ãƒ«ã€‚
ProtTest3: ML,Bayesç³»çµ±æ¨¹ã‚’æ›¸ãã¨ãã®ç½®æ›ãƒ¢ãƒ‡ãƒ«ã‚’æ¨å®šã—ã¦ãã‚Œã‚‹ãƒ„ãƒ¼ãƒ«ã€‚

2017.8.3
https://besshomanabulumi.wixsite.com/manabuhome/itol
iTOLã®ä½¿ã„æ–¹
ç³»çµ±æ¨¹ã®Figureã‚’ã‹ã£ã“ã‚ˆãä»•ä¸Šã’ã‚‰ã‚Œã‚‹ãƒ„ãƒ¼ãƒ«ã€‚

2017.8.10
https://besshomanabulumi.wixsite.com/manabuhome/itol-1
ProtTest3ã®ä½¿ã„æ–¹
ProtTest 3ã¯å¡©åŸºãƒ»ã‚¢ãƒŸãƒé…¸ã®ç½®æ›ãƒ¢ãƒ‡ãƒ«ã‚’æ¨å®šã—ã¦ãã‚Œã‚‹ç„¡æ–™ãƒ„ãƒ¼ãƒ«ã§ã™ã€‚

----------
### SeaView

https://www.ncbi.nlm.nih.gov/pubmed/19854763
Mol Biol Evol. 2010 Feb;27(2):221-4. doi: 10.1093/molbev/msp259. Epub 2009 Oct 23.
SeaView version 4: A multiplatform graphical user interface for sequence alignment and phylogenetic tree building.
Gouy M, Guindon S, Gascuel O.

https://www.mybiosoftware.com/seaview-4-2-12-sequence-alignment-phylogenetic-tree-building.html
SeaView 4.7 - Sequence Alignment and Phylogenetic Tree Building

Dec 9, 2014
https://www.youtube.com/watch?v=-XfF8sJaZ-c
Exercise 3 Seaview to visually align sequences - YouTube


http://www2.tba.t-com.ne.jp/nakada/takashi/phylogeny/seaview2.html
SeaView ä½¿ç”¨æ³•
SeaView ã‚’ç”¨ã„ãŸã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆç·¨é›†
ä½œæˆï¼šä»²ç”°å´‡å¿—
æ›´æ–°ï¼š2008å¹´03æœˆ10æ—¥

----------
### DupliPHY

www.bioinf.manchester.ac.uk/dupliphy

https://www.ncbi.nlm.nih.gov/pubmed/25294920
Bioinformatics. 2015 Feb 1;31(3):416-7. doi: 10.1093/bioinformatics/btu645. Epub 2014 Oct 7.
DupliPHY-Web: a web server for DupliPHY and DupliPHY-ML.
Ames RM1, Lovell SC1.

https://www.ncbi.nlm.nih.gov/pubmed/22039210
Bioinformatics. 2012 Jan 1;28(1):48-55. doi: 10.1093/bioinformatics/btr592. Epub 2011 Oct 28.
Determining the evolutionary history of gene families.
Ames RM1, Money D, Ghatge VP, Whelan S, Lovell SC.
https://academic.oup.com/bioinformatics/article/28/1/48/218268
- tree reconciliation techniques (Akerborg et al., 2009; Chen et al., 2000; Page, 1998; Tofigh et al., 2010). These methods infer gene trees for each gene family, and then reconcile these trees with a known species tree to infer gain and loss events.
- The probabilistic model implemented in COUNT (Csuros, 2010) only annotates ancestral species as containing 0, 1 or more members of a family, whereas the parsimony method produces indistinguishable results from DupliPHY (Supplementary Material). The method of (Iwasaki and Takagi, 2007) allows a maximum gene family size of three.
- COUNT (Csuros, 2010) provides a parsimony reconstruction of the ancestral size of gene families. We find that there is very little difference between DupliPHY and COUNT for simulations under any of the three models (Supplementary Fig. S2). 

----------
### Count

http://www.iro.umontreal.ca/~csuros/gene_content/count.html
Count: analysis of gene content evolution

https://www.ncbi.nlm.nih.gov/pubmed/20551134
Bioinformatics. 2010 Aug 1;26(15):1910-2. doi: 10.1093/bioinformatics/btq315. Epub 2010 Jun 15.
Count: evolutionary analysis of phylogenetic profiles with parsimony and likelihood.
CsurÃ¶s M1.

----------
## MEGA
MEGA: Molecular Evolutionary Genetics Analysis software

- https://www.megasoftware.net/

Documentation
[Online Manual](https://www.megasoftware.net/web_help)
First Time User

[walkthrough tutorial](https://www.megasoftware.net/web_help_10/Introduction.htm)
Introduction

1. [Mega Basics](https://www.megasoftware.net/web_help_10/Part_I_Getting_Started/A_Walk_Through_MEGA/MEGA_Basics.htm)
2. [Aligning Sequences](https://www.megasoftware.net/web_help_10/Part_I_Getting_Started/A_Walk_Through_MEGA/Aligning_Sequences.htm)
3. [Estimating Evolutionary Distances](https://www.megasoftware.net/web_help_10/Part_I_Getting_Started/A_Walk_Through_MEGA/Estimating_Evolutionary_Distances.htm)
4. [Building Trees from Sequence Data](https://www.megasoftware.net/web_help_10/Part_I_Getting_Started/A_Walk_Through_MEGA/Building_Trees_From_Sequence_Data.htm)
5. [Testing Tree Reliability](https://www.megasoftware.net/web_help_10/Part_I_Getting_Started/A_Walk_Through_MEGA/Testing_Tree_Reliability.htm)

Site Links
[Videos](https://www.megasoftware.net/videos)
Instructional Videos

### TUTORIALS

#### KUMAR LAB VIDEOS

Molecular Dating with MEGA

Choosing and Acquiring Sequences Part 1
https://www.youtube.com/watch?v=raaOgtvMJWw
2018/10/09

Choosing and Acquiring Sequences Part 2
https://www.youtube.com/watch?v=cVdmH7nNboE
2018/10/09

Reconstructing Ancestral Sequences
https://www.youtube.com/watch?v=djju9WFMvn0
2018/10/09

Relative Rate Framework for Efficient, Reliable, and Relaxed Clock Dating in MEGA

Inferring Selection with MEGA

MEGA X CC For Analysis Pipelines

Tree Inference with MEGA
https://www.youtube.com/watch?v=_ch8YIPvR-M
2019/08/01
This is presentation that took place at SMBE 2018 in Japan about a protocol on how to reconstruct phylogenetic trees with MEGA 10.

Bootstrap with MEGA
https://www.youtube.com/watch?v=qTmfQxT9HJc
2019/08/01
This is presentation that took place at SMBE 2018 in Japan about a protocol on how to bootstrap trees with MEGA 10.

#### MEGA X

Align Sequences then Compute Substitution Matrix, Compute Pairwise distance matrix, Construct ML Tree, and Find Gene Duplication
https://www.youtube.com/watch?v=m3qwVttzsKA
How to use MEGA software for evolutionary studies of HSP
2018/07/18

Build a Mega File from Multiple Fasta Files, then Construct a Neighbor Joining Tree
https://www.youtube.com/watch?v=hlgB3aRyE_8
How to use Molecular evolutionary Genetic Analysis (MEGA) software
2018/07/31

### MEGA_Tamura

7:10 PM Â· Apr 26, 2021
https://twitter.com/fiddler_K/status/1386623833897984003
ï¼»ãƒ¡ãƒ¢ï¼½MEGA11ãŒãƒªãƒªãƒ¼ã‚¹ã•ã‚Œã¦ã‚‹ã€‚æ©Ÿæ¢°å­¦ç¿’ã§é€²åŒ–é€Ÿåº¦ã®è‡ªå·±ç›¸é–¢ã‚’ãƒ†ã‚¹ãƒˆã™ã‚‹æ–¹æ³•ã‚„ãƒ™ã‚¤ã‚ºæ³•ã§é€²åŒ–ç¢ºç‡(EP)ã‚’æ¨å®šã™ã‚‹æ–¹æ³•ãŒè¿½åŠ ã•ã‚Œã¦ã‚‹ã‚ˆã†/MEGA11: Molecular Evolutionary Genetics Analysis version 11

https://pubmed.ncbi.nlm.nih.gov/33892491/
Mol Biol Evol
. 2021 Apr 23;msab120. doi: 10.1093/molbev/msab120. Online ahead of print.
MEGA11: Molecular Evolutionary Genetics Analysis version 11
Koichiro Tamura 1 2, Glen Stecher 3, Sudhir Kumar 3 4 5
https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab120/6248099

https://www.ncbi.nlm.nih.gov/pubmed/29722887
Mol Biol Evol. 2018 Jun 1;35(6):1547-1549. doi: 10.1093/molbev/msy096.
MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms.
Kumar S1,2,3, Stecher G1, Li M1, Knyaz C1, Tamura K4,5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5967553/

https://biosciencedbc.jp/blog/20200303-01.html
æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ï¼ˆSARS-CoV-2ï¼‰ãŠã‚ˆã³æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹æ„ŸæŸ“ç—‡ï¼ˆCOVID-19ï¼‰ã«é–¢ã™ã‚‹ç ”ç©¶ãƒ‡ãƒ¼ã‚¿ãƒ»ãƒªã‚½ãƒ¼ã‚¹ - NBDC
DBCLSãŒNBDCã¨ã®å…±åŒç ”ç©¶ã®ä¸€ç’°ã§é‹å–¶ã™ã‚‹ã€Œçµ±åˆTVã€ã«ã¯ã€MEGAã®æœ€æ–°ãƒãƒ¼ã‚¸ãƒ§ãƒ³ã‚’ç”¨ã„ãŸåˆ†å­ç³»çµ±æ¨¹ã®æ¨å®šã«ã¤ã„ã¦ã®è¬›ç¾©å‹•ç”»ï¼ˆæ—¥æœ¬èªï¼‰ãŒæ²è¼‰ã•ã‚Œã¦ã„ã¾ã™ã€‚ç™»å£‡è€…ã¯MEGAã®é–‹ç™ºè€…ã§ã‚ã‚‹é¦–éƒ½å¤§å­¦æ±äº¬ï¼ˆæ±äº¬éƒ½ç«‹å¤§å­¦ï¼‰ã®ç”°æ‘æ•™æˆã§ã™ã€‚

MEGA X ã‚’ç”¨ã„ãŸåˆ†å­ç³»çµ±è§£æ @ åˆ†å­ç³»çµ±æ¨¹æ¨å®šæ³•:ç†è«–ã¨å¿œç”¨ ãƒ¯ãƒ¼ã‚¯ã‚·ãƒ§ãƒƒãƒ— | çµ±åˆTV
https://doi.org/10.7875/togotv.2019.193
https://twitter.com/hashtag/ws222?f=live

https://evolgen.biol.se.tmu.ac.jp/MEGA/default.html
MEGA3 ãƒãƒ¥ãƒ¼ãƒˆãƒªã‚¢ãƒ«ï¼ˆActin gene coding region ã‚’ä¾‹é¡Œã¨ã—ã¦ï¼‰
- å…¥åŠ›ãƒ‡ãƒ¼ã‚¿ã®æº–å‚™
- é…åˆ—ãƒ‡ãƒ¼ã‚¿ã®ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ
- åˆ†å­ç³»çµ±æ¨¹ã®ä½œæˆ

https://evolgen.biol.se.tmu.ac.jp/MEGA/data-mining.htm
å…¥åŠ›ãƒ‡ãƒ¼ã‚¿ã®æº–å‚™
ï¼‘ï¼é…åˆ—ãƒ‡ãƒ¼ã‚¿ã®ãƒ€ã‚¦ãƒ³ãƒ­ãƒ¼ãƒ‰
ï¼’ï¼ã‚·ãƒ¼ã‚±ãƒ³ã‚µãƒ¼ãƒ‡ãƒ¼ã‚¿ã®å…¥åŠ›
ï¼“ï¼ãƒ†ã‚­ã‚¹ãƒˆãƒ‡ãƒ¼ã‚¿ã®å…¥åŠ›ï¼ˆAlignmentExplorerã®åŸºæœ¬æ“ä½œæ³•ï¼‰
ï¼”ï¼ç›¸åŒé…åˆ—ã®æ¤œç´¢

https://evolgen.biol.se.tmu.ac.jp/MEGA/align-protocol.htm
é…åˆ—ãƒ‡ãƒ¼ã‚¿ã®ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ

https://evolgen.biol.se.tmu.ac.jp/MEGA/tree-protocol.htm
åˆ†å­ç³»çµ±æ¨¹ã®ä½œæˆ
ï¼‘ï¼è¿‘éš£çµåˆæ³•ã«ã‚ˆã‚‹ç³»çµ±æ¨¹ã®ä½œæˆã¨ãƒ–ãƒ¼ãƒˆã‚¹ãƒˆãƒ©ãƒƒãƒ—ãƒ»ãƒ†ã‚¹ãƒˆ
ï¼’ï¼ç³»çµ±æ¨¹ã‚’åŸºã«ã—ãŸè§£æä¾‹
ï¼ˆï¼‘ï¼‰ åˆ†å­æ™‚è¨ˆã‚’ç”¨ã„ãŸåˆ†å²å¹´ä»£ã®æ¨å®š ï¼ˆå“ºä¹³é¡ ãƒãƒˆã‚¯ãƒ­ãƒ¼ãƒ ï½‚éºä¼å­ã®å ´åˆï¼‰
ï¼ˆï¼’ï¼‰åˆ†é¡ç¾¤ã¨åˆ†å­ç³»çµ±ã¨ã®é–¢é€£ï¼ˆäººé¡é›†å›£ã®D-loopé…åˆ—ã®å ´åˆï¼‰

### Janecka Genomics

2020/12/19
https://www.youtube.com/watch?v=xKS5qZwl1GY
Making a Phylogenetic Tree with Bootstrap Support Values in MEGA
This video shows the steps in MEGA for opening a multiple sequence alignment, determining the best sequence evolution model, reconstructing a maximum likelihood phylogenetic tree with bootstrap support values, and saving the tree in three different formats. 

2020/12/13
https://www.youtube.com/watch?v=Pq4Cy76zNew
Making a Multiple Sequence Alignment in MEGA
This video shows the steps of obtaining nucleotide gene sequences from NCBI, editing the downloaded fasta file, importing the sequences into MEGA, and building a multiple sequence alignment.

### MEGA_unclassified

2020/07/06
https://www.youtube.com/watch?v=8eTkVIdmHbg
Phylogenetic Analysis with MEGA X - YouTube
KENNETH FRANCIS RODRIGUES -
This video tutorial describes the application of MEGA X for the alignment and phylogenetic reconstruction using a single locus.


https://sites.google.com/view/enter-the-fungi/phylogenetic-analysis/alignment
Enter the FUNGI - Fasta ã®æ•´ç†ã¨MUSCLE/ MAFFT ã§ã®ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ
ãƒ¡ãƒ¢ MEGA X ã‚¤ãƒ³ã‚¹ãƒˆãƒ¼ãƒ«ã‚³ãƒãƒ³ãƒ‰ on Linux 

https://bio-and.info/post/DNA_Analysis/PhylogeneticTreeConstruct
ç³»çµ±æ¨¹ã®æ›¸ãæ–¹ï¼ˆMEGAã‚’ä½¿ç”¨ã—ãŸç³»çµ±æ¨¹ã®æ›¸ãæ–¹ï¼‰ - ãƒã‚¤ã‚ªã¨ã‚¤ãƒ³ãƒ•ã‚©ã¾ã¨ã‚ -
â‘ . MEGAã®ã‚¤ãƒ³ã‚¹ãƒˆãƒ¼ãƒ«
â‘¡. ã‚¢ãƒŸãƒé…¸é…åˆ—ã®æº–å‚™
â‘¢. MEGAã«ã‚¢ãƒŸãƒé…¸é…åˆ—ã‚’å–ã‚Šè¾¼ã¿ã€ãƒãƒ«ãƒãƒ—ãƒ«ã‚¢ãƒ©ã‚¤ãƒ¡ãƒ³ãƒˆ
â‘£. ç³»çµ±æ¨¹ã®ä½œæˆ



https://sci-tech.ksc.kwansei.ac.jp/~tohhiro/bioinfo18/bioinfo2018-4.pdf
ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹ç¬¬å››å›
å¾—ã‚‰ã‚ŒãŸã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã‚’ä½¿ã£ã¦MEGAã§ç³»çµ±æ¨¹ã‚’ä½œæˆ
(1) MEGAã‚’èµ·å‹•ã—ã¦mafftã§ä½œæˆã—ãŸã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆä›¾èª­ã¿è¾¼ã¿

ãƒ‡ãƒ•ã‚©ãƒ«ãƒˆã§ã¯Original TreeãŒè¡¨ç¤ºã•ã‚Œã¦ã„ã‚‹
Bootstrap consensus treeã®ã‚¿ãƒ–ã‚’é¸æŠ

2018/01/20
https://ultrabem-branch3.com/informatics/bioinformatics/tree_make_mega_ml
ç³»çµ±æ¨¹ã®ä½œã‚Šæ–¹: MEGA ã§æœ€å°¤æ³•ã‚’ä½¿ã†
MEGA ã§ã¯ã€ç³»çµ±æ¨¹ã¯ .mts ã¨ã„ã†æ‹¡å¼µå­ã§ä¿å­˜ã•ã‚Œã‚‹ã€‚
Original Tree ã¨ Bootstrap concensus tree ã¨ã„ã† 2 ã¤ã®ã‚¿ãƒ–ãŒã‚ã‚‹ã€‚

å‹•ç”»
https://doi.org/10.7875/togotv.2017.106
2017-11-06 MEGA7ã‚’ä½¿ã£ã¦é…åˆ—ã®ã‚¢ãƒ©ã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆãƒ»ç³»çµ±è§£æã‚’è¡Œã†

http://www.tezuru-mozuru.com/?tag=mega
MEGA â€“ ãƒãƒ¼ãƒ ã¦ã¥ã‚‹ã‚‚ã¥ã‚‹
2017å¹´3æœˆ24æ—¥
http://www.tezuru-mozuru.com/?p=9939
MEGAã«ã‚ˆã‚‹æœ€å°¤æ³•ç³»çµ±æ¨¹æ¨å®š

----------
